export const fullQuizData = [
  {
    question: `What is the length of a motif, in terms of amino acids residue?`,
    options: ["30- 60", "10- 20", "70- 90", "1- 10"],
    correct: "b",
    explanation: `A typical motif is 10-20 amino acids long. For e.g. Zn-finger motif. Hence it is also referred to as super secondary structure. This motif is seen in transcription factors.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `On average, what is the length of a typical domain?`,
    options: ["About 100 residues", "About 300 residues", "About 500 residues", "About 900 residues"],
    correct: "a",
    explanation: `The predicted optimal number of residues, which corresponds to the maximum free energy of unfolding, is 100. This is in agreement with a statistical analysis derived from their experimental structures of motifs. For too short chain, change in enthalpy of internal interactions is not favorable enough for folding because of the limited number of inter-residue contacts. And a long chain is also unfavorable for a single domain.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is false about the ‘loop’ structure in proteins?`,
    options: ["They connect helices and sheets", "They are more tolerant of mutations", "They are more flexible and can adopt multiple conformations", "They are never the components of active sites"],
    correct: "d",
    explanation: `Loops are frequently components of active sites as they are flexible in nature and as they are situated on the surface of the structure. Besides, they vary in length and 3-D configurations which give even more chances to be component of active sites.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the common structural motifs are described wrongly?`,
    options: ["\u03b2-hairpin \u2013 adjacent antiparallel strands", "Greek key \u2013 4 adjacent antiparallel strand", "\u03b2-\u03b1-\u03b2 \u2013 2 parallel strands connected by helix", "\u03b2-\u03b1-\u03b2 \u2013 2 antiparallel strands connected by helix"],
    correct: "d",
    explanation: `In motif, two adjacent β parallel strands are connected by an α helix from the C-terminus of strand 1 to the N-terminus of strand. Most protein structures that contain parallel beta-sheets are built up from combinations of such β-α-βmotifs.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following least describes Long Loop β-hairpins?`,
    options: ["They are Often referred to as a \u2018random coil\u2019 conformation", "Generally they are referred to as the \u03b2-meander supersecondary structure", "Loop looks similar to the Greek Letter \u2126", "Wide-range of conformations with very specific sequence preferences"],
    correct: "d",
    explanation: `They are wide-range of conformations with no particular sequence preferences. As the name suggests ‘meander’ the conformation they possess is also quite unspecified. Addition to that Long loop β-hairpins are special case of Ω loops, that explains a lot about their structural preferences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Motifs that can form α/β horseshoes conformation are rich with which protein residue?`,
    options: ["Proline", "Arginine", "Valine", "Leucine"],
    correct: "d",
    explanation: `Specific pattern of Leucine residues, strands form a curved sheet with helices on the outside. Leucine-rich repeats (LRRs) are 20-29-residue sequence motifs present in a number of proteins with diverse functions. The primary function of these motifs appears to be to provide a versatile structural framework for the formation of protein-protein interactions.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following wrongly describes protein domains?`,
    options: ["They are made up of one secondary structure", "Defined as independently foldable units", "They are stable structures as compared to motifs", "They are separated by linker regions"],
    correct: "a",
    explanation: `Protein domains are made up of two or more motifs i.e. the secondary structure to form stable and folded 3-D structures. They are conserved part of the protein sequence and can evolve, function, and exist independently of the rest of the protein chain.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The protein structural motif domain- helix loop helix are contained by all of the following except ________`,
    options: ["Scleraxis", "Neurogenins", "Transcription Factor 4", "Leucine zipper"],
    correct: "d",
    explanation: `Leucine zipper is associated with gene regulation and contains alpha helix with leucine at every 7th amino acid. While rest of them are under one of the largest families of dimerizing transcription factors.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not the function of Short Linear Motifs?`,
    options: ["Irreversible cleavage of the peptide at the SLiM", "Reversible cleavage of the peptide at the SLiM", "Moiety addition at targeted sites on SLiM", "Structural modifications of the peptide backbone"],
    correct: "a",
    explanation: `Short Linear Motifs are short stretches of protein sequence that mediate protein-protein interaction. SLiMs can act as recognition sites of endo-peptidases resulting in the irreversible cleavage of the peptide at the SLiM.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In the zinc finger, which residues in this sequence motif form ligands to a zinc ion?`,
    options: ["Cysteine and histidine", "Cysteine and arginine", "Histidine and proline", "Histidine and arginine"],
    correct: "a",
    explanation: `In the zinc finger, which is found in a widely varying family of DNA-binding proteins, cysteine and histidine residues in this sequence motif form ligands to a zinc ion whose coordination is essential to stabilize the tertiary structure.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `While scanning for similarities in motifs, how regular expressions’ techniques work?`,
    options: ["It represents a sequence family by a string of characters and further compares them", "An algorithm similar to dynamic programming is used", "Dot matrix analysis is used in this type of sequence analysis", "Matrix analysis methods are used in this type"],
    correct: "a",
    explanation: `In regular expressions’ techniques Pattern matching is defined as true or false in answer or outcome. In other words, if the pattern described in regex is found in a string of letters, the answer is true.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following best defines regular expressions?`,
    options: ["They are made up of terms, operators and modifiers", "They describe string or set of strings to find matching patterns", "They are strictly restricted to alignment and corresponding score", "They consist of set of rules for the connotations of various amino acid residues"],
    correct: "b",
    explanation: `Regular expressions are powerful notable algebra that describe string or set of strings to find matching patterns. Pattern matching is defined as true or false in answer or outcome. And it is true that they are made up of terms, operators and modifiers but they are terminologies further used in matching process.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In terminologies related to regular expressions which of the following is false about terms and operators?`,
    options: ["Terms are strings or substrings", "Operators combine terms and expressions", "Operators do not have precedence", "Operators have precedence like arithmetic operators"],
    correct: "c",
    explanation: `For harmonious, efficient and error-free functioning of the matching preocess, operators have precedence in order to set the priority of the operations to be carried out during the alignment.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In regular expressions, which of the following pair of pattern is wrongly matched with its significance?`,
    options: ["\u2018-\u2019 \u2013 separator", "< \u2013 N-terminal", "> \u2013 C-terminal", "\u2018>>\u2019 \u2013 end"],
    correct: "d",
    explanation: `Regular Expression Symbols have their own significances in regular expressions’ system. For e.g. x(2,3) means x-x or x-x-x. Similarly, ‘.’ means end.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Emotif uses which databases for alignment of sequences?`,
    options: ["BLOCKS and PRINTS databases", "PROSITE", "BLOCKS", "PRINTS"],
    correct: "a",
    explanation: `Emotif is a motif database that uses multiple sequence alignments from both the BLOCKS and PRINTS databases with an alignment collection much larger than PROSITE. It identifies patterns by allowing fuzzy matching of regular expressions. Therefore, it produces fewer false negatives than PROSITE.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `While analysing motif sequences, what is the major disadvantageous feature of PROSITE?`,
    options: ["The database constructs profiles to complement some of the sequence patterns", "The functional information of these patterns is primarily based on published literature", "Some of the sequence patterns are too short to be specific", "Lack of specificity about probability and variation and relation between them"],
    correct: "c",
    explanation: `The major pitfall with the PROSITE patterns is that some of the sequence patterns are too short to be specific. Rest of the options are advantages. The problem with these short sequence patterns is that the resulting match is very likely to be a result of random events. Overall, PROSITE has a greater than 20% error rate. Thus, either a match or non-match in PROSITE should be treated with caution.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a characteristic of Fuzzy or approximate matches in regular expression?`,
    options: ["This method is able to include more variant forms of a motif with a conserved function", "the rule of matching is based on observations, not actual assumptions", "with the more relaxed matching, there is increase of the noise level and false positives", "the rule of matching is based on assumptions not actual observations"],
    correct: "b",
    explanation: `The rule of matching is based on assumptions not actual observations in Fuzzy or approximate matches in regular expression. This provides more permissive matching by allowing more flexible matching of residues of similar biochemical properties. For example, if an original alignment only contains phenylalanine at a particular position, fuzzy matching allows other aromatic residues (including unobserved tyrosine and tryptophan) in a sequence to match with the expression.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a characteristic of exact matches in regular expression?`,
    options: ["There must be a strict match of sequence patterns", "Any variations in the query sequence from the predefined patterns are not allowed", "Provide more permissive matching by allowing more flexible matching of residues of similar biochemical properties", "Searching a motif database using this approach results in either a match or non-match"],
    correct: "c",
    explanation: `In this type of matching, there has to be a strict match of sequence patterns. This way of searching has a good chance of missing truly relevant motifs that have slight variations, thus generating false-negative results. As new sequences of motif are being accumulated, the rigid regular expression tends to become obsolete if not updated regularly to reflect the changes.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `What does this representation mean- R.L.[EQD]?`,
    options: ["An arginine- Amino acid- Leucine- Amino acid- Either Apartic acid, glutamic acid or glutamine", "An arginine- Leucine- Either Apartic acid, glutamic acid or glutamine", "An arginine- Leucine- Amino acid- Either Apartic acid, glutamic acid or glutamine", "An arginine- Leucine- Apartic acid and glutamic acid and glutamine"],
    correct: "a",
    explanation: `This is an example of pexel motif. Here, the ‘.’ represents the ‘end’ i.e. the amino acid as mentioned in the answer and the [ ] means ‘or’ i.e. either of the mentioned residue is present in the given position.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not an advantage of Statistical models’ methods in analyzing protein motifs?`,
    options: ["Sequence information is preserved from a multiple sequence alignment and expresses it with probabilistic models", "Statistical models allow partial matches and compensate for unobserved sequence patterns using pseudo-counts", "Statistical models have stronger predictive power than the regular expression based approach, even when they are derived from a limited set of sequences", "The comparative flexibility is less in case of these methods when compared to regular expressions methods"],
    correct: "d",
    explanation: `The major limitation of regular expressions is that this method does not take into account sequence probability information about the multiple alignment from which it is modeled making them less flexible. If a regular expression is derived from an incomplete sequence set, it has less predictive power because many more sequences with the same type of motifs are not represented. Unlike regular expressions, position-specific scoring matrices (PSSMs), profiles, and HMMs preserve the sequence information from a multiple sequence alignment and express it with probabilistic models.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `For motif scanning which of the following programs or databases is for regulated sites curated from scientific literature?`,
    options: ["ENSEMBL", "ORegAnno", "MAST", "Clover"],
    correct: "b",
    explanation: `Clover identifies overrepresented motifs in protein sequences whereas; MAST allows users to scan different databases for matches to motifs. ENSEMBL is another online genomic sequence repository which also includes online tools for data mining as well as BLAST searches.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not an advantageous feature or algorithm of the database PRINTS?`,
    options: ["This program breaks down a motif into even smaller non-overlapping units called \u2018fingerprints\u2019, which are represented by unweighted PSSMs", "To define a motif, at least a majority of fingerprints are required to match with a query sequence", "A query that has simultaneous high-scoring matches to a majority of fingerprints belonging to a motif is a good indication of containing the functional motif", "The difficulty to recognize short motifs when they reach the size of single fingerprints"],
    correct: "d",
    explanation: `PRINTS is a protein fingerprint database containing ungapped, manually curated alignments corresponding to the most conserved regions among related sequences. The drawbacks of PRINTS are–the difficulty to recognize short motifs when they reach the size of single fingerprints and a relatively small database, which restricts detection of many motifs.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In which of the following multipurpose packages Gibbs sampling algorithm is used?`,
    options: ["Consensus", "BEST", "AlignACE", "PhyloCon"],
    correct: "c",
    explanation: `The Gibbs sampling algorithm can identify multiple motifs in a sequence in a sequence set using iterative masking procedure. It is used in AlignACE whereas BEST is a suite of four motif discovery tools integrated in a graphical user interface. Also, Consensus program finds motifs in a set of unaligned sequences and PhyloCon builds on this framework by modeling conservation across orthologous genes from multiple species.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue in case of the database BLOCKS?`,
    options: ["The alignments are automatically generated using the same data sets used for deriving the BLOSUM matrices", "The derived ungapped alignments are called \u2018blocks\u2019, which are usually longer than motifs, are subsequently converted to PSSMs", "A weighting scheme and pseudo counts are subsequently applied to the PSSMs to account for underrepresented and unobserved residues in alignments", "The functional annotation of blocks is not consistent with that for the motifs"],
    correct: "d",
    explanation: `BLOCKS is a database that uses multiple alignments derived from the most conserved, ungapped regions of homologous protein sequences. Because blocks often encompass motifs, the functional annotation of blocks is thus consistent with that for the motifs. A query sequence can be used to align with pre-computed profiles in the database to select the highest scored matches. Because of the use of the weighting scheme, the signal-to-noise ratio is improved relative to PRINTS.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is false in case of the database Pfam and its algorithm?`,
    options: ["Each motif or domain is represented by an HMM profile generated from the seed alignment of a number of conserved homologous proteins", "Since the probability scoring mechanism is more complex in HMM than in a profile-based approach the use of HMM yields further increases in sensitivity of the database matches", "Pfam-B only contains sequence families not covered in Pfam", "The functional annotation of motifs in Pfam-A is often related to that in UNIPROT"],
    correct: "d",
    explanation: `Pfam is a database with protein domain alignments derived from sequences in SWISSPROT and TrEMBL. The Pfam database is composed of two parts, Pfam-A and Pfam-B. Pfam-A involves manual alignments and Pfam-B, automatic alignment in a way similar to ProDom. The functional annotation of motifs in Pfam-A is often related to that in PROSITE. Because of the automatic nature, Pfam-B has a much larger coverage but is also more error prone because some HMMs are generated from unrelated sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is false in case of the database SMART and its algorithm?`,
    options: ["Contains HMM profiles constructed from manually refined protein domain alignments", "Alignments in the database are built based on tertiary structures whenever available or based on PSI-BLAST profiles", "Alignments are further checked but not refined by human annotators before HMM profile construction", "SMART stands for Simple Modular Architecture Research Tool"],
    correct: "c",
    explanation: `Alignments are further checked and refined by human annotators before HMM profile construction. Protein functions are also manually curated. Thus, the database may be of better quality than Pfam with more extensive functional annotations. Compared to Pfam,
The SMART database contains an independent collection of HMMs, with emphasis on signaling, extracellular, and chromatin-associated motifs and domains. Sequence searching in this database produces a graphical output of domains with well-annotated information with respect to cellular localization, functional sites, super-family, and tertiary structure.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is false in case of the database InterPro and its algorithm?`,
    options: ["InterPro is an integrated pattern database designed to unify multiple databases for protein domains and functional sites", "This database integrates information from PROSITE, Pfam, PRINTS, ProDom, and SMART databases", "Only overlapping motifs and domains in a protein sequence derived by all five databases are included", "All the motifs and domains in a protein sequence derived by all five databases are included"],
    correct: "d",
    explanation: `The only overlapping motifs and domains in a protein sequence derived by all five databases are included in the database. The InterPro entries use a combination of regular expressions, fingerprints, profiles, and HMMs in pattern matching. However, an InterPro search does not obviate the need to search other databases because of its unique criteria of motif inclusion and thus may have lower sensitivity than exhaustive searches in individual databases. A popular feature of this database is a graphical output that summarizes motif matches and has links to more detailed information.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is false in case of the CDART and its algorithm?`,
    options: ["CDART is a domain search program that combines the results from RPS-BLAST, SMART, and Pfam", "The program is now an integral part of the regular BLAST search function", "CDART is a substitute for individual database searches", "It stands for Conserved Domain Architecture"],
    correct: "c",
    explanation: `CDART is a domain search program that combines the results from various database searches. As with InterPro, CDART is not a substitute for individual database searches because it often misses certain features that can be found in SMART and Pfam.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Point out the wrong or irrelevant mathematical method in motif analysis.`,
    options: ["Enumeration", "Probabilistic Optimization", "Deterministic Optimization", "Literature mining"],
    correct: "d",
    explanation: `All the rest of the options are indeed valid and proven mathematical methods that contain efficient algorithms in finding motifs in protein sequences. Literature mining is not a mathematical algorithm or tool as such to be used in identifying motifs. But it is definitely a part of research when it comes to find a function of various protein sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about COG is incorrect regarding its features?`,
    options: ["Currently, there are 4,873 clusters in the COG databases derived from unicellular organisms", "It is constructed by comparing protein sequences encoded in forty-three completely sequenced genomes, which are mainly from prokaryotes, representing thirty major phylogenetic lineages", "The interface for sequence searching in the COG database is the COGnitor program, which is based on gapped BLAST", "It is a protein family database based on structural classification"],
    correct: "d",
    explanation: `COG which stands for Cluster of Orthologous Groups, is a protein family database based on phylogenetic classification. Because orthologous proteins shared by three or more lineages are considered to have descended through a vertical evolutionary scenario, if the function of one of the members is known, functionality of other members can be assigned.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about InterPro is incorrect regarding its features?`,
    options: ["Protein relatedness is defined by the P-values from the BLAST alignments", "The most closely related sequences are grouped into the lowest level clusters", "More distant protein groups are merged into higher levels of clusters", "The outcome of this cluster merging is a tree-like structure of functional categories"],
    correct: "a",
    explanation: `InterPro is a database of clusters of homologous proteins similar to COG. Protein relatedness is defined by the E-values from the BLAST alignments. The database further provides gene ontology information for protein cluster at each level as well as keywords from InterPro domains for functional prediction.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Pfam is available at four locations around the world. Which of the following is not one of them?`,
    options: ["UK", "Sweden", "US", "Japan"],
    correct: "d",
    explanation: `Pfam is available at four locations around the world each providing a core set of functionality for accessing each family. They are US, UK, Sweden and France. Documentation on the content and use of Pfam is available via the web.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a member database of InterPro?`,
    options: ["SCOP", "HAMAP", "PANTHER", "Pfam"],
    correct: "a",
    explanation: `The signatures from InterPro come from 11 member databases viz. CATH-Gene3D, HAMAP, PANTHER, Pfam, PIRSF, PRINTS, ProDom, PROSITE, SMART, SUPERFAMILY, TIGRFAMs.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about SCOP is incorrect regarding its features?`,
    options: ["Proteins with the same shapes but having little sequence or functional similarity are placed in different super families, and are assumed to have only a very distant common ancestor", "Proteins having the same shape and some similarity of sequence and/or function are placed in \u2018families\u2019, and are assumed to have a closer common ancestor", "SCOP was created in 1994 in the Centre of Protein Engineering and the University College London", "It aims to determine the evolutionary relationship between proteins"],
    correct: "c",
    explanation: `SCOP, Structural Classification of Proteins, was created in 1994 in the Centre of Protein Engineering and the Laboratory of Molecular Biology. It was maintained by Alexey G. Murzin and his colleagues in the Centre for Protein Engineering until its closure in 2010 and subsequently at the Laboratory of Molecular Biology in Cambridge, England.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `What is the source of protein structures in SCOP and CATH?`,
    options: ["Uniprot", "Protein Data Bank", "Ensemble", "InterPro"],
    correct: "b",
    explanation: `The source of protein structures in SCOP is PDB (Protein Data Bank). PDB is a secondary database which means it has protein structures derived from primary databases that have the protein sequences. UNIPROT is a primary database.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about SUPERFAMILY database is incorrect regarding its features?`,
    options: ["Sequences can be submitted raw or FASTA format", "Sequences must be submitted in FASTA format only", "It searches the database using a superfamily, family, or species name plus a sequence, SCOP, PDB or HMM ID\u2019s", "It has generated GO annotations for evolutionarily closed domains and distant domains"],
    correct: "b",
    explanation: `SUPERFAMILY is a database of structural and functional annotation for all proteins and genomes. It classifies amino acid sequences into known structural domains, especially into SCOP super families. Sequences can be amino acids, a fixed frame nucleotide sequence, or all frames of a submitted nucleotide sequence. Up to 1000 sequences can be run at a time.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about PRINTS and ProDom databases is incorrect regarding its features?`,
    options: ["PRINTS is a compendium of protein fingerprints", "Usually the motifs do not overlap, but are separated along a sequence, though they may be contiguous in 3D-space", "Current versions of ProDom are built using a novel procedure based on recursive BLAST searches", "ProDom domain database consists of an automatic compilation of homologous domains"],
    correct: "c",
    explanation: `Current versions of ProDom are built using a novel procedure based on recursive PSI-BLAST searches and not just BLAST searches. And PRINTS is indeed a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family; its diagnostic power is refined by iterative scanning of UniProt.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about CATH-Gene3D and HAMAP databases is incorrect regarding its features?`,
    options: ["CATH-Gene3D describes protein families and domain architectures in complete genomes", "In CATH-Gene3D the functional annotation is provided to proteins from single resource", "HAMAP profiles are manually created by expert curators they identify proteins that are part of well-conserved bacterial, archaeal and plastid-encoded proteins families or subfamilies.", "HAMAP stands for High-quality Automated and Manual Annotation of microbial Proteomes"],
    correct: "b",
    explanation: `In CATH-Gene3D Protein families are formed using a Markov clustering algorithm, followed by multi-linkage clustering according to sequence identity. Mapping of predicted structure and sequence domains is undertaken using hidden Markov models libraries representing CATH and Pfam domains. Functional annotation is provided to proteins from multiple resources. Functional prediction and analysis of domain architectures is available at the website.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following statements about PANTHER and TIGRFAMs databases is incorrect regarding its features?`,
    options: ["TIGRFAMs provides a tool for identifying functionally related proteins based on sequence homology", "TIGRFAMs is a collection of protein families, featuring curated multiple sequence alignments, hidden Markov models (HMMs) and annotation", "Hidden Markov models (HMMs) are not used in PANTHER", "PANTHER is a large collection of protein families that have been subdivided into functionally related subfamilies, using human expertise"],
    correct: "c",
    explanation: `In PANTHER the subfamilies model the divergence of specific functions within protein families, allowing more accurate association with function (human-curated molecular function and biological process classifications and pathway diagrams), as well as inference of amino acids important for functional specificity. Hidden Markov models (HMMs) are built for each family and subfamily for classifying additional protein sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `When did Needleman-Wunsch first describe the algorithm for global alignment?`,
    options: ["1899", "1970", "1930", "1950"],
    correct: "b",
    explanation: `Needleman and Wunsch were among the first to describe dynamic programming algorithm for global sequence. In global sequence alignment, an attempt to align the entirety of two different sequences is made, up to and including the ends of sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe dynamic programming?`,
    options: ["The approach compares every pair of characters in the two sequences and generates an alignment, which is the best or optimal", "Global alignment algorithm is based on this method", "Local alignment algorithm is based on this method", "The method can be useful in aligning protein sequences to protein sequences only"],
    correct: "d",
    explanation: `The method can be useful in aligning nucleotide to protein sequences as well. These programs first perform pair-wise alignment on each pair of sequences. Then, they perform local re-arrangements on these results, in order to optimize overlaps between multiple sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not an advantage of Needleman-Wunsch algorithm?`,
    options: ["New algorithmic improvements as well as increasing computer capacity make it possible to align a query sequence against a large DB in a few minutes", "Similar sequence region is of same order and orientation", "This does not help in determining evolutionary relationship", "If you have 2 genes that are already understood as closely related, then this type of algorithm can be used to understand them in further details"],
    correct: "c",
    explanation: `Needleman-Wunsch algorithm is used when 2 genes that are already understood as closely related and can be used to understand them in further details. This is quite helpful in finding orthologs, paralogs and homologs in evolutionary studies.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a disadvantage of Needleman-Wunsch algorithm?`,
    options: ["This method is comparatively slow", "There is a need of intensive memory", "This cannot be applied on genome sized sequences", "This method can be applied to even large sized sequences"],
    correct: "d",
    explanation: `This method cannot be applied on genome sized sequences. But this is indeed useful in determining similarities and evolutionary relationships.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe global alignment algorithm?`,
    options: ["It attempts to align every residue in every sequence", "It is most useful when the aligning sequences are similar and of roughly the same size", "It is useful when the aligning sequences are dissimilar", "It can use Needleman-Wunsch algorithm"],
    correct: "c",
    explanation: `Performing global alignment is most useful when the aligning sequences are similar and of roughly the same size. This is most useful to find the similarities among the organisms that are roughly connected on the timeline.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe PAM matrices?`,
    options: ["These matrices are used in optimal alignment scoring", "It stands for Point Altered Mutations", "It stands for Point Accepted Mutations", "It was first developed by Margaret Dayhoff"],
    correct: "b",
    explanation: `PAM stands for Point Accepted Mutations. PAM matrices are calculated by observing the differences in closely related proteins. One PAM unit (PAM1) specifies one accepted point mutation per 100 amino acid residues, i.e. 1% change and 99% remains as such.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding the scoring system used in dynamic programming?`,
    options: ["If the residues are same in both the sequences the match score is assumed as +5 which is added to the diagonally positioned cell of the current cell", "If the residues are not same, the mismatch score is assumed as -3", "If the residues are not same, the mismatch score is assumed as 3", "The score should be added to the diagonally positioned cell of the current cell"],
    correct: "c",
    explanation: `If the residues are not same, the mismatch score is assumed as -3 and it has to be negative. However, these scores are not unique, they can be user defined also, but the mismatch and gap penalty should be the negative values.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is wrong in case of substitution matrices?`,
    options: ["They determine likelihood of homology between two sequences", "They use system where substitutions that are more likely should get a higher score", "They use system where substitutions that are less likely should get a lower score", "BLOSUM-X type uses logarithmic identity to find similarity"],
    correct: "d",
    explanation: `BLOSUM-X type identifies sequences that are X% similar to the query sequence i. e. score 54 corresponds to 54% similarity hence reducing the complexity of the output and giving the similarity in percentage. Also, these matrices are popular in bioinformatics due to their speed and accuracy.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `When did Smith–Waterman first describe the algorithm for local alignment?`,
    options: ["1950", "1970", "1981", "1925"],
    correct: "c",
    explanation: `The algorithm was first proposed by Temple F. Smith and Michael S. Waterman in 1981. The Smith–Waterman algorithm performs local sequence alignment; that is, for determining similar regions between two strings of nucleic acid sequences or protein sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe local alignment?`,
    options: ["A local alignment aligns a substring of the query sequence to a substring of the target sequence", "A local alignment is defined by maximizing the alignment score, so that deleting a column from either end would reduce the score, and adding further columns at either end would also reduce the score", "Local alignments have terminal gaps", "The substrings to be examined may be all of one or both sequences; if all of both are included then the local alignment is also global"],
    correct: "c",
    explanation: `Local alignments never have terminal gaps, because a higher score could be obtained by deleting the gaps (which always have negative scores, i.e. penalties). In case of global alignment there are terminal gaps while analyzing.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe local alignment algorithm?`,
    options: ["Score can be negative", "Negative score is set to 0", "First row and first column are set to 0 in initialization step", "In traceback step, beginning is with the highest score, it ends when 0 is encountered"],
    correct: "a",
    explanation: `Score can be negative. When any element has a score lower than zero, it means that the sequences up to this position have no similarities; this element will then be set to zero to eliminate influence from previous alignment. In this way, calculation can continue to find alignment in any position afterward.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Local alignments are more used when _____________`,
    options: ["There are totally similar and equal length sequences", "Dissimilar sequences are suspected to contain regions of similarity", "Similar sequence motif with larger sequence context", "Partially similar, different length and conserved region containing sequences"],
    correct: "a",
    explanation: `The given description is suitable for global alignment. It attempts to align maximum of the entire sequence unlike local alignment where the partially similar sequences are analyzed.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe BLOSUM matrices?`,
    options: ["It stands for BLOcks SUbstitution Matrix", "It was developed by Henikoff and Henikoff", "The year it was developed was 1992", "These matrices are logarithmic identity values"],
    correct: "d",
    explanation: `These matrices are actual percentage identity values. Or simply, they depend on similarity. Blosum 62 means there is 62 % similarity.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding the gap penalty used in dynamic programming?`,
    options: ["Gap penalty is subtracted for each gap that has been introduced", "Gap penalty is added for each gap that has been introduced", "The gap score defines a penalty given to alignment when we have insertion or deletion", "Gap open and gap extension has been introduced when there are continuous gaps (five or more)"],
    correct: "b",
    explanation: `Dynamic programming algorithms use gap penalties to maximize the biological meaning. The open penalty is always applied at the start of the gap, and then the other gaps following it is given with a gap extension penalty which will be less compared to the open penalty. Typical values are –12 for gap opening, and –4 for gap extension.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Among the following which one is not the approach to the local alignment?`,
    options: ["Smith-Waterman algorithm", "K-tuple method", "Words method", "Needleman-Wunsch algorithm"],
    correct: "d",
    explanation: `Local alignment can be distinguished on two broad approaches, Smith-Waterman algorithm and word methods, also known as k-tuple methods and they are implemented in the well-known families of programs FASTA and BLAST.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe k-tuple methods?`,
    options: ["k-tuple methods are best known for their implementation in the database search tools FASTA and the BLAST family", "They are also known as words methods", "They are basically heuristic methods to find local alignment", "They are useful in small scale databases"],
    correct: "d",
    explanation: `k-tuple or word methods are especially useful in large-scale database searches where a large proportion of stored sequences will have essentially no significant match with the query sequence. They are heuristic methods that are not guaranteed to find an optimal alignment solution but are significantly more efficient than Smith-Waterman algorithm.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following does not describe BLAST?`,
    options: ["It stands for Basic Local Alignment Search Tool", "It uses word matching like FASTA", "It is one of the tools of the NCBI", "Even if no words are similar, there is an alignment to be considered"],
    correct: "d",
    explanation: `If no words are similar, there is no alignment i. e. it will not find matches for very short sequences. But it is considerably accurate as compared to other tools and hence is quite popular.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding BLAST and FASTA?`,
    options: ["FASTA is faster than BLAST", "FASTA is the most accurate", "BLAST has limited choices of databases", "FASTA is more sensitive for DNA-DNA comparisons"],
    correct: "a",
    explanation: `BLAST is faster than FASTA and most other tools. The speed and relatively good accuracy of BLAST is the key why the tool is the most popular bioinformatics search tool.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `For what type of sequences Gibbs sampling is used?`,
    options: ["Closely related sequences", "Distinctly related sequences", "Distinctly related sequences that share common motifs", "Closely related sequences that share common motifs"],
    correct: "c",
    explanation: `Often, distantly related sequences that share common motifs cannot be readily aligned. For example, the sequences for the helix-turn-helix motif in transcription factors can be subtly different enough that traditional multiple sequence alignment approaches fail to generate a satisfactory answer. For detecting such subtle motifs, more sophisticated algorithms such as expectation maximization (EM) and Gibbs sampling are used.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about Expectation Maximization (EM) method?`,
    options: ["It is used to find hidden motifs", "The method works by first making a random or guessed alignment of the sequences to generate a trial PSSM", "The trial PSSM is used to compare with each sequence individually", "The log odds scores of the PSSM are modified at the end of the process"],
    correct: "d",
    explanation: `The log odds scores of the PSSM are modified in each iteration to maximize the alignment of the matrix to each sequence. During the iterations, the sequence pattern for the conserved motifs is gradually “recruited” to the PSSM.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is true about Expectation Maximization (EM) method?`,
    options: ["The log odds scores of the PSSM are modified at the end of the process", "The procedure stops prematurely if the scores reach convergence", "The final result is not sensitive to the initial alignment", "Local optimum is an advantage of EM method"],
    correct: "b",
    explanation: `The final result is sensitive to the initial alignment. The Local optimum is actually a drawback of EM method. It is same as the fact that the procedure stops prematurely if the scores reach convergence.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `MEME stands for _____________`,
    options: ["Multiple Expectation Maximization for Motif Elicitation", "Multiple Expectation Maximization for Motif Extraction", "Mega Expectation Maximization for Motif Elicitation", "Micro Expectation Maximization for Motif Extraction"],
    correct: "a",
    explanation: `Multiple Expectation Maximization for Motif Elicitation is a web-based program that uses the EM algorithm to find motifs either for DNA or protein sequences. It uses amodified EM algorithm to avoid the local minimum problem.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In the web-based program MEME, the computation is a _____ step procedure.`,
    options: ["one", "two", "three", "four"],
    correct: "b",
    explanation: `In constructing a probability matrix, it allows multiple starting alignments and does not assume that there are motifs in every sequence. Also, the computation is a two-step procedure that includes generation of sequence motif and finding highest score.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Gibbs is a web-based program that uses the Gibbs sampling approach to look for _____ gap-free segments for either DNA or protein sequences.`,
    options: ["short, partially conserved", "long, partially conserved", "long, conserved", "short, not conserved"],
    correct: "a",
    explanation: `Gibbs sampling approach to look for short, partially conserved gap-free segments for either DNA or protein sequences. To ensure accuracy, more than twenty sequences of the exact same length should be used.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `A multiple sequence alignment or a motif is often represented by a graphic representation called a ______`,
    options: ["logo", "motto", "algorithm", "algo"],
    correct: "a",
    explanation: `In a logo, each position consists of stacked letters representing the residues appearing in a particular column of a multiple alignment. This graphic representation called a logo.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The overall height of a logo position reflects how conserved the position is, and the _____ of each letter in a position reflects the _______ of the residue in the alignment.`,
    options: ["height, relative frequency", "width, relative frequency", "height, amplitude", "width, amplitude"],
    correct: "a",
    explanation: `The height expresses the data about the extent of the conservation of the position and each letter shows the frequency of that particular residue. The amplitude, here in this case, is irrelevant option.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Conserved positions have _____ residues and bigger symbols.`,
    options: ["fewer", "more", "maximum", "minimum"],
    correct: "a",
    explanation: `The options maximum and minimum are comparatively obsolete as there involves the studies of alignment. Conserved positions have fewer residues and bigger symbols; whereas less conserved positions have a more heterogeneous mixture of smaller symbols stacked together. In general, a sequence logo provides a clearer description of a consensus sequence.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `_______ is an interactive program for generating sequence logos.`,
    options: ["EMBOSS", "WebLogo", "LOGOLY", "BLAST"],
    correct: "b",
    explanation: `In WebLogo, a user needs to enter the sequence alignment in FASTA format to allow the program to compute the logos. A graphic file is returned to the user as a result.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a software for dot plot analysis?`,
    options: ["SIMMI", "DOTLET", "DOTMATCHER", "LALIGN"],
    correct: "a",
    explanation: `For the purpose of dot plot interpretation there are various softwares currently present. Among these SIM is used for these kinds of alignments through dot-plot method that is wrongly abbreviated.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The softwares for dot plot analysis perform several tasks. Which one of them is not performed by them?`,
    options: ["Gap open penalty", "Gap extend penalty", "Expectation threshold", "Change or mutate residues"],
    correct: "d",
    explanation: `The gap penalties mentioned above are for the determination of score of the aligning sequences. The change in residue barely takes place as there are number of other softwares for that purpose and also the main objective is to find the score of the alignment.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `For palindromic sequences, what is the structure of the dot plot?`,
    options: ["2 intersecting diagonal lines at the midpoint", "One diagonal", "Two parallel diagonals", "No diagonal"],
    correct: "a",
    explanation: `For perfectly aligned sequences there is a diagonal formation of dot plot. For palindromic sequences i. e. for sequences that are symmetrical from the midpoint of the sequence, there exist 2 intersecting diagonals on the plot.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `For significantly aligning sequences what is the resulting structure on the plot?`,
    options: ["Intercrossing lines", "Crosses everywhere", "Vertical lines", "A diagonal and lines parallel to diagonal"],
    correct: "d",
    explanation: `If there is alignment of sequences there is a significantly bold diagonal visible on the plot. And if the is a bit imperfect, the diagonal is shattered too to an extent and forms small parallel lines to it.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `When was this method, first described?`,
    options: ["1959", "1966", "1970", "1982"],
    correct: "c",
    explanation: `This method was first described in 1970. Briefly, this method involves constructing a matrix with one of the sequences to be compared running horizontally across the bottom, and the other running vertically along the left-hand side.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Who were the inventors of this method?`,
    options: ["Smith-Waterman", "Margaret Preston", "Gibbs and McIntyre", "Needleman-Wunsch"],
    correct: "c",
    explanation: `The first computer aided sequence comparison is called “dot-matrix analysis” or simply dot-plot. The first published account of this method is by Gibbs and McIntyre (1970 The diagram, a method for comparing sequences. Eur. J. Biochem 16: 1-11).`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is true for EMBOSS Dottup?`,
    options: ["Allows you to specify threshold", "Doesn\u2019t allow you to specify threshold", "Doesn\u2019t allow you to specify window size", "If all cells in the window are identity, it colors in some specific cells in the window"],
    correct: "b",
    explanation: `The EMBOSS Dottup doesn’t allow you to specify threshold but allows you to specify window size. Also, if all cells in the window are identity, it colors in all the cells in the window.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Isolated dots that are not on the diagonal represent exact matches.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Those isolated dots represent random matches. The dots on the diagonal represent the perfect alignment and the dots with vertical and horizontal shifts show insertions and deletions.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Vertical frame shifts show ______ while the horizontal ones show _______`,
    options: ["insertion, insertion", "insertion, deletion", "deletion, deletion", "deletion, insertion"],
    correct: "b",
    explanation: `Deletion and insertion of nucleotides is quite common in alignment process. The dot plot easily represents them with vertical and horizontal shifts. And the mutations are totally out of the diagonal zone.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Dot plot of repeating elements would be small crosses on plot.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The repeating elements would be represented in parallel lines in repetitive manner. Better is the repetition; better is the nature of parallel lines. Also, the intersections show the pallindromic sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Use of the dynamic programming method requires a scoring system for the comparison of symbol pairs, and a scheme for GAP penalties.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Once those parameters have been set, the resulting alignment for two sequences should always be the same. Hence, the use of the dynamic programming method requires a scoring system for the comparison of symbol pairs (nucleotides for DNA sequences and amino acids for protein sequences), and a scheme for insertion/deletion (GAP) penalties.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `After the derivation, the outputs of the dynamic programming are the ratios are called even scores.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `After the derivation, the outputs of the dynamic programming are the ratios are called odd scores. The ratios are transformed to logarithms of odds scores, called log odds scores, so that scores of sequential pairs may be added to reflect the overall odds of a real to chance alignment of an alignment. This happens in Dayhoff PAM250 and BLOSUM62.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The matrices PAM250 and BLOSUM62 contain _______`,
    options: ["positive and negative values", "positive values only", "negative values only", "neither positive nor negative values, just the percentage"],
    correct: "a",
    explanation: `These matrices contain positive and negative values, reflecting the likelihood of each amino acid substitution in related proteins. Using these tables, an alignment of a sequential set of amino acid pairs with no gaps receives an overall score that is the sum of the positive and negative log odds scores for each individual amino acid pair in the alignment.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The higher is the score in the alignment _________`,
    options: ["the more significant is the alignment", "or the less it resembles alignments in related proteins", "the less significant is the alignment", "the less it aligns with the related protein sequence"],
    correct: "a",
    explanation: `In the scoring system, the higher this score, the more significant is the alignment, or the more it resembles alignments in related proteins. Also, the score given for gaps in aligned sequences is negative, because such misaligned regions should be uncommon in sequences of related proteins. Such a score will reduce the score obtained from an adjacent, matching region upstream in the sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Gaps are added to the alignment because it ______`,
    options: ["increases the matching of identical amino acids at subsequent portions in the alignment", "increases the matching of or dissimilar amino acids at subsequent portions in the alignment", "reduces the overall score", "enhances the area of the sequences"],
    correct: "a",
    explanation: `In the alignment process, gaps are added to the alignment in a manner that increases the matching of identical or similar amino acids at subsequent portions in the alignment. Ideally, when two similar protein sequences are aligned, the alignment should have long regions of identical or related amino acid pairs and very few gaps. As the sequences become more distant, more mismatched amino acid pairs and gaps should appear.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a description of dynamic programming algorithm?`,
    options: ["A method of sequence alignment", "A method that can take gaps into account", "A method that requires a manageable number of comparisons", "This method doesn\u2019t provide an optimal (highest scoring) alignment"],
    correct: "d",
    explanation: `The method of sequence alignment by dynamic programming provides an optimal (highest scoring) alignment as an output. The quality of the alignment between two sequences is calculated using a scoring system that favors the matching of related or identical amino acids and penalizes for poorly matched amino acids and gaps.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is not a site on internet for alignment of sequence pairs?`,
    options: ["BLASTX", "BLASTN", "SIM", "BCM Search Launcher"],
    correct: "a",
    explanation: `BLASTP is used under BLAST 2 sequence alignment. Also, The BLAST algorithm normally used for database similarity searches can also be used to align two sequences. SIM is known as Local similarity program for finding alternative alignments.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Dayhoff PAM matrices, are based on an evolutionary model of protein change, whereas, BLOSUM matrices, are designed to identify members of the same family.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `There is a very large number amino acid scoring matrices in use, some much more popular than others, and these scoring matrices are designed for different purposes. Some, such as the Dayhoff PAM matrices, are based on an evolutionary model of protein change, whereas others, such as the BLOSUM matrices, are designed to identify members of the same family. Alignments between DNA sequences require similar kinds of considerations.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `A feature of the dynamic programming algorithm is that the alignments obtained depend on the choice of a scoring system for comparing character pairs and penalty scores for gaps.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For an algorithm, the output depends on the choice of a scoring system. For protein sequences, the simplest system of comparison is one based on identity. A match in an alignment is only scored if the two aligned amino acids are identical. However, one can also examine related protein sequences that can be aligned easily and find which amino acids are commonly substituted for each other.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding dynamic programming algorithm?`,
    options: ["The method compares every pair of characters in the two sequences and generates an alignment", "The output alignment will include matched and mismatched characters and gaps in the two sequences that are positioned so that the number of matches between identical or related characters is the maximum possible", "The dynamic programming algorithm provides a reliable computational method for aligning DNA and protein sequences", "This doesn\u2019t allow making evolutionary predictions on the basis of sequence alignments"],
    correct: "d",
    explanation: `Optimal alignments provide useful information to biologists concerning sequence relationships by giving the best possible information as to which characters in a sequence should be in the same column in an alignment, and which are insertions in one of the sequences (or deletions on the other). This information is important for making functional, structural, and evolutionary predictions on the basis of sequence alignments.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In scoring matrices, for convenience, odds scores are converted to log odds scores.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The odds scores are converted to log odds scores so that the values for amino acid pairs in an alignment may be summed to obtain the log odds score of the alignment. In this case, the logarithms are calculated to the base 2 and multiplied by 2 to give values designated as half-bits (a bit is the unit of an odds score that has been converted to a logarithm to the base 2). The value of 4 indicates that the 4 amino acid alignment is 2(4/2) = 4 four-fold more likely than expected by chance.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following doesn’t describe PAM matrices?`,
    options: ["This family of matrices lists the likelihood of change from one amino acid to another in homologous protein sequences during evolution", "There is presently no other type of scoring matrix that is based on such sound evolutionary principles as are these matrices", "Even though they were originally based on a relatively small data set, the PAM matrices remain a useful tool for sequence alignment", "It stands for Percent Altered Mutation"],
    correct: "d",
    explanation: `PAM stands for Percent Accepted Mutation. In this, each matrix gives the changes expected for a given period of evolutionary time, evidenced by decreased sequence similarity as genes encoding the same protein diverge with increased evolutionary time.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The assumption in this evolutionary model is that the amino acid substitutions observed over short periods of evolutionary history can be extrapolated to longer distances.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The BLOSUM matrices are based on scoring substitutions found over a range of evolutionary periods and reveal that substitutions are not always as predicted by the PAM model. The purpose of assumption in this evolutionary model is to make predictions.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about the modification of PAM matrices?`,
    options: ["At one time, the PAM250 scoring matrix was modified in an attempt to improve the alignment obtained", "All scores for matching a particular amino acid were normalized to the same mean and standard deviation, and all amino acid identities were given the same score to provide an equal contribution for each amino acid in a sequence alignment", "This took place in 1976", "These modifications were included as the default matrices for the GCG sequence alignment programs in versions 8 and earlier and are optional in later versions"],
    correct: "c",
    explanation: `This event took place in 1986 by Gribskov and Burgess. However, they are not recommended because they will not give an optimal alignment that is in accordance with the evolutionary model.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The Dayhoff model of protein evolution is not a Markov process.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The Dayhoff Model of Protein Evolution as Used in PAM Matrices is a Markov process. In Analysis of the Dayhoff Model, each amino acid site in a protein can change at any time to any of the other 20 amino acids with probabilities given by the PAM table, and the changes that occur at each site are independent of the amino acids found at other sites in the protein and depend only on the current amino acid at the site.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is true regarding the assumptions in the method of constructing the Dayhoff scoring matrix?`,
    options: ["it is assumed that each amino acid position is equally mutable", "it is assumed that each amino acid position is not equally mutable", "it is assumed that each amino acid position is not mutable at all", "sites do not vary in their degree of mutability"],
    correct: "a",
    explanation: `In this process, first, it is assumed that each amino acid position is equally mutable, whereas, in fact, sites vary considerably in their degree of mutability. Mutagenesis hot spots are well known in molecular genetics, and variations in mutability of different amino acid sites in proteins are well known.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The more conserved amino acids in similar proteins from different species are ones that play an essential role in structure and function and the less conserved are in sites that can vary without having a significant effect on function.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `There are many factors that influence both the location and types of amino acid changes that occur in proteins. Wilbur (1985) has tested the Markov model of evolution and has shown that it can be valid if certain changes are made in the way that the PAM matrices are calculated.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `A gap opening penalty for any gap (g) and a gap extension penalty for each element in the gap (r) are most often used, to give a total gap score wx, according to the equation ______`,
    options: ["wx \u2013 rx = -g", "wx = g \u2013 rx", "wx = g + rx", "wx + g + rx = 0"],
    correct: "c",
    explanation: `wx = g + rx is the equation where x is the length of the gap. in some formulations of the gap penalty, the equation wx = g + r (x – 1) is used. Thus, the gap extension penalty is not added to the gap opening penalty until the gap size is 2.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In the GCG and FASTA program suites, the scoring matrix itself is formatted in a way that includes default ______`,
    options: ["gap additions", "alignment scores", "score penalties", "gap penalties"],
    correct: "d",
    explanation: `These program suites include default gap penalties. When deciding gap penalties for local alignment programs, a consideration is that the penalties should be large enough to provide a local alignment of the sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In case of the varying alignment, penalizing gaps heavily might occur. Then the best scoring local alignment between the sequences will be one that optimizes the score between matches and mismatches, without any gaps.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If both mismatches and gaps are heavily penalized, the resulting alignment will also be a local alignment that contains the longest region of exact matches. In the above two cases, the alignment score of the highest-scoring local alignment will increase as the logarithm of the length of the sequences. Under these same conditions, the score of the corresponding global alignment between the sequences will be negative.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `On analysis of the alignment scores of random sequences will reveal that the scores follow a different distribution than the normal distribution called the _________`,
    options: ["Gumbel equal value distribution", "Gumbel extreme value distribution", "Gumbel end value distribution", "Gumbel distribution"],
    correct: "b",
    explanation: `Originally, the significance of sequence alignment scores was evaluated on the basis of the assumption that alignment scores followed a normal statistical distribution. If sequences are randomly generated in a computer by a Monte Carlo or sequence shuffling method, as in generating a sequence by picking marbles representing four bases or 20 amino acids out of a bag, the distribution may look normal at first glance. But on further analysis the above result was obtained.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The statistical analysis of alignment scores is much better understood for ________ than for _______`,
    options: ["global alignments, local alignments", "local alignments, global alignments", "global alignments, any other alignment method", "Needleman-Wunsch alignment, Smith-Waterman alignment"],
    correct: "b",
    explanation: `Smith-Waterman alignment algorithm and the scoring system used to produce a local alignment are designed to reveal regions of closely matching sequence with a positive alignment score. In random or unrelated sequence alignments, these regions are rarely found. Hence, their presence in real sequence alignments is significant, and the probability of their occurring by chance alignment of unrelated sequences can be readily calculated.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `When random or unrelated sequences are compared using a global alignment method, they can have ____________ reflecting the tendency of the global algorithm to match as many characters as possible.`,
    options: ["very low scores", "very high scores", "moderate scores", "low scores"],
    correct: "b",
    explanation: `The significance of the scores of global alignments, is more difficult to determine. Using the Needleman-Wunsch algorithm and a suitable scoring system, there are many ways to produce a global alignment between any pair of sequences, and the scores of many different alignments may be quite similar hence the scores obtained might be unusually high.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following are not related to Needleman-Wunsch alignment algorithm?`,
    options: ["Global alignment programs use this algorithm", "The output is a positive number", "Small changes in the scoring system can produce a different alignment", "Changes in the scoring system can produce the same alignment"],
    correct: "d",
    explanation: `In general, global alignment programs use the Needleman-Wunsch alignment algorithm and a scoring system that scores the average match of an aligned nucleotide or amino acid pair as a positive number. Hence, the score of the alignment of random or unrelated sequences grows proportionally to the length of the sequences. In addition, there are many possible different global alignments depending on the scoring system chosen, and small changes in the scoring system can produce a different alignment.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Waterman, in1989, provided a set of means and standard deviations of global alignment scores between random DNA sequences, using mismatch and gap penalties that produce a linear increase in score with _______ a distinguishing feature of global alignments.`,
    options: ["alignment score", "sequence score", "sequence length", "scoring system"],
    correct: "c",
    explanation: `In the algorithm provided by Waterman, the score of the alignment of random or unrelated sequences grows proportionally to the length of the sequences. However, these values are of limited use because they are based on a simple gap scoring system.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Who suggested that the global alignment scores between unrelated protein sequences followed the extreme value distribution, similar to local alignment scores? And when?`,
    options: ["Abagyan and Batalov, in 1981", "Chv\u00e1tal and Lipman, in 1984", "Abagyan and Batalov, in 1997", "Chv\u00e1tal and Sankoff, in 1995"],
    correct: "c",
    explanation: `Abagyan and Batalov, in 1997, suggested the above observation. However, since the scoring system that they used favored local alignments, these alignments they produced may not be global but local. Unfortunately, there is no equivalent theory on which to base an analysis of global alignment scores as there is for local alignment scores.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `_______ analyzed the distribution of scores among 100 vertebrate nucleic acid sequences and compared these scores with randomized sequences prepared in different ways.`,
    options: ["Lipman, in 1984", "Batalov, in 1964", "Waterman, in 1987", "Lipman, in 1967"],
    correct: "a",
    explanation: `When the randomized sequences were prepared by shuffling the sequence to conserve base composition, as was done by Dayhoff and others, the standard deviation was approximately one-third less than the distribution of scores of the natural sequences. Thus, natural sequences are more variable than randomized ones, and using such randomized sequences for a significance test may lead to an overestimation of the significance.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `If the random sequences were prepared in a way that maintained the local base composition by producing them from overlapping fragments of sequence, the distribution of scores has a _______ standard deviation that is closer to the distribution of the natural sequences.`,
    options: ["lowest", "higher", "lower", "moderate"],
    correct: "c",
    explanation: `The conclusion from the above is that the presence of conserved local patterns can influence the score in statistical tests such that an alignment can appear to be more significant than it actually is. Although this study was done using the Smith-Waterman algorithm with nucleic acids, the same cautionary note applies for other types of alignments.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `The GCG alignment programs have a RANDOMIZATION option, which shuffles the second sequence and calculates similarity scores between the unshuffled sequence and each of the shuffled copies.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If the new similarity scores are significantly smaller than the real alignment score, the alignment is considered significant. This analysis is only useful for providing a rough approximation of the significance of an alignment score and can easily be misleading.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Dayhoff, 1978- 1983, devised a second method for testing the relatedness of two protein sequences that can accommodate some local variation. Where this method is useful?`,
    options: ["For finding repeated regions within a sequence", "For finding similar regions that are in a different order in two sequences", "For finding small conserved region such as an active site", "For finding huge regions within sequences"],
    correct: "d",
    explanation: `As used in a computer program called RELATE (Dayhoff 1978), all possible segments of a given length of one sequence are compared with all segments of the same length from another. An alignment score using a scoring matrix is obtained for each comparison to give a score distribution among all of the segments. A segment comparison score in standard deviation units is calculated as the difference between the values for real sequences minus the average value for random sequences divided by the standard deviation of the scores from the random sequences.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `By whom and when were the Bayesian methods applied first?`,
    options: ["Smith-Waterman, 1981", "Agarwal and States, 1996", "Smith-Waterman, 1996", "Agarwal and States, 1981"],
    correct: "b",
    explanation: `Agarwal and States, in1996, have applied Bayesian methods to provide the best estimate of the evolutionary distance between two DNA sequences. For example, sequences of the same length that have a certain level of mismatches.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `With the application of Bayesian methods, the most probable repeat length and evolutionary time since the repeat was formed may be derived.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Sequences of this type originated from gene duplication events in the yeast and Caenorhabditis elegans genomes. When there are multiple mismatches between such repeated sequences, it is difficult to determine the most likely length of the repeats. Here the methods can be used.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `If the purpose is to calculate the probability of one event AND a second event, the odds scores for the events are _________`,
    options: ["added", "multiplied", "multiplied and added", "subtracted"],
    correct: "b",
    explanation: `An example is the calculation of the odds of an alignment of two sequences from the alignment scores for each of the matched pairs of bases or amino acids in the alignment. The odds scores for the pairs are multiplied. Usually, the log odds score for the first pair is added to that for the second, etc., until the scores for every pair have been added.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In a type of probability, analysis is to calculate the odds score for one event OR a second event, or of a series of events. In this case, the odds scores are _______`,
    options: ["multiplied", "subtracted", "added and multiplied", "added"],
    correct: "d",
    explanation: `An example is the calculation of the odds score for a given sequence alignment using a series of alternative PAM scoring matrices. The alignment scores are calculated in log odds units and then converted into odds scores.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `In Bayesian methods, difficulty with making estimations is that the estimate depends on the following Assumption. (Assumption – The mutation rate in sequences has been constant with time and that the rate of mutation of all nucleotides is the same.)`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The assumption mentioned above (the molecular clock hypothesis) is made to reduce the complications. Such problems may be solved by scoring different portions of a sequence with a different scoring matrix, and then using the above Bayesian methods to calculate the best evolutionary distance.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Another difficulty in Bayesian methods is deciding on the length of sequence that was duplicated.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In genomes, the presence of repeats may be revealed by long regions of matched sequence positions dispersed among regions of sequence positions that do not match. However, as the frequency of mismatches is increased, it becomes difficult to determine the extent of the repeated region.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `A length and distance that gives the highest overall probability may then be determined. Such alignments are initially found using ________`,
    options: ["a particular scoring matrix only", "an alignment algorithm only", "an alignment algorithm and a particular scoring matrix", "dot method"],
    correct: "c",
    explanation: `Analysis of the yeast and C. elegans genomes for such repeats has underscored the importance of using a range of DNA scoring matrices such as PAM1 to
PAM120 if most repeats are to be found. The application of the above Bayesian analysis allows a determination of the probability distributions as a function of both length of the repeated region and evolutionary distance.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following feature of Bayesian methods is the disadvantage of it?`,
    options: ["A length and distance that gives the highest overall probability may be determined", "They are used to calculate evolutionary distance", "Computationally Bayesian methods are better", "A specific mutational model is required"],
    correct: "d",
    explanation: `One disadvantage of the Bayesian approach is that a specific mutational model is required, whereas other methods, such as the maximum likelihood approach, can be used to estimate the best mutational model as well as the distance. Computationally, however, the Bayesian method is much more practical.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Zhu (1998) have devised a computer program called the Bayes block aligner which in effect slides ____ sequences along each other to find the ______ ungapped regions or blocks.`,
    options: ["two, least scoring", "two, highest scoring", "multiple, highest scoring", "multiple, least scoring"],
    correct: "b",
    explanation: `These blocks are then joined in various combinations to produce alignments.
There is no need for gap penalties because only the aligned sequence positions in blocks are scored. Instead of using a given substitution matrix and gap scoring system to find the highest scoring alignment, a Bayesian statistical approach is used.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Unlike the commonly used methods for aligning a pair of sequences, the Bayesian method _______ using a particular scoring matrix or designated gap penalties.`,
    options: ["does not depend on", "depends on", "is based on", "involves"],
    correct: "a",
    explanation: `Because it doesn’t depend on the mentioned techniques, there is no need to choose a particular scoring system or gap penalty. Instead, a number of different scoring matrices and range of block numbers up to some reasonable maximum are examined, and the most probable alignments are determined. The Bayesian method provides a distribution of alignments weighted according to probability and can also provide an estimate of the evolutionary distance between the sequences that is independent of scoring matrix and gaps.`,
    super_topic: `Sequence Alignment`,
  },
  {
    question: `Which of the following is incorrect regarding pair wise sequence alignment?`,
    options: ["The most fundamental process in this type of comparison is sequence alignment", "It is an important first step toward structural and functional analysis of newly determined sequences", "This is the process by which sequences are compared by searching for common character patterns and establishing residue-residue correspondence among related sequences", "It is the process of aligning multiple sequences"],
    correct: "d",
    explanation: `Pair wise sequence alignment is the process of aligning two sequences and is the basis of database similarity searching and multiple sequence alignment. As new biological sequences are being generated at exponential rates, sequence comparison is becoming increasingly important to draw functional and evolutionary inference of a new protein with proteins already existing in the database.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Which of the following is incorrect about evolution?`,
    options: ["The macromolecules can be considered molecular fossils that encode the history of millions of years of evolution", "The building blocks of these biological macromolecules, nucleotide bases, and amino acids form linear sequences that determine the primary structure of the molecules", "DNA and proteins are products of evolution", "The molecular sequences barely undergo changes"],
    correct: "d",
    explanation: `During this time period, the molecular sequences undergo random changes, some of which are selected during the process of evolution. As the selected sequences gradually accumulate mutations and diverge over time, traces of evolution may still remain in certain portions of the sequences to allow identification of the common ancestry.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `The presence of evolutionary traces is because some of the residues that perform key functional and structural roles tend to be preserved by natural selection; other residues that may be less crucial for structure and function tend to mutate more frequently.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The residues that perform key functional and structural roles tend to be preserved by natural selection. For example, active site residues of an enzyme family tend to be conserved because they are responsible for catalytic functions. Therefore, by comparing sequences through alignment, patterns of conservation and variation can be identified.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `The degree of sequence variation in the alignment reveals evolutionary relatedness of different sequences, whereas the conservation between sequences reflects the changes that have occurred during evolution in the form of substitutions, insertions, and deletions.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The degree of sequence conservation in the alignment reveals evolutionary relatedness of different sequences, whereas the variation between sequences reflects the changes that have occurred during evolution in the form of substitutions, insertions, and deletions. Identifying the evolutionary relationships between sequences helps to characterize the function of unknown sequences. When a sequence alignment reveals significant similarity among a group of sequences, they can be considered as belonging to the same family.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `If the two sequences share significant similarity, it is extremely ______ that the extensive similarity between the two sequences has been acquired randomly, meaning that the two sequences must have derived from a common evolutionary origin.`,
    options: ["unlikely", "possible", "likely", "relevant"],
    correct: "a",
    explanation: `Sequence alignment provides inference for the relatedness of two sequences under study. Regions that are aligned but not identical represent residue substitutions; regions which residues from one sequence correspond to nothing in the other represent insertions or deletions that have taken place on one of the sequences during evolution.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Sometimes, it is also possible that two sequences have derived from a common ancestor, but may have diverged to such an extent that the common ancestral relationships are not recognizable at the sequence level.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `There are examples of such paralogous genes that have distinct functions but similar origin. In that case, the distant evolutionary relationships have to be detected using other methods.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect regarding sequence homology?`,
    options: ["Two sequences can homologous relationship even if have do not have common origin", "It is an important concept in sequence analysis", "When two sequences are descended from a common evolutionary origin, they are said to have a homologous relationship", "When two sequences are descended from a common evolutionary origin, they are said to share homology"],
    correct: "a",
    explanation: `Homologous relationships are more certain when the sequences have common evolutionary origin. A related but different term is sequence similarity, which is the percentage of aligned residues that are similar in physiochemical properties such as size, charge, and hydrophobicity.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Sequence similarity can be quantified using ________ homology is a ______ statement.`,
    options: ["percentages, quantitative", "percentages, qualitative", "ratios, qualitative", "ratios, quantitative"],
    correct: "b",
    explanation: `Similarity is a direct result of observation from the sequence Alignment. For example, one may say that two sequences share 40% similarity. It is incorrect to say that the two sequences share 40% homology. They are either homologous or nonhomologous.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Shorter sequences require higher cutoffs for inferring homologous relationships than longer sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For determining a homology relationship of two protein sequences, for example, if both sequences are aligned at full length, which is 100 residues long, an identity of 30% or higher can be safely regarded as having close homology. If their identity level falls between 20% and 30%, determination of homologous relationships in this range becomes less certain.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Sequence similarity and sequence identity are synonymous for nucleotide sequences and protein sequences as well.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Sequence similarity and sequence identity are synonymous for nucleotide sequences. For protein sequences, however, the two concepts are very different. In a protein sequence alignment, sequence identity refers to the percentage of matches of the same amino acid residues between two aligned sequences. Similarity refers to the percentage of aligned residues that have similar physicochemical characteristics and can be more readily substituted for each other.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `The overall goal of pair wise sequence alignment is to find the best pairing of two sequences, such that there is maximum correspondence among residues.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The goal of pair wise sequence alignment is to find the best pairing and to achieve this goal; one sequence needs to be shifted relative to the other to find the position where maximum matches are found. There are two different alignment strategies that are often used: global alignment and local alignment.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `In local alignment, the two sequences to be aligned cannot be of unequal lengths.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The two sequences to be aligned can be of different lengths. This approach is more appropriate for aligning divergent biological sequences containing only modules that are similar, which are referred to as domains or motifs. This approach can be used for aligning more divergent sequences with the goal of searching for conserved patterns in DNA or protein sequences.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Alignment algorithms, both global and local, are fundamentally similar and only differ in the optimization strategy used in aligning similar residues.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Both types of algorithms can be based on one of the three methods: the dot matrix method, the dynamic programming method, and the word method. The word method is used in fast database similarity searching.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `In a dot matrix, two sequences to be compared are written in the _____________ of the matrix.`,
    options: ["horizontal and vertical axes", "2 parallel horizontal axes", "2 parallel vertical axes", "horizontal axis (one preceding another)"],
    correct: "a",
    explanation: `The comparison is done by scanning each residue of one sequence for similarity with all residues in the other sequence. If a residue match is found, a dot is placed within the graph. Otherwise, the matrix positions are left blank.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `When the two sequences have substantial regions of similarity, many dots line up to form contiguous _______ lines.`,
    options: ["crossings on", "horizontal", "diagonal", "vertical"],
    correct: "c",
    explanation: `The dots line up to form contiguous diagonal lines, which reveal the sequence alignment. If there are interruptions in the middle of a diagonal line, they indicate insertions or deletions. Parallel diagonal lines within the matrix represent repetitive regions of the sequences.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `A problem exists when comparing _____ sequences using the dot matrix method, namely, the _______`,
    options: ["small, amplification", "large, amplification", "small, high noise level", "large, high noise level"],
    correct: "d",
    explanation: `In most dot plots, dots are plotted all over the graph obscuring identification of the true alignment. For DNA sequences, the problem is particularly acute because there are only four possible characters in DNA and each residue therefore has a one-in-four chance of matching a residue in another sequence.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `If the selected window size is too long, sensitivity of the alignment is lost.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Dots are only placed when a stretch of residues equal to the window size from one sequence matches completely with a stretch of another sequence. This method has been shown to be effective in reducing the noise level. The window is also called a tuple, the size of which can be manipulated so that a clear pattern of sequence match can be plotted. However, if the selected window size is too long, sensitivity of the alignment is lost.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `A sequence can be aligned with itself to identify internal repeat elements.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In the self comparison, there is a main diagonal for perfect matching of each residue. If repeats are present, short parallel lines are observed above and below the main diagonal.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Self complementarity of DNA sequences cannot be identified using a dot plot.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Self complementarity of DNA sequences, also called inverted repeats can be identified using a dot plot. For example, those that forms the stems of a hairpin structure. In this case, a DNA sequence is compared with its reverse-complemented sequence. Parallel diagonals represent the inverted repeats.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about dot plot method and its applications?`,
    options: ["This method gives a direct visual statement of the relationship between two sequences", "One of its advantages is the identification of sequence repeat regions based on the presence of parallel diagonals of the same size vertically or horizontally in the matrix", "It is not useful in identifying chromosomal repeats", "The method can be used in identifying nucleic acid secondary structures through detecting self-complementarity of a sequence"],
    correct: "c",
    explanation: `It is useful in identifying chromosomal repeats and in comparing gene order conservation between two closely related genomes. The dot matrix method gives a direct visual statement of the relationship between two sequences and helps easy identification of the regions of greatest similarities. The method thus has some applications in genomics.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `The truly statistically significant sequence alignment will be able to provide evidence of homology between the sequences involved.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `When given a sequence alignment showing a certain degree of similarity, it is often important to determine whether the observed sequence alignment can occur by random chance or the alignment is indeed statistically sound. When a statistically significant sequence alignment is under consideration, it will be able to provide evidence of homology between the sequences involved.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `By calculating alignment scores of a large number of ______ sequence pairs, a distribution model of the ______ sequence scores can be derived.`,
    options: ["related, randomized", "unrelated, randomized", "unrelated, unrandomized", "related, unrandomized"],
    correct: "b",
    explanation: `Solving the statistical significance problem requires a statistical test of the alignment scores of two unrelated sequences of the same length. From the distribution, a statistical test can be performed based on the number of standard deviations from the average score.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Many studies have demonstrated that the distribution of similarity scores assumes a peculiar shape that resembles a highly skewed normal distribution with a long tail on one side. The distribution matches the _______`,
    options: ["Gumble elective value distribution", "Gumble extreme void distribution", "Gumble end value distribution", "Gumble extreme value distribution"],
    correct: "d",
    explanation: `The mentioned Distribution pattern matches the Gumble extreme value distribution for which a mathematical expression is available. This means that, given a sequence similarity value, by using the mathematical formula for the extreme distribution, the statistical significance can be accurately estimated.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Which of the following is a part of the statistical test of sequences?`,
    options: ["An optimal alignment between two chosen sequences is obtained at the end", "Unrelated sequences of the same length are then generated through a randomization process", "Unrelated sequences of the different length are then generated through a randomization process", "Related sequences of the same length are then generated through a randomization process"],
    correct: "b",
    explanation: `Unrelated sequences of the same length are then generated through a randomization process in which one of the two sequences is randomly shuffled. And the next step is that a new alignment score is computed for the shuffled sequence pair.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `In the statistical test, randomization process in which one of the two given sequences is randomly shuffled.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `After the mentioned step, computation for the alignment score for the shuffled sequence pair is done. Further, More such scores are similarly obtained through repeated shuffling.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `What is used to generate parameters for the extreme distribution?`,
    options: ["The pool of alignment scores from the shuffled sequences", "A single score of a shuffled sequence", "The pool of alignment scores from the unshuffled sequences", "The basic optimal score computed at the beginning of the test"],
    correct: "a",
    explanation: `Maximum scores are obtained through repeated shuffling. Then the pool of alignment scores from the shuffled sequences is used to generate parameters for the extreme distribution. The original alignment score is then compared against the distribution of random alignments to determine whether the score is beyond random chance.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `If the score is located in the extreme margin of the distribution, that means that the alignment between the two sequences is ______ due to random chance and is thus considered ______`,
    options: ["unlikely, significant", "unlikely, insignificant", "unlikely, insignificant", "very likely, significant"],
    correct: "a",
    explanation: `The extreme margin of the distribution denotes the likeliness and thus significance. A P-value is given to indicate the probability that the original alignment is due to random chance.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `It is not known whether the Gumble distribution applies equally well to gapped alignments.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The statistics in the test were derived from ungapped local sequence alignments. Hence, it is not known whether the Gumble distribution applies equally well to gapped alignments. However, for all practical purposes, it is reasonable to assume that scores for gapped alignments essentially fit the same distribution. A frequently used software program for assessing statistical significance of a pairwise alignment is the PRSS program.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about the PRSS program?`,
    options: ["It stands for Probability of Random Shuffles", "It is a web-based program that can be used to evaluate the statistical significance of DNA or protein sequence alignment", "It first aligns two sequences using the Needleman-Wunsch algorithm and calculates the score", "It holds one sequence in its original form and randomizes the order of residues in the other sequence."],
    correct: "c",
    explanation: `It first aligns two sequences using the Smith–Waterman algorithm and calculates the score. The shuffled sequence is realigned with the unshuffled sequence. The resulting alignment score is recorded. This process is iterated many (normally 1,000) times to help generate data for fitting the Gumble distribution.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `The major disadvantage of the PRSS program is that it doesn’t allow partial shuffling.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The major feature of the program is that it allows partial shuffling. For example, shuffling can be restricted to residues within a local window of 25–40, whereas the residues outside the window remain unchanged.`,
    super_topic: `Pairwise Sequence Alignment`,
  },
  {
    question: `Related sequences are identified through the database similarity searching and as the process generates multiple matching sequence pairs, it is often necessary to convert the numerous pair wise alignments into a single alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A natural extension of pair wise alignment is multiple sequence alignment, which is to align multiple related sequences to achieve optimal matching of the sequences. Related sequences are identified through the database similarity searching. As the process generates multiple matching sequence pairs, it is often necessary to convert the numerous pair wise alignments into a single alignment, which arranges sequences in such a way that evolutionarily equivalent positions across all sequences are matched.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `There is a unique advantage of multiple sequence alignment because it reveals more biological information than many pair wise alignments can.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is truly an advantage of multiple sequence alignment. For example, it allows the identification of conserved sequence patterns and motifs in the whole sequence family, which are not obvious to detect by comparing only two sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following cannot be related to multiple sequence alignment?`,
    options: ["Many conserved and functionally critical amino acid residues can be identified in a protein multiple alignment", "Multiple sequence alignment is also an essential prerequisite to carrying out phylogenetic analysis of sequence families and prediction of protein secondary and tertiary structures", "Multiple sequence alignment also has applications in designing degenerate polymerase chain reaction (PCR) primers based on multiple related sequences", "This method does not contribute much to degenerate polymerase chain reaction (PCR) primers creation"],
    correct: "d",
    explanation: `In practice, heuristic approaches are most often used. Multiple sequence alignment has applications in designing degenerate (PCR) primers based on multiple related sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The scoring function for multiple sequence alignment is based on the concept of sum of pairs (SP).`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Multiple sequence alignment is to arrange sequences in such a way that a maximum number of residues from each sequence are matched up according to a particular scoring function and is based on the concept of sum of pairs (SP). As the name suggests, it is the sum of the scores of all possible pairs of sequences in a multiple alignment based on a particular scoring matrix.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following scores are not considered while calculating the SP scores?`,
    options: ["All possible pair wise matches", "All possible mismatches", "All possible gap costs", "Number of gap penalties"],
    correct: "d",
    explanation: `In calculating the SP scores, each column is scored by summing the scores for all possible pair wise matches, mismatches and gap costs. The score of the entire alignment is the sum of all of the column scores. The score of the entire alignment is the sum of all of the column scores. In that case, option d becomes irrelevant choice here.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Given a multiple alignment of three sequences, the sum of scores is calculated as the sum of the dissimilarity scores of every pair of sequences at each position.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Given a multiple alignment of three sequences, the sum of scores is calculated as the sum of the similarity scores of every pair of sequences at each position. The scoring is based on the BLOSUM62 matrix. If the total score for the alignment is 5, which means that the alignment is 25 = 32 times more likely to occur among homologous sequences than by random chance.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `There are two approaches viz. exhaustive and heuristic approaches used in multiple sequence alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The exhaustive alignment method involves examining all possible aligned positions simultaneously. Similar to dynamic programming in pair wise alignment, which involves the use of a two-dimensional matrix to search for an optimal alignment, to use dynamic programming for multiple sequence alignment, extra dimensions are needed to take all possible ways of sequence matching into consideration.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In a multidimensional search matrix, for aligning N sequences, an (N+2)-dimensional matrix is needed to be filled with alignment scores.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In a multidimensional search matrix, for aligning N sequences, an N-dimensional matrix is needed to be filled with alignment scores. For instance, for three sequences, a three-dimensional matrix is required to account for all possible alignment scores. Back-tracking is applied through the three-dimensional matrix to find the highest scored path that represents the optimal alignment.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `As the amount of computational time and memory space required increases exponentially with the number of sequences, it makes the multidimensional search matrix method computationally prohibitive to use for a large data set.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is indeed the drawback of that method. For this reason, full dynamic programming is limited to small datasets of less than ten short sequences. For the same reason, few multiple alignment programs employing this “brute force” approach are publicly available.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about DCA?`,
    options: ["It stands for Divide-and-Conquer Alignment", "It works by breaking each of the sequences into two smaller sections", "The breaking points during the process are determined based on regional similarity of the sequences", "If the sections are not short enough, further divisions are restricted as well"],
    correct: "d",
    explanation: `This is a web-based program that is in fact semi exhaustive because certain steps of computation are reduced to heuristics. If the sections are not short enough, further divisions are carried out. When the lengths of the sequences reach a predefined threshold, dynamic programming is applied for aligning each set of subsequences. The resulting short alignments are joined together head to tail to yield a multiple alignment of the entire length of all sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding the Progressive Alignment Method?`,
    options: ["Progressive alignment depends on the stepwise assembly of multiple alignments and is heuristic in nature", "It speeds up the alignment of multiple sequences through a multistep process", "It first conducts pair wise alignments for each possible pair of sequences using the Needleman\u2013Wunsch global alignment method and records these similarity scores from the pair wise comparisons", "Its drawback is it slows down the alignment of multiple sequences through a single step process"],
    correct: "d",
    explanation: `Progressive alignment speeds up the alignment of multiple sequences through a multistep process further, the scores can either be percent identity or similarity scores based on a particular substitution matrix. Both scores correlate with the evolutionary distances between sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Clustal is a progressive multiple alignment program available either as a stand-alone or on-line program.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Probably the most well-known progressive alignment program is Clustal. The stand-alone program, which runs on UNIX and Macintosh, has two variants, Clustal W and Clustal X. The W version provides a simple text-based interface and the X version provides a more user-friendly graphical interface.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding the progressive alignment method?`,
    options: ["The program also applies a weighting scheme to increase the reliability of aligning divergent sequences (sequences with less than 25% identity)", "The progress is done by down weighting redundant and closely related groups of sequences in the alignment by a certain factor", "This scheme is useful in enhancing similar sequences from dominating the alignment", "This scheme is useful in enhancing similar sequences from dominating the alignment"],
    correct: "c",
    explanation: `This scheme is useful in enhancing similar sequences from dominating the alignment. Further, the weight factor for each sequence is determined by its branch length on the guide tree. The branch lengths are normalized by how many times sequences share a basal branch from the root of the tree.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not a drawback of the progressive alignment method?`,
    options: ["The progressive alignment method is not suitable for comparing sequences of different lengths because it is a global alignment\u2013based method", "In this method the use of affine gap penalties, long gaps are not allowed, and, in some cases, this may limit the accuracy of the method", "In this method the use of affine gap penalties, long gaps is allowed, and, in some cases, this may limit the accuracy of the method", "The final alignment result is also influenced by the order of sequence addition"],
    correct: "c",
    explanation: `Another major limitation is the “greedy” nature of the algorithm: it depends on initial pair wise alignment. Once gaps introduced in the early steps of alignment, they are fixed. The final alignment could be far from optimal. The problem can be more glaring when dealing with divergent sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about iterative approach?`,
    options: ["The iterative approach is based on the idea that an optimal solution can be found by repeatedly modifying existing suboptimal solutions", "Because the order of the sequences used for alignment is different in each iteration", "This method is also heuristic in nature and does not have guarantees for finding the optimal alignment", "This method is not based on heuristic methods"],
    correct: "d",
    explanation: `This method is based on heuristic methods. The procedure starts by producing a low-quality alignment and gradually improves it by iterative realignment through well-defined procedures until no more improvements in the alignment scores can be achieved.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about PRRN?`,
    options: ["PRRN is a web-based program that uses a double nested iterative strategy for multiple alignment", "It performs multiple alignments through two sets of iterations: inner iteration and outer iteration", "In the outer iteration, an initial random alignment is generated that is used to derive a UPGMA tree", "In the inner iteration, the sequences are randomly divided into multiple groups"],
    correct: "d",
    explanation: `In the inner iteration, the sequences are randomly divided into two groups. Randomized alignment is used for each group in the initial cycle, after which the alignment positions in each group are fixed. The two groups, each treated as a single sequence, are then aligned to each other using global dynamic programming. The process is repeated through many cycles until the total SP score no longer increases. At this point, the resulting alignment is used to construct a new UPGMA tree.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The major drawback of the progressive and iterative alignment strategies is that they are largely global alignment based and may therefore fail to recognize conserved domains and motifs among highly divergent sequences of varying lengths.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For such divergent sequences that share only regional similarities, a local alignment based approach has to be used. The strategy identifies a block of ungapped alignment shared by all the sequences, hence, the block-based local alignment strategy.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about DIALIGN2?`,
    options: ["It is a web based program designed to detect local similarities", "It is designed to detect global similarities", "It does not apply gap penalties and thus is not sensitive to long gaps", "The method breaks each of the sequences down to smaller segments and performs all possible pair wise alignments between the segments"],
    correct: "b",
    explanation: `High-scoring segments, called blocks, among different sequences are then compiled in a progressive manner to assemble a full multiple alignment. It places emphasis on block-to-block comparison rather than residue-to-residue comparison. The sequence regions between the blocks are left unaligned. The program has been shown to be especially suitable for aligning divergent sequences with only local similarity.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Match-Box compares segments of some of the nine residues of possible Pair wise alignments.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Match-Box compares segments of every nine residues of all possible pair wise alignments. It is a web-based server that also aims to identify conserved blocks (or boxes) among sequences. The program compares segments of every nine residues of all possible pair wise alignments. If the similarity of particular segments is above a certain threshold across all sequences, they are used as an anchor to assemble multiple alignments; residues between blocks are unaligned.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not the objective to perform sequence comparison?`,
    options: ["To observe patterns of conservation", "To find the common motifs present in both sequences", "To study the physical properties of molecules", "To study evolutionary relationships"],
    correct: "c",
    explanation: `To assess whether it is likely that two sequences evolved from the same sequence comparison is required. Also, to find out which sequences from the database are similar to the sequence at hand, sequence comparison is carried out.`,
    super_topic: `Quizzone`,
  },
  {
    question: `A dotplot is a visual and qualitative technique whereas the sequence alignment is an exact and quantitative measure of similarity of alignments.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The sequence alignment is an exact and quantitative measure of similarity of alignments. It involves Construction of the best alignment between the sequences and assessment of the similarity from the alignment.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The global sequence alignment is suitable when the two sequences are of dissimilar length, with a negligible degree of similarity throughout.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The global sequence alignment is suitable when the two sequences are of similar length, with a significant degree of similarity throughout. It gives the best alignment over the entire length of two sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The alignment score is the sum of substitution scores and gap penalties in this type of algorithm.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Use of +1 as a reward for a match, -1 as the penalty for a mismatch, and ignoring gaps is the scoring scheme of this type of method. Hence, the alignment score is the sum of substitution scores and gap penalties in this type of algorithm.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The substitution matrices are rarely used in this type of matching.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The substitution matrices are quite commonly used in this type of matching. A concise way to express the residue substitution costs can be achieved with a N x N matrix where, N is 4 for DNA and 20 for proteins as 4 nucleotides in DNA and 20 amino acid residues in proteins are in picture respectively.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about Protein substitution matrices?`,
    options: ["They are significantly more complex than DNA scoring matrices", "They have the N x N matrices of the amino acids", "Protein substitution matrices have quite important role in evolutionary studies", "They are significantly quite less complex than DNA scoring matrices"],
    correct: "d",
    explanation: `Protein substitution matrices are significantly more complex than DNA scoring matrices. Proteins are composed of twenty amino acids, and physico-chemical properties of individual amino acids vary considerably. A protein substitution matrix can be based on any property of amino acids: size, polarity, charge, hydrophobicity.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In Needleman-Wunsch algorithm, the gaps are scored -2.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In Needleman-Wunsch algorithm, the gaps are ignored. Amount of gap penalty is zero here. A gap corresponds to an insertion or a deletion of a Residue.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The number of possible global alignments between two sequences of length N is _____`,
    options: ["2N\u03c0N\u221a", "22N\u03c0N\u221a", "2(N\u22121)\u03c0N\u221a", "22NN\u221a"],
    correct: "b",
    explanation: `By the total number of permutations and combinations option b gives the accurate number of possible global alignments between two sequences of length N. For two sequences of 250 residues this is 10149.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about Needleman-Wunsch algorithm?`,
    options: ["It is an example of dynamic programming", "Basic idea here is to build up the best alignment by using optimal alignments of larger subsequences", "It was first used by Saul Needleman and Christian Wunsch", "It was first used in 1970"],
    correct: "b",
    explanation: `In case of Needleman-Wunsch algorithm, the basic idea here is to build up the best alignment by using optimal alignments of smaller subsequences. It is based on dynamic programming, a discipline invented by Richard Bellman in 1953.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `There are two types matrices involved in the study- score matrices and trace matrices.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The Needleman-Wunsch algorithm consists of three steps where these matrices play their role as follows:
i. Initialization of the score matrix
ii. Calculation of scores and filling the traceback matrix
iii. Deducing the alignment from the traceback matrix`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Progressive alignment methods use the dynamic programming method to build an MSA starting with the most related sequences and then progressively adding less related sequences or groups of sequences to the initial alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The progressive alignment methods use the dynamic programming method. Relationships among the sequences are modeled by an evolutionary tree in which the outer branches or leaves are the sequences. The tree is based on pair-wise comparisons of the sequences using one of the phylogenetic methods.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Progenitor sequences represented by the ______ branches of the tree are derived by alignment of the _______ sequences.`,
    options: ["outer, outermost", "inner, outermost", "inner, innermost", "outer, innermost"],
    correct: "b",
    explanation: `Progenitor sequences represented by the inner branches of the tree are derived by alignment of the outermost sequences. These inner branches will have uncertainties where positions in the outermost sequences are dissimilar.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `CLUSTALW is a more recent version of CLUSTAL with the W standing for ________`,
    options: ["weakening", "winding", "weighting", "wiping"],
    correct: "c",
    explanation: `The W in CLUSTALW stands for ‘weighting’ to represent the ability of the program to provide weights to the sequence and program parameters. CLUSTAL has been around for more than 10 years and lots of improvements in the program have been made.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The CLUSTALX provides a graphic interface.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Two examples of programs that use progressive methods are CLUSTALW and the Genetics Computer Group program PILEUP. CLUSTALX provides a graphic interface.
These changes provide more realistic alignments that should reflect the evolutionary changes in the aligned sequences and the more appropriate distribution of gaps between conserved domains.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about CLUSTAL program?`,
    options: ["CLUSTAL performs a global-multiple sequence alignment by a different method than MSA (Multiple Sequence Alignment)", "The initial heuristic alignment obtained by MSA is calculated in a different way", "The initial step includes performing pair-wise alignments of all of the sequences", "The intermediate step includes use the alignment scores to produce a phylogenetic tree"],
    correct: "b",
    explanation: `The initial heuristic alignment obtained by MSA is calculated the same way, although it performs a global-multiple sequence alignment by a different method than MSA (Multiple Sequence Alignment). As the mentioned options are first two steps, the last is aligning the sequences sequentially, guided by the phylogenetic relationships indicated by the tree.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The initial alignments used to produce the guide tree may be obtained by various methods. Which of the following is not one of them?`,
    options: ["Fast k-tuple", "pattern-finding approach similar", "FASTA", "Faster, full dynamic programming method"],
    correct: "d",
    explanation: `The methods used, might be fast k-tuple or pattern-finding approach similar to FASTA that is useful for many sequences and the full dynamic programming method as well. But the option d becomes incorrect as full dynamic programming method is slower as compared to rest of the methods in options.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The scoring of gaps in a MSA (Multiple Sequence Alignment) has to be performed in a different manner from scoring gaps in a pair-wise alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `As more sequences are added to a profile of an existing MSA, gaps accumulate and influence the alignment of further sequences. CLUSTALW calculates gaps in a novel way designed to place them between conserved domains.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Like other alignment programs, CLUSTAL uses a null score for opening a gap in a sequence alignment and a penalty for extending the gap by one residue.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `CLUSTAL uses a penalty for opening a gap in a sequence alignment and an additional penalty for extending the gap by one residue. These penalties are user-defined. Gaps found in the initial alignments remain fixed. New gaps introduced as more sequences are added also receive this same gap penalty, even when they occur within an existing gap, but the gap penalties for alignment are then modified according to the average match value in the substitution matrix, the percent identity between the sequences, and the sequence lengths.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue about PILEUP program?`,
    options: ["It is the MSA program that is a part of the Genetics Computer Group package of sequence analysis programs", "It is owned since 1997 by Oxford Communications and is widely used due to the popularity and availability of this package", "It uses a method for MSA that is polar opposite to CLUSTALW", "The sequences are aligned pair-wise using the Needleman- Wunsch dynamic programming algorithm"],
    correct: "c",
    explanation: `PILEUP uses a method for MSA that is very similar to CLUSTALW. The sequences are aligned pair-wise using the Needleman- Wunsch dynamic programming algorithm, and the scores are used to produce a tree by the unweighted pair-group method using arithmetic averages. The resulting tree is then used to guide the alignment of the most closely related sequences and groups of sequences. The resulting alignment is a global alignment produced by the Needleman-Wunsch algorithm.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The resulting tree is then used to guide the alignment of the most closely related sequences and groups of sequences. The resulting alignment is a global alignment produced by the Needleman-Wunsch algorithm.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The very first sequences to be aligned are the most closely related on the sequence tree. If these sequences align very well, there will be few errors in the initial alignments. However, the more distantly related these sequences, the more errors will be made, and these errors will be propagated to the MSA. There is no simple way to circumvent this problem. A second problem with the progressive alignment method is the choice of suitable scoring matrices and gap penalties that apply to the set of sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Iterative methods include repeatedly realigning subgroups of the sequences and then by aligning these subgroups into a local alignment of all of the sequences.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Subgroups are aligned into a global alignment of all of the sequences. The objective is to improve the overall alignment score, such as a sum of pairs score. Selection of these groups may be based on the ordering of the sequences on a phylogenetic tree predicted in a manner similar to that of progressive alignment, separation of one or two of the sequences from the rest, or a random selection of the groups.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is incorrect regarding PRRP?`,
    options: ["The program PRRP uses iterative methods to produce an alignment", "An initial pair-wise alignment is made to predict a tree", "Only one cycle is performed", "The whole process is repeated until there is no further increase in the alignment score"],
    correct: "c",
    explanation: `As mentioned, an initial pair-wise alignment is made to predict a tree, the tree is used to produce weights for making alignments in the same manner as MSA except that the sequences are analyzed for the presence of aligned regions that include gaps rather than being globally aligned, and these regions are iteratively recalculated to improve the alignment score. The best scoring alignment is then used in a new cycle of calculations to predict a new tree, new weights, and new alignments.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In the program DIALIGN, pairs of sequences are aligned to locate aligned regions that do not include gaps, much like continuous diagonals in a dot matrix plot.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The program DIALIGN finds an alignment by a different iterative method. Pairs of sequences are aligned to locate aligned regions that do not include gaps, much like continuous diagonals in a dot matrix plot. Diagonals of various lengths are identified. A consistent collection of weighted diagonals that provides an alignment which is a maximum sum of weights is then found.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The Genetic Algorithm method has been recently adapted for MSA(Multiple Sequence Alignment) by Corpet (1998).`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The genetic algorithm is a general type of machine-learning algorithm that has no direct relationship to biology and that was invented by computer scientists. The method has been recently adapted for MSA (Multiple Sequence Alignment) by Notredame and Higgins (1996) in a computer program package called SAGA (Sequence Alignment by Genetic Algorithm).`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `An approach for obtaining a higher-scoring MSA (Multiple Sequence Alignment) by rearranging an existing alignment uses a probability approach called simulated annealing.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The program MSASA (Multiple Sequence Alignment by Simulated Annealing) starts with a heuristic MSA (Multiple Sequence Alignment). Further, it changes the alignment by following an algorithm designed to identify changes that increase the alignment score.
Free 30-Day C++ Certification Bootcamp is Live. Join Now!`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The first step in Genetic Algorithm is arranging the sequences to be aligned in rows.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The sequences to be aligned are written in rows, as on a page, except that they are made to overlap by a random amount of sequence, up to 50 residues long for sequences about 200 in length. The ends are then padded with gaps. A typical population of 100 of these MSAs is made, although other numbers may be set.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The second step in the Genetic Algorithm comprises of scoring of the 100 initial MSAs by the sum of pairs method.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The 100 initial MSAs are scored by the sum of pairs method, except that both natural and quasi-natural gap-scoring schemes are used. Recall that the best SSP score for a MSA is the minimum one and the one that is closest to the sum of the pair-wise sequence alignment. Standard amino acid scoring matrices and gap opening and extension penalties are used.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In Genetic Algorithm, in the mutation process _______`,
    options: ["sequence is changed", "gaps are not inserted", "sequence is not changed", "gaps are not rearranged"],
    correct: "c",
    explanation: `In the mutation process, the sequence is not changed (else it would no longer be an alignment), but gaps are inserted and rearranged in an attempt to create a better-scoring MSA. In the gap insertion process, the sequences in a given MSA are divided into two groups based on an estimated phylogenetic tree, and gaps of random length are inserted into random positions in the alignment.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The HMM is a statistical model that considers few combinations of matches and gaps to generate an alignment of a set of sequences.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The HMM is a statistical model that considers all possible combinations of matches, mismatches, and gaps to generate an alignment of a set of sequences. A localized region of similarity, including insertions and deletions, may also be modeled by an HMM. Analysis of sequences by an HMM is discussed on page 185 along with other statistical methods.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not true about iterative methods?`,
    options: ["Genetic Algorithm is method used for under this", "Hidden Markov Models are used for Multiple Sequence Alignment", "The objective is to improve the overall alignment score", "MultAlin recalculates global scores"],
    correct: "d",
    explanation: `MultAlin (Corpet 1988) recalculates pair-wise scores during the production of a progressive Alignment. In addition, it uses these scores to recalculate the tree, which is then used to refine the alignment in an effort to improve the score.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not among the methods for finding localized sequence similarity?`,
    options: ["Profile Analysis", "Block Analysis", "Extraction of Blocks from a Global or Local MSA", "Pattern blocking"],
    correct: "d",
    explanation: `Pattern Searching is the correct name of the method for finding localized sequence similarity. This type of analysis was performed on groups of related proteins, and the amino acid patterns that were located may be found in the Prosite catalog.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Profiles are found by performing the _____ MSA of a group of sequences and then removing the _______ regions in the alignment into a smaller MSA.`,
    options: ["local, more highly conserved", "global, low conserved", "global, more highly conserved", "local, low conserved"],
    correct: "c",
    explanation: `Profiles are found by performing the global MSA of a group of sequences and then removing the more highly conserved regions in the alignment into a smaller MSA. A scoring matrix for the MSA, called a profile, is then made. The profile is composed of columns much like a mini-MSA and may include matches, mismatches, insertions, and deletions.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The program Profilemake can be used to produce a profile from an MSA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A version of the Profilesearch program, which performs a database search for matches to a profile, is available at the University of Pittsburgh Supercomputer Center. A special grant application may be needed to use this facility. Profile-generating programs are available by FTP and are included in the Genetics Computer Group suite of programs.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding the block analysis method?`,
    options: ["Blocks represent a conserved region in the MSA", "Blocks differ from profiles in lacking insert and delete positions in the sequences", "Every column includes only matches and mismatches", "Blocks may be made by searching for a section of an MSA alignment that is low conserved"],
    correct: "d",
    explanation: `Like profiles, blocks may be made by searching for a section of an MSA alignment that is highly conserved. However, aligned regions may also be found by searching each sequence in turn for similar patterns of the same length. These patterns may include a region with one or a few matching characters followed by a short spacer region of unmatched characters and then by another set of a few matching characters, and so on, until the sequences start to be different.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Block analysis methods use substitution matrices such as the PAM and BLOSUM matrices to score matches.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `These methods do not use substitution matrices such as the PAM and BLOSUM matrices to score matches. Rather, they are based on finding exact matches that have the same spacing in at least some of the input sequences, and that may be repeated in a given sequence.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In the method of extraction of blocks from a global or local MSA, a global MSA of related protein sequences usually includes regions that have been aligned without gaps in any of the sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These ungapped patterns may be extracted from these aligned regions and used to produce blocks. Blocks found in this manner are only as good as the MSA from which they are derived. A global MSA of related protein sequences usually includes regions that have been aligned without gaps in any of the sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not true regarding the BLOCKS?`,
    options: ["The BLOCKS server can extract a conserved, ungapped region from a MSA to produce a sequence block", "The server can also find blocks in a set of unaligned, input sequences and maintains a large database of blocks based on an analysis of proteins in the Prosite catalog", "Blocks are found by the Protomat program", "The program MOTIF doesn\u2019t locate spaced patterns"],
    correct: "d",
    explanation: `Blocks are found in two steps: First, the program MOTIF described on the previous page is used to locate spaced patterns. The second step takes the best and most consistent patterns found in step 1 and uses the program MOTOMAT to merge overlapping triplets and extend them, orders the resulting blocks, and chooses those that are in the largest subset of sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The pattern searching method type of analysis was performed on groups of related proteins, and the amino acid patterns that were located may be found in the Prosite catalog.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This Prosite catalog groups proteins that have similar biochemical functions on the basis of amino acid patterns such as those in the active site. Subsequently, these families were searched for amino acid patterns by the MOTIF program (Smith et al. 1990), which finds patterns of the type aa1 d1 aa2 d2 aa3, where aa1 and aa2 are conserved amino acids and d1 and d2 are stretches of intervening sequence up to 24 amino acids long.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Although MOTIF program is used successfully for making the BLOCKS database, it is limited in the pattern sizes that can be found.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The MOTIF program distinguishes true motifs from random background patterns by requiring that motifs occur in a number of the input sequences and tend not to be internally repeated in any one sequence. As the length of the motif increases, there are many possible combinations of patterns of a given length where only a few characters match.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The Expectation Maximization algorithm has been used to identify conserved domains in unaligned proteins only.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `This algorithm has been used to identify both conserved domains in unaligned proteins and protein-binding sites in unaligned DNA sequences (Lawrence and Reilly 1990), including sites that may include gaps (Cardon and Stormo 1992). Given are a set of sequences that are expected to have a common sequence pattern and may not be easily recognizable by eye.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is untrue regarding Expectation Maximization algorithm?`,
    options: ["An initial guess is made as to the location and size of the site of interest in each of the sequences, and these parts of the sequence are aligned", "The alignment provides an estimate of the base or amino acid composition of each column in the site", "The column-by-column composition of the site already available is used to estimate the probability of finding the site at any position in each of the sequences", "The row-by-column composition of the site already available is used to estimate the probability"],
    correct: "d",
    explanation: `The EM algorithm then consists of two steps, which are repeated consecutively. In step 1, the expectation step, the column-by-column composition of the site already available is used to estimate the probability of finding the site at any position in each of the sequences. These probabilities are used in turn to provide new information as to the expected base or amino acid distribution for each column in the site.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Out of the two repeated steps in EM algorithm, the step 2 is ________`,
    options: ["the maximization step", "the minimization step", "the optimization step", "the normalization step"],
    correct: "a",
    explanation: `In step 2, the maximization step, the new counts of bases or amino acids for each position in the site found in step 1 are substituted for the previous set. Step 1 is then repeated using these new counts. The cycle is repeated until the algorithm converges on a solution and does not change with further cycles. At that time, the best location of the site in each sequence and the best estimate of the residue composition of each column in the site will be available.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In EM algorithm, as an example, suppose that there are 10 DNA sequences having very little similarity with each other, each about 100 nucleotides long and thought to contain a binding site near the middle 20 residues, based on biochemical and genetic evidence. the following steps would be used by the EM algorithm to find the most probable location of the binding sites in each of the ______ sequences.`,
    options: ["30", "10", "25", "20"],
    correct: "b",
    explanation: `When examining the EM program MEME, the size and number of binding sites, the location in each sequence, and whether or not the site is present in each sequence do not necessarily have to be known. For the present example, the following steps would be used by the EM algorithm to find the most probable location of the binding sites in each of the 10 sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In the initial step of EM algorithm, the 20-residue-long binding motif patterns in each sequence are aligned as an initial guess of the motif.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The base composition of each column in the aligned patterns is then determined. The composition of the flanking sequence on each side of the site provides the surrounding base or amino acid composition for comparison. Each sequence is assumed to be the same length and to be aligned by the ends.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `In the intermediate steps of EM algorithm, the number of each base in each column is determined and then converted to fractions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, that there are four Gs in the first column of the 10 sequences, then the frequency of G in the first column of the site, fSG = 4/10 = 0.4. This procedure is repeated for each base and each column.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `For the 10-residue DNA sequence example, there are _______ possible starting sites for a 20-residue-long site.`,
    options: ["30", "21", "81", "60"],
    correct: "c",
    explanation: `For the 10-residue DNA sequence example, there are 100 – 20 +1 possible starting sites for a 20-residue-long site. Where the first one is at position 1 in the sequence ending one at 20 and the last beginning at position 81 and ending at 100 (there is not enough sequence for a 20-residue-long site beyond position 81).`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `An alternative method is to produce an odds scoring matrix calculated by dividing each base frequency by the background frequency of that base.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In this method, the probability of each location is then found by multiplying the odds scores from each column. An even simpler method is to use log odds scores in the matrix. The column scores are then simply added. In this case, the log odds scores must be converted to odds scores before position probabilities are calculated.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following about MEME is untrue?`,
    options: ["It is a Web resource for performing local MSAs (Multiple Sequence Alignment) by the above expectation maximization method is the program MEME", "It stands for Multiple EM for Motif Elicitation", "It was developed at developed at the University of California at San Diego Supercomputing Center", "The Web page has multiple versions for searching blocks by an EM algorithm"],
    correct: "d",
    explanation: `The Web page for two versions of MEME, ParaMEME, a Web program that searches for blocks by an EM algorithm (Described below), and a similar program MetaMEME (which searches for profiles using HMMs, described below).The Motif Alignment and Search Tool (MAST) for searching through databases for matches to motifs.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following about the Gibbs sampler is untrue?`,
    options: ["It is a statistical method for finding motifs in sequences", "It is dissimilar to the principle of the EM method", "It searches for the statistically most probable motifs", "It can find the optimal width and number of given motifs in each sequence"],
    correct: "b",
    explanation: `It is another statistical method for finding motifs in sequences is the Gibbs sampler. The method is similar in principle to the EM method described above, but the algorithm is different. A combinatorial approach of the Gibbs sampler and MOTIF may be used to make blocks at the BLOCKS Web site.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Analysis of s for conserved blocks of sequence leads to production of the position-specific scoring matrix.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The analysis of MSAs (Multiple Sequence Alignment) for conserved blocks of sequence leads to production of the position-specific scoring matrix or PSSM. The PSSM may be used to search a sequence to obtain the most probable location or locations of the motif represented by the PSSM. Alternatively, the PSSM may be used to search an entire database to identify additional sequences that also have the same motif.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The quality and quantity of information provided by the PSSM also varies for ________ in the motif.`,
    options: ["each row", "each column", "rows and columns", "neither the rows nor the columns"],
    correct: "b",
    explanation: `The quality and quantity of information provided by the PSSM also varies for each column in the motif, and this variation profoundly influences the matches found with sequences. This situation can be accurately described by information theory, and the results can be displayed by a colored graph called a sequence logo.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Two considerations arise in trying to tune the PSSM so that it adequately represents the training sequences. Which of the following is not their description?`,
    options: ["If a given column in 20 sequences has only isoleucine, it is not very likely that different amino acid will be found in other sequences with that motif because the residue is probably important for function", "If a given column in 20 sequences has only isoleucine, it is very likely that different amino acid will be found in other sequences with that motif because the residue is probably important for function", "If the number of sequences with the found motif is large and reasonably diverse, the sequences represent a good statistical sampling of all sequences that are ever likely to be found with that same motif", "Another column in the motif from the 20 sequences may have several amino acids, and some amino acids may not be represented at all"],
    correct: "b",
    explanation: `The PSSM is constructed by a simple logarithmic transformation of a matrix giving the frequency of each amino acid in the motif. Even more variation may be expected at that position in other sequences, although the more abundant amino acids already found in that column would probably be favored.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `If a good sampling of sequences is _______ the number of sequences is _________ and the motif structure is ________ it should, in principle, be possible to obtain frequencies highly representative of the same motif in other sequences also.`,
    options: ["available, sufficiently large, not too complex", "unavailable, sufficiently large, not too complex", "unavailable, sufficiently small, not too complex", "available, sufficiently large, too complex"],
    correct: "a",
    explanation: `The more abundant amino acids already found in that column would probably be favored. Thus, if a good sampling of sequences is available, the number of sequences is sufficiently large, and the motif structure is not too complex, it should, in principle, be possible to obtain frequencies highly representative of the same motif in other sequences also (Henikoff and Henikoff 1996).`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `If the data set is _______ then unless the motif has __________ amino acids in each column, the column frequencies in the motif may not be highly representative of all other occurrences of the motif.`,
    options: ["small, distinct", "small, almost identical", "large, almost identical", "large, distinct"],
    correct: "b",
    explanation: `The number of sequences for producing the motif may be small, highly diverse, or complex, giving rise to a second level of consideration. If the data set is small, then unless the motif has almost identical amino acids in each column, the column frequencies in the motif may not be highly representative of all other occurrences of the motif. In such cases, it is desirable to improve the estimates of the amino acid frequencies by adding extra amino acid counts, called pseudocounts, to obtain a more reasonable distribution of amino acid frequencies in the column.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Even if many pseudocounts are added in comparison to real sequence counts, the amino acid frequencies will not have any effect or influence.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Knowing how many counts to add is a difficult but fortunately solvable problem. On the one hand, if too many pseudocounts are added in comparison to real sequence counts, the pseudocounts will become the dominant influence in the amino acid frequencies and searches using the motif will not work. On the other hand, if there are relatively few real counts, many amino acid variations may not be present because of the small sample of sequences.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Which of the following is not a feature of editors and formatters?`,
    options: ["provision for displaying the sequence on a color monitor with residue colors to aid in a clear visual representation of the alignment", "recognition of the multiple sequence format that was output by the MSA (Multiple Sequence Alignment) program", "maintenance of the alignment in a suitable format when the editing is completed", "disallowing shading conserved residues in the alignment"],
    correct: "d",
    explanation: `In addition to this, provision of a suitable windows interface, allowing use of the mouse to add, delete, or move sequence followed by an updated display of the alignment, is a feature. In addition, there are other types of editing that are commonly performed on MSAs (Multiple Sequence Alignment) program such as, for example, shading conserved residues in the alignment.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `GDE (Genetic Data Environment) provides a general interface on UNIX machines for sequence analysis, sequence alignment editing, and display.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is available from several anonymous FTP sites. This interface requires communication with a host UNIX machine running the Genetics Computer Group software. Interface with MS-DOS or Macintosh is possible if the computer is equipped with the appropriate X-Windows client software.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `MACAW is a local multiple sequence alignment program only.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `MACAW is both a local multiple sequence alignment program and a sequence editing tool. Given a set of sequences, the program finds ungapped blocks in the sequences and gives their statistical significance. Later versions of the program find blocks by one of three user-chosen methods.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `Two commonly encountered examples are the Genetics Computer Group’s MSF format and the CLUSTALW ALN format.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is because these formats follow a precise outline, one may be readily converted to another by computer programs. READSEQ by D.G.Gilbert at Indiana University at Bloomington is one such program.`,
    super_topic: `Multiple Sequence Alignment`,
  },
  {
    question: `The main application of pairwise alignment is retrieving biological sequences in databases based on similarity.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This process involves submission of a query sequence and performing a pairwise comparison of the query sequence with all individual sequences in a database. Thus, database similarity searching is pairwise alignment on a large scale. This type of searching is one of the most effective ways to assign putative functions to newly determined sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Dynamic programming method is the fastest and most practical method.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Dynamic programming method is slow and impractical to use in most cases. Special search methods are needed to speed up the computational process of sequence comparison.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not one of the requirements for implementing algorithms for sequence database searching?`,
    options: ["Size of the dataset", "Sensitivity", "Specificity", "Speed"],
    correct: "a",
    explanation: `There are unique requirements for implementing algorithms for sequence database searching out of which, the later three play an important role. However, speed can vary with the size of database. achieving all three at a time is nearly impossible.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Sensitivity refers to the ability to find as many correct hits as possible.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Among the unique requirements for implementing algorithms for sequence database Searching, the first criterion is sensitivity, which refers to the ability to find as many correct hits as possible. It is measured by the extent of inclusion of correctly identified sequence members of the same family. These correct hits are considered ‘true positives’ in the database searching exercise.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The specificity refers to the ability to include incorrect hits.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In heuristic database searching methods, The second requirement criterion is 1 also called specificity, which refers to the ability to exclude incorrect hits. These incorrect hits are unrelated sequences mistakenly identified in database searching and are considered ‘false positives.’`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In heuristic methods, speed doesn’t vary with the size of database.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The speed is the time it takes to get results from database searches. Depending on the size of the database, speed sometimes can be a primary concern in the search methods.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `An increase in sensitivity is associated with _______ in selectivity.`,
    options: ["no specific change", "increase", "decrease", "exponential increase"],
    correct: "c",
    explanation: `Ideally, one wants to have the greatest sensitivity, selectivity, and speed in database searches. However, satisfying all three requirements is difficult in reality. What generally happens is that an increase in sensitivity is associated with decrease in selectivity. A very inclusive search tends to include many false positives. Similarly, an improvement in speed often comes at the cost of lowered sensitivity and selectivity. A compromise between the three criteria often has to be made.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is incorrect?`,
    options: ["Initial DNA sequencing reactions generate short sequence reads from DNA clones", "To assemble a whole genome sequence, these short fragments are joined to form larger fragments", "The average length of the reads is about 50 bases", "A number of overlapping contigs can be further merged to form scaffolds"],
    correct: "c",
    explanation: `The average length of the reads is about 500 bases. To assemble a whole genome sequence, these short fragments are joined to form larger fragments after removing overlaps. These longer, merged sequences are termed contigs, which are usually 5,000 to 10,000 bases long. A number of overlapping contigs can be further merged to form scaffolds (30,000–50,000 bases, also called supercontigs), which are unidirectionally oriented along a physical map of a chromosome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Currently, there are two major heuristic algorithms for performing database searches: BLAST and FASTA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These methods are not guaranteed to find the optimal alignment or true homologs, but are 50–100 times faster than dynamic programming. The increased computational speed comes at a moderate expense of sensitivity and specificity of the search, which is easily tolerated by working molecular biologists. Both programs can provide a reasonably good indication of sequence similarity by identifying similar sequence segments.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is incorrect the ‘word’ method?`,
    options: ["Both BLAST and FASTA use a heuristic word method", "Word method is used for fast pairwise sequence alignment in BLAST and FASTA", "The basic assumption is that two related sequences must have at least one word in common", "Two related sequences must have at zero word in common while assuming"],
    correct: "d",
    explanation: `This is the third method of pairwise sequence alignment. It works by finding short stretches of identical or nearly identical letters in two sequences. These short strings of characters are called words, which are similar to the windows used in the dot matrix method. The basic assumption is that two related sequences must have at least one word in common. By first identifying word matches, a longer alignment can be obtained by extending similarity regions from the words. Once regions of high sequence similarity are found, adjacent high-scoring regions can be joined into full alignment.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `1.The BLAST program was developed in _______`,
    options: ["1992", "1995", "1990", "1991"],
    correct: "c",
    explanation: `The BLAST program was developed by Stephen Altschul of NCBI in 1990 and hassince become one of the most popular programs for sequence analysis. BLAST uses heuristics to align a query sequence with all sequences in a database.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In sequence alignment by BLAST, each word from query sequence is typically _______ residues for protein sequences and _______ residues for DNA sequences.`,
    options: ["ten, eleven", "three, three", "three, eleven", "three, ten"],
    correct: "c",
    explanation: `The first step is to create a list of words from the query sequence. Each word is typically three residues for protein sequences and eleven residues for DNA sequences. The list includes every possible word extracted from the query sequence. This step is also called seeding.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In sequence alignment by BLAST, the second step is to search a sequence database for the occurrence of these words.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This step is to identify database sequences containing the matching words. The matching of the words is scored by a given substitution matrix. A word is considered a match if it is above a threshold.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The final step involves pairwise alignment by extending from the words in both directions while counting the alignment score using the same substitution matrix.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The extension continues until the score of the alignment drops below a threshold due to mismatches (the drop threshold is twenty-two for proteins and twenty for DNA). The resulting contiguous aligned segment pair without gaps is called high-scoring segment pair. In the original version of BLAST, the highest scored HSPs are presented as the final report. They are also called maximum scoring pairs.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `A recent improvement in the implementation of BLAST is the ability to provide gapped alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In gapped BLAST, the highest scored segment is chosen to be extended in both directions using dynamic programming where gaps may be introduced. The extension continues if the alignment score is above a certain threshold; otherwise it is terminated. However, the overall score is allowed to drop below the threshold only if it is temporary and rises again to attain above threshold values. Final trimming of terminal regions is needed before producing a report of the final alignment.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not a variant of BLAST?`,
    options: ["BLASTN", "BLASTP", "BLASTX", "TBLASTNX"],
    correct: "d",
    explanation: `BLAST is a family of programs that includes BLASTN, BLASTP, BLASTX TBLASTN, and TBLASTX. BLASTN queries nucleotide sequences with a nucleotide sequence database. The alignment scoring is based on the BLOSUM62 matrix.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `BLASTX uses protein sequences as queries to search against a protein sequence database.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `BLASTP, and not BLASTX, uses protein sequences as queries to search against a protein sequence database. BLASTX uses nucleotide sequences as queries and translates them in all six reading frames to produce translated protein sequences, which are used to query a protein sequence database.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `TBLASTX queries protein sequences to anucleotide sequence database with the sequences translated in all six reading frames.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `TBLASTN queries protein sequences to anucleotide sequence database with the sequences translated in all six reading frames. TBLASTX uses nucleotide sequences, which are translated in all six frames, to search against a nucleotide sequence database that has all the sequences translated in six frames. In addition, there is also a bl2seq program that performs local alignment of two user-provided input sequences. The graphical output includes horizontal bars and a diagonal in a two-dimensional diagram showing the overall extent of matching between the two sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not correct about BLAST?`,
    options: ["The BLAST web server has been designed in such away as to simplify the task of program selection", "The programs are organized based on the type of query sequences", "The programs are organized based on the type of nucleotide sequences, or nucleotide sequence to be translated", "BLAST is not based on heuristic searching methods"],
    correct: "d",
    explanation: `BLAST and FASTA are based on heuristic searching methods. In addition, programs for special purposes are grouped separately; for example, bl2seq, immunoglobulin BLAST, and VecScreen, a program for removing contaminating vector sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `If one is looking for protein homologs encoded in newly sequenced genomes, one may use TBLASTN, which translates nucleotide database sequences in all six open reading frames.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This may help to identify protein coding genes that have not yet been annotated. If a DNA sequence is to be used as the query, a protein-level comparison can be done with TBLASTX. However, both programs are very computationally intensive and the search process can be very slow.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not correct about FASTA?`,
    options: ["Its stands for FAST ALL", "It was in fact the first database similarity search tool developed, preceding the development of BLAST", "FASTA uses a \u2018hashing\u2019 strategy to find matches for a short stretch of identical residues with a length of k", "The string of residues is known as blocks"],
    correct: "d",
    explanation: `The string of residues is known as ktuples or ktups, which are equivalent to words inBLAST, but are normally shorter than the words. Typically, a ktup is composed of two residues for protein sequences and six residues for DNA sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The first step in FASTA alignment id to arrange the sequences in matrices’ rows and columns in order to be analyzed.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The first step in FASTA alignment is to identify ktups between two sequences by using the hashing strategy. This strategy works by constructing a look up table that shows the position of each ktup for the two sequences under consideration. The mentioned method is similar to that of BLAST.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The positional difference for each word between the two sequences is obtained by ________ the position of the ______ sequence from that of the _______ sequence and is expressed as the offset.`,
    options: ["subtracting, second, first", "adding, second, first", "adding, first, second", "subtracting, first, second"],
    correct: "d",
    explanation: `The positional difference for each word between the two sequences is obtained by subtracting the position of the first sequence from that of the second sequence and is expressed as the offset. The ktups that have the same offset values are then linked to reveal a contiguous identical sequence region that corresponds to a stretch of diagonal in a two-dimensional matrix.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The second step in FASTA is to narrow down the high similarity regions between the two sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Normally, many diagonals between the two sequences can be identified in the hashing step. The top ten regions with the highest density of diagonals are identified as high similarity regions. The diagonals in these regions are scored using a substitution matrix.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In FASTA, neighboring high-scoring segments along the same diagonal are selected and joined to form a single alignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This step allows introducing gaps between the diagonals while applying gap penalties. The score of the gapped alignment is calculated again. In step 3, the gapped alignment is refined further using the Smith–Waterman algorithm to produce a final alignment.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The last step is to perform a statistical evaluation of the final alignment as in BLAST, which produces the E-value.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The last step of FASTA is similar to that of BLAST. The determination of E-value is the most important part of the whole analysis as it gives the degree of alignment between the sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The web-based FASTA program is offered by the European Bioinformatics Institute.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Similar to BLAST, FASTA has a number of subprograms. The web-based FASTA program offered by the European Bioinformatics Institute allows the use of either DNA or protein sequences as the query to search against a protein database or nucleotide database.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `FASTX, which compares a protein query sequence to a translated DNA database.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Some available variants of the program are FASTX and TFASTX. FASTX translates a DNA sequence and uses the translated protein sequence to query a protein database. TFASTX compares a protein query sequence to a translated DNA database.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `FASTA doesn’t use bit scores.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `FASTA uses E-values and bit scores as well. Estimation of the two parameters in FASTA is essentially the same as in BLAST. However, the FASTA output provides one more statistical parameter, the Z-score.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Z-score describes the number of standard deviations from the mean score for the database search.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because most of the alignments with the query sequence are with unrelated sequences, the higher the Z-score for a reported match, the further away from the mean of the score distribution, hence, the more significant the match. For a Z-score > 15, the match can be considered extremely significant, with a certainty of a homologous relationship. If Z is in the range of 5 to 15, the sequence pair can be described as highly probable homologs. If Z<5, their relationship is described as less certain.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In FASTA, For a Z-score > 15, the match can be considered extremely ______ with _____ of a homologous relationship.`,
    options: ["insignificant, uncertainty", "significant, uncertainty", "significant, certainty", "insignificant, certainty"],
    correct: "c",
    explanation: `If Z is in the range of 5 to 15, the sequence pair can be described as highly probable homologs. If Z<5, their relationship is described as less certain.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `BLAST uses a _______ to find matching words, whereas FASTA identifies identical matching words using the _____`,
    options: ["substitution matrix, hashing procedure", "substitution matrix, blocks", "hashing procedure, substitution matrix", "ktups, substitution matrix"],
    correct: "a",
    explanation: `BLAST and FASTA have been shown to perform almost equally well in regular database searching; However, there are some notable differences between the two approaches. The major difference is in the seeding step– BLAST uses a substitution matrix to find matching words, whereas FASTA identifies identical matching words using the hashing procedure.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not a benefit or a factual of FASTA over BLAST?`,
    options: ["FASTA scans smaller window sizes", "It gives more sensitive results", "It gives less sensitive results", "It gives results with a better coverage rate for homologs"],
    correct: "c",
    explanation: `By default, FASTA scans smaller window sizes. Thus, it gives more sensitive results than BLAST, with a better coverage rate for homologs. However, it is usually slower than BLAST.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The use of low-complexity masking in the BLAST procedure means that it may have higher specificity than FASTA because potential false positives are reduced.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In addition to the given statement, BLAST sometimes gives multiple best-scoring alignments from the same sequence. FASTA returns only one final alignment.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Which of the following is not a benefit of BLAST?`,
    options: ["Handling of gaps", "Speed", "More sensitive", "Statistical rigor"],
    correct: "a",
    explanation: `In addition to this, user friendly UI of BLAST is also one of its benefits. However, it does not handle gaps well. In that case gapped BLAST is better.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `BLAST might not find matches for very short sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In BLAST, similarity matching of words is involved. If no words are found similar, then no alignment is detected and hence it might not find matches for very short sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `BLAST often produces several short HSPs rather than a single aligned region.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The results of the word matching and attempts to extend the alignment are segments. They are called as HSPs (High-Scoring Segment Pairs). BLAST often produces several short HSPs rather than a single aligned region.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `FASTA is derived from logic of the dot plot.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because of this, it computes best diagonals from all frames of alignment. The method looks for exact matches between words in query and test sequence.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The gapped portion in the diagonals represents matches in FASTA.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The diagonal’s nature indicates the matching of the sequences. After all diagonals are found, it tries to join diagonals by adding gaps. Further, it Computes alignments in regions of best diagonals.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The initiation of FASTA format has ____ symbol.`,
    options: [">", "<", "/", "*"],
    correct: "a",
    explanation: `Its format is simple as used by almost all programs. Header line has > at the beginning. Also no specific requirements are there for line length, characters, etc.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The rigorous dynamic programming method is normally not used for database searching because it is slow and computationally expensive.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Heuristics such as BLAST and FASTA are developed for faster speed. However, the heuristic methods are limited in sensitivity and are not guaranteed to find the optimal alignment. They often fail to find alignment for distantly related sequences.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `FASTA and BLAST are __________ but __________ for larger datasets.`,
    options: ["faster, more sensitive", "faster, less sensitive", "slower, less sensitive", "slower, more sensitive"],
    correct: "b",
    explanation: `Empirical tests have indeed shown that the exhaustive method produces superior results over the heuristic methods like BLAST and FASTA. But heuristic methods are better and practical when it comes to assess larger datasets with comparatively low sensitivity.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Scan PS is a web-based program that implements a modified version of the Needleman-Wunsch algorithm.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `ScanPS (Scan Protein Sequence) is a web-based program that implements a modified version of the Smith–Waterman algorithm optimized for parallel processing. The major feature is that the program allows iterative searching similar to PSI-BLAST, which builds profiles from one round of search results and uses them for the second round of database searching. Full dynamic programming is used in each cycle for added sensitivity.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Par Align is a web-based server that uses parallel processors to perform exhaustive sequence comparisons using either a parallelized version of the Smith–Waterman algorithm or a heuristic program for further speed gains.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The heuristic subprogram first finds exact ungapped alignments and uses them as anchors for extension into gapped alignments by combining the scores of several diagonals in the alignment matrix. The search speed of ParAlign approaches to that of BLAST, but with higher sensitivity.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In Smith–Waterman algorithm, in initialization Step, the _________ row and ________ column are subject to gap penalty.`,
    options: ["first, first", "first, second", "second, First", "first, last"],
    correct: "a",
    explanation: `In Smith–Waterman algorithm, first row and first column are set to 0. In the Needleman Wunsch algorithm, First row and first column are subject to gap penalty.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Local sequence alignments are necessary for many cases out of which one is repeated.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It can also be used for modular organization of genes and proteins (exons, domains, etc.) Also it can be used in cases of sequences diverged so that similarity was retained, or can be detected, just in some sub-regions.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `In SW algorithm, to align two sequences of lengths of m and n _________ time is required.`,
    options: ["O(mn)", "O(m2n)", "O(m2n3)", "O(mn2)"],
    correct: "b",
    explanation: `The Smith–Waterman algorithm is quite demanding of time. Hence if two sequences of lengths of m and n have to be aligned, the required time is O(m2n). It requires O(mn) calculation steps.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `One of the challenges in SWA is obtaining correct alignments in regions of low similarity between distantly related biological sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is because mutations have added too much ‘noise’ over evolutionary time to allow for a meaningful comparison of those regions. Local alignment avoids such regions altogether and focuses on those with a positive score, i.e. those with an evolutionarily conserved signal of similarity.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `Score can be negative in Smith–Waterman algorithm.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Negative score is set to 0. In Needleman–Wunsch algorithm, the Score can be negative. Also, in Smith–Waterman algorithm, in tracing back step, it begins with the highest score, ends when 0 is encountered.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The function of the scoring matrix is to conduct one-to-one comparisons between all components in two sequences and record the optimal alignment results.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The scoring process reflects the concept of dynamic programming. The final optimal alignment is found by iteratively expanding the growing optimal alignment.`,
    super_topic: `Database Similarity Searching`,
  },
  {
    question: `The building blocks of proteins are ______ naturally occurring amino acids, small molecules that contain a free amino group (NH2) and a free carboxyl group (COOH).`,
    options: ["ten", "twenty", "nine", "nineteen"],
    correct: "b",
    explanation: `Both of these groups are linked to a central carbon (Cα), which is attached to hydrogen and a side chain group (R). Amino acids differ only by the side chain R group. The chemical reactivities of the R groups determine the specific properties of the amino acids. Amino acids can be grouped into several categories based on the chemical and physical properties of the side chains, such as size and affinity for water.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Within the hydrophobic set of amino acids, they can be further divided into aliphatic and aromatic.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Aliphatic side chains are linear hydrocarbon chains and aromatic side chains are cyclic rings. Within the hydrophilic set, amino acids can be subdivided into polar and charged. Charged amino acids can be either positively charged (basic) or negatively charged (acidic).`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `______ the smallest amino acid, has a hydrogen atom as the R group.`,
    options: ["valine", "proline", "Glycine", "threonine"],
    correct: "c",
    explanation: `Of particular interest within the twenty amino acids are glycine and proline. It can therefore adopt more flexible conformations that are not possible for other amino acids. Proline is on the other extreme of flexibility. Its side chain forms a bond with its own backbone amino group, causing it to be cyclic. The cyclic conformation makes it very rigid, unable to occupy many of the main chain conformations adopted by other amino acids.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The peptide formation involves two amino acids covalently joined together between the carboxyl group of one amino acid and the amino group of another.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This reaction is a condensation reaction involving removal of elements of water from the two molecules. The resulting product is called a dipeptide. The newly formed covalent bond connecting the two amino acids is called a peptide bond. Once an amino acid is incorporated into a peptide, it becomes an amino acid residue. Multiple amino acids can be joined together to form a longer chain of amino acid polymer.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `A linear polymer of more than fifty amino acid residues is referred to as a ________`,
    options: ["dipeptide", "oligopeptide", "peptide", "polypeptide"],
    correct: "d",
    explanation: `A polypeptide, also called a protein, has a well-defined three-dimensional arrangement. On the other hand, a polymer with fewer than fifty residues is usually called a peptide without a well-defined three-dimensional structure. The residues a peptide or polypeptide are numbered beginning with the residue containing the amino group, referred to as the N-terminus, and ending with the residue containing the carboxyl group, known as the C-terminus.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct?`,
    options: ["The rigid double bond structure forces atoms associated with the peptide bond to lie in the same plane, called the dipeptide plane", "A peptide bond is actually a partial double bond owing to shared electrons between", "Because of the planar nature of the peptide bond and the size of the R groups, there are considerable restrictions on the rotational freedom by the two bonded pairs of atoms around the peptide bond", "The angle of rotation about the bond is referred to as the dihedral angle (also called the torsional angle)"],
    correct: "a",
    explanation: `The rigid double bond structure forces atoms associated with the peptide bond to lie in the same plane, called the peptide plane. For a peptide unit, the atoms linked to the peptide bond can be moved to a certain extent by the rotation of two bonds flanking the peptide bond.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the stabilizing Forces?`,
    options: ["Protein structures from secondary to quaternary are maintained by noncovalent forces", "They include electrostatic interactions but not van der Waals forces, and hydrogen bonding", "Electrostatic interactions are a significant stabilizing force in a protein structure", "Electrostatic interactions occur when excess negative charges in one region are neutralized by positive charges in another region"],
    correct: "b",
    explanation: `It includes electrostatic interactions, van der Waals forces, and hydrogen bonding. The result is the formation of salt bridges between oppositely charged residues. The electrostatic interactions can function within a relatively long range (15 Å). Hydrogen bonds are a particular type of electrostatic interactions similar to dipole–dipole interactions involving hydrogen from one residue and oxygen from another. Hydrogen bonds can occur between main chain atoms as well as side chain atoms.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the α-Helices?`,
    options: ["An \u03b1-helix has a main chain backbone conformation that resembles a corkscrew", "Nearly all known \u03b1-helices are right handed, exhibiting a leftward spiral form", "Nearly all known \u03b1-helices are right handed, exhibiting a rightward spiral form", "In right handed helix, there are 3.6 amino acids per helical turn"],
    correct: "b",
    explanation: `The structure is stabilized by hydrogen bonds formed between the main chain atoms of residues i and i + 4. The hydrogen bonds are nearly parallel with the helical axis. The average φ and ψ angles are 60◦ and 45◦, respectively, and are distributed in a narrowly defined region in the lower left region of a Ramachandran plot.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the β-sheet?`,
    options: ["A \u03b2-sheet is a fully extended configuration built up from several spatially adjacent regions of a polypeptide chain", "Each region involved in forming the \u03b2-sheet is a \u03b2-strand", "The \u03b2-strand conformation is pleated with main chain backbone zigzagging and side chains positioned on same sides of the sheet", "\u03b2-Strands are stabilized by hydrogen bonds between residues of adjacent strands"],
    correct: "c",
    explanation: `The β-strand conformation is pleated with main chain backbone zigzagging and side chains positioned alternately on opposite sides of the sheet. β-strands near the surface of the protein tend to show an alternating pattern of hydrophobic and hydrophilic regions, whereas strands buried at the core of a protein are nearly all hydrophobic. The β-strands can run in the same direction to form a parallel sheet or can run every other chain in reverse orientation to form an antiparallel sheet or a mixture of both.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the Coils and Loops?`,
    options: ["They are regular structures", "They are irregular structures", "The loops are often characterized by sharp turns or hairpin-like structures", "If the connecting regions are completely irregular, they belong to random coils"],
    correct: "a",
    explanation: `Residues in the loop or coil regions tend to be charged and polar and located on the surface of the protein structure. They are often the evolutionarily variable regions where mutations, deletions, and insertions frequently occur. They can be functionally significant because these locations are often the active sites of proteins.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Globular proteins are usually insoluble.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Globular proteins are usually soluble and surrounded by water molecules. They tend to have an overall compact structure of spherical shape with polar or hydrophilic residues on the surface and hydrophobic residues in the core. Such an arrangement is energetically favorable because it minimizes contacts with water by hydrophobic residues in the core and maximizes interactions with water by surface polar and charged residues. Common examples of globular proteins are enzymes, myoglobins, cytokines, and protein hormones.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the Integral Membrane Proteins?`,
    options: ["Membrane proteins exist in lipid bilayers of cell membranes", "The exterior of the proteins spanning the membrane must be very hydrophobic to be stable", "The exterior of the proteins spanning the membrane must be very hydrophilic to be stable", "Most typical transmembrane segments are \u03b1-helices"],
    correct: "c",
    explanation: `Because they are surrounded by lipids, the exterior of the proteins spanning the membrane must be very hydrophobic to be stable. Occasionally, for some bacterial periplasmic membrane proteins, they are composed of β-strands. The loops connecting these segments sometimes lie in the aqueous phase, in which they can be entirely hydrophilic.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the X-ray Crystallography?`,
    options: ["In x-ray protein crystallography, proteins need to be grown into large crystals in which their positions are fixed in a repeated, ordered fashion", "The protein crystals are illuminated with an intense x-ray beam", "The x-rays are deflected by the electron clouds surrounding the atoms in the crystal producing a regular pattern of diffraction", "The protein crystals are illuminated with an intense infrared beam"],
    correct: "d",
    explanation: `The diffraction pattern is composed of thousands of tiny spots recorded on a x-ray film.
The diffraction pattern can be converted into an electron density map using a mathematical procedure known as Fourier transform. To interpret a three-dimensional structure from two-dimensional electron density maps requires solving the phases in the diffraction data.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is not correct about the NMR?`,
    options: ["It stands for Nuclear Magnetic Resonance", "NMR spectroscopy detects spinning patterns of atomic nuclei in a electric field", "NMR spectroscopy detects spinning patterns of atomic nuclei in a magnetic field", "Protein samples are labeled with radioisotopes such as 13C and 15N"],
    correct: "b",
    explanation: `Radiofrequency radiation is used to induce transitions between nuclear spin states in a magnetic field. Interactions between spinning isotope pairs produce radio signal peaks that correlate with the distances between them. By interpreting the signals observed using NMR, proximity between atoms can be determined.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `One can search a structure in PDB using the four-letter code or keywords related to its annotation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Each entry is given a unique code, PDB id, consisting of four characters of either letters A to Z or digits 0 to 9 such as 1LYZ and 4RCR. The identified structure can be viewed directly online or downloaded to a local computer for analysis.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The main feature of computer visualization programs is interactivity, which allows users to visually manipulate the structural images through a graphical user interface.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `At the touch of a mouse button, a user can move, rotate, and zoom an atomic model on a computer screen in real time, or examine any portion of the structure in great detail, as well as draw it in various forms in different colors. Further manipulations can include changing the conformation of a structure by protein modeling or matching a ligand to an enzyme active site through docking exercises.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `A Protein Data Bank (PDB) data file for a protein structure contains only x, and z coordinates of atoms.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Because a Protein Data Bank (PDB) data file for a protein structure contains only x, y, and z coordinates of atoms, the most basic requirement for a visualization program is to build connectivity between atoms to make a view of a molecule. The visualization program should also be able to produce molecular structures in different styles, which include wire frames, balls and sticks, space-filling spheres, and ribbons.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `A wire-frame diagram is a line drawing representing bonds between atoms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The wire frame is the simplest form of model representation. It is useful for localizing positions of specific residues in a protein structure, or for displaying a skeletal form of a structure when Cα atoms of each residue are connected.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Balls and sticks are solid spheres and rods, representing atoms and bonds, respectively.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These diagrams can also be used to represent the backbone of a structure. In a space-filling representation, each atom is described using large solid spheres with radii corresponding to the van der Waals radii of the atoms.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Ribbon diagrams use cylinders or spiral ribbons to represent α-helices and broad, flat arrows to represent β-strands.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This type of representation is very attractive in that it allows easy identification of secondary structure elements and gives a clear view of the overall topology of the structure. The resulting images are also visually appealing.
Free 30-Day C++ Certification Bootcamp is Live. Join Now!`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Different representation styles can be used in combination to highlight a certain feature of a structure while deemphasizing the structures surrounding it.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, a cofactor of an enzyme can be shown as space-filling spheres while the rest of the protein structure is shown as wire frames or ribbons. Some widely used and freely available software programs are there for molecular graphics.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is wrong about Swiss-PDB Viewer?`,
    options: ["It is a structure viewer for multiple platforms", "It is a structure viewer for single platforms", "It is essentially a Swiss-Army knife for structure visualization and modeling", "It is capable of structure visualization, analysis, and homology modeling"],
    correct: "b",
    explanation: `It is essentially a Swiss-Army knife for structure visualization and modeling because it incorporates so many functions in a small shareware program. It allows display of multiple structures at the same time in different styles, by charge distribution, or by surface accessibility. It can measure distances, angles, and even mutate residues. In addition, it can calculate molecular surface, electrostatic potential, Ramachandran plot, and so on. The homology modeling part includes energy minimization and loop modeling.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is an incorrect statement?`,
    options: ["The last column contains the sequences themselves", "It is quite tough making an index of the information in each of these fields so that a search query can locate all the occurrences through the index", "If one imagines a large table with each sequence entry occupying one row, then each column will include one of the above types of information for each sequence, and each column is called a FIELD", "The DNA, protein, and reference databases have all been cross-referenced so that moving between them is readily accomplished"],
    correct: "b",
    explanation: `It is very easy to make an index of the information in each of these fields so that a search query can locate all the occurrences through the index. Even related sequences are cross-referenced. In addition, the information in one database can be cross-referenced to that in another database.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Ribbons another UNIX program similar to Molscript, generates ribbon diagrams depicting protein secondary structures.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Aesthetically appealing images can be produced that are of publication quality. However, the program, which is also command-line-based, is extremely difficult to use.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is incorrect about protein structure comparison?`,
    options: ["The comparative approach is important in finding remote protein homologs", "Protein structures have a much higher degree of conservation than the sequences", "Protein structures have a much lesser degree of conservation than the sequences", "Proteins can share common structures even without sequence similarity"],
    correct: "c",
    explanation: `Structure comparison is one of the fundamental techniques in protein structure analysis. Structure comparison can often reveal distant evolutionary relationships between proteins, which is not feasible using the sequence-based alignment approach alone. In addition, protein structure comparison is a prerequisite for protein structural classification into different fold classes.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The intermolecular approach is normally applied to relatively _____ structures.`,
    options: ["distinctive", "dissimilar", "similar", "different"],
    correct: "c",
    explanation: `The algorithmic approaches to comparing protein geometric properties can be divided into three categories: the first superposes protein structures by minimizing intermolecular distances; the second relies on measuring intramolecular distances of a structure; and the third includes algorithms that combine both intermolecular and intramolecular approaches.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is incorrect about intermolecular approach?`,
    options: ["This procedure starts with identifying equivalent residues or atoms", "After residue\u2013residue correspondence is established, one of the structures is moved laterally and vertically toward the other structure to allow the two structures to be in the same location", "The structures are rotated relative to each other around the three-dimensional axes", "The rotation doesn\u2019t depend on the intermolecular distance"],
    correct: "d",
    explanation: `The rotation continues until the shortest intermolecular distance is reached. At this point, an optimal superimposition of the two structures is reached. After superimposition, equivalent residue pairs can be identified, which helps to quantitate the fitting between the two structures.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The root mean square deviation (RMSD), without the size dependency correction is?`,
    options: ["\u2211Ni=1D2iN", "\u2211Ni=1D2iN+1", "\u2211Ni=1D2iNxN", "\u2211Ni=1D2iN\u20131"],
    correct: "a",
    explanation: `An important measurement of the structure fit during superposition is the distance between equivalent positions on the protein structures. This requires using a least square-fitting function called root mean square deviation (RMSD), which is the square root of the averaged sum of the squared differences of the atomic distances. Here D is the distance between coordinate data points and N is the total number of corresponding residue pairs.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The root mean square deviation (RMSD), with the size dependency correction is?`,
    options: ["RMSD\u22121.3+0.5ln(N+1)", "RMSD\u22121.3+0.5ln(NxN)", "RMSD\u22121.3+0.5ln(N\u22121)", "RMSD\u22121.3+0.5ln(N)"],
    correct: "d",
    explanation: `In practice, only the distances between Cα carbons of corresponding residues are measured. The goal of structural comparison is to achieve a minimum RMSD. However, the problem with RMSD is that it depends on the size of the proteins being compared. For the same degree of sequence identity, large proteins tend to have higher RMSD values than small proteins when an optimal alignment is reached. Recently, a logarithmic factor has been proposed to correct this size-dependency problem. This new measure is called RMSD100.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The intramolecular approach does not depend on sequence similarity between the proteins to be compared.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The intramolecular approach relies on structural internal statistics and therefore does not depend on sequence similarity between the proteins to be compared. In addition, this method does not generate a physical superposition of structures but instead provides a quantitative evaluation of the structural similarity between corresponding residue pairs.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is incorrect about the intramolecular approach?`,
    options: ["The method works by generating a distance matrix between residues of the same protein", "It generates a string between residues of the same protein", "In comparing two protein structures, the distance matrices from the two structures are moved relative to each other to achieve maximum overlaps", "By overlaying two distance matrices, similar intramolecular distance patterns representing similar structure folding regions can be identified"],
    correct: "b",
    explanation: `For the ease of comparison, each matrix is decomposed into smaller submatrices consisting of hexapeptide fragments. To maximize the similarity regions between two structures, a Monte Carlo procedure is used. By reducing three-dimensional information into two-dimensional information, this strategy identifies overall structural resemblances and common structure cores.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is incorrect about Multiple Structure Alignment?`,
    options: ["The alignment strategy is different than the Clustal sequence alignment using a progressive approach", "All structures are first compared in a pairwise fashion", "A distance matrix is developed based on structure similarity scores such as RMSD", "The aligned structures create a median structure that allows other structures to be progressively added for comparison based on the hierarchy described in the guide tree"],
    correct: "a",
    explanation: `In addition to pairwise alignment, a number of algorithms can also perform multiple structure alignment. The alignment strategy is similar to the Clustal sequence alignment using a progressive approach. When all the structures in the set are added, this eventually creates a multiple structure alignment.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is incorrect about SSAP?`,
    options: ["It is a web server that uses an intramolecular distance\u2013based method", "Matrices are built based on the C\u03b2 distances of all residue pairs", "Dynamic programming approach is not used here", "Dynamic programming approach is used"],
    correct: "c",
    explanation: `When comparing two different matrices, a dynamic programming approach is used to find the path of residue positions with optimal scores. The dynamic programming is applied at two levels, one at a lower level in which all residue pairs between the proteins are compared and another at an upper level in which subsequently identified equivalent residue pairs are processed to refine the matching positions. An SSAP score is reported for structural similarity. A score above 70 indicates a good structural similarity.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `VAST is a web server that performs alignment using intramolecular approaches only.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `VAST (Vector Alignment Search Tool) is a web server that performs alignment using both the inter- and intramolecular approaches. The superposition is based on information of directionality of secondary structural elements (represented as vectors). Optimal alignment between two structures is defined by the highest degree of vector matches.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The classification results from both systems, SCOP and CATH are quite dissimilar.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Due to the differences in classification criteria, one might expect that there would be huge differences in classification results. In fact, the classification results from both systems are quite similar. Exhaustive analysis has shown that the results from the two systems converge at about 80% of the time. In other words, only about 20% of the structure fold assignments are different.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The first step in structure classification is to remove redundancy from databases.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Among the tens of thousands of entries in PDB, the majority of the structures are redundant as they correspond to structures solved at different resolutions, or associated with different ligands or with single-residue mutations. The redundancy can be removed by selecting representatives through a sequence alignment–based approach.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The second step in structure classification is to separate structurally distinct domains within a structure.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because some proteins are composed of multiple domains, they must be subdivided before a sensible structural comparison can be carried out. This domain identification and separation can be done either manually or based on special algorithms for domain recognition.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The last step in structure classification involves grouping proteins/domains of similar structures.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Once multidomain proteins are split into separate domains, structure comparison can be conducted at the domain level, either through manual inspection, or automated structural alignment, or a combination of both. This step involves grouping proteins/domains of similar structures and clustering them based on different levels of resemblance in secondary structure composition and arrangement of the secondary structures in space.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is untrue about SCOP?`,
    options: ["It is a database for comparing and classifying protein structures", "It is constructed almost entirely based on manual examination of protein structures", "The proteins are grouped into hierarchies of classes, folds, superfamilies, and families", "The SCOP families consist of proteins having low sequence identity (>30%)"],
    correct: "d",
    explanation: `The SCOP families consist of proteins having high sequence identity (>30%). Thus, the proteins within a family clearly share close evolutionary relationships and normally have the same functionality. The protein structures at this level are also extremely similar.
Free 30-Day C++ Certification Bootcamp is Live. Join Now!`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Members within the ____ fold ______ have evolutionary relationships.`,
    options: ["same, always", "same, do not always", "one, always", "different, do not"],
    correct: "b",
    explanation: `Folds consist of superfamilies with a common core structure, which is determined manually. This level describes similar overall secondary structures with similar orientation and connectivity between them. Members within the same fold do not always have evolutionary relationships. Some of the shared core structure may be a result of analogy. Classes consist of folds with similar core structures.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `In CATH, Structural domain separation is carried by ___________`,
    options: ["manual comparison only", "computer programs only", "human expertise only", "a combined effort of a human expert and computer programs"],
    correct: "d",
    explanation: `CATH classifies proteins based on the automatic structural alignment program SSAP as well as manual comparison. Structural domain separation is carried out also as a combined effort of a human expert and computer programs. Individual domain structures are classified at five major levels: class, architecture, fold/topology, homologous superfamily, and homologous family.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `Which of the following is untrue about SCOP and CATH?`,
    options: ["The definition for class in CATH is quite dissimilar to that in SCOP", "The definition for class in CATH is based on secondary structure content", "Architecture is a unique level in CATH, intermediate between fold and class", "The definition for class in CATH is similar to that in SCOP"],
    correct: "d",
    explanation: `The topology level is equivalent to the fold level in SCOP, which describes overall orientation of secondary structures and takes into account the sequence connectivity between the secondary structure elements. The homologous superfamily and homologous family levels are equivalent to the superfamily and family levels in SCOP with similar evolutionary definitions, respectively.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `SCOP is _______ based on manual comparison of structures by human experts with no quantitative criteria to group proteins.`,
    options: ["entirely", "almost entirely", "not", "partially"],
    correct: "b",
    explanation: `It is argued that this approach offers some flexibility in recognizing distant structural relatives, because human brains may be more adept at recognizing slightly dissimilar structures that essentially have the same architecture. However, this reliance on human expertise also renders the method subjective. The exact boundaries between levels and groups are sometimes arbitrary.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `CATH is a combination of manual curation and automated procedure, which makes the process less subjective.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, in defining domains, CATH first relies on the consensus of three different algorithms to recognize domains. When the computer programs disagree, human intervention will take place. In addition, the extra Architecture level in CATH makes the structure classification more continuous. The drawback of the systems is that the fixed thresholds in structural comparison may make assignment less accurate.`,
    super_topic: `Structural Bioinformatics`,
  },
  {
    question: `The formation of _____ is determined by ______ interactions, whereas the formation of ____ is strongly influenced by ______ interactions.`,
    options: ["\u03b1-helices, long-range, \u03b1-helices, short-range", "\u03b1-helices, long-range, \u03b2-strands, short-range", "\u03b1-helices, short-range, \u03b2-strands, long-range", "\u03b2-strands, short-range, \u03b2-strands, long-range"],
    correct: "c",
    explanation: `Protein secondary structure prediction with high accuracy is not a trivial ask. It remained a very difficult problem for decades. This is because protein secondary structure elements are context dependent. The formation of α-helices is determined by short-range interactions, whereas the formation of β-strands is strongly influenced by long-range interactions. Prediction for long-range interactions is theoretically difficult. After more than three decades of effort, prediction accuracies have only been improved from about 50% to about 75%.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The secondary structure prediction methods can be either ab initio based, which make use of single sequence information only, or homology based, which makes use of multiple sequence alignment information.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The ab initio methods, which belong to early generation methods, predict secondary structures based on statistical calculations of the residues of a single query sequence. The homology-based methods do not rely on statistics of residues of a single sequence, but on common secondary structural patterns conserved among multiple homologous sequences.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Ab Initio–Based Methods?`,
    options: ["This type of method predicts the secondary structure based on a single query sequence", "This type of method predicts the secondary structure based on a multiple query sequence", "It measures the relative propensity of each amino acid belonging to a certain secondary structure element", "The propensity scores are derived from known crystal structures"],
    correct: "b",
    explanation: `Examples of ab initio prediction are the Chou–Fasman and Garnier, Osguthorpe, Robson (GOR) methods. The ab initio methods were developed in the 1970s when protein structural data were very limited. The statistics derived from the limited data sets can therefore be rather inaccurate. However, the methods are simple enough that they are often used to illustrate the basics of secondary structure prediction.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The Chou–Fasman algorithm determines the propensity or intrinsic tendency of each residue to be in the helix, strand, and β-turn conformation using observed frequencies found in protein crystal structures.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It determines the propensity or intrinsic tendency of each residue using observed frequencies found in protein crystal structures (conformational values for coils are not considered). For example, it is known that alanine, glutamic acid, and methionine are commonly found in α-helices, whereas glycine and proline are much less likely to be found in such structures.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The GOR method is based on the “propensity” of each residue to be in one of the two conformational states, helix (H), strand(E).`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The GOR method is based on the “propensity” of each residue to be in one of the two conformational states, helix (H), strand(E), turn(T), and coil (C). However, instead of using the propensity value from a single residue to predict a conformational state, it takes short-range interactions of neighboring residues into account.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Chou–Fasman and GOR methods?`,
    options: ["Both are the first-generation methods", "They are developed in the 1970s,", "They suffer from the fact that the prediction rules are somewhat arbitrary", "They are based on single sequence statistics with clear relation to known protein-folding theories"],
    correct: "d",
    explanation: `They are based on single sequence statistics without clear relation to known protein-folding theories. The predictions solely rely on local sequence information and fail to take into account long range interactions. A Chou-Fasman–based prediction does not even consider the short-range environmental information.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Homology-Based Methods?`,
    options: ["The third generation of algorithms was developed in the late 1990s", "They were developed by making use of evolutionary information", "This type of method uses the ab initio secondary structure prediction of individual sequences only", "This type of method combines the ab initio secondary structure prediction of individual sequences and alignment information from multiple homologous sequences (>35% identity)"],
    correct: "c",
    explanation: `The idea behind this approach is that close protein homologs should adopt the same secondary and tertiary structure. When each individual sequence is predicted for secondary structure using a method similar to the GOR method, errors and variations may occur. However, evolutionary conservation dictates that there should be no major variations for their secondary structure elements.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Because residues in the same aligned position are assumed to have the same secondary structure, any inconsistencies or errors in prediction of individual sequences can be corrected using a majority rule.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `By aligning multiple sequences, information of positional conservation is revealed. This homology based method has helped improve the prediction accuracy by another 10% over the second-generation methods.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Prediction with Neural Networks?`,
    options: ["When multiple sequence alignments and neural networks are combined, the result is further improved accuracy", "A neural network is trained by a single sequence", "A neural network is trained by a sequence profile derived from the multiple sequence alignment", "When the sufficiently trained network processes an unknown sequence, it applies the rules learned in training to recognize particular structural patterns"],
    correct: "b",
    explanation: `A neural network is trained not by a single sequence but by a sequence profile derived from the multiple sequence alignment. This combined approach has been shown to improve the accuracy to above 75%, which is a breakthrough in secondary structure prediction. The improvement mainly comes from enhanced secondary structure signals through consensus drawing. The following lists several frequently used third generation prediction algorithms available as web servers.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is untrue regarding PHD?`,
    options: ["It stands for Profile network from Heidelberg", "It is a web-based program that combines neural network only", "It first performs a BLASTP of the query sequence against a non redundant protein sequence database", "In initial steps it finds a set of homologous sequences, which are aligned with the MAXHOM program (a weighted dynamic programming algorithm performing global alignment)"],
    correct: "b",
    explanation: `It is a web-based program that combines neural network with multiple sequence alignment. After the initial steps, the resulting alignment in the form of a profile is fed into a neural network that contains three hidden layers. The first hidden layer makes raw prediction based on the multiple sequence alignment by sliding a window of thirteen positions.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding the transmembrane proteins?`,
    options: ["Constitute up to 30% of all cellular proteins", "They are responsible for performing a wide variety of important functions in a cell, such as signal transduction, cross-membrane transport, and energy conversion", "The membrane proteins are also of tremendous biomedical importance", "They are not drug targets or receptors"],
    correct: "d",
    explanation: `The membrane proteins are also of tremendous biomedical importance, as they often serve as drug targets for pharmaceutical development. There are two types of integral membrane proteins: α-helical type and β-barrel type. Most transmembrane proteins contain solely α-helices, which are found in the cytoplasmic membrane. A few membrane proteins consist of β-strands forming a β- barrel topology, a cylindrical structure composed of antiparallel β-sheets.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The structures of this group of proteins, however, are comparatively a lot difficult to resolve either by x-ray crystallography or nuclear magnetic resonance (NMR) spectroscopy.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For this group of proteins, prediction of the transmembrane secondary structural elements and their organization is particularly important. Fortunately, the prediction process is somewhat easier because of the hydrophobic environment of the lipid bilayers, which restricts the transmembrane segments to be hydrophobic as well.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Prediction of Helical Membrane Proteins?`,
    options: ["For membrane proteins consisting of transmembrane \u03b1\u2013helices, these transmembrane helices are predominantly hydrophobic with a specific distribution of positively charged residues", "The \u03b1-helices generally run perpendicular to the membrane plane", "The \u03b1-helices generally run parallel to the membrane plane", "The \u03b1-helices have an average length between seventeen and twenty-five residues"],
    correct: "c",
    explanation: `The hydrophobic helices are normally separated by hydrophilic loops with average lengths of fewer than sixty residues. The residues bordering the transmembrane spans are more positively charged. Another feature indicative of the presence of transmembrane segments is that residues at the cytosolic side near the hydrophobic anchor are more positively charged than those at the lumenal or periplasmic side. This is known as the positive-inside rule.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The early algorithms based their prediction on hydrophobicity scales.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A number of algorithms for identifying transmembrane helices have been developed where the early algorithms based their prediction on hydrophobicity scales. They typically scan a window of seventeen to twenty-five residues and assign membrane spans based on hydrophobicity scores. Some are also able to determine the orientation of the membrane helices based on the positive-inside rule.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Predictions solely based on hydrophobicity profiles have lowest error rates.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Predictions solely based on hydrophobicity profiles have high error rates. As with the third-generation predictions for globular proteins, applying evolutionary information with the help of neural networks or HMMs can improve the prediction accuracy significantly.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `The presence of ______ signal peptides can significantly compromise the prediction _______ because the programs tend to confuse hydrophobic signal peptides with membrane helices.`,
    options: ["hydrophobic, accuracy", "hydrophobic, error", "hydrophilic, accuracy", "hydrophilic, error"],
    correct: "a",
    explanation: `Predicting transmembrane helices is relatively easy. The accuracy of Some of the best predicting programs, such as TMHMM or HMMTOP, can exceed 70%. To minimize errors, the presence of signal peptides can be detected using a number of specialized programs and then manually excluded.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding TMHMM?`,
    options: ["It is a web-based program based on an HMM algorithm", "It is trained to recognize transmembrane helical patterns", "It is not trained to recognize transmembrane helical patterns", "When a query sequence is scanned, the probability of having an \u03b1-helical domain is given"],
    correct: "c",
    explanation: `It is trained to recognize transmembrane helical patterns based on a training set of 160 well-characterized helical membrane proteins. The orientation of the α-helices is predicted based on the positive-inside rule. The prediction output returns the number of transmembrane helices, the boundaries of the helices, and a graphical representation of the helices. This program can also be used to simply distinguish between globular proteins and membrane proteins.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Phobius?`,
    options: ["It is a web-based program designed to overcome false positives caused by the presence of signal peptides", "The program incorporates distinct HMM models for signal peptides only", "The program incorporates distinct HMM models for signal peptides as well as transmembrane helices", "After distinguishing the putative signal peptides from the rest of the query sequence, prediction is made on the remainder of the sequence"],
    correct: "b",
    explanation: `In addition to the given data, it has been shown that the prediction accuracy can be significantly improved compared to TMHMM (94% by Phobius compared to 70% by TMHMM). In addition to the normal prediction mode, the user can also define certain sequence regions as signal peptides or other nonmembrane sequences based on external knowledge.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is true regarding Prediction of β-Barrel Membrane Proteins?`,
    options: ["For membrane proteins with \u03b2-strands only, the \u03b2-strands forming the transmembrane segment are amphipathic in nature", "For membrane proteins with \u03b2-strands only, the \u03b2-strands forming the transmembrane segment are only hydrophilic in nature", "For membrane proteins with \u03b2-strands only, the \u03b2-strands forming the transmembrane segment are only hydrophobic in nature", "They contain six to nine residues"],
    correct: "a",
    explanation: `As stated, for membrane proteins with β-strands only, the β-strands forming the transmembrane segment are amphipathic in nature. They contain ten to twenty-two residues with every second residue being hydrophobic and facing the lipid bilayers whereas the other residues facing the pore of the β-barrel are more hydrophilic.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Scanning a sequence by hydrophobicity does not reveal transmembrane β-strands.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These programs for predicting transmembrane α-helices are not applicable for this unique type of membrane proteins. To predict the β-barrel type of membrane proteins, a small number of algorithms have been made available based on neural networks and related techniques.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Coiled coils are super helical structures involving two to more interacting α-helices from the same or different proteins.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The individual α-helices twist and wind around each other to form a coiled bundle structure. The coiled coil conformation is important in facilitating inter- or intra protein interactions. Proteins possessing these structural domains are often involved in transcription regulation or in the maintenance of cytoskeletal integrity.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is true regarding Coiled coil?`,
    options: ["They have an integral repeat of twenty residues", "They have an integral repeat of seven residues", "They have an integral repeat of thirty residues", "The sequence periodicity doesn\u2019t contribute in designing algorithms to predict the structural domain."],
    correct: "b",
    explanation: `Coiled coils have an integral repeat of seven residues (heptads) which assume a side-chain packing geometry at facing residues. For every seven residues, the first and fourth are hydrophobic, facing the helical interface; the others are hydrophilic and exposed to the solvent. The sequence periodicity forms the basis for designing algorithms to predict this important structural domain.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding Coils?`,
    options: ["It is a web-based program that detects coiled coil regions in proteins", "It scans a window of fourteen, twenty-one, or twenty-eight residues", "It scans a window of fourteen or twenty-one residues only", "It compares the sequence to a probability matrix compiled from known parallel two-stranded coiled coils"],
    correct: "c",
    explanation: `By comparing the similarity scores, the program calculates the probability of the sequence to adopt a coiled coil conformation. The program is accurate for solvent-exposed, left-handed coiled coils, but less sensitive for other types of coiled coil structures, such as buried or right-handed coiled coils.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `In Multicoil, The scoring matrix is constructed based on a database of known three-stranded coiled coils only.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Multicoil is a web-based program for predicting coiled coils. The scoring matrix is constructed based on a database of known two-stranded and three-stranded coiled coils. The program is more conservative than Coils. It has been recently used in several genome-wide studies to screen for protein–protein interactions mediated by coiled coil domains.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Leucine zipper domains are a special type of coiled coils found in transcription regulatory proteins which contain two anti parallel α-helices held together by hydrophobic interactions of leucine residues.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The heptad repeat pattern is L-X(6)-L-X(6)-L–X(6)-L. This repeat pattern alone can sometimes allow the domain detection, albeit with high rates of false positives. The reason for the high false-positive rates is that the condition of the sequence region being a coiled coil conformation is not satisfied. To address this problem, algorithms have been developed that take into account both leucine repeats and coiled coil conformation to give accurate prediction.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding PSIPRED?`,
    options: ["It is a web-based program that predicts protein secondary structures", "It uses a combination of evolutionary information and neural networks", "It uses a combination of evolutionary information only", "The multiple sequence alignment is derived from a PSI-BLAST database search"],
    correct: "c",
    explanation: `A profile is extracted from the multiple sequence alignment generated from three rounds of automated PSI-BLAST. The profile is then used as input for a neural network prediction similar to that in PHD, but without the jury layer. To achieve higher accuracy, a unique filtering algorithm is implemented to filter out unrelated PSI-BLAST hits during profile construction.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Prof is not similar to PHD.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Prof stands for Protein forecasting. It is an algorithm that combines PSI-BLAST profiles and a multistaged neural network, similar to that in PHD. In addition, it uses a linear discriminant function to discriminate between the three states.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Jpred combines the analysis results from six prediction algorithms, including PHD, PREDATOR, DSC, NNSSP, Jnet, and ZPred.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The query sequence is first used to search databases with PSI-BLAST for three iterations. Redundant sequence hits are removed. The resulting sequence homologs are used to build a multiple alignment from which a profile is extracted. The profile information is submitted to the six prediction programs. If there is sufficient agreement among the prediction programs, the majority of the prediction is taken as the structure.`,
    super_topic: `Secondary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about homology modeling?`,
    options: ["Homology modeling predicts protein structures based on sequence homology with known structures", "It is also known as comparative modeling", "The principle behind it is that if two proteins share a high enough sequence similarity, they are likely to have very similar three-dimensional structures", "It doesn\u2019t involve the evolutionary distances anywhere"],
    correct: "d",
    explanation: `As the name suggests, homology modeling predicts protein structures based on sequence homology with known structures. Homology modeling produces an all-atom model based on alignment with template proteins.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about template Selection Step?`,
    options: ["The first step in protein structural modeling is to select appropriate structural templates", "This forms the foundation for rest of the modeling process", "There is no use of heuristic alignment search programs", "The template selection involves searching the Protein Data Bank (PDB) for homologous proteins with determined structures"],
    correct: "c",
    explanation: `The search can be performed using a heuristic pair wise alignment search program such as BLAST or FASTA. However, the use of dynamic programming based search programs such as SEARCH or ScanPS can result in more sensitive search results. The relatively small size of the structural database means that the search time using the exhaustive method is still within reasonable limits, while giving a more sensitive result to ensure the best possible similarity hits.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about Sequence Alignment Step?`,
    options: ["Once the structure with the highest sequence similarity is identified as a template, the full-length sequences of the template and target proteins need to be realigned using refined alignment algorithms to obtain optimal alignment", "The realignment is the most critical step in homology modeling", "The realignment directly affects the quality of the final model", "Errors made in the alignment step can be corrected in the following modeling steps"],
    correct: "d",
    explanation: `Incorrect alignment at this stage leads to incorrect designation of homologous residues and therefore to incorrect structural models. Errors made in the alignment step cannot be corrected in the following modeling steps. Therefore, the best possible multiple alignment algorithms, such as Praline and T-Coffee should be used for this purpose.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about Backbone Model Building Step?`,
    options: ["Once optimal alignment is achieved, residues in the aligned regions of the target protein can assume a similar structure as the template proteins", "Coordinates of the corresponding residues of the template proteins can be simply copied onto the target protein", "If the two residues differ, everything other than the backbone atoms can be copied", "If the two aligned residues are identical, coordinates of the side chain atoms are copied along with the main chain atoms"],
    correct: "c",
    explanation: `Option “Once optimal alignment is achieved, residues in the aligned regions of the target protein can assume a similar structure as the template proteins” and “Coordinates of the corresponding residues of the template proteins can be simply copied onto the target protein” mean the same. If the two residues differ, only the backbone atoms can be copied. The side chain atoms are rebuilt in a subsequent procedure.
In backbone modeling, it is simplest to use only one template structure. The structure with the best quality and highest resolution is normally chosen if multiple options are available.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about Loop Modeling Step?`,
    options: ["In the sequence alignment for modeling, there are often regions caused by insertions and deletions producing gaps in sequence alignment", "In the sequence alignment for modeling, there are no regions producing gaps in sequence alignment", "The gaps cannot be directly modeled", "Loop modeling is required for closing the gaps requires"],
    correct: "b",
    explanation: `Closing the gaps requires loop modeling, which is a very difficult problem in homology modeling and is also a major source of error. Loop modeling can be considered a mini–protein modeling problem by itself. Unfortunately, there are no mature methods available that can model loops reliably. Currently, there are two main techniques used to approach the problem: the database searching method and the ab initio method.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `The procedure begins by measuring the orientation and distance of the anchor regions in the stems and searching PDB for segments of the same length that also match the above endpoint conformation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Usually, many different alternative segments that fit the endpoints of the stems are available. The best loop can be selected based on sequence similarity as well as minimal steric clashes with the neighboring parts of the structure. The conformation of the best matching fragments is then copied onto the anchoring points of the stems.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about specialized programs for loop modeling?`,
    options: ["PETRA is a web server that models loops using the database approach", "FREAD is a web server that models loops using the database approach", "CODA is a web server that uses a consensus method based on the prediction results from FREAD and PETRA", "For loops of three to eight residues, CODA uses consensus conformation of both methods"],
    correct: "a",
    explanation: `PETRA is a web server that uses the ab initio method to model loops. For nine to thirty residues, CODA uses FREAD prediction only.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `In Side Chain Refinement step, A side chain can be built by searching every possible conformation at every torsion angle of the side chain to select the one that has the lowest interaction energy with neighboring atoms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `However, this approach is computationally prohibitive in most cases. In fact, most current side chain prediction programs use the concept of rotamers, which are favored side chain torsion angles extracted from known protein crystal structures. In prediction of side chain conformation, only the possible rotamers with the lowest interaction energy with nearby atoms are selected.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `In the step of Model Refinement Using Energy Function, the structural irregularities can be corrected by applying the energy minimization procedure on the entire model, which moves the atoms in such a way that the overall conformation has the lowest energy potential.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In these loop modeling and side chain modeling steps, potential energy calculations are applied to improve the model. However, this does not guarantee that the entire raw homology model is free of structural irregularities such as unfavorable bond angles, bond lengths, or close atomic contacts. There, this step is used. The goal of energy minimization is to relieve steric collisions and strains without significantly altering the overall structure.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Energy minimization has to be used with caution because excessive energy minimization often moves residues away from their correct positions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Only limited energy minimization is recommended (a few hundred iterations) to remove major errors, such as short bond distances and close atomic clashes. Key conserved residues and those involved in cofactor binding have to be restrained if necessary during the process.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about Ab initio prediction?`,
    options: ["The limited knowledge of protein folding forms the basis of ab initio prediction", "The ab initio prediction method attempts to produce all-atom protein models based on sequence information alone without the aid of known protein structures", "The ab initio prediction method attempts to produce all-atom protein models based on sequence information alone with some aid of known protein structures", "The perceived advantage of this method is that predictions are not restricted by known folds and that novel protein folds can be identified"],
    correct: "c",
    explanation: `Alongside the advantages, because the physicochemical laws governing protein folding are not yet well understood, the energy functions used in the ab initio prediction are, at present, rather inaccurate. The folding problem remains one of the greatest challenges in bioinformatics today.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `The prediction programs are thus designed using the energy minimization principle.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Current ab initio algorithms are not yet able to accurately simulate the protein folding process. They work by using some type of heuristics. Because the native state of a protein structure is near energy minimum, the prediction programs are thus designed using the energy minimization principle.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Searching for a fold with the absolute minimum energy may not be valid in reality.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These algorithms search for every possible conformation to find the one with the lowest global energy. However, searching for a fold with the absolute minimum energy may not be valid in reality. This contributes to one of the fundamental flaws of this approach. In addition, searching for all possible structural conformations is not yet computationally feasible.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Rosetta is a web server that predicts protein three-dimensional conformations using the ab initio method.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This in fact relies on a “mini-threading” method. The method first breaks down the query sequence into many very short segments (three to nine residues) and predict the secondary structure of the small segments using a Hidden Markov model–based program, HMMSTR.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `In Rosetta, The segments with assigned _______ structures are subsequently assembled into a ______ dimensional configuration.`,
    options: ["primary, three", "secondary, three", "secondary, two", "primary, three"],
    correct: "b",
    explanation: `Through random combinations of the fragments, a large number of models are built and their overall energy potentials calculated. The conformation with the lowest global free energy is chosen as the best model.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `There are a large number of protein folds available, compared to millions of protein sequences.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `There are only small number of protein folds available (<1,000), compared to millions of protein sequences. This means that protein structures tend to be more conserved than protein sequences. Consequently, many proteins can share a similar fold even in the absence of sequence similarities. This allowed the development of computational methods to predict protein structures beyond sequence similarities.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Threading or structural fold recognition predicts the structural fold of an unknown protein sequence by fitting the sequence into a structural database and selecting the best-fitting fold.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `To determine whether a protein sequence adopts a known three-dimensional structure fold relies on threading and fold recognition methods. The comparison emphasizes matching of secondary structures, which are most evolutionarily conserved. Therefore, this approach can identify structurally similar proteins even without detectable sequence similarity.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `The algorithms used here can be classified into two categories, pairwise energy based and profile based.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The pairwise energy–based method was originally referred to as threading and the profile-based method was originally defined as fold recognition. However, the two terms are now often used interchangeably without distinction in the literature.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `In the pairwise energy based method, a protein sequence is searched for in a structural fold database to find the best matching structural fold using ______ criteria.`,
    options: ["sequence-based", "structure-based", "energy-based", "residue-based"],
    correct: "c",
    explanation: `The detailed procedure involves aligning the query sequence with each structural fold in a fold library. The alignment is performed essentially at the sequence profile level using dynamic programming or heuristic approaches. Local alignment is often adjusted to get lower energy and thus better fitting. The adjustment can be achieved using algorithms such as double-dynamic programming.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `The next step in the pairwise energy based method is to build a crude model for the target sequence by replacing aligned residues in the template structure with the corresponding residues in the query.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `After the mentioned step, the last step is to calculate the energy terms of the raw model, which include pairwise residue interaction energy, solvation energy, and hydrophobic energy. Finally, the models are ranked based on the energy terms to find the lowest energy fold that corresponds to the structurally most compatible fold.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about profile method?`,
    options: ["A profile is constructed for a group of related protein structures", "The propensity of amino acids in not in picture of this method", "Statistical information from these aligned residues is then used to construct a profile", "The structural profile is generated by the superimposition of the structures to expose corresponding residues"],
    correct: "b",
    explanation: `The profile contains scores that describe the propensity of each of the twenty amino acid residues to be at each profile position. The profile scores contain information for secondary structural types, the degree of solvent exposure, polarity, and hydrophobicity of the amino acids. To predict the structural fold of an unknown query sequence, the query sequence is first predicted for its secondary structure, solvent accessibility, and polarity.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Because threading and fold recognition detect structural homologs ________ relying on sequence similarities, they have been shown to be _______ than PSI-BLAST in finding distant evolutionary relationships`,
    options: ["without completely, far more sensitive", "completely, far more sensitive", "completely, less sensitive", "without completely, less sensitive"],
    correct: "a",
    explanation: `In many cases, they can identify more than twice as many distant homologs than PSI-BLAST. However, this high sensitivity can also be their weakness because high sensitivity is often associated with low specificity. The predictions resulting from threading and fold recognition often come with very high rates of false positives. Therefore, much caution is required in accepting the prediction results.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about threading and fold recognition?`,
    options: ["It assess the compatibility of an amino acid sequence with a known structure in a fold library", "If the protein fold to be predicted does not exist in the fold library, the method won\u2019t necessarily fail", "If the protein fold to be predicted does not exist in the fold library, the method will fail", "Threading and fold recognition do not generate fully refined atomic models for the query sequences"],
    correct: "b",
    explanation: `A disadvantage compared to homology modeling lies in the fact that threading and fold recognition do not generate fully refined atomic models for the query sequences. This is because accurate alignment between distant homologs is difficult to achieve. Instead, threading and fold recognition procedures only provide a rough approximation of the overall topology of the native structure.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about 3D-PSSM?`,
    options: ["It is a web-based program that employs the structural profile method to identify protein folds", "The profiles for each protein superfamily are constructed by combining multiple smaller profiles", "A protein structural superfamily doesn\u2019t have sequence-based PSI-BLAST profile", "In initial steps, protein structures in a superfamily based on the SCOP classification are superimposed"],
    correct: "c",
    explanation: `First, protein structures in a superfamily based on the SCOP classification are superimposed and are used to construct a structural profile by incorporating secondary structures and solvent accessibility information for corresponding residues. In addition, each member in a protein structural superfamily has its own sequence-based PSI-BLAST profile computed. These sequence profiles are used in combination with the structure profile to form a large superfamily profile in which each position contains both sequence and structural information.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `Which of the following is true about Gen Threader?`,
    options: ["It is a web-based program that uses a hybrid of the profile and pairwise energy methods", "It is a web-based program that uses profile methods only", "It is a web-based program that uses pairwise energy methods only", "The initial step is quite dissimilar to 3D-PSSM"],
    correct: "a",
    explanation: `The initial step is similar to 3D-PSSM; the query protein sequence is subject to three rounds of PSI-BLAST. The resulting multiple sequence hits are used to generate a profile. Its secondary structure is predicted using PSIPRED. Both are used as input for threading computation based on a pairwise energy potential method. The threading results are evaluated using neural networks that combine energy potentials, sequence alignment scores, and length information to create a single score representing the relationship between the query and template proteins.`,
    super_topic: `Tertiary Structure Prediction`,
  },
  {
    question: `RNA structures can be experimentally determined using _____`,
    options: ["X-ray crystallography techniques only", "NMR techniques only", "X-ray crystallography or NMR techniques", "Gel electrophoresis"],
    correct: "c",
    explanation: `However, the two approaches are extremely time consuming and expensive. As a result, computational prediction has become an attractive alternative. Option Gel electrophoresis, here, becomes irrelevant as it comes to the structure of RNA.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is not a form of RNA?`,
    options: ["mRNA", "tRNA", "qRNA", "rRNA"],
    correct: "c",
    explanation: `It is known that RNA is a carrier of genetic information and exists in three main forms. They are messenger RNA (mRNA), ribosomal RNA (rRNA), and transfer RNA (tRNA). Their main roles are as follows: mRNA is responsible for directing protein synthesis; rRNA provides structural scaffolding within ribosomes; and tRNA serves as a carrier of amino acids for polypeptide synthesis.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Unlike DNA, which is mainly double stranded, RNA is double stranded, although an RNA molecule can self-hybridize at certain regions to form partial double-stranded structures.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `RNA is single stranded, although an RNA molecule can self-hybridize at certain regions to form partial double-stranded structures. Generally, mRNA is more or less linear and non-structured, whereas rRNA and tRNA can only function by forming particular secondary and tertiary structures.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Structures are not much important when it comes to the study of the functions.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Knowledge of the structures of these molecules is particularly important for understanding their functions. Difficulties in experimental determination of RNA structures make theoretical prediction a very desirable approach. In fact, computational-based analysis is the main tool in RNA-based drug design in pharmaceutical industry. In addition, knowledge of the secondary structures of rRNA is key for RNA-based phylogenetic analysis.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is not a base of RNA?`,
    options: ["Thymine (T)", "Adenine (A)", "Cytosine (C)", "Guanine (G)"],
    correct: "a",
    explanation: `RNA structures can be described at three levels as in proteins: primary, secondary, and tertiary. The primary structure is the linear sequence of RNA, consisting of four bases, adenine (A), cytosine (C), guanine (G), and uracil (U).`,
    super_topic: `Quizzone`,
  },
  {
    question: `Base pairing, in RNA, is A–G and U–C.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The secondary structure refers to the planar representation that contains base-paired regions among single-stranded regions. The base pairing is mainly composed of traditional Watson–Crick base pairing, which is A–U and G–C.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In addition to the canonical base pairing, there often exists non-canonical base pairing such as _________ and ________ base paring.`,
    options: ["G, U", "G, C", "U, C", "A, C"],
    correct: "a",
    explanation: `There often exists non-canonical base pairing such as G and U base paring. The G–U base pair is less stable and normally occurs within a double-strand helix surrounded by Watson–Crick base pairs. Finally, the tertiary structure is the three-dimensional arrangement of bases of the RNA molecule.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Four main subtypes of secondary structures can be identified. They are hairpin loops, bulge loops, interior loops, and multi-branch loops.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because the RNA tertiary structure is very difficult to predict, attention has been mainly focused on secondary structure prediction. Based on the arrangement of helical base pairing in secondary structures, the mentioned four main subtypes of secondary structures can be identified.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The ______ refers to a structure with two ends of a single-stranded region (loop) connecting a base-paired region (stem).`,
    options: ["helical junctions", "hairpin loop", "bulge loop", "interior loop"],
    correct: "b",
    explanation: `The bulge loop refers to a single stranded region connecting two adjacent base-paired segments so that it “bubbles” out in the middle of a double helix on one side. The multi-branch loop is also called helical junctions.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The __________ refers to two single-stranded regions on opposite strands connecting two adjacent base-paired segments.`,
    options: ["hairpin loop", "interior loop", "pseudoknot loop", "helical junctions"],
    correct: "b",
    explanation: `In addition to the traditional secondary structural elements, base pairing between loops of different secondary structural elements can result in a higher level of structures such as pseudoknots, kissing hairpins, and hairpin–bulge contact. A pseudoknot loop refers to base pairing formed between loop residues within a hairpin loop and residues outside the hairpin loop.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `At present, there are essentially two types of method of RNA structure prediction. One is the minimum free energy approach and the Second one is a comparative approach.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `One is based on the calculation of the minimum free energy of the stable structure derived from a single RNA sequence. This can be considered an ab initio approach. The second is a comparative approach which infers structures based on an evolutionary comparison of multiple related RNA sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Ab initio approach makes structural predictions based on ______`,
    options: ["a single RNA sequence", "comparing RNA sequences", "evolutionary basis", "pure phylogenetics"],
    correct: "a",
    explanation: `The rationale behind this method is that the structure of an RNA molecule is solely determined by its sequence. Thus, algorithms can be designed to search for a stable RNA structure with the lowest free energy.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In ab initio approach, generally, when a base pairing is formed, the energy of the molecule is _________ because of attractive interactions between the two strands.`,
    options: ["lowered", "increased", "multiplied", "kept stable"],
    correct: "a",
    explanation: `Here, the algorithms can be designed to search for a stable RNA structure with the lowest free energy. Thus, to search for a most stable structure, ab initio programs are designed to search for a structure with the maximum number of base pairs.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In ab initio methods, free energy can be calculated based on parameters empirically derived for small molecules.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `G–C base pairs are more stable than A–U base pairs, which are more stable than G–U base pairs. It is also known that base-pair formation is not an independent event.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The energy necessary to form individual base pairs is not quite affected by adjacent base pairs.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The energy necessary to form individual base pairs is influenced by adjacent base pairs through helical stacking forces. This is known as co-operativity in helix formation.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The attractive interactions lead to _________ energy.`,
    options: ["increased", "higher", "lower", "no change in"],
    correct: "c",
    explanation: `If a base pair is next to other base pairs, the base pairs tend to stabilize each other through attractive stacking interactions between aromatic rings of the base pairs. The attractive interactions lead to even lower energy. Parameters for calculating the co-operativity of the base-pair formation have been determined and can be used for structure prediction.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `If the base pair is adjacent to loops or bulges, the neighboring loops and bulges tend to ___________ the base-pair formation.`,
    options: ["have no change on", "decrease the energy", "stabilize", "destabilize"],
    correct: "d",
    explanation: `This is because there is a loss of entropy when the ends of the helical structure are constrained by unpaired loop residues. The destabilizing force to a helical structure also depends on the types of loops nearby.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The scoring scheme based on the combined stabilizing and destabilizing interactions forms the foundation of the ab initio RNA secondary structure prediction method.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Parameters for calculating different destabilizing energies have also been determined and can be used as penalties for secondary structure calculations. This method works by first finding all possible base-pairing patterns from a sequence and then calculating the total energy of a potential secondary structure by taking into account all the adjacent stabilizing and destabilizing forces.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Ab initio methods are energetically least favorable.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `These methods are energetically most favorable. If there are multiple alternative secondary structures, the method finds the conformation with the lowest energy.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The dot matrix method and the dynamic programming method can be used in detecting self-complementary regions of a sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A simple dot matrix can find all possible base-paring patterns of an RNA sequence when one sequence is compared with itself. In this case, dots are placed in the matrix to represent matching complementary bases instead of identical ones.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In dot matrix in ab initio methods, the diagonals _____________ to the main diagonal represent regions that can self hybridize.`,
    options: ["parallel", "cutting in random fashion", "perpendicular", "adjacent parallel"],
    correct: "c",
    explanation: `The diagonals perpendicular to the main diagonal represent regions that can self hybridize to form double-stranded structure with traditional A–U and G–C base pairs. In reality, the pattern detection in a dot matrix is often obscured by high noise levels.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In dot matrix in ab initio methods, one way to reduce the noise in the matrix is to select an appropriate window size of a maximum number of contiguous base matches.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `To reduce the noise in the matrix is to select an appropriate window size of a minimum number of contiguous base matches. Normally, only a window size of four consecutive base matches is used. If the dot plot reveals more than one feasible structure, the lowest energy one is chosen.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In dynamic programming, in ab initio methods, the use of a dot plot can be effective in finding a ____ secondary structure in a ________ molecule.`,
    options: ["multiple, large", "single, large", "single, small", "multiple, small"],
    correct: "c",
    explanation: `Mostly, The use of a dot plot can be effective in finding a single secondary structure in a small Molecule. However, if a large molecule contains multiple secondary structure segments, choosing a combination that is energetically most stable among a large number of possibilities can be a daunting task.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In ab initio methods, a quantitative approach such as dynamic programming can be used to assemble a final structure with optimal base-paired regions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In this approach, an RNA sequence is compared with itself. A scoring scheme is applied to fill the matrix with match scores based on Watson–Crick base complementarity.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In dynamic programming, in ab initio methods, Often ________ base pairing and energy terms of the base pairing _____ incorporated into the scoring process.`,
    options: ["G-C, are", "G\u2013U, are not", "G\u2013U, are also", "G-C, are not"],
    correct: "c",
    explanation: `Although the traditional structure comprises of A–U and G–C base pairs, G-U base pairing is incorporated into the scoring process. A path with the maximal score within a scoring matrix after taking into account the entire sequence information represents the most probable secondary structure form.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The dynamic programming method produces ____ structure with _____ score.`,
    options: ["one, single best", "multiple, single best", "multiple, multiple", "single, multiple"],
    correct: "a",
    explanation: `However, this is potentially a drawback of this approach. In reality an RNA may exist in multiple alternative forms with near minimum energy but not necessarily the one with maximum base pairs.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The problem of dynamic programming to select one single structure can be complemented by adding a probability distribution function, known as the _________ which calculates a mathematical distribution of probable base pairs in thermodynamic equilibrium.`,
    options: ["partition function", "division function", "increment function", "fold function"],
    correct: "a",
    explanation: `This function helps to select a number of suboptimal structures within a certain energy range. The MFOLD and RNAFold are two well-known programs using the ab initio prediction method.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is correct about MFOLD?`,
    options: ["It uses the dynamic programming only", "It uses the thermodynamic calculations only", "It uses the both dynamic programming and thermodynamic calculations as well", "It doesn\u2019t take into account the themoststablility of the secondary structures"],
    correct: "b",
    explanation: `It is a web-based program for RNA secondary structure prediction. It combines dynamic programming and thermodynamic calculations for identifying themostable secondary structures with the lowest energy. It also produces dot plots coupled with energy terms. This method is reliable for short sequences but becomes less accurate as the sequence length increases.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Like Mfold, RNAfold only examines the energy terms of the optimal alignment in a dot plot.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `It is one of the web programs in the Vienna package. Unlike Mfold, which only examines the energy terms of the optimal alignment in a dot plot, RNAfold extends the sequence alignment to the vicinity of the optimal diagonals to calculate thermodynamic stability of alternative structures.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following about the RNAFold is incorrect?`,
    options: ["It extends the sequence alignment to the vicinity of the optimal diagonals to calculate thermodynamic stability of alternative structures", "It incorporates a partition function", "It doesn\u2019t necessarily use a partition function", "It aims to select a number of statistically most probable structures in one of its steps"],
    correct: "c",
    explanation: `Based on both thermodynamic calculations and the partition function, a number of alternative structures that may be suboptimal are provided. The collection of the predicted structures may provide a better estimate of plausible foldings of an RNA molecule than the predictions by Mfold.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In ab initio methods for RNA structure prediction, the prediction results from RNAfold are not always guaranteed to be better than those predicted by Mfold.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because of the much larger number of secondary structures to be computed, a more simplified energy rule has to be used to increase computational speed. Thus, the prediction results are not always guaranteed to be better than those predicted by Mfold.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The comparative approach uses basic RNA structure based predictions to infer a consensus structure.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The comparative approach uses multiple evolutionarily related RNA sequences to infer a consensus structure. This approach is based on the assumption that RNA sequences that deem to be homologous fold into the same secondary structure.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `To distinguish the conserved secondary structure among multiple related RNA sequences, a concept of “covariation” is used.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is known that RNA functional motifs are structurally conserved. To maintain the secondary structures while the homologous sequences evolve, a mutation occurring in one position that is responsible for base pairing should be compensated for by a mutation in the corresponding base-pairing position so to maintain base pairing and the stability of the secondary structure.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `___________ of covariation can be ___________ to the RNA structure and functions.`,
    options: ["Any lack, deleterious", "Any lack, benign", "Any abundance, deleterious", "Any inadequacy, advantageous"],
    correct: "a",
    explanation: `Based on this rule, algorithms can be written to search for the covariation patterns after a set of homologous RNA sequences are properly aligned. The detected correlated substitutions help to determine conserved base pairing in a secondary structure.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `An aspect of the comparative method is to select a __________ structure through consensus drawing.`,
    options: ["relatively distinct", "remote", "common", "least abundant"],
    correct: "c",
    explanation: `Predicting secondary structures for each individual sequence may produce errors, by comparing all predicted structures of a group of aligned RNA sequences and drawing a consensus. Hence, the commonly adopted structure can be selected; many other possible structures can be eliminated in the process.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The comparative-based algorithms can be further divided into two categories based on the type of input data.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The comparative-based algorithms can be further divided as mentioned. One requires predefined alignment and the other does not.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The type of algorithm that requires predefined alignment, requires the user to provide _______ alignment as input.`,
    options: ["not necessarily an alignment", "multiple only", "pairwise or multiple", "pair wise only"],
    correct: "c",
    explanation: `As the name suggests, it does require predefined alignment, the option ‘not necessarily an alignment’ becomes irrelevant. The sequence alignment can be obtained using standard alignment programs such as T-Coffee, PRRN, or Clustal. Based on the alignment input, the prediction programs compute structurally consistent mutational patterns such as covariation and derive a consensus structure common for all the sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The type of algorithm that _____ predefined alignment is ______ for reasonably conserved sequences.`,
    options: ["doesn\u2019t require, more successful", "requires, less successful", "doesn\u2019t require, relatively successful", "requires, relatively successful"],
    correct: "d",
    explanation: `The requirement for using this type of program is an appropriate set of homologous sequences that have to be similar enough to allow accurate alignment, but divergent enough to allow covariations to be detected. If this condition is not met, correct structures cannot be inferred.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The predefined alignment requiring method also depends on the quality of the input alignment. If there ___ errors in the alignment, covariation signals ____ detected.`,
    options: ["are, will be", "are, will not be", "are not, will not be", "are not, possibly will not be"],
    correct: "b",
    explanation: `The selection of one single consensus structure is also a drawback because alternative and evolutionarily unconserved structures are not predicted. The RNAalifold is an example of this type of program based on predefined aligned sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is true about the RNAalifold?`,
    options: ["Dynamic programming is not involved", "Minimum free energy method is not used", "Only minimum free energy is used", "Covariation information is taken into consideration"],
    correct: "d",
    explanation: `It is a program in the Vienna package. It uses a multiple sequence alignment as input to analyze covariation patterns on the sequences. A scoring matrix is created that combines minimum free energy and covariation information. Dynamic programming is used to select the structure that has the minimum energy for the whole set of aligned RNA sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Rigorously evaluating the performance of RNA prediction programs has traditionally been hindered by the dearth of three-dimensional structural information for RNA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The availability of recently solved crystal structures of the entire ribosome provides a wealth of structural details relating to diverse types of RNA molecules. The high resolution structural information can then be used as a benchmark for evaluating state-of-the-art RNA structure prediction programs in all categories.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `If prediction accuracy can be represented using a ______ the ______ programs score roughly 20% to 60% depending on the length of the sequences.`,
    options: ["multiple parameter, ab initio\u2013based", "single parameter, ab initio\u2013based", "multiple parameter, comparative\u2013based", "single parameter, comparative\u2013based"],
    correct: "b",
    explanation: `As mentioned, the scores depend on the length of the sequences. Generally speaking, the programs perform better for shorter RNA sequences than for longer ones.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `For _________ RNA sequences, such as tRNA, some programs may be able to produce _________% accuracy.`,
    options: ["small, 70", "small, 40", "large, 90", "large, 75"],
    correct: "a",
    explanation: `The number of the percentage may vary but the qualitative idea is that for small RNA sequences, some programs may produce better accuracy. The major limitation for performance gains of this category appears to be dependence on energy parameters alone, which may not be sufficient to distinguish different structural possibilities of the same molecule.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The pre-alignment independent programs fare __________ for predicting long sequences.`,
    options: ["slight better", "much better", "a bit worse", "much worse"],
    correct: "d",
    explanation: `For small RNA sequences such as tRNA, both subtypes can achieve very high accuracy (up to 100%). This illustrates that the comparative approach is consistently more accurate than the ab initio one.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Based on recent benchmark comparisons, the comparative-type algorithms can reach an accuracy range of 20% to 80%.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The results depend on whether a program is pre-alignment dependent or not. Most of the superior performance comes from pre-alignment-dependent programs such as RNAalifold.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In comparative approach to RNA structure prediction, algorithms that do not use pre-alignment, align multiple input sequences and infers a consensus structure.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The alignment is produced using dynamic programming with a scoring scheme that incorporates sequence similarity as well as energy terms. Because the full dynamic programming for multiple alignment is computationally too demanding, currently available programs limit the input to two sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In comparative approach to RNA structure prediction, Foldalign is a web-based only program for RNA alignment`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Foldalign is a web-based program for RNA alignment and structure prediction. The user provides a pair of unaligned sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In comparative approach to RNA structure prediction, the Foldalign program doesn’t use the covariation information.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The program uses a combination of clustal and dynamic programming with a scoring scheme that includes covariation information to construct the alignment. A commonly conserved structure for both sequences is subsequently derived based on the alignment. To reduce computational complexity, the program ignores multi-branch loops and is only suitable for handling short RNA sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In comparative approach to RNA structure prediction, Dynalign is a ________ program.`,
    options: ["Windows based", "Fedora", "UNIX", "iOS based"],
    correct: "c",
    explanation: `Is a UNIX program with a free source code for downloading. Here, the user again provides two input sequences. The program calculates the possible secondary structures of each using a method similar to Mfold.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In comparative approach to RNA structure prediction, in Dynalign program–by comparing _________ from each sequence, a ______ structure common to both sequences is selected that serves as the basis for sequence alignment.`,
    options: ["multiple alternative structures, lowest energy", "single structure, lowest energy", "single structure, highest energy", "multiple alternative structures, highest energy"],
    correct: "a",
    explanation: `The unique feature of this program is that it does not require sequence similarity and therefore can handle very divergent sequences. However, because of the computation complexity, the program only predicts small RNA sequences such as tRNA with reasonable accuracy.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is incorrect about the RNA structure prediction?`,
    options: ["Given the sequence, it provides an ab initio prediction of secondary structure", "From the many possible choices of complementary sequences that can potentially base-pair, the compatible sets that provide the highest energy molecules are chosen", "Structures with energies almost as stable as the most stable one may also be produced", "Regions whose predictions are the most reliable can be identified from such an analysis"],
    correct: "b",
    explanation: `Stable structures are the ones with least or relatively quite low energy. Sequence variations found in related sequences may also be used to predict which base pairs are likely to be found in each of the molecules. One variation of RNA structure prediction methods will predict a set of sequences that are able to form a particular structure.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `A type of RNA secondary structure prediction method takes into account conserved patterns of base-pairing that are conserved during evolution of a given class of RNA molecules.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Sequence positions that base-pair are found to vary at the same time during evolution of RNA molecules so that structural integrity is maintained. For example, if two positions G and C form a base pair in a given type of molecule, then sequences that have C and G reversed, or A and U or U and A at the corresponding positions, would be considered reasonable matches.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `RNA secondary structure is composed primarily of triple-stranded RNA regions formed by folding the single-stranded molecule back twice on itself.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `RNA secondary structure is composed primarily of double-stranded RNA regions formed by folding the single-stranded molecule back on itself. To produce such double-stranded regions, a run of bases downstream in the RNA sequence must be complementary to another upstream run so that Watson–Crick base-pairing between the complementary nucleotides G/C and A/U (analogous to the G/C and A/T base pairs in DNA) can occur.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `____ wobble pairs may be produced in these double-stranded regions.`,
    options: ["A/A", "A/U", "G/C", "G/U"],
    correct: "d",
    explanation: `As in DNA, the G/C base pairs contribute the greatest energetic stability to the molecule, with A/U base pairs contributing less stability than G/C, and G/U wobble base pairs contributing the least. From the RNA structures that have been solved, these base pairs and a number of additional ones have been identified.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In predicting RNA secondary structure, some simplifying assumptions are usually made, like the _____ structure is similar to the _____`,
    options: ["most likely, energetically most unstable structure", "most unlikely, energetically most stable structure", "most likely, energetically most stable structure", "least likely, energetically most stable structure"],
    correct: "c",
    explanation: `The assumption here is that the most predicted or likely structure has to be similar to the energetically most stable structure. The abundance is not taken into consideration in this.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The Second assumption in predicting RNA secondary structure is that the energy associated with any position in the structure is _____ influenced by local sequence and structure.`,
    options: ["only", "not at all", "partially", "never"],
    correct: "a",
    explanation: `The energy associated with a particular base pair in a double-stranded region is assumed to be influenced only by the previous base pair and not by the base pairs farther down the double-stranded region or anywhere else in the structure. These energies can be reliably estimated by experimentation with small, synthetic RNA oligonucleotides recently improved to include sequence dependence.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The third assumption in predicting RNA secondary structure is that the structure is assumed to be formed by _____ of the chain back on itself in a manner that ________`,
    options: ["crossing, produces knots", "crossing, does not produce any knots", "folding, produces knots", "folding, does not produce any knots"],
    correct: "d",
    explanation: `The best way of representing this requirement is to draw the sequence in a circular form. The paired bases are then joined by arcs. If the total structure with all predicted base pairs is to be free of knots, none of the arcs must cross.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In 1971, first estimation of the energy associated with regions of secondary structure by extrapolation from studies with small molecules was done and then an attempt was made to predict which configurations of larger molecules were the most energetically stable.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Energy estimates included the stabilizing energy associated with stacking base pairs in a double-stranded region and the destabilizing influence of regions that were not paired. Pipas and McMahon (1975) developed computer programs that listed all possible helical regions in tRNA sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Nussinov and Jacobson (1980) were the first to design a precise and efficient algorithm for predicting secondary structure.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The algorithm generates two scoring matrices—one M (i,j) to keep track of the maximum number of base pairs that can be formed in any interval i to j in the sequence. The second K (i,j) is to keep track of the base position k that is paired with j.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is incorrect about the prediction of RNA secondary structure?`,
    options: ["Every base is first compared to every other base by a type of analysis very similar to the dot matrix analysis", "A row of matches in the RNA matrix indicates a succession of complementary nucleotides that can potentially form a double-stranded region", "A row of matches in the RNA matrix indicates a failure of complementary nucleotides that can potentially form a double-stranded region", "The sequence is listed across the top and down the side of the page, and G/C, A/U, and G/U base pairs are scored"],
    correct: "c",
    explanation: `The energy of each predicted structure is estimated by the nearest- neighbor rule by summing the negative base-stacking energies for each pair of bases in double-stranded regions. By adding the estimated positive energies of destabilizing regions such as loops at the end of hairpins, bulges within hairpins, internal bulges, and other unpaired regions.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Through a single scoring matrix, evaluation of all the different possible configurations is done.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `To evaluate all the different possible configurations and to find the most energetically favorable, several types of scoring matrices are used. The complementary regions are evaluated by a dynamic programming algorithm to predict the most energetically stable molecule. The method is similar to the dynamic programming method used for sequence alignment.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The object is to find a diagonal row of matches that goes from upper left to lower right.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The object is to find a diagonal row of matches that goes from upper right to lower left. In general, each matrix value is obtained by considering the minimum energy values, obtained by all previous complementary pairs, decreased by the stacking energy of any additional complementary base pairs or increased by the destabilizing energy associated with non-complementary bases.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The increase depends on the type and length of loop that is introduced by the non-complementary base pair, whether internal loop, bulge loop, or hairpin loop.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This comparison of all possible matches and energy values is continued until all nucleotides have been compared. There is a pattern followed in comparing bases within the RNA molecule.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The sequence is listed down the first column of base comparisons’ table and free energy calculations’ table in the 5’→3’ orientation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The first four bases of the sequence are also listed in the first row of the tables in the 5’→3’ direction. Several complementary base pairs between the first and last four bases that could lead to secondary structure are shown in the tables.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `A general theory for modeling strings of symbols, such as bases in DNA sequences, has been developed by linguists. There is a hierarchy of these so-called transformational grammars that deal with situations of increasing complexity.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The application of these grammars to sequence analysis has been extensively discussed elsewhere. The context-free grammar is suitable for finding groups of symbols in different parts of the input sequence that thus are not in the same context.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `______ regions in sequences, such as those in RNA that will form secondary structures, are an example of such context-free sequences.`,
    options: ["non-interlocking", "non-Complementary", "complementary", "non-compatible"],
    correct: "c",
    explanation: `Stochastic context-free grammars (SCFG) introduce uncertainty into the definition of such regions. It allows them to use alternative symbols as found in the evolution of RNA molecules.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The use of SCFGs in RNA secondary structure production analysis is in fact very similar to that of the covariance model, with the grammatical productions resembling the nodes in the ordered binary tree.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `As with hidden Markov models, the probability distribution of each production must be derived by training with known sequences. The algorithms used for training the SCFG and for aligning a sequence with the SCFG are somewhat different from those used with hidden Markov models, and the time and memory requirements are greater.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In a SCFG, each production of a non-terminal symbol has an associated probability for giving rise to the resulting product, and there are a set of productions, each giving a different result.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, the production S1 → C S2 G could also be represented by 15 other base-pair combinations, and each of these has a corresponding probability. Thus, each production can be considered to be represented by a probability distribution over the possible outcomes.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The application of SCFGs to RNA secondary structure analysis is very similar in form to the probabilistic covariance models.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For RNA, the symbols of the alphabet are A, C, G, and U. The context-free grammar establishes a set of rules called productions for generating the sequence from the alphabet, in this case an RNA molecule with sections that can base-pair and others that cannot base-pair.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Originally, the FOLD program of _____ predicted _____ having the minimum free energy.`,
    options: ["M. Zuker, two structures", "M. Wunsch, two structures", "M. Zuker, only one", "M. Wunsch, only one"],
    correct: "c",
    explanation: `However, changes in a single nucleotide can result in drastic changes in the predicted structure. A later version, called MFOLD, has improved prediction of non-base paired interactions and predicts several structures having energies close to the minimum free energy.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `To find suboptimal structures, the dynamic programming method was modified.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It was so done to evaluate parts of a new scoring matrix in which the sequence is represented in two tandem copies on both the vertical and horizontal axes. The regions from i 1 to n and j 1 to n are used to calculate an energy V (i,j) for the best structure that includes an i, j base pair and is called the included region.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In the programs related to FOLD, the second region, the ________ region, is used to calculate the energy of the ________ structure.`,
    options: ["included, most likely", "included, best", "excluded, least likely", "excluded, best"],
    correct: "d",
    explanation: `A second region, the excluded region, is used to calculate the energy of the best structure that includes i, j but is not derived from the structure at i+1, j-1 After certain corrections are made, the difference between the included and excluded values is the most energetic structure that includes the base pair i, j.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `In the programs related to FOLD, All complementary base pairs can be sampled in a fashion to determine which are present in a suboptimal structure.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `All complementary base pairs can be sampled in a fashion to determine which are within a certain range of the optimal one. An energy dot plot is produced showing the locations of alternative base pairs that produce the most stable or suboptimally stable structures.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The program may be instructed to find structures within a certain percentage of the minimum free energy.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Parameter d provides a measure of similarity between two structures. When MFOLD is established on a suitable local host machine, the window is interactive, and clicking a part of the display will lead to program output of the corresponding structure.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Three scores, Pnum (i), Hnum (i,j), and Ssum, have been derived to assist with a determination of the reliability of a secondary structure prediction for a particular base i or a base pair i,j. Which of the following is not a correct regarding Pnum?`,
    options: ["is the total number of energy dots regardless of color in the i th row and i th column of the energy dot plot", "is the total number of energy dots considering the color in the ith row and ith column of the energy dot plot", "represents in an unfiltered dot plot", "represents the number of base pairs"],
    correct: "b",
    explanation: `It represents the number of base pairs that the ith base can form with all other base pairs in structures within the defined energy range. The lower this value, the better defined or “well determined” the local structure because there are few competitive foldings.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Three scores, Pnum (i), Hnum (i,j), and Ssum, have been derived to assist with a determination of the reliability of a secondary structure prediction for a particular base i or a base pair i,j. Which of the following is not a correct regarding Hnum?`,
    options: ["is the sum of Pnum(i) and Pnum(j) -1", "is the sum of Pnum(i) and Pnum(j) + 1", "is the total number of dots in the ith row and jth column", "represents the total number of base pairs with the ith or jth base in the predicted structures"],
    correct: "b",
    explanation: `The Hnum for a double stranded region is the average Hnum value for the base pairs in that helix. The lower this number, the more well determined the double-stranded region.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Three scores, Pnum (i), Hnum (i,j), and Ssum, have been derived to assist with a determination of the reliability of a secondary structure prediction for a particular base i or a base pair i,j. Which of the following is not a correct regarding Ssum?`,
    options: ["is also called as ss-count", "is the number of foldings in which base i is single-stranded divided by m, the number of folding", "is the number of foldings in which base i is single-stranded multiplied by m, the number of folding", "gives the probability that base i is single-stranded"],
    correct: "c",
    explanation: `If Snum is approximately 1, then base i is probably in a single-stranded region, and if Snum is approximately 0, then base i is probably not in such a region. This reliability information has been used to annotate output files of MFOLD and other RNA display programs.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `A limitation of the Zuker method and other methods (Nakaya et al. 1995) for computing suboptimal RNA structures is that they do not compute all the structures within a given energy range of the minimum free-energy structure.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, no alternative structures are produced that have the absence of base pairs in the best structure, and, if two substructures are joined by a stretch of unpaired bases, no structures are produced that are suboptimal for both structures. These factors limit the number of alternative structures predicted compared to known variations based on sequence variations in tRNAs.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `These limitations of the Zuker method and other methods have been largely overcome by using an algorithm originally described by Waterman and Byers (1985) for finding sequence alignments within a certain range of the optimal one by modifications of the trace-back procedure used in dynamic programming.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This method efficiently calculates a large number of alternative structures, up to a very large number, within a given energy range of the minimum free-energy structure. The method has been used to demonstrate that natural tRNA sequences can form many alternative structures that are close to the minimum free-energy structure and that base modification plays a major role in this energetic stability.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `______ molecules can simply be identified based on their sequence similarity with already-known sequences.`,
    options: ["Larger, less conserved", "Larger, highly conserved", "smaller, highly conserved", "shorter, highly conserved"],
    correct: "b",
    explanation: `For smaller sequences with more sequence variation, this method does not work. A number of methods for finding small RNA genes have been described and are available on the Web. A major problem with these methods in searches of large genomes is that a small false positive rate becomes quite unacceptable because there are so many false positives to check out.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `One of the first methods used to find tRNA genes was to search for sequences that are complementary and can fold into a knot like the three found in tRNAs.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `One of the first methods used to find tRNA genes was to search for sequences that are self-complementary and can fold into a hairpin like the three found in tRNAs (Staden 1980). Through the regions of self-complimentarity it was first possible to find the tRNA.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Fichant and Burks (1991) described a program, tRNAscan, that searches a genomic sequence with a sliding window searching simultaneously for matches to a set of invariant bases and conserved self-complementary regions in tRNAs with an accuracy of 97.5%.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A method for finding the RNA polymerase III transcriptional control regions of tRNA genes using a scoring matrix derived from known control regions, was derived. That is also very accurate. Finally, Lowe and Eddy (1997) have devised a search algorithm tRNAscan-SE that uses a combination of three methods to find tRNA genes in genomic sequences—tRNAscan, the Pavesi algorithm, and the COVELS program based on sequence covariance analysis (Eddy and Durbin 1994). This method is reportedly 99–100% accurate with an extremely low rate of false positives.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The probabilistic model was used to identify small nucleolar (sno) RNAs in the yeast genome that methylate ribosomal RNA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The model is not used to search genomic sequences directly. Instead, a list of candidate sequences is first found by searching for patterns that match the sequences in the model (Lowe and Eddy 1999).`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The probability model mentioned above was a hybrid combination of HMMs and SCFGs trained on sno RNAs.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These RNAs vary sufficiently in sequence and structure that they are not found by straightforward similarity searches. The RNAs found were shown to be sno RNAs by insertional mutagenesis.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is untrue regarding RNA structure?`,
    options: ["RNA structure 4.6 is a Windows implementation of the Zuker algorithm", "It includes additional options for other folding algorithms and incorporation of experimental data", "The authors of RNA structure collaborate very closely with the Turner laboratory and keep the most up-to-date thermodynamic parameters", "The OligoWalk program cannot be used for siRNA design"],
    correct: "d",
    explanation: `Two unique ways of incorporating experimental data in the RNA folding is done with Dynalign and chemical modification. The Dynalign program computes the lowest free-energy sequence alignment and secondary structure common to two RNA sequences.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about Vienna RNA Websuite?`,
    options: ["It introduced the Wuchty algorithm, developed applications of the McCaskill algorithm", "It also offers a wide variety of algorithms and functions", "The Wuchty algorithm generates a small but complete set of suboptimal structures", "The Wuchty algorithm computes some possible tertiary structures within a narrow free-energy range"],
    correct: "d",
    explanation: `The Wuchty algorithm computes all possible secondary structures within a narrow free-energy range. The Wuchty algorithm generates a small but complete set of suboptimal structures that may include some very different secondary structures but also very many highly similar structures. However, structures containing more than one suboptimal region may occur in the Wuchty set of structures but would be absent if the Zuker method for sampling suboptimal structures were used.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about the Sfold algorithm?`,
    options: ["It uses a unique algorithm to aid in the design of siRNA", "The algorithm combines thermodynamic stabilities, calculations of target accessibility, and empirical rules", "The website offers specialized programs for the design of siRNA, antisense RNA, trans-cleaving RNA, and mRNA-microRNA interactions", "The website doesn\u2019t offer programs for the design of a general program for statistically sampling suboptimal RNA structures"],
    correct: "d",
    explanation: `The algorithm uses a partition function calculation and then groups suboptimal structures by similarity. The centroid structure is the most-representative structure that is closest in similarity to all the other structures.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `If the centroid structure is different from the minimum free-energy structure, the centroid structure is often closer to the phylogenetic prediction and contains fewer base pairs, or fewer false-positive base pair predictions, than the minimum free-energy prediction.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The point is to show a structure that represents a group of structures rather than a single predicted structure. Many long RNA sequences, such as viral genomes or mRNA, may not have a single structure but instead have a dynamic structure that has some conserved features but also varies and changes, and these many conformations may all exist simultaneously in the cell.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `The ILM program uses an iterative loop matching algorithm to maximize base pairs and allows pseudoknots to form by allowing base. pairs to be added or removed in successive rounds.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The Nussinov algorithm, or maximum loop matching algorithm, is the basic framework for generating a structure with the most possible base pairs. The base pairs are ranked using both thermodynamic parameters and covariation data for aligned sequences. ILM requires the RnaViz program to visualize the RNA secondary structure with pseudoknots.`,
    super_topic: `RNA Structure Prediction`,
  },
  {
    question: `Which of the following is untrue about the genome mapping?`,
    options: ["It doesn\u2019t lead to the understanding of a genome structure", "It involves identifying relative locations of genes", "It involves identifying traits", "It involves identifying mutations"],
    correct: "a",
    explanation: `The first step to understanding a genome structure is through genome mapping, which is a process of identifying relative locations of genes, mutations or traits on a chromosome. A low-resolution approach to mapping genomes is to describe the order and relative distances of genetic markers on a chromosome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Genetic markers are ______ portions of a _______ whose inheritance patterns can be followed.`,
    options: ["unidentifiable, genes", "unidentifiable, chromosome", "identifiable, chromosome", "identifiable, genes"],
    correct: "c",
    explanation: `For many eukaryotes, genetic markers represent morphologic phenotypes. In addition to genetic linkage maps, there are also other types of genome maps such as physical maps and cytologic maps, which describe genomes at different levels of resolution.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Genetic linkage maps, also called genetic maps, identify the relative positions of genetic markers on a chromosome and are based on how frequent the markers are inherited together.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The rationale behind genetic mapping is that the closer the two genetic markers are, the more likely it is that they are inherited together and are not separated in a genetic crossing event. The distance between the two genetic markers is measured in centiMorgans (cM), which is the frequency of recombination of genetic markers.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `One centiMorgan is defined as _________ percentage of the total recombination events.`,
    options: ["one", "ten", "0.1", "0.01"],
    correct: "a",
    explanation: `One centiMorgan is one percentage of the total recombination events when separation of the two genetic markers is observed in a genetic crossing experiment. One centiMorgan is approximately 1 Mb in humans and 0.5 Mb in Drosophila.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Physical maps are maps of locations of identifiable landmarks on a genomic DNA _______ inheritance patterns.`,
    options: ["remotely related to", "related to", "regardless of", "associated with"],
    correct: "c",
    explanation: `The distance between genetic markers is measured directly as kilobases (Kb) or megabases (Mb). Because the distance is expressed in physical units, it is more accurate and reliable than centiMorgans used in genetic maps.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Physical maps are constructed by using a chromosome walking technique.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It uses a number of radio labeled probes to hybridize to a library of DNA clone fragments. By identifying overlapping clones probed by common probes, a relative order of the cloned fragments can be established.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about cytologic maps?`,
    options: ["They cannot be directly observed under microscope", "They refer to banding patterns", "They can be viewed on stained chromosomes", "They can be directly observed under microscope"],
    correct: "a",
    explanation: `Cytologic maps refer to banding patterns seen on stained chromosomes, which can be directly observed under a microscope. The observable light and dark bands are the visually distinct markers on a chromosome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Cytologic maps can be considered to be of _____ resolution and hence somewhat ______ physical maps.`,
    options: ["very high, inaccurate", "very low, accurate", "very high, accurate", "very low, inaccurate"],
    correct: "d",
    explanation: `The banding patterns, however, are not always constant and are subject to change depending on the extent of chromosomal contraction. Thus, cytologic maps can be considered to be of very low resolution and hence somewhat inaccurate physical maps. The distance between two bands is expressed in relative units (Dustin units).`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `In medical applications, the ultimate goal of gene mapping is to disease genes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Once the gene is cloned, the determination of DNA sequence is possible. Further, the study of target protein is carried out.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `One of the fundamental events that occur in meiosis is crossing over in which homologous chromosomes exchange segments causing a reshuffling of genes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If genes are far apart on the same chromosome, it is likely that recombination occurs. Conversely, if they are very close together, they are more likely to be transmitted as a block.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The _____ resolution genome map is the genomic DNA sequence that can be considered as a type of ______ map describing a genome at the single base-pair level.`,
    options: ["highest, physical", "lowest, physical", "highest, cytological", "lowest, cytological"],
    correct: "a",
    explanation: `Cytological maps have quite low resolution, when compared to physical maps. They can be viewed under microscopes as well.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about DNA sequencing?`,
    options: ["It is now routinely carried out using the Sanger method", "This doesn\u2019t make use of DNA polymerases", "This involves the synthesis of DNA chains of varying length", "The DNA synthesis is stopped by adding dideoxynucleotides"],
    correct: "b",
    explanation: `DNA polymerases are used to synthesize DNA chains. The dideoxynucleotides are labeled with fluorescent dyes, which terminate the DNA synthesis at positions containing all four bases, resulting in nested fragments that vary in length by a single base. When the labeled DNA is subjected to electrophoresis, the banding patterns in the gel reveal the DNA sequence.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `In DNA sequencing, the fluorescent traces of the DNA sequences are read by a computer program that assigns bases for each peak in a chromatogram.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This process is called base calling. Automated base calling may generate errors and human intervention is often required to correct the sequence calls.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The shotgun approach _______ sequences clones from _____ of cloned DNA.`,
    options: ["randomly, one end", "randomly, both ends", "specifically, both ends", "specifically, one end"],
    correct: "b",
    explanation: `There are two major strategies for whole genome sequencing: the shotgun approach and the hierarchical approach. The shotgun approach generates a large number of sequenced DNA fragments. The number of random fragments has to be very large, so large that the DNA fragments overlap sufficiently to cover the entire genome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The shotgun approach does not require knowledge of physical mapping of the clone fragments, but rather a robust computer assembly program to join the pieces of random fragments into a single, whole-genome sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Generally, the genome has to be redundantly sequenced in such a way that the overall length of the fragments covers the entire genome multiple times. This is designed to minimize sequencing errors and ensure correct assembly of a contiguous sequence. Overlapping sequences with an overall length of six to ten times the genome size are normally obtained for this purpose.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Despite the multiple coverage, sometimes certain genomic regions remain unsequenced, mainly owing to cloning difficulties.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In such mentioned cases, the remainder gap sequences can be obtained through extending sequences from regions of known genomic sequences using a more traditional PCR technique. That which requires the use of custom primers and performs genome walking in a stepwise fashion. This step of genome sequencing is also known as finishing, which is followed by computational assembly of all the sequence data into a final complete genome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The hierarchical genome sequencing approach is ______`,
    options: ["entirely dissimilar to the shotgun approach", "dissimilar to the shotgun approach", "similar to the shotgun approach, but on a larger scale", "similar to the shotgun approach, but on a smaller scale"],
    correct: "d",
    explanation: `In this, the chromosomes are initially mapped using the physical mapping strategy. Longer fragments of genomic DNA (100 to 300 kB) are obtained and cloned into a high-capacity bacterial vector called bacterial artificial chromosome (BAC).`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `In hierarchical genome sequencing approach, based on the results of _______ mapping _______of the BAC clones on a chromosome can be determined.`,
    options: ["physical, the locations and orders", "physical, only the locations", "cytological, only the locations", "physical, only the orders"],
    correct: "a",
    explanation: `By successively sequencing adjacent BACclone fragments, the entire genome can be covered. The complete sequence of each individual BAC clone can be obtained using the shotgun approach. Overlapping BAC clones are subsequently assembled into an entire genome sequence.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The hierarchical approach is ____ and _____ than the shotgun approach because it involves an initial clone-based physical mapping step.`,
    options: ["slower, less costly", "faster, more costly", "faster, less costly", "slower, more costly"],
    correct: "d",
    explanation: `During the era of human genome sequencing, there was a heated debate on the merits of each of the two strategies. Despite the mentioned fact, once the map is generated, assembly of the whole genome becomes relatively easy and less error prone.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The whole genome shotgun approach can produce a draft sequence very rapidly because it is based on the direct sequencing approach.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `However, it is computationally very demanding to assemble the short random fragments. Although the approach has been successfully employed in sequencing small microbial genomes, for a complex eukaryotic genome that contains high levels of repetitive sequences, such as the human genome, the full shotgun approach becomes less accurate and tends to leave more “holes” in the final assembled sequence than the hierarchical approach. Current genome sequencing of large organisms often uses a combination of both approaches.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The major challenges in genome assembly are sequence errors, contamination by bacterial vectors, and repetitive sequence regions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Sequence errors can often be corrected by drawing a consensus from an alignment of multiple overlapped sequences. Bacterial vector sequences can be removed using filtering programs prior to assembly. To overcome the problem of sequence repeats, programs such as RepeatMasker can be used to detect and mask repeats. Additional constraints on the sequence reads can be applied to avoid miss-assembly caused by repeat sequences.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `When a sequence is generated from ____ ends of a single clone, the distance between the two opposing fragments of a clone is fixed to ________ meaning that they are always separated by a distance defined by a _____ length (normally 1,000 to 9,000 bases).`,
    options: ["both, an uncertain range, clone", "one, an uncertain range, clone", "both, a certain range, clone", "both, a certain range, gene"],
    correct: "c",
    explanation: `A commonly used constraint to avoid errors caused by sequence repeats is the so called forward–reverse constraint. When the constraint is applied, even when one of the fragments has a perfect match with a repetitive element outside the range, it is not able to be moved to that location to cause miss-assembly.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about base calling and assembly programs?`,
    options: ["The first step toward genome assembly includes derive base calls", "The first step toward genome assembly includes assigning associated quality scores", "One of the steps is to assemble the sequence reads into contiguous sequences", "There is no identifying overlap between sequence fragments"],
    correct: "d",
    explanation: `One of the steps includes identifying overlaps between sequence fragments, assigning the order of the fragments and deriving a consensus of an overall sequence. Assembling all shotgun fragments into a full genome is a computationally very challenging step. There are a variety of programs available for processing the raw sequence data.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect about Phred?`,
    options: ["It is a UNIX program", "It doesn\u2019t give a probability score in output", "It is used for base calling", "It uses a Fourier analysis to resolve fluorescence traces and predict actual peak locations of bases"],
    correct: "b",
    explanation: `It also gives a probability score for each base call that may be attributable to error. The commonly accepted score threshold is twenty, which corresponds to a 1% chance of error. The higher the score, the better the quality of the sequence reads. If the score value falls below the threshold, human intervention is required.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect about Phrap?`,
    options: ["It aligns individual fragments in a pairwise fashion using the Smith\u2013Waterman algorithm", "It doesn\u2019t take input from Phred", "It is used for sequence assembly", "It is a UNIX program"],
    correct: "b",
    explanation: `It takes Phred base-call files with quality scores as input and aligns individual fragments in a pairwise fashion using the Smith–Waterman algorithm. The base quality information is taken into account during the pairwise alignment. After all the pair wise sequence similarity is identified, the program performs assembly by progressively merging sequence pairs with decreasing similarity scores while removing overlapped regions. Consensus contigs are derived after joining all possible overlapped reads.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `VecScreen is a primarily aimed for sequence assembly.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `It is a web-based Program that helps detect contaminating bacterial vector sequences. It scans an input nucleotide sequence and compares it with a database of known vector sequences by using the BLAST program.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect about EULER?`,
    options: ["It is an assembly algorithm", "It uses a Eulerian Superpath approach, which is a polynomial algorithm", "In this approach, a sequence fragment is broken down to tuples of five nucleotides", "The tuples are distributed in a diagram with numerous nodes that are all interconnected"],
    correct: "c",
    explanation: `The tuples are converted to binary vectors in the nodes. By using a Viterbi algorithm, the shortest path among the vectors can be found, which is the best way to connect the tuples into a full sequence. Because this approach does not directly rely on detecting overlaps, it may be advantageous in assembling sequences with repeat motifs.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `TIGR Assembler is a UNIX program from TIGR for assembly of large shotgun sequence fragments.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It treats the sequence input as clean reads without consideration of the sequence quality. A main feature of the program is the application of the forward–reverse constraints to avoid miss-assembly caused by sequence repeats. The sequence alignment in the assembly stage is performed using the Smith–Waterman algorithm.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect about ARACHNE?`,
    options: ["It accepts base calls with associated quality scores assigned by Phred as input", "It is a free UNIX program", "It is for the assembly of whole-genome shotgun reads", "It doesn\u2019t involve heuristic approach"],
    correct: "d",
    explanation: `Its unique features include using a heuristic approach similar to FASTA to align overlapping fragments, evaluating alignments using statistical scores, correcting sequencing errors based on multiple sequence alignment, and using forward-reverse constraints. It accepts base calls with associated quality scores assigned by Phred as input and produces scaffolds or a fully assembled genome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The genome annotation process involves two steps: gene prediction and functional assignment.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Before the assembled sequence is deposited into a database, it has to be analyzed for useful biological features. The genome annotation process provides comments for the features.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect regarding gene annotation?`,
    options: ["The gene annotation of the human genome employs a combination of theoretical prediction and experimental verification", "Gene structures are first predicted by ab initio exon prediction programs", "The predicted genes are compared with experimentally determined cDNA and EST sequences", "The pairwise alignment programs are not involved"],
    correct: "d",
    explanation: `The predictions are verified by BLAST searches against a sequence database. The predicted genes are further compared with experimentally determined cDNA and EST sequences using the pairwise alignment programs such as GeneWise, Spidey, SIM4, and EST2 Genome.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect regarding gene ontology?`,
    options: ["There is standardization of the names and activities", "There is no standardization of associated pathways", "It provides consistency in describing overall protein functions", "It facilitates grouping of proteins of related functions"],
    correct: "b",
    explanation: `A GO description of a protein provides three sets of information: biological process, cellular component, and molecular function, each of which uses a unique set of non-overlapping vocabularies. The standardization of the names, activities, and associated pathways provides consistency in describing overall protein functions.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect regarding Automated Genome Annotation?`,
    options: ["It exists because of the need to develop fast and automated methods to annotate the genomic sequences", "The automated approach relies on homology detection", "The automated approach doesn\u2019t rely on heuristic sequence similarity searching", "Automation brings speed in gene annotation process"],
    correct: "c",
    explanation: `If a newly sequenced gene or its gene product has significant matches with a database sequence beyond a certain threshold, a transfer of functional assignment is taking place. In addition to sequence matching at the full length, detection of conserved motifs often offers additional functional clues.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Conserved functional sites can be identified by profile and hidden Markov model–based motif and domain search tools such as SMART and InterPro.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Detecting remote homologs typically involves combined searches of protein motifs and domains and prediction for secondary and tertiary structures. The prediction can also be performed using structure-based approaches such as threading and fold recognition.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `The remote homology detection helps to shed light on the possible functions of the proteins that previously have no functional information at all.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The bioinformatic analysis can spur an important advance in knowledge in many cases. Some hypothetical proteins, because of their novel structural folds, still cannot be predicted even with the advanced bioinformatics approaches and remain challenges for both experimental and computational work.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect regarding genome economy?`,
    options: ["It is a phenomenon of synthesizing more proteins from fewer genes", "This is a major strategy that eukaryotic organisms use to achieve a myriad of genotypic diversities only", "This is a major strategy that eukaryotic organisms use to achieve a myriad of phenotypic diversities", "There are numerous underlying genetic mechanisms to help account for genome economy"],
    correct: "b",
    explanation: `A major mechanism responsible for the protein diversity is alternative splicing, which refers to the splicing event that joins different exons from a single gene to form different transcripts. A related mechanism, known as exon shuffling, which joins exons from different genes to generate more transcripts, is also common in eukaryotes. It is known that, in humans, about two thirds of the genes exhibit alternative splicing and exon shuffling during expression, generating 90% of the total proteins.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `In some circumstances, one mRNA transcript can lead to the translation of more than one protein.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, human dentin phosphoprotein and dentin sialoprotein are proteins involved in tooth formation. An mRNA transcript that includes coding regions from both proteins is translated into a precursor protein that is cleaved to produce two different mature proteins.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is incorrect regarding GeneQuiz?`,
    options: ["It is a web server for protein DNA annotation", "It is a web server for protein sequence annotation", "It compares a query sequence against databases using BLAST and FASTA to identify homologs with high similarities", "It performs domain analysis using the PROSITE and Blocks databases"],
    correct: "a",
    explanation: `It performs domain analysis using the PROSITE and Blocks databases as well as analysis of secondary structures and super-secondary structures that include prediction of coiled coils and transmembrane helices. Multiple search and analysis results are compiled to produce a summary of protein function with an assigned confidence level (clear, tentative, marginal, and negligible).`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about comparative genomics?`,
    options: ["It is comparison of whole genomes from different organisms", "It includes comparison of gene number, gene location, and gene content from these genomes", "It provides insights into the mechanism of genome evolution and gene transfer among genomes", "It doesn\u2019t help to reveal the extent of conservation among genomes"],
    correct: "c",
    explanation: `It helps to understand the pattern of acquisition of foreign genes through lateral gene transfer. It also helps to reveal the core set of genes common among different genomes, which should correspond to the genes that are crucial for survival. This knowledge can be potentially useful in future metabolic pathway engineering.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about Whole Genome Alignment?`,
    options: ["This helps to reveal the presence of conserved functional elements", "It doesn\u2019t help to understand sequence conservation between genomes", "It be accomplished through direct genome comparison or genome alignment", "The alignment at the genome level is fundamentally no different from the basic sequence alignment"],
    correct: "b",
    explanation: `Regular alignment programs tend to be error prone and inefficient when dealing with long stretches of DNA containing hundreds or thousands of genes. Another challenge of genome alignment is effective visualization of alignment results. Because it is obviously difficult to sift through and make sense of the extremely large alignments, a graphical representation is a must for interpretation of the result.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about LAGAN?`,
    options: ["It stands for Limited Area Global Alignment of Nucleotides", "It is a web-based program designed for pairwise alignment of small fragments of genomes only", "It first finds anchors between two genomic sequences using an algorithm that identifies short, exactly matching words", "Regions that have high density of words are selected as anchors"],
    correct: "b",
    explanation: `It is a web-based program designed for pairwise alignment of large genomes. The unique feature of this program is that it is able to take into account degeneracy of the genetic codes and is therefore able to handle more distantly related genomes.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `A minimal constitutes genome, which is a _____ set of genes required for maintaining a free living cellular organism.`,
    options: ["maximum", "maximal", "highest number of set of", "minimal"],
    correct: "d",
    explanation: `Finding minimal genomes helps provide an understanding of genes constituting key metabolic pathways, which are critical for a cell’s survival. This analysis involves identification of orthologous genes shared between a number of divergent genomes.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Coregenes is a web-based program that determines a ________ set of genes based on comparison of ________ small genomes.`,
    options: ["vast, four", "core, fifteen", "core, four", "vast, fifteen"],
    correct: "c",
    explanation: `The user supplies NCBI accession numbers for the genomes of interest. The program performs an iterative BLAST comparison to find orthologous genes by using one genome as a reference and another as a query. This pairwise comparison is performed for all four genomes. As a result, the common genes are compiled as a core set of genes from the genomes.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about Lateral gene transfer?`,
    options: ["It is also known as vertical gene transfer", "There is exchange of genetic materials between species", "It mainly occurs among prokaryotic organisms when foreign genes are acquired through mechanisms", "It is one of the examples is transformation"],
    correct: "a",
    explanation: `It is defined as the exchange of genetic materials between species in a way that is incongruent with commonly accepted vertical evolutionary pathway. Examples are transformation (direct uptake of foreign DNA from environment), conjugation (gene uptake through mating behavior), and transduction (gene uptake mediated by infecting viruses). The transmission of genes between organisms can occur relatively recently or as a more ancient event.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `A way to discern lateral gene transfer is through phylogenetic analysis, referred to as an ‘among-genome’ approach, which can be used to discover __________`,
    options: ["recent lateral gene transfer events but almost negligible ancient events", "recent lateral gene transfer events", "ancient lateral gene transfer events", "both recent and ancient lateral gene transfer events"],
    correct: "d",
    explanation: `Abnormal groupings in phylogenetic trees are often interpreted as the possibility of lateral gene transfer events. There are some basic tools for identifying genomic regions that may be a result of lateral gene transfer events using the within-genome approach, namely, ACT, Swaap.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Within-Genome Approach is to identify regions within a genome with unusual compositions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Single or oligonucleotide statistics, such as G–C composition, codon bias, and oligonucleotide frequencies are used. Unusual nucleotide statistics in certain genomic regions versus the rest of the genome may help to identify “foreign” genes in a genome. A commonly used parameter is GC skew ((G − C)/(G + C)), which is compositional bias for G in a DNA sequence and is a commonly used indicator for newly acquired genetic elements.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue about Gene Order Comparison?`,
    options: ["When the order of a number of linked genes is conserved between genomes, it is called synteny", "Generally, gene order is much more conserved compared with gene sequences", "Generally, gene order is much less conserved compared with gene sequences", "It is in fact rarely observed among divergent species"],
    correct: "b",
    explanation: `Gene order conservation is in fact rarely observed among divergent species. Therefore, comparison of syntenic relationships is normally carried out between relatively close lineages. However, if syntenic relationships for certain genes are indeed observed among divergent prokaryotes, they often provide important clues to functional relationships of the genes of interest.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Genes involved in the same metabolic pathway tend to be clustered among phylogenetically diverse organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The preservation of the gene order is a result of the selective pressure to allow the genes to be co-regulated and function as an operon. Furthermore, the synteny of genes from divergent groups often associates with physical interactions of the encoded gene products.`,
    super_topic: `Genome Mapping, Assembly and Comparison`,
  },
  {
    question: `Which of the following is untrue regarding expressed sequence tags (ESTs)?`,
    options: ["EST libraries can be easily generated from various cell lines, tissues, organs, and at various developmental stages", "Although individual ESTs are prone to error, an entire collection of ESTs contains valuable information", "Identification of cDNA clone is difficult", "ESTs can also facilitate the unique identification of a gene from a cDNA library"],
    correct: "c",
    explanation: `A short tag can lead to a cDNA clone. Often, after consolidation of multiple EST sequences, a full-length cDNA can be derived. By searching a non-redundant EST collection, one can identify potential genes of interest.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `To generate EST data, clones in the cDNA library are randomly selected for sequencing from either end of the inserts.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The EST data are able to provide a rough estimate of genes that are actively expressed in a genome under a particular physiological condition. This is because the frequencies for particular ESTs reflect the abundance of the corresponding mRNA in a cell, which corresponds to the levels of gene expression at that condition. Another potential benefit of EST sampling is that, by randomly sequencing cDNA clones, it is possible to discover new genes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding the drawbacks of expressed sequence tags (ESTs)?`,
    options: ["They are often of lowquality because they are automatically generated without verification", "Many bases are ambiguously determined, represented by N\u2019s", "Frame shift errors and artifactual stop codons are some common errors", "Despite of all the failures, the translation the sequences is smooth"],
    correct: "d",
    explanation: `Common errors also include frameshift errors and artifactual stop codons, resulting in failures of translating the sequences. In addition, there is often contamination by vector sequence, introns (fromunspliced RNAs), ribosomal RNA (rRNA), mitochondrial RNA, among others. ESTs represent only partial sequences of genes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `It has been estimated that up to 11% of cDNA clones may be chimeric.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A problem of ESTs is the presence of chimeric clones owing to cloning artifacts in library construction, in which more than one transcript is ligated in a clone resulting in the 5_ end of a sequence representing one gene and the 3’ end another gene. Another fundamental problem with EST profiling is that it predominantly represents highly expressed, abundant transcripts. Weakly expressed genes are hardly found in a EST sequencing survey.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `GenBank has a special EST database, dbEST that contains EST collections for a large number of organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The rapid accumulation of EST sequences has prompted the establishment of public and private databases to archive the data. The mentioned database is regularly updated to reflect the progress of various EST sequencing projects. Each newly submitted EST sequence is subject to a database search. If a strong similarity to a known gene is found, it is annotated accordingly.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding EST Index Construction?`,
    options: ["The goal of the EST databases is to organize and consolidate the largely redundant EST data", "The process includes a preprocessing step that removes masks repeats", "There is no screening of vector contaminants", "The goal of the EST databases is to improve the quality of the sequence information so the data can be used to extract full-length cDNAs"],
    correct: "c",
    explanation: `The process includes a preprocessing step that removes vector contaminants and masks repeats. Vecscreen, can be used to screen out bacterial vector sequences. This is followed by a clustering step that associates EST sequences with unique genes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding UniGene?`,
    options: ["It is an NCBI EST cluster database", "Overlapping EST sequences are computationally processed to represent a single expressed gene", "Each cluster is a set of overlapping EST sequences", "The overlapping EST sequences are computationally processed to represent a set of expressed genes"],
    correct: "d",
    explanation: `The database is constructed based on combined information from dbEST, GenBank mRNA database, and “electronically spliced” genomic DNA. Only ESTs with 3’poly-A ends are clustered to minimize the problem of chimerism. The resulting 3’EST sequences provide more unique representation of the transcripts.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding TIGR Gene Indices?`,
    options: ["It is an EST database that the similar type of clustering method from UniGene", "It is an EST database that uses a different clustering method from UniGene", "It compiles data from dbEST, GenBank mRNA and genomic DNA data, and TIGR\u2019s own sequence databased Sequences are only clustered if they are more than 95% identical for over a fortynucleotide region in pairwise comparisons", "None of the mentioned"],
    correct: "a",
    explanation: `BLAST and FASTA are used to identify sequence overlaps. In the sequence assembly stage, both TIGR Assembler and CAP3 are used to construct contigs, producing a so-called tentative consensus (TC). To prevent chimerism, transcripts are clustered only if they match fully with known genes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding SAGE?`,
    options: ["It stands for Serial analysis of gene expression", "It is another high throughput, sequence-based approach for global gene expression profile analysis", "It stands for Squared analysis of gene expression", "Unlike EST sampling, SAGE is more quantitative in determining mRNA expression in a cell"],
    correct: "c",
    explanation: `In this method, short fragments of DNA (usually 15 base pairs [bp]) are excised from cDNA sequences and used as unique markers of the gene transcripts. The sequence fragments are termed tags. They are subsequently concatenated (linked together), cloned, and sequenced.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect about a microarray?`,
    options: ["It is a slide attached with a high-density array of immobilized DNA oligomers representing the entire genome of the species under study", "Array of immobilized DNA oligomers cannot be cDNAs", "Each oligomer is spotted on the slide and serves as a probe for binding to a unique complementary cDNA", "It is the most commonly used global gene expression profiling method"],
    correct: "b",
    explanation: `The entire cDNA population, labeled with fluorescent dyes or radioisotopes, is allowed to hybridize with the oligo probes on the chip. The amount of fluorescent or radiolabels at each spot position reflects the amount of corresponding mRNA in the cell. Using this analysis, patterns of global gene expression in a cell can be examined.`,
    super_topic: `Quizzone`,
  },
  {
    question: `which of the following is incorrect about Oligonucleotide Design in A microarray?`,
    options: ["DNA microarrays are generated by fixing oligonucleotides onto a solid support", "The oligonucleotide array slide represents thousands of preselected genes from an organism", "The length of oligonucleotides is typically in the range of twenty-five to seventy bases long", "The oligonucleotides don\u2019t react with cDNA samples"],
    correct: "d",
    explanation: `The oligonucleotides are called probes that hybridize to labeled cDNA samples. Shorter oligo probes tend to be more specific in hybridization because they are better at discriminating perfect complementary sequences fromsequences containing mismatches. However, longer oligos can be more sensitive in binding cDNAs.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect about Data Collection?`,
    options: ["The two-color microarray uses multiple dyes at times", "The most common type of microarray protocol is the two-color microarray", "The cDNAs are obtained by extracting total RNA or mRNA from tissues or cells and incorporating fluorescent dyes in the DNA strands during the cDNA biosynthesis", "The expression of genes is measured via the signals from cDNAs hybridizing with the specific oligonucleotide probes on the microarray"],
    correct: "a",
    explanation: `The most common type of microarray protocol is the two-color microarray, which involves labeling one set of cDNA from an experimental condition with one dye (Cy5, red fluorescence) and another set of cDNA from a reference condition (the controls) with another dye (Cy3, green fluorescence). When the two differently labeled cDNA samples are mixed in equal quantity and allowed to hybridize with the DNA probes on the chips, gene expression patterns of both samples can be measured simultaneously.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `In the analysis of microarray data–If replicated datasets are available, rigorous statistical tests such as t-test and analysis of variance (ANOVA) can be performed to test the null hypothesis that a given data point is not significantly different from the mean of the data distribution.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For such tests, it is common to use a P-value cutoff of .05, which means a confidence level of 95% to distinguish the data groups. This level also corresponds to a gene expression level with two standard deviations from the mean of distribution.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect about Classification of microarray data?`,
    options: ["For microarray data, clustering analysis identifies coexpressed and coregulated genes", "For microarray data, clustering analysis identifies coexpressed but not coregulated genes", "For microarray data, clustering analysis identifies and coregulated but not coexpressed genes", "Genes within a category have more similarity in expression than genes from different categories."],
    correct: "a",
    explanation: `When genes are co-regulated, they normally reflect related functionality. Through gene clustering, functions of previously uncharacterized genes may be discovered. Clustering methods include hierarchical clustering and partitioning clustering (e.g., k-means, self-organizing maps [SOMs]).
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `A supervised analysis refers to classification of data into a set of predefined categories. For example, depending on the purpose of the experiment, the data can be classified into predefined ‘diseased’ or ‘normal’ categories.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Similarly, an unsupervised analysis does not assume predefined categories, but identifies data categories according to actual similarity patterns. The unsupervised analysis is also called clustering, which is to group patterns into clusters of genes with correlated profiles.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect about Hierarchical Clustering?`,
    options: ["The tree-branching pattern illustrates a higher degree of relationship between related gene groups", "It is not similar to the distance phylogenetic tree-building method", "It produces a treelike structure that represents a hierarchy or relative relatedness of data groups", "In the tree leaves, similar gene expression profiles are placed more closely together than dissimilar gene expression profiles"],
    correct: "b",
    explanation: `A hierarchical clustering method is in principle similar to the distance phylogenetic tree-building method. When genes with similar expression profiles are grouped in such a way, functions for unknown genes can often be inferred. Hierarchical clustering uses the agglomerative approach that works in much the same way as the UPGMA method, in which the most similar data pairs are joined first to form a cluster.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect about k-Means Clustering?`,
    options: ["k-means clustering produces a dendrogram", "It classifies data through a single step partition", "It is a divisive approach", "In this method, data are partitioned into k-clusters, which are prespecified at the outset"],
    correct: "a",
    explanation: `In contrast to hierarchical clustering algorithms, k-means clustering does not produce a dendrogram, but instead classifies data through a single step partition. The value of k is normally randomly set but can be adjusted if results are found to be unsatisfactory.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect about Self-Organizing Maps?`,
    options: ["Clustering by SOMs is in principle similar to the k-means method", "It doesn\u2019t involve neural networks", "The data points are initially assigned to the nodes at random", "It starts by defining a number of nodes"],
    correct: "b",
    explanation: `This pattern recognition algorithm employs neural networks. The distance between the input data points and the centroids are calculated. The data points are successively adjusted among the nodes, and their distances to the centroids are recalculated. After many iterations, a stabilized clustering pattern are reached with the minimum distances of the data points to the centroids. The differences between SOM and k-means are that, in SOM, the nodes are not treated as isolated entities, but as connected to other nodes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `TIGR TM4 is a suite of multiplatform programs for analyzing microarray data.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This comprehensive package includes four interlinked programs, TIGR spot finder (for image analysis), MIDAS (for data normalization), MeV (for clustering analysis and visualization), and MADAM (for data management). The package provides different data normalization schemes and clustering options. Other Similar Clustering Programs are EPCLUST, SOTA.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue about SAGE?`,
    options: ["Sequencing is the most costly and time-consuming step", "Here, sequencing is economical but time-consuming step", "Sequencing is economical but time-reducing step", "It is difficult to know how many tags need to be sequenced to get a good coverage of the entire transcriptome"],
    correct: "a",
    explanation: `It is generally determined on a case-by-case basis. As a rule of thumb, 10,000 clones representing approximately 500,000 tags from each sample are sequenced. The scale and cost of the sequencing required for SAGE analysis are prohibitive for most laboratories. Only large sequencing centers can afford to carry out SAGE analysis routinely.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue about the drawbacks of SAGE?`,
    options: ["One or two sequencing errors in the tag sequence can lead to ambiguous or erroneous tag identification", "Correctly sequenced SAGE tag sometimes may correspond to several genes or no gene at all", "Correctly sequenced SAGE tag always corresponds to several genes", "The drawback with this approach is the sensitivity to sequencing errors"],
    correct: "c",
    explanation: `To improve the sensitivity and specificity of SAGE detection, the lengths of the tags need to be increased for the technique. There are some comprehensive software tools for SAGE analysis viz. SAGEmap, SAGExProfiler.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `SAGEmap is a SAGE database created by NCBI.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Given a cDNA sequence, one can search SAGE libraries for possible SAGE tags and perform “virtual” Northern blots that indicate the relative abundance of a tag in a SAGE library. Each output is hyperlinked to a particular UniGene entry with sequence annotation.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `SAGExProfiler doesn’t provide information about overexpressed or silenced genes.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `It is a web-based program that allows a “virtual subtraction” of an expression profile of one library (e.g., normal tissue) from another (e.g., diseased tissue). Comparison of the two libraries can provide information about overexpressed or silenced genes in normal versus diseased tissues.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue about SAGE Genie?`,
    options: ["It is an NCBI web-based program", "It allows matching of experimentally obtained SAGE tags to known genes", "It provides an interface for visualizing human gene expression", "It doesn\u2019t filter out linker sequences from experimentally obtained SAGE tags"],
    correct: "d",
    explanation: `It has a filtering function that filters out linker sequences from experimentally obtained SAGE tags and allows expression pattern comparison between normal and diseased human tissues. The data output can be presented using subprograms such as the Anatomic Viewer, Digital Northern, and Digital Gene Expression Display.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `DNA microarrays measure “absolute” mRNA expression levels without arbitrary reference standards, whereas SAGE indicates the relative expression levels.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `SAGE measures “absolute” mRNA expression levels without arbitrary reference standards, whereas DNA microarrays indicate the relative expression levels. Therefore, SAGE expression data are more comparable across experimental conditions and platforms. This makes public SAGE databases more informative by allowing comparison of data from reference conditions with various experimental treatments.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `The PCR amplification step involved in the SAGE procedure means that it requires a large quantity of sample mRNA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The PCR amplification step involved in the SAGE procedure means that it requires only a minute quantity of sample mRNA. This compares favorably to the requirement for a much larger quantity of mRNA for microarray experiments, which may be impossible to obtain under certain circumstances.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `The classic protein separation methods involve two-dimensional gel electrophoresis followed by gel image analysis.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Further characterization involves determination of amino acid composition, peptide mass fingerprints, and sequences using mass spectrometry (MS). Finally, database searching is needed for protein identification.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect regarding 2D-Page?`,
    options: ["Not all proteins can be separated by this method or stained properly", "The stained gel can be scanned and digitized for image analysis", "Membrane proteins are largely hydrophilic and readily solubilized", "One of the challenges of this technique is the separation of membrane proteins"],
    correct: "c",
    explanation: `Membrane proteins are largely hydrophobic and not readily solublized. They tend to aggregate in the aqueous medium of a two-dimensional gel. To overcome this problem, membrane proteins can be fractionated using specialized protocols and then electrophoresed using optimized buffers containing zwitterionic detergents. Subfractionation can be carried out to separate nuclear, cytosol, cytoskeletal, and other subcellular fractions to boost the concentrations of rare proteins and to reveal subcellular localizations of the proteins.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Comparing two-dimensional gel images from various experiments can sometimes pose a challenge because the gels, unlike DNA microarrays, may shrink or warp.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This requires the software programs to be able to stretch or maneuver one of the gels relative to the other to find a common geometry. When the reference spots are aligned properly, the rest of the spots can be subsequently compared automatically.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect regarding Mass Spectrometry Protein Identification?`,
    options: ["The proteolysis doesn\u2019t generate a pattern according to molecular weight", "Proteins can be identified and characterized using MS", "The proteins from a two dimensional gel system are first digested in situ with a protease", "Protein spots of interest are excised from the two-dimensional gel"],
    correct: "a",
    explanation: `The proteolysis generates a unique pattern of peptide fragments of various MWs, which is termed a peptide fingerprint. The fragments can be analyzed with MS, a high-resolution technique for determining molecular masses. Currently, electro-spray ionization MS and matrix-assisted laser desorption ionization (MALDI) MS are commonly used.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Electrospray ionization MS and matrix-assisted laser desorption ionization (MALDI) MS only differ in the ionization procedure used.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In MALDI-MS, for example, the peptides are charged with positive ions and forced through an analyzing tube with a magnetic field. Peptides are analyzed in the gas phase. Because smaller peptides are deflected more than larger ones in a magnetic field, the peptide fragments can be separated according to molecular mass and charges. A detector generates a spectrum that displays ion intensity as a function of the mass-to-charge ratio.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect regarding the Protein Identification through Database Searching?`,
    options: ["MS characterization of proteins is highly dependent on bioinformatic analysis", "Bioinformatics programs can be used to search for the identity of a protein in a database of theoretically digested proteins", "Even in reality, the protease digestion is always perfect in MS", "The purpose of the database search is to find exact or nearly exact matches"],
    correct: "c",
    explanation: `In reality, protease digestion is rarely perfect, often generating partially digested products as a result of missed cuts at expected cutting sites. Peptides resulting from MALDI-MS are also charged, which increases their mass slightly.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `ExPASY is a comprehensive proteomics web server with a suite of programs for searching peptide information from the SWISS-PROT and TrEMBL databases.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `There are twelve database search tools in this server dedicated to protein identification based on MS data. For example, the AACompIdent program identifies proteins based on pI, MW, and amino acid composition and compares these values with theoretical compositions of all proteins in SWISS-PROT/TrEMBL.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect regarding Mascot and ProFound?`,
    options: ["ProFound is a web server with a set of interconnected programs", "ProFound searches a protein sequence database using MS fingerprinting information", "Bayesian algorithm is not involved in ProFound", "Mascot is a web server that identifies proteins based on peptide mass fingerprints, sequence entries, or raw MS/MS data from one or more peptides"],
    correct: "c",
    explanation: `In ProFound, A Bayesian algorithm is used. It ranks the database matches according to the probability of database sequences producing the peptide mass fingerprints.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is incorrect regarding Differential In-Gel Electrophoresis?`,
    options: ["Proteins are mixed together before electrophoresis on a two-dimensional gel", "Differentially expressed proteins in both conditions can\u2019t be visualized in the same gel", "In this, Differences in protein expression patterns can be detected in a similar way as in fluorescent-labeled DNA microarrays", "Proteins from experimental and control samples are labeled with differently colored fluorescent dyes"],
    correct: "b",
    explanation: `Differentially expressed proteins in both conditions can be co-separated and visualized in the same gel. Compared to regular 2D-PAGE, the process reduces the noise and improves the reproducibility and sensitivity of detection. In principle, it resembles the two-color DNA microarray analysis. The drawbacks of this approach are that different proteins take up fluorescent tags to different extents and that some proteins labeled with the fluorophores may become less soluble and precipitate before electrophoresis.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is a wrong statement?`,
    options: ["Prokaryotes include bacteria and Archaea", "Prokaryotes have relatively large genomes", "Prokaryotes have relatively small genomes", "In Prokaryotes, The gene density in the genomes is high, with more than 90% of a genome sequence containing coding sequence"],
    correct: "b",
    explanation: `Prokaryotes have relatively small genomes with sizes ranging from0.5 to 10Mbp (1Mbp=106 bp). Each prokaryotic gene is composed of a single contiguous stretch of ORF coding for a single protein or RNA with no interruptions within a gene.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is a wrong about AutoMotif?`,
    options: ["It is a web server predicting protein sequence motifs", "It doesn\u2019t use SVM approach", "In this process, the query sequence is chopped up into a number of overlapping fragments", "The overlapping fragments from are query sequence are fed into different kernels (similar to nodes)"],
    correct: "b",
    explanation: `Hyperplane, which has been trained to recognize known protein sequence motifs, separates the kernels into different classes. Each separation is compared with known motif classes, most of which are related to posttranslational modification. The best match with a known class defines the functional motif.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `It is important to use bioinformatics tools to predict sites for posttranslational modifications based on specific protein sequences. However, prediction of such modifications can often be difficult because the short lengths of the sequence motifs associated with certain modifications.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This often leads to many false-positive identifications. One such example is the known consensus motif for protein phosphorylation, [ST]-x-[RK]. Such a short motif can be found multiple times in almost every protein sequence. Most of the predictions based on this sequence motif alone are likely to be wrong, producing very high rates of false-positives.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `To minimize false-positive results, a statistical learning process called support vector machine (SVM) can be used to increase the specificity of prediction.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is a data classification method similar to the linear or quadratic discriminant analysis. In this method, the data are projected in a three-dimensional space or even a multidimensional space.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `In a statistical learning process called support vector machine (SVM), a hyperplane is _____`,
    options: ["a linear or nonlinear mathematical function", "nonlinear mathematical function", "linear mathematical function", "exponential mathematical function"],
    correct: "a",
    explanation: `It is used to best separate true signals from noise. The algorithm has more environmental variables included that may be required for the enzyme modification. After training the algorithm with sufficient structural features, it is able to correctly recognize many posttranslational modification patterns.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Uncategorized`,
  },
  {
    question: `A disulfide bridge is a unique type of _____ modification in which _____ bonds are formed between cysteine residues.`,
    options: ["posttranslational, covalent", "translational, covalent", "translational, ionic", "posttranslational, ionic"],
    correct: "a",
    explanation: `Disulfide bonds are important for maintaining the stability of certain types of proteins. The disulfide prediction is the prediction of paring potential or bonding states of cysteines in a protein.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Accurate prediction of _____ bonds may also help to predict the _____ dimensional structure of the protein of interest.`,
    options: ["nitrogen, two", "nitrogen, three", "disulfide, three", "oxygen, three"],
    correct: "c",
    explanation: `This problem can be tackled by using profiles constructed from multiple sequence alignment. It can also be tackled by using residue contact potentials calculated based on the local sequence environment.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Only Advanced neural networks are used to discern long-distance pairwise interactions among cysteine residues.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Advanced neural networks or SVM or hidden Markov model (HMM) algorithms are often used to discern long-distance pairwise interactions among cysteine residues. Cysteine is one of the publicly available programs specialized in disulfide prediction.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Cysteine doesn’t make predictions by building profiles.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Is a web server that predicts the disulfide bonding states of cysteine residues in a protein sequence by building profiles based on multiple sequence alignment information. A recursive neural network ranks the candidate residues for disulfide formation.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `ExPASY contains a number of programs to determine posttranslational modifications based on MS molecular mass data.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Find Mod is a subprogram that uses experimentally determined peptide fingerprint information to compare the masses of the peptide fragments with those of theoretical peptides. If a difference is found, it predicts a particular type of modification basedona set of predefined rules. It can predict twenty-eight types of modifications, including methylation, phosphorylation, lipidation, and sulfation.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is an incorrect statement about the terminologies related to protein sorting?`,
    options: ["Subcellular localization is an integral part of protein functionality", "Many proteins exhibit functions only after being transported to certain compartments of the cell", "All the proteins exhibit functions after being transported to certain compartments of the cell", "Protein sorting is also known as protein targeting"],
    correct: "c",
    explanation: `The study of the mechanism of protein trafficking and subcellular localization is the field of protein sorting, which has become one of the central themes in modern cell biology. Identifying protein subcellular localization is an important aspect of functional annotation, because knowing the cellular localization of a protein often helps to narrow down its putative functions.`,
    super_topic: `Quizzone`,
  },
  {
    question: `For many eukaryotic proteins, newly synthesized protein precursors have to be transported to specific membrane-bound compartments and be proteolytically processed to become functional.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These compartments include chloroplasts, mitochondria, the nucleus, and peroxisomes. To carry out protein translocation, unique peptide signals have to be present in the nascent proteins, which function as “zip codes” that direct the proteins to each of these compartments.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Once the proteins are translocated within the organelles, protease cleavage takes place to remove the signal sequences and generate mature proteins.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is an example of posttranslational modification. Even in prokaryotes, proteins can be targeted to the inner or outer membranes, the periplasmic space between these membranes, or the extracellular space. The sorting of these proteins is similar to that in eukaryotes and relies on the presence of signal peptides.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `The signal sequences have a _________ consensus but contain some specific features. They all have a ______ core region preceded by one or more positively charged residues.`,
    options: ["weak, hydrophilic", "weak, hydrophobic", "strong, hydrophilic", "strong, hydrophilic"],
    correct: "b",
    explanation: `However, the length and sequence of the signal sequences vary tremendously. Peptides targeting mitochondria, for example, are located in the N-terminal region.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `The signal sequences are typically _______ residues long, rich in _____ charged residues such as arginines as well as hydroxyl residues such as serines and threonines, but devoid of ______ charged residues.`,
    options: ["28 to 80, positively, negatively", "300 to 800, negatively, positively", "28 to 80, negatively, positively", "300 to 500, positively, negatively"],
    correct: "a",
    explanation: `They have the tendency to form amphiphilic α-helices. These targeting sequences are cleaved once the precursor proteins are inside the mitochondria.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Chloroplast localization signals are also located in the __________ terminus and are about 25 to 100 residues in length, containing very few _______ charged residues but many hydroxylated residues such as serine.`,
    options: ["N, negatively", "C, negatively", "C, positively", "N, positively"],
    correct: "a",
    explanation: `Chloroplast localization signals are also called transit Peptides. An interesting feature of the proteins targeted for the chloroplasts is that the transit signals are bipartite.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Chloroplast localization signals consist of two adjacent signal peptides, one for targeting the proteins to the stromaportion of the chloroplast before being cleaved and the other for targeting the remaining portion of the proteins to the thylakoids.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Localization signals targeting to the nucleus are variable in length (seven to forty-one residues) and are found in the internal region of the proteins. They typically consist of one or two stretches of basic residues with a consensus motif K(K/R)X(K/R). Nuclear signal sequences are not cleaved after protein transport.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is an incorrect statement about SignalP?`,
    options: ["It only uses neural networks", "It only uses HMMs", "It is a web-based program that predicts subcellular localization signals", "It uses both neural networks and HMMs"],
    correct: "b",
    explanation: `The neural network algorithm combines two different scores, one for recognizing signal peptides and the other for protease cleavage sites. The HMM-based analysis discriminates between signal peptides and the N-terminal transmembrane anchor segments required for insertion of the protein into the membrane.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is not one of the training sets in SignalP?`,
    options: ["Prokaryotes", "Eukaryotes", "Gram-positive bacteria", "Gram-negative bacteria"],
    correct: "a",
    explanation: `This distinction is necessary because there are significant differences in the characteristics of the signal peptides from these organisms. Therefore, appropriate datasets need to be selected before analyzing the sequence. The program predicts both the signal peptides and the protease cleavage sites of the query sequence.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `TargetP is a neural network-based program, similar to SignalP.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It predicts the subcellular locations of eukaryotic proteins based on their N-terminal amino acid sequence only. It uses analysis output from SignalP and feeds it into a decision neural network, which makes a final choice regarding the target compartment.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding the classic yeast two-hybrid method?`,
    options: ["Protein\u2013protein interaction networks of yeast and a small number of other species have been subsequently determined using this method", "This technique is a high throughput approach", "Each bait and prey construct has to be prepared individually to map interactions between all proteins", "It has been systematically applied to study interactions at the whole proteome level"],
    correct: "b",
    explanation: `This technique is essentially a low throughput approach. A major flaw in this method is that it is an indirect approach to probe protein–protein interaction and has a tendency to generate false positives (spurious interactions) and false negatives (undetected interactions). It has been estimated from proteome-wide characterizations that the rate of false positives can be as high as 50%.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `If the bait and prey proteins _______ they bring the DNA-binding and trans-activation domains in such close proximity that they reconstitute the function of the transcription activator, turning ____ the expression of a reporter gene as a result. Which of the following is not the correct pair of blanks?`,
    options: ["physically interact, on", "do not interact, on", "do not interact, off", "stop interacting, off"],
    correct: "b",
    explanation: `Molecular constructs are made such that each of the two domains is covalently attached to each of the two candidate proteins. If the two candidate proteins do not interact, the reporter gene expression remains switched off.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `An alternative approach to determining protein–protein interactions is to use a large-scale affinity purification technique that involves attaching fusion tags to proteins and purifying the associated protein complexes in an affinity chromatography column.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The purified proteins are then analyzed by gel electrophoresis followed by MS for identification of the interacting components. The protein microarray systems mentioned above also provide a high throughput alternative for studying protein–protein interactions.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding the Predicting Interactions Based on Domain Fusion?`,
    options: ["It is based on gene fusion events", "Predicting protein\u2013protein interactions is called the \u201cRosetta stone\u201d method", "A fused protein often reveals relationships between its domain components", "A fused protein doesn\u2019t necessarily reveal about the relationships between its domain components"],
    correct: "d",
    explanation: `The rationale goes like this: if A and B exist as interacting domains in a fusion protein in one proteome, the gene encoding the protein is a fusion gene. Their homologous gene sequences A and B existing separately in another genome most likely encode proteins interacting to perform a common function. Conversely, if ancestral genes A and B encode interacting proteins, they may have a tendency to be fused together in other genomes during evolution to enhance their effectiveness.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `When the two domains are located in two different proteins, to preserve the same functionality, their close proximity and interaction have to be preserved as well.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In this method, by studying gene/protein fusion events, protein–protein interactions can be predicted. This prediction rule has been proven to be rather reliable and since successfully applied to a large number of proteins from both prokaryote and eukaryotes.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `The justification behind Rosetta stone method is that when two domains are fused in a single protein, they have to be in _______ proximity to perform a common function.`,
    options: ["distant", "close", "extremely distant", "extremely close"],
    correct: "d",
    explanation: `When the two domains are located in two different proteins, to preserve the same functionality, their close proximity and interaction have to be preserved as well. Therefore, by studying gene/protein fusion events, protein–protein interactions can be predicted.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `In Predicting Interactions Based on Gene Neighbors if a certain gene linkage is found to be indeed conserved across divergent genomes, it can be used as a strong indicator of formation of an operon that encodes proteins that are functionally and even physically coupled.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This rule of predicting protein–protein interactions holds up for most prokaryotic genomes. For eukaryotic genomes, gene order may be a less potent predictor of protein interactions than a tight co-regulation for gene expression.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding the predicting Interactions Based on Phylogenetic Information?`,
    options: ["Proteins do not operate as a complex", "This method detects the co-presence or co-absence of orthologs across a number of genomes", "Protein interactions can be predicted using phylogenetic profiles", "Phylogenetic profile are defined as patterns of gene pairs that are concurrently present or absent across genomes"],
    correct: "a",
    explanation: `The logic behind the co-occurrence approach is that proteins normally operate as a complex. If one of the components of the complex is lost, it results in the failure of the entire complex. Under the selective pressure, the rest of the nonfunctional interacting partners in the complex are also lost during evolution because they have become functionally unnecessary.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Which of the following is untrue regarding the STRING?`,
    options: ["Search Tool for the Retrieval of Interacting Genes/Proteins", "Functional associations include only the direct protein-protein interactions", "It is based on combined evidence of gene linkage, gene fusion and phylogenetic profiles", "It is a web server that predicts gene and protein functional associations"],
    correct: "b",
    explanation: `Functional associations include both direct and indirect protein-protein interactions.
Indirect interactions can mean enzymes in the same pathway sharing a common substrate or proteins regulating each other in the genetic pathway.`,
    super_topic: `Functional Genomics & Proteomics`,
  },
  {
    question: `Phylogenetics is the study of the evolutionary history of living organisms using treelike diagrams to represent pedigrees of these organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Tree branching patterns representing the evolutionary divergence are referred to as phylogeny. Phylogenetics can be studied in various ways. It is often studied using fossil records, which contain morphological information about ancestors of current species and the timeline of divergence.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `The descriptions of morphological traits are often _____ which are due to _____`,
    options: ["ambiguous, multiple genetic factors", "lucid, more than one genetic factors", "clear, multiple genetic factors", "ambiguous, one or two genetic factors"],
    correct: "a",
    explanation: `Thus, using fossil records to determine phylogenetic relationships can often be biased. For microorganisms, fossils are essentially nonexistent, which makes it impossible to study phylogeny with this approach.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is incorrect regarding the advantages of Molecular data for phylogenetics study?`,
    options: ["They are more numerous than fossil records", "They are easier to obtain as compared to fossil records", "Sampling bias is involved", "More clear-cut and robust phylogenetic trees can be constructed with the molecular data"],
    correct: "c",
    explanation: `There is no sampling bias involved, which helps to mend the gaps in real fossil records. Therefore, they have become favorite and sometimes the only information available for researchers to reconstruct evolutionary history. The advent of the genomic era with tremendous amounts of molecular sequence data has led to the rapid development of molecular phylogenetics.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `To use molecular data to reconstruct evolutionary history requires making a number of reasonable assumptions. Which of the following is incorrect about it?`,
    options: ["The molecular sequences used in phylogenetic construction are homologous", "The molecular sequences used in phylogenetic construction share a common origin", "Phylogenetic divergence cannot be bifurcating", "Parent branch splits into two daughter branches at any given point"],
    correct: "c",
    explanation: `Here, option “Phylogenetic divergence cannot be bifurcating” and “Parent branch splits into two daughter branches at any given point” contradict. Another assumption in phylogenetics is that each position in a sequence evolved independently. The variability among sequences is sufficiently informative for constructing unambiguous phylogenetic trees.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Building phylogenetic tree involves bifurcation and multifurcation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Multifurcation is normally a result of insufficient evidence to fully resolve the tree or a result of an evolutionary process known as radiation. Sometimes, a branch point on a phylogenetic tree may have more than two descendents, resulting in a multifurcating node.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is incorrect regarding the terminologies of phylogenetics?`,
    options: ["The branching pattern in a tree is called tree topology", "When all branches bifurcate on a phylogenetic tree, it is referred to as dichotomy", "In case of dichotomy, each ancestor divides and gives rise to multiple descendants", "An unrooted phylogenetic tree does not assume knowledge of a common ancestor"],
    correct: "c",
    explanation: `Sometimes, a branch point on a phylogenetic tree may have more than two descendents, resulting in a multifurcating node. The phylogeny with multifurcating branches is called polytomy. A polytomy is an be a result of either an ancestral taxon giving rise to more than two immediate descendants simultaneously during evolution, a process known as radiation, or an unresolved phylogeny in which the exact order of bifurcations cannot be determined precisely.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Because there is no indication of which node represents an ancestor, there is no direction of an evolutionary path in an unrooted tree.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `To define the direction of an evolution path, a tree must be rooted. In a rooted tree, all the sequences under study have a common ancestor or root node from which a unique evolutionary path leads to all other nodes.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Molecular clock is an assumption by which molecular sequences evolve at varying rates.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Molecular clock is an assumption by which molecular sequences evolve at constant rates so that the amount of accumulated mutations is proportional to evolutionary time. Based on this hypothesis, branch lengths on a tree can be used to estimate divergence time. This assumption of uniformity of evolutionary rates, however, rarely holds true in reality.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `A gene phylogeny only describes the evolution of a particular gene or encoded protein.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `One of the objectives of building phylogenetic trees based on molecular sequences is to reconstruct the evolutionary history of the species involved. However, strictly speaking, a gene phylogeny (phylogeny inferred from a gene or protein sequence) only describes the evolution of that particular gene or encoded protein.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Evolution of a particular sequence _______ correlate with the evolutionary path of the species.`,
    options: ["does not", "always", "does not necessarily", "invariably"],
    correct: "c",
    explanation: `The sequence may evolve more or less rapidly than other genes in the genome or may have a different evolutionary history from the rest of the genome owing to horizontal gene transfer events. Thus, the evolution of a particular sequence does not necessarily correlate with the evolutionary path of the species.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `The species evolution is the ______ of evolution by _____ in a genome.`,
    options: ["combined result, multiple genes", "result, single genes", "result, sole genes", "distinct results, single gene"],
    correct: "a",
    explanation: `In a species tree, the branching point at an internal node represents the speciation event whereas, in a gene tree, the internal node indicates a gene duplication event. The two events may or may not coincide.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `To obtain a species phylogeny, phylogenetic trees from a variety of gene families need to be constructed.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `To obtain a species phylogeny, phylogenetic trees from a variety of gene families need to be constructed to give an overall assessment of the species evolution. Phylogenetic trees drawn as cladograms (top) and phylograms (bottom).`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `It is often desirable to define the root of a tree. There are two ways to define the root of a tree. One is to use an outgroup, which ______`,
    options: ["is a sequence that is homologous to the sequences under consideration", "is separated from those sequences at an early evolutionary time", "is generally determined from independent sources of information", "is generally determined from similar or related sources of information"],
    correct: "d",
    explanation: `Here, the option “is generally determined from independent sources of information” and contradict and it can be explained as follows. For example, a bird sequence can be used as a root for the phylogenetic analysis of mammals based on multiple lines of evidence that indicate that birds branched off prior to all mammalian taxa in the in group. Outgroups are required to be distinct from the in group sequences, but not too distant from the in group.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is incorrect statement about the Kimura model?`,
    options: ["It is a model to correct evolutionary distances and is a more sophisticated model", "In this, the mutation rates for transitions and transversion are assumed to be different", "According to this model, occur more frequently than transversions", "According to this model, transversions occur more frequently than transitions"],
    correct: "d",
    explanation: `This provides a more realistic estimate of evolutionary distances. The Kimura model uses the following formula dAB = −(1/2) ln(1 − 2pti − ptv) − (1/4) ln(1 − 2ptv). dAB is the evolutionary distance between sequences Aand B, pti is the observed frequency for transition, and ptv the frequency of transversion.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is incorrect statement about Choosing Substitution Models?`,
    options: ["There is one substitution at a particular position, in divergent sequences", "The evolutionary divergence is beyond the ability of the statistical models to correct", "The statistical models used to correct homoplasy are called substitution models or evolutionary models", "For constructing DNA phylogenies, there are nucleotide substitution models available"],
    correct: "a",
    explanation: `The caveat of using these models is that if there are too many multiple substitutions at a particular position, which is often true for very divergent sequences, the position may become saturated. This means that the evolutionary divergence is beyond the ability of the statistical models to correct. In this case, true evolutionary distances cannot be derived. Therefore, only reasonably similar sequences are to be used in phylogenetic comparisons.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `The second step in phylogenetic analysis is to construct sequence alignment. This is probably the most critical step in the procedure because it establishes positional correspondence in evolution.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Incorrect alignment leads to systematic errors in the final tree or even a completely wrong tree. For that reason, it is essential that the sequences are correctly aligned. Multiple state-of-the-art alignment programs such as T-Coffee should be used. The alignment results from multiple sources should be inspected and compared carefully to identify the most reasonable one. Automatic sequence alignments almost always contain errors and should be further edited or refined if necessary.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is an incorrect statement about Newick Format?`,
    options: ["It was designed to provide information of tree topology to computer programs without having to draw the tree itself", "In this format, trees are represented by taxa excluded in nested parentheses", "In this linear representation, each internal node is represented by a pair of parentheses", "For a tree with scaled branch lengths, the branch lengths in arbitrary units are placed immediately after the name of the taxon separated by a colon"],
    correct: "b",
    explanation: `Trees are represented by taxa included in nested parentheses. In this linear representation, each internal node is represented by a pair of parentheses that enclose all member of a monophyletic group separated by a comma.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Sometimes a tree-building method may result in several equally optimal trees. A consensus tree can be built by showing the commonly resolved bifurcating portions and collapsing the ones that disagree among the trees, which results in a polytomy.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Combining the nodes can be done either by strict consensus or by majority rule.
In a strict consensus tree, all conflicting nodes are collapsed into polytomies. In a consensus tree based on a majority rule, among the conflicting nodes, those that agree by more than 50% of the nodes are retained whereas the remaining nodes are collapsed into multifurcation.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `The number of rooted trees (NR) for n taxa is ______`,
    options: ["NR = (2n\u2212 3)!/2n+2 (n\u2212 2)!", "NR = (2n\u2212 3)!/2n (n\u2212 2)!", "NR = (2n\u2212 3)!/2n\u22122 (n\u2212 5)!", "NR = (2n\u2212 3)!/2n\u22122 (n\u2212 2)!"],
    correct: "d",
    explanation: `The number of potential tree topologies can be enormously large even with a moderate number of taxa. The increase of possible tree topologies follows an exponential function. In this formula, (2n−3)! Is a mathematical expression of factorial, which is the product of positive integers from 1 to 2n − 3. For example, 5! = 1 × 2 × 3 × 4 × 5 = 120.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `For unrooted trees, the number of unrooted tree topologies (NU) is ________`,
    options: ["NU = (2n\u2212 5)!/2n\u22123(n\u2212 5)!", "NU = (2n\u2212 5)!/2n\u22123(n\u2212 3)!", "NU = (2n\u2212 5)!/2\u22122(n\u2212 3)!", "NU = (2n\u2212 5)!/2n(n\u2212 3)!"],
    correct: "b",
    explanation: `The number of possible topologies increases extremely rapidly with the number of taxa. For six taxa, there are 105 unrooted trees and 945 rooted trees. If there are ten taxa, there can be 2,027,025 unrooted trees and 34,459,425 rooted ones.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `It can be computationally very demanding to find a true phylogenetic tree when the number of sequences is large.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because the number of rooted topologies is much larger than that for unrooted ones, the search for a true phylogenetic tree can be simplified by calculating the unrooted trees first. Once an optimal tree is found, rooting the tree can be performed by designating a number of taxa in the data set as an outgroup based on external information to produce a rooted tree.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is incorrect statement about Molecular Markers?`,
    options: ["For studying very closely related organisms, protein sequences are preferred", "The decision to use nucleotide or protein sequences depends on the purposes of the study", "For constructing molecular phylogenetic trees, one can use either nucleotide or protein sequence data", "The decision to use nucleotide or protein sequences depends on the properties of the sequences"],
    correct: "a",
    explanation: `The choice of molecular markers is an important matter because it can make a major difference in obtaining a correct tree. For studying very closely related organisms, nucleotide sequences, which evolve more rapidly than proteins, can be used. For example, for evolutionary analysis of different individuals within a population, noncoding regions of mitochondrial DNA are often used.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `For studying the evolution of ________ divergent groups of organisms, one may choose either ______ nucleotide sequences, such as ribosomal RNA or protein sequences.`,
    options: ["less widely, slowly evolving", "more widely, slowly evolving", "more widely, rapidly evolving", "less widely, rapidly evolving"],
    correct: "b",
    explanation: `If the phylogenetic relationships to be delineated are at the deepest level, such as between bacteria and eukaryotes, using conserved protein sequences makes more sense than using nucleotide sequences.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `In many cases ______ sequences are preferable to ______ sequences because they are relatively ____ conserved.`,
    options: ["protein, nucleotide, less", "nucleotide, protein, less", "protein, nucleotide, more", "nucleotide, protein, more"],
    correct: "c",
    explanation: `Protein sequences are preferable to nucleotide sequences because protein sequences are relatively more conserved as a result of the degeneracy of the genetic code in which sixty-one codons encode for twenty amino acids, meaning thereby a change in a codon may not result in a change in amino acid.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Protein sequences can remain the same while the corresponding DNA sequences have more room for variation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The protein sequences can remain the same while the corresponding DNA sequences have more room for variation, especially at the third codon position. The significant difference in evolutionary rates among the three nucleotide positions also violates one of the assumptions of tree-building. In contrast, the protein sequences do not suffer from this problem, even for divergent sequences.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `DNA sequences are sometimes more biased than protein sequences because of preferential codon usage in different organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In this case, different codons for the same amino acid are used at different frequencies, leading to sequence variations not attributable to evolution. In addition, the genetic code of mitochondria varies from the standard genetic code. Therefore, for comparison of mitochondria protein-coding genes, it is necessary to translate the DNA sequences into protein sequences.`,
    super_topic: `Molecular Phylogenetics`,
  },
  {
    question: `Which of the following is untrue about distance based methods?`,
    options: ["The computed evolutionary distances can be used to construct a matrix of distances between all individual pairs of taxa", "Clustering is the only method among the algorithms for the distance-based tree-building method", "The clustering-type algorithms compute a tree based on a distance matrix starting from the most similar sequence pairs", "Based on the pairwise distance scores in the matrix, a phylogenetic tree can be constructed for all the taxa involved"],
    correct: "b",
    explanation: `The algorithms for the distance-based tree-building method can be subdivided into either clustering based or optimality based. These algorithms include an unweighted pair group method using arithmetic average (UPGMA) and neighbor joining. The optimality-based algorithms compare many alternative tree topologies and select one that has the best fit between estimated distances in the tree and the actual evolutionary distances.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue about the Unweighted Pair Group Method Using Arithmetic Average?`,
    options: ["The simplest clustering method is UPGMA, which builds a tree by a sequential clustering method", "Given a distance matrix, it starts by grouping two taxa with the largest pairwise distance in the distance matrix", "The distances between this new composite taxon and all remaining taxa are calculated to create a reduced matrix", "The grouping process is repeated and another newly reduced matrix is created"],
    correct: "b",
    explanation: `It starts by grouping two taxa with the smallest pairwise distance in the distance matrix. A node is placed at the midpoint or half distance between them. It then creates a reduced matrix by treating the new cluster as a single taxon.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The basic assumption of the UPGMA method is that all taxa evolve at a constant rate and that they are equally distant from the root, implying that a molecular clock is in effect.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `However, real data rarely meet this assumption. Thus, UPGMA often produces erroneous tree topologies. However, owing to its fast speed of calculation, it has found extensive usage in clustering analysis of DNA microarray data.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In the Neighbor Joining step, The UPGMA method uses unweighted distances and assumes that all taxa have constant evolutionary rates.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Since this molecular clock assumption is often not met in biological sequences, to build a more accurate phylogenetic trees, the neighbor joining (NJ) method can be used, which is somewhat similar to UPGMA in that it builds a tree by using stepwise reduced distance matrices. However, the NJ method does not assume the taxa to be equidistant from the root.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Corrects for unequal evolutionary rates between sequences by using a conversion step. This conversion requires the calculations of “r-values” and “transformed r-values” using the following formula ______`,
    options: ["dAB\u2019 = dAB \u2212 1/4 \u00d7 (rA + rB)", "dAB\u2019 = dAB \u2212 1/2 \u00d7 (rA + rB)", "dAB\u2019 = dAB \u2212 1/3 \u00d7 (rA + rB)", "dAB\u2019 = (dAB/3) \u2212 1/2 \u00d7 (rA + rB)"],
    correct: "b",
    explanation: `AB is the converted distance between A and B and dAB is the actual evolutionary distance between A and B. The value of rsub>A (or rB) is the sum of distances of A (or B) to all other taxa.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `A generalized expression of the r-value is ri calculated based on the following formula _______`,
    options: ["ri = \u2211dij + dj2", "ri = \u2211dij", "ri = \u2211dij + di", "ri = \u2211dij + dj"],
    correct: "b",
    explanation: `i and j are two different taxa. The r-values are needed to create a modified distance matrix. The transformed r-values (r ‘) are used to determine the distances of an individual taxon to the nearest node: r i2 = ri/ (n−2).`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The tree construction process is somewhat similar to that used UPGMA.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Rather than building trees from the closest pair of branches and progressing to the entire tree, the NJ tree method begins with a completely unresolved star tree by joining all taxa onto a single node and progressively decomposes the tree by selecting pairs of taxa based on the above modified pairwise distances. This allows the taxa with the shortest corrected distances to be joined first as a node.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue about the Optimality-Based Methods?`,
    options: ["The clustering-based methods produce multiple trees as output", "Optimality-based methods select a tree that best fits the actual evolutionary distance matrix", "There is no criterion in judging how this tree is compared to other alternative trees", "Optimality-based methods have a well-defined algorithm to compare all possible tree topologies"],
    correct: "a",
    explanation: `The clustering-based methods produce a single tree as output. Based on the differences in optimality criteria, there are two types of algorithms, Fitch–Margoliash and minimum evolution, that are described next. The exhaustive search for an optimal tree necessitates a slow computation, which is a clear drawback especially when the dataset is large.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue about the Fitch–Margoliash?`,
    options: ["Method selects a best tree among all possible trees based on minimal deviation between the distances calculated in the overall branches in the tree and the distances in the original dataset", "It starts by randomly clustering two taxa in a node", "It starts by creating three equations to describe the distances", "The method searches for some specific tree topologies"],
    correct: "d",
    explanation: `It solves the three algebraic equations for unknown branch lengths. The clustering of the two taxa helps to create a newly reduced matrix. This process is iterated until a tree is completely resolved. The method searches for all tree topologies and selects the one that has the lowest squared deviation of actual distances and calculated tree branch lengths.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Minimum evolution (ME) constructs a tree with a similar procedure, but uses a different optimality criterion that finds a tree among all possible trees with a minimum overall branch length. The optimality criterion relies on the formula S = ∑bi where bi is the (i)th branch length.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Searching for the minimum total branch length is an indirect approach to achieving the best fit of the branch lengths with the original dataset. Analysis has shown that minimum evolution in fact slightly outperforms the least square-based FM method.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement about Character-based methods?`,
    options: ["They are also called discrete methods", "They are based directly on the sequence characters rather than on pairwise distances", "They doesn\u2019t count mutational events accumulated on the sequences", "They may avoid the loss of information when characters are converted to distances"],
    correct: "c",
    explanation: `They count mutational events accumulated on the sequences. This preservation of character information means that evolutionary dynamics of each character can be studied. Ancestral sequences can also be inferred. The two most popular character-based approaches are the maximum parsimony (MP) and maximum likelihood (ML) methods.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement about Maximum Parsimony Method?`,
    options: ["By cutting off the unnecessary variables, model development may become difficult, and there may be more chances of introducing inconsistencies, ambiguities, and redundancies, hence, the name Occam\u2019s razor", "In dealing with problems that may have an infinite number of possible solutions, choosing the simplest model may help to \u2018cut off\u2019 those variables that are not really necessary to explain the phenomenon", "This method chooses a tree that has the fewest evolutionary changes or shortest overall branch lengths", "It is based on a principle related to a medieval philosophy called Occam\u2019s razor"],
    correct: "a",
    explanation: `The theory was formulated by William of Occam in the thirteenth century and states that the simplest explanation is probably the correct one. This is because the simplest explanation requires the fewest assumptions and the fewest leaps of logic.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement about Building Work of MP tree?`,
    options: ["It works by searching for all possible tree topologies and reconstructing ancestral sequences that require the minimum number of changes to evolve to the current sequences", "Other than informative sites are non-informative, which are constant sites or sites that have changes occurring only once", "Informative sites are the ones that can often be explained by a unique tree topology", "Constant sites have the same state in all taxa and are quite useful in evaluating the various topologies"],
    correct: "d",
    explanation: `Constant sites have the same state in all taxa and are obviously useless in evaluating the various topologies. The sites that have changes occurring only once are not very useful either for constructing parsimony trees because they can be explained by multiple tree topologies. The non-informative sites are thus discarded in parsimony tree construction.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Because these ancestral character states are not known directly, multiple possible solutions may exist. In this case, the parsimony principle applies to choose the character states that result in a minimum number of substitutions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The inference of an ancestral sequence is made by first going from the leaves to internal nodes and to the common root to determine all possible ancestral character states. Then it goes back from the common root to the leaves to assign ancestral sequences that require the minimum number of substitutions.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The unweighted method treats all mutations as equivalent.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This may be an oversimplification; mutations of some sites are known to occur less frequently than others, for example, transversions versus transitions, functionally important sites versus neutral sites. Therefore, a weighting scheme that takes into account the different kinds of mutations helps to select tree topologies more accurately. The MP method that incorporates a weighting scheme is called weighted parsimony.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement about Tree-Searching Methods?`,
    options: ["The choice of the first three taxa can be random", "Parsimony method examines all possible tree topologies to find the maximally parsimonious tree", "It starts by building a three taxa unrooted tree, for which only one topology is available", "This is different than exhaustive search method"],
    correct: "d",
    explanation: `This is an exhaustive search method. The next step is to add a fourth taxon to the existing branches, producing three possible topologies. The remaining taxa are progressively added to form all possible tree topologies. Obviously, this brute-force approach only works if there are relatively few sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement about branch-and-bound?`,
    options: ["It uses a shortcut to find an MP tree", "It establishes an upper limit (or upper bound) for the number of allowed sequence variations", "It solely uses UPGMA method", "It starts by building a distance tree for all taxa involved"],
    correct: "c",
    explanation: `It starts by building a distance tree for all taxa involved using either NJ or UPGMA and then computing the minimum number of substitutions for this tree. The resulting number defines the upper bound to which any other trees are compared. The rationale is that a maximally parsimonious tree must be equal to or shorter than the distance-based tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The branch-and-bound method starts building trees in a similar way as in the exhaustive method.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The difference is that the previously established upper bound limits the tree growth. Whenever the overall tree length at every single stage exceeds the upper bound, the topology search toward a particular direction aborts. By doing so, it dramatically reduces the number of trees considered hence the computing time while at the same time guaranteeing to find the most parsimonious tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `When the number of taxa exceeds twenty, even the branch-and-bound method becomes computationally unfeasible.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A more heuristic search method must be used. A computer heuristic procedure is an approximation strategy to find an empirical solution for a complicated problem. This strategy generates quick answers, but not necessarily the best answer.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In a heuristic tree search, only a small subset of all possible trees is examined.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This method starts by carrying out a quick initial approximation, which is to build an NJ tree and subsequently modifying it slightly into a different topology to see whether that leads to a shorter tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect about Bootstrapping?`,
    options: ["It is a statistical technique that tests the sampling errors of a phylogenetic tree", "It does the tests by repeatedly sampling trees through slightly perturbed datasets", "A newly constructed tree is not biased at all", "The robustness of the original tree can be assessed here"],
    correct: "c",
    explanation: `The rationale for bootstrapping is that a newly constructed tree is possibly biased owing to incorrect alignment or chance fluctuations of distance measurements. To determine the robustness or reproducibility of the current tree, trees are repeatedly constructed with slightly perturbed alignments that have some random fluctuations introduced.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `A truly robust phylogenetic relationship should have enough characters to support the relationship even if the dataset is perturbed in such away.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Otherwise, the noise introduced in the resampling process is sufficient to generate different trees, indicating that the original topology may be derived from weak phylogenetic signals. Thus, this type of analysis gives an idea of the statistical confidence of the tree topology.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect about nonparametric bootstrapping?`,
    options: ["All the bootstrapped trees are summarized into a consensus tree based on a majority rule", "The most supported branching patterns shown at each node are labeled with bootstrap values", "The most supported branching patterns are the percentage of appearance of a particular clade", "This test doesn\u2019t provide a measure for evaluating the confidence levels of the tree topology"],
    correct: "d",
    explanation: `The bootstrap test provides a measure for evaluating the confidence levels of the tree topology. Analysis has shown that a bootstrap value of 70% approximately corresponds to 95% statistical confidence, although the issue is still a subject of debate.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect about Caveats?`,
    options: ["Unusually high GC content in the original dataset is the potential cause for generating biased trees", "Unusually accelerated evolutionary rates is the potential cause for generating biased trees", "Unusually accelerated evolutionary rates is the potential cause for generating biased bootstrap estimates", "Not a large number of bootstrap re-sampling steps are needed to achieve yielding results"],
    correct: "d",
    explanation: `In addition, from a statistical point of view, a large number of bootstrap resampling steps are needed to achieve meaningful results. It is generally recommended that a phylogenetic tree should be bootstrapped 500 to 1,000 times. However, this presents a practical dilemma.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect statement?`,
    options: ["In this method one half of the sites in a dataset are randomly deleted", "It creates datasets half as long as the original", "Each new dataset is subjected to phylogenetic tree construction using the different methods as the original", "One criticism of this approach is that the size of datasets has been changed into one half and that the datasets are no longer considered replicates"],
    correct: "c",
    explanation: `Each new dataset is subjected to phylogenetic tree construction using the same method as the original. The advantage of jackknifing is that sites are not duplicated relative to the original dataset and that computing time is much shortened because of shorter sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is incorrect about Bayesian Simulation?`,
    options: ["It does not require bootstrapping", "It requires bootstrapping", "The MCMC procedure itself involves thousands or millions of steps of resampling", "Posterior probabilities are assigned at each node of a best Bayesian tree as statistical support"],
    correct: "b",
    explanation: `Because of fast computational speed of MCMC tree searching, the Bayesian method offers a practical advantage over regular ML and makes the statistical evaluation of ML trees more feasible. Unlike bootstrap values, Bayesian probabilities are normally higher because most trees are sampled near a small number of optimal trees. Therefore, they have a different statistical meaning from bootstrap.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In phylogenetic analysis, it is also important to test whether two competing tree topologies can be distinguished and whether one tree is significantly better than the other.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The task is different from bootstrapping in that it tests the statistical significance of the entire phylogeny, not just portions of it. For that purpose, several statistical tests have been developed specifically for each of the three types of tree reconstruction methods, distance, parsimony, and likelihood. A test devised specifically for MP trees is called the Kishino–Hasegawa (KH) test.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The KH test sets out to test the null hypothesis that the two competing tree topologies are not significantly different.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A paired Student t-test is used to assess whether the null hypothesis can be rejected at a statistically significant level. In this test, the difference of branch lengths at each informative site between the two trees is calculated.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In Shimodaira–Hasegawa Test, The degree of freedom used for the analysis depends on the substitution model used. It relies on the following test formula d = 2(ln LA – ln LB) = 2 ln(LA/LB). Here, is the log likelihood ratio score and ln LA and ln LB are likelihood scores for tree A and tree B, respectively.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A frequently used statistical test for ML trees is the Shimodaira–Hasegawa (SH) test (likelihood ratio test). It tests the goodness of fit of two competing trees using the χ2 test. For this test, log likelihood scores of two competing trees have to be obtained first.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `PAUP is a Macintosh program (UNIX version available in the GCG package) with a very user-friendly graphical interface.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It stands for Phylogenetic analysis using parsimony. It is a commercial phylogenetic package. It is probably one of the most widely used phylogenetic programs available from Sinauer Publishers. PAUP was originally developed as parsimony program, but expanded to a comprehensive package that is capable of performing distance, parsimony, and likelihood analyses.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In PAUP, The distance options include NJ, ME, FM, and UPGMA.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For distance or ML analyses, PAUP has the option for detailed specifications of substitution models, base frequencies, and among site rate heterogeneity (γ -shape parameters, proportion of invariant sites). PAUP is also able to perform nonparametric bootstrapping, jackknifing, KH testing, and SH testing.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Phylip stands for Phylogenetic inference package (by Joe Felsenstein).`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Is a free multiplatform comprehensive package containing thirty-five subprograms for performing distance, parsimony, and likelihood analysis, as well as bootstrapping for both nucleotide and amino acid sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In PAUP, to complete an analysis the user is not required to move between different subprograms while keeping modifying names of the intermediate output files.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The only problem is that to complete an analysis the user is required to move between different subprograms while keeping modifying names of the intermediate output files. It is command-line based, but relatively easy to use for each single program.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding TREE-PUZZLE?`,
    options: ["The resulting puzzle trees are automatically assigned puzzle support values to internal branches", "The support values are percentages of consistent quartet trees", "The support values do not have the same meaning as bootstrap values", "The support values have the same meaning as bootstrap values"],
    correct: "d",
    explanation: `Because of the heuristic nature of the program, it allows ML analyses of large datasets. TREE-PUZZLE version 5.0 is available for Mac, UNIX, and Windows.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `PHYML is a web-based _______ program using the _____`,
    options: ["phylogenetic, GA (Genetic Algorithm)", "sequence based alignment, GA (Genetic Algorithm)", "phylogenetic, dynamic programming", "sequence based alignment, dynamic programming"],
    correct: "a",
    explanation: `It first builds an NJ tree. Further it uses it as a starting tree for subsequent iterative refinement through subtree swapping. Branch lengths are simultaneously optimized during this process.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In PHYML, The tree searching _____ when the total ML score no longer ______`,
    options: ["ceases, increases", "stops, decreases", "terminates, decreases", "stops, increases"],
    correct: "d",
    explanation: `PHYML is a web-based phylogenetic program using the GA. The main advantage of this program is the ability to build trees from very large datasets with hundreds of taxa and to complete tree searching within a relatively short time frame.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `MrBayes is a Bayesian phylogenetic inference program.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It randomly samples tree topologies using the MCMC procedure. Next it infers the posterior distribution of tree topologies.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `MrBayes has a range of probabilistic models available to search for a set of trees with the lowest posterior probability.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `MrBayes has a range of probabilistic models available to search for a set of trees with the highest posterior probability. It is fast and capable of handling large datasets. The program is available in multi platform versions. A web program that also employs Bayesian inference for phylogenetic analysis is BAMBE.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding the maximum parsimony method?`,
    options: ["The analysis steps are continued for every position in the sequence alignment", "This method is used for large numbers of sequences", "Those trees that produce the smallest number of changes overall for all sequence positions are identified", "This method is used for sequences that are quite similar"],
    correct: "b",
    explanation: `The algorithm followed is not particularly complicated, but it is guaranteed to find the best tree, because all possible trees relating a group of sequences are examined. For this reason, the method is quite time-consuming and is not useful for data that include a large number of sequences or sequences with a large amount of variation.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding the programs for analysis of nucleic acid sequences?`,
    options: ["DNAPARS treats gaps as a fifth nucleotide state", "DNAPENNY performs parsimonious phylogenies by branch-and-bound search", "DNAPENNY can analyze sequences up to 11 or 12", "Compatibility criterion is not involved in DNACOMP"],
    correct: "d",
    explanation: `DNACOMP performs phylogenetic analysis using the compatibility criterion.
Rather than searching for overall parsimony at all sites in the multiple sequence alignment, this method finds the tree that supports the largest number of sites. This method is recommended when the rate of evolution varies among sites.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `PROTPARS counts the minimum number of mutations to change a codon for the first amino acid into a codon for the second amino acid, but only scores those mutations in the mutational path that actually change the amino acid.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `PROTPARS is used For analysis of protein sequences. As mentioned, Silent mutations that do not change the amino acid are not scored on the grounds that they have little evolutionary significance.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Parsimony can give ________ information when rates of sequence change _______ in the different branches of a tree that are represented by the sequence data.`,
    options: ["misleading, vary", "useful, change", "misleading, are constant", "sometimes contradicting, are constant"],
    correct: "a",
    explanation: `These variations produce a range of branch lengths, long ones representing more extended periods of time and short ones representing shorter times. Although other columns in the sequence alignment that show less variation may provide the correct tree, the columns representing greater variation dominate the analysis.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding the distance methods?`,
    options: ["The distance method was pioneered by Feng and Doolittle", "A collection of programs by authors Feng and Doolittle will produce both an alignment and tree of a set of protein sequences", "The program CLUSTALW uses the neighbor-joining distance method as a guide to multiple sequence alignments", "Among the Programs of the PHYLIP package, DNADIST is not one of them"],
    correct: "d",
    explanation: `DNADIST and PROTDIST are the Programs of the PHYLIP package that perform a distance analysis. They automatically read in a sequence in the PHYLIP in file format and automatically produce a file called outfile with a distance table.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding the Distance analysis programs in PHYLIP?`,
    options: ["FITCH estimates a phylogenetic tree assuming additivity of branch lengths", "FITCH uses the Fitch-Margoliash method", "FITCH assumes a molecular clock but KITSCH does not", "NEIGHBOR estimates phylogenies using the neighbor-joining or unweighted pair group method with arithmetic mean (UPGMA)"],
    correct: "c",
    explanation: `KITSCH assumes a molecular clock but FITCH does not. Also, in NEIGHBOR the neighbor-joining method does not assume a molecular clock and produces an unrooted tree. The UPGMA method assumes a molecular clock and produces a rooted tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is untrue regarding the neighbor-joining method?`,
    options: ["It is very much like the Fitch-Margoliash method", "It is totally dissimilar than the Fitch-Margoliash method", "It is especially suitable when the rate of evolution of the separate lineages under consideration varies", "When the branch lengths of trees of known topology are allowed to vary in a manner that simulates varying levels of evolutionary change, it is the most reliable method"],
    correct: "b",
    explanation: `The neighbor-joining method is very much like the Fitch-Margoliash method except that the choice as to which sequences to pair is determined by a different algorithm. In the situation mentioned in option d, the neighbor-joining method and the Sattath and Taversky method, are the most reliable in predicting the correct tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Neighbor-joining chooses the sequences that should be joined to give the best least squares estimates of the branch lengths that most closely reflect the actual distances between the sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is not necessary to compare all possible trees to find the least squares fit as in the Fitch-Margoliash method. The method pairs sequences based on the effect of the pairing on the sum of the branch lengths of the tree.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is wrong statement about the maximum likelihood approach?`,
    options: ["This method doesn\u2019t always involve probability calculations", "It finds a tree that best accounts for the variation in a set of sequences", "The method is similar to the maximum parsimony method", "The analysis is performed on each column of a multiple sequence alignment"],
    correct: "a",
    explanation: `This method involve probability calculations to find a tree that best accounts for the variation in a set of sequences. All possible trees are considered. Hence, the method is only feasible for a small number of sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In about the maximum likelihood approach, for each tree, the number of sequence changes or mutations that may have occurred to give the sequence variation is considered.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Because the rate of appearance of new mutations is very small, the more mutations needed to fit a tree to the data, the less likely that tree (Felsenstein 1981). The maximum likelihood method resembles the maximum parsimony method in that trees with the least number of changes will be the most likely.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The maximum likelihood method can be used to explore relationships among more diverse sequences, conditions that are not well handled by maximum parsimony methods.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The maximum likelihood method presents an additional opportunity to evaluate trees with variations in mutation rates in different lineages. Also it provides opportunity to use explicit evolutionary models such as the Jukes-Cantor and Kimura models with allowances for variations in base composition.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The main disadvantage of maximum likelihood methods is that they are _____`,
    options: ["mathematically less folded", "mathematically less complex", "computationally lucid", "computationally intense"],
    correct: "d",
    explanation: `The main disadvantage of maximum likelihood methods is that they are computationally intense. However, with faster computers, the maximum likelihood method is seeing wider use and is being used for more complex models of evolution.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Maximum likelihood has also been used for an analysis of mutations in overlapping reading frames in viruses.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `PAUP version 4 can be used to perform a maximum likelihood analysis on DNA sequences. The method has also been applied for changes from one amino acid to another in protein sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is wrong statement about DNAML and DNAMLK?`,
    options: ["PHYLIP includes mentioned two programs for this maximum likelihood analysis", "DNAML estimates phylogenies from nucleotide sequences by the maximum likelihood method", "DNAMLK estimates phylogenies in the same manner as DNAML", "DNAMLK estimates phylogenies without molecular clock"],
    correct: "d",
    explanation: `DNAMLK estimates phylogenies from nucleotide sequences by the maximum likelihood method in the same manner as DNAML, but assumes a molecular clock. DNAML allows for variable frequencies of the four nucleotides, for unequal rates of transitions and transversions, and for different rates of change in different categories of sites, as specified by the program.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is wrong statement about the maximum likelihood method’s steps?`,
    options: ["It starts with an evolutionary model of sequence change that provides estimates of rates of substitution of one base for another", "In the beginning there is an evolutionary model of sequence change that provides estimates of transitions and transversions in a set of nucleic acid sequences", "The rates of all possible substitutions are chosen so that the base composition differs", "The set of sequences is then aligned"],
    correct: "c",
    explanation: `The rates of all possible substitutions are chosen so that the base composition remains the same. The set of sequences is then aligned, and the substitutions in each column are examined for their fit to a set of trees that describe possible phylogenetic relationships among the sequences.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Once all positions in the sequence alignment have been examined, the likelihoods given by each column in the alignment for each tree are _____ to give the likelihood of the tree.`,
    options: ["multiplied", "added", "divided", "squared"],
    correct: "a",
    explanation: `Because these likelihoods are very small numbers, their logarithms are usually added to give the logarithm likelihood of each tree. The most likely tree given the data is then identified.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `A method of sequence alignment based on a Model (Bishop and Thompson 1986) was introduced that predicts the manner in which DNA sequences change during evolution. Which of the following is wrong about it?`,
    options: ["This method is different from the Smith-Waterman local alignment algorithm", "This method is quite similar to the Smith-Waterman local alignment algorithm", "The underlying mutational theory is like those used to produce the PAM matrices for predicting changes in DNA and protein sequences", "Sequences are predicted to change by a Markov process such that each mutation in the sequence is independent of previous mutations at that site or at other sites"],
    correct: "b",
    explanation: `This method is different from the Smith-Waterman local alignment algorithm in identifying the most probable (maximum likelihood probability alignment) based on an evolutionary model of change in sequences, as opposed to a score based on observed substitutions in related proteins and a gap scoring system. An example for option d can be–a given nucleotide at any sequence position can mutate into another at the same rate or may not change at all during a period of evolutionary time.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Phylogenetic analysis of a set of sequences that aligns ______ is straightforward because the positions that correspond in the sequences can be readily identified in a ______ of the sequences.`,
    options: ["very well, multiple sequence alignment", "in a haphazard manner, multiple sequence alignment", "in a distorted way, multiple sequence alignment", "very well, self alignment"],
    correct: "a",
    explanation: `Option “very well, self alignment” here, becomes irrelevant as there is phylogenetic analysis involved. The types of changes in the aligned positions or the numbers of changes in the alignments between pairs of sequences then provide a basis for a determination of phylogenetic relationships among the sequences by the above methods of phylogenetic analysis.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `For sequences that have ________ a phylogenetic analysis is __________`,
    options: ["diverged considerably, more challenging", "not diverged, more challenging", "diverged considerably, less challenging", "diverged considerably, a less work to do"],
    correct: "a",
    explanation: `For diverged sequences, the analysis steps increase as well. A determination of the sequence changes that have occurred is more difficult because the multiple sequence alignment may not be optimal and because multiple changes may have occurred in the aligned sequence positions.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The choice of a suitable multiple sequence alignment method depends on the degree of variation among the sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The degree of variation also sometimes affecs the efficiency and nature of output. Once a suitable alignment has been found, one may also ask how well the predicted phylogenetic relationships are supported by the data in the multiple sequence alignment.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In the bootstrap method, the data are resampled by _____ choosing _____ columns from the aligned sequences to produce, in effect, a new sequence alignment of the _____`,
    options: ["randomly, horizontal, same length", "specifically, vertical, different lengths", "randomly, vertical, same length", "randomly, vertical, different lengths"],
    correct: "c",
    explanation: `Each column of data may be used more than once and some columns may not be used at all in the new alignment. Trees are then predicted from many of these alignments of resampled sequences (Felsenstein 1988).`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In the bootstrap method, for branches in the predicted tree topology to be significant, the resampled data sets should frequently predict the same branches.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Bootstrap analysis is supported by most of the commonly used phylogenetic inference software packages and is commonly used to test tree branch reliability. Another method of testing the reliability of one part of the tree is to collapse two branches into a common node (Maddison and Maddison 1992).`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `In the final steps of the bootstrap method, the _____ the decay value, the _______ significant the original branches.`,
    options: ["greater, less", "greater, more", "lesser, more", "more, less"],
    correct: "b",
    explanation: `In the final steps of the bootstrap method,The tree length is again evaluated and compared to the original length, and any increase is the decay value. The greater the decay value, the more significant the original branches. In addition to these methods, there are some additional recommendations that increase confidence in a phylogenetic prediction.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `A common recommendation is to use at least two of the methods—maximum parsimony, distance, or maximum likelihood, for the analysis.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If two of these methods provide the same prediction, confidence in the prediction is much higher. Another recommendation is to pay careful attention to the evolutionary assumptions and models that are used for both sequence alignment and tree construction.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `The traditional use of phylogenetic analysis is to discover evolutionary relationships among species.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In such cases, a suitable gene or DNA sequence that shows just enough, but not too much, variation among a group of organisms is selected for phylogenetic analysis. For example, analysis of mitochondrial sequences is used to discover evolutionary relationships among mammals.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Two more recent uses of phylogenetic analysis are to analyze ______ and to trace the evolutionary history of specific genes. Which of the following could not be the correct blank?`,
    options: ["gene families", "genomes", "proteomes", "physical separation methods"],
    correct: "d",
    explanation: `Options here, indicate the laboratory operations unlike the computational data mentioned in other options. For example, database similarity searches may identify several proteins in a plant genome that are similar to a yeast query protein.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Tracking the evolutionary history of individual genes in a group of species can reveal which genes have remained in a genome for a long time and which genes have been horizontally transferred between species.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Thus, phylogenetic analysis can also contribute to an understanding of genome evolution. Or e.g. from a phylogenetic analysis of the protein family, the plant gene most closely related to the yeast gene and therefore most likely to have the same function can be determined.`,
    super_topic: `Phylogenetic Tree Construction`,
  },
  {
    question: `Which of the following is true regarding the methods of gene prediction?`,
    options: ["They solely consist of a type called ab initio\u2013based methods", "The ab initio\u2013based approach predicts genes based on the given sequence alone", "The ab initio\u2013based approach predicts genes based on the given sequence and relative homology data", "They solely consist of a type called homology-based approaches"],
    correct: "b",
    explanation: `The current gene prediction methods can be classified into two major categories, ab initio–based and homology-based approaches. The ab initio–based approach predicts genes based on the given sequence alone.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `In the ab initio–based approaches—they rely on two major features associated with genes: one of them being the existence of gene signals, which include start and stop codons, intron splice signals, transcription factor binding sites, etc.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `They also include ribosomal binding sites, and polyadenylation (poly-A) sites. In addition, the triplet codon structure limits the coding frame length to multiples of three, which can be used as a condition for gene prediction.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `In the ab initio–based approaches—they rely on two major features associated with genes: one of them being gene content, which is statistical description of coding regions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It has been observed that nucleotide composition and statistical patterns of the coding regions tend to vary significantly from those of the non-coding regions. The unique features can be detected by employing probabilistic models such as Markov models or hidden Markov models to help distinguish coding from non-coding regions.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The homology-based method makes predictions based on significant matches of the query sequence with sequences of known genes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For instance, if a translated DNA sequence is found to be similar to a known protein or protein family from a database search, this can be strong evidence that the region codes for a protein. Alternatively, when possible exons of a genomic DNA region match a sequenced cDNA, this also provides experimental evidence for the existence of a coding region.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `FGENESB is a web-based program that is also based on fifth-order HMMs for detecting coding regions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The program is specifically trained for bacterial sequences. It uses the Vertibi algorithm to find an optimal match for the query sequence with the intrinsic model. A linear discriminant analysis (LDA) is used to further distinguish coding signals from non-coding signals.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue about GeneMark?`,
    options: ["It is a suite of gene prediction programs based on the fifth-order HMMs", "The main program is trained on a number of complete microbial genomes", "A GeneMark heuristic program can be used to improve accuracy", "If the sequence to be predicted is from a non-listed organism, the most closely related organism can be chosen as the basis for computation"],
    correct: "c",
    explanation: `Another option for predicting genes from a new organism is to use a self-trained program GeneMarkS as long as the user can provide at least 100 kbp of sequence on which to train the model. If the query sequence is shorter than 100 kbp, a GeneMark heuristic program can be used with some loss of accuracy. In addition to predicting prokaryotic genes, GeneMark also has a variant for eukaryotic gene prediction using HMM.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue about Glimmer?`,
    options: ["It stands for Gene Locator and Interpolated Markov Modeler", "It is a UNIX program from TIGR", "It does not necessarily use the IMM algorithm", "It is used to predict potential coding regions"],
    correct: "c",
    explanation: `The computation consists of two steps, namely model building and gene prediction. The model building involves training by the input sequence, which optimizes the parameters of the model. In an actual gene prediction, the overlapping frames are “flagged” to alert the user for further inspection. Glimmer also has a variant, GlimmerM, for eukaryotic gene prediction.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `RBS finder is a UNIX program that uses the prediction output from Glimmer and searches for the Shine–Delgarno sequences in the vicinity of predicted start sites.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A high-scoring site is found by the intrinsic probabilistic model, a start codon is confirmed. Otherwise the program moves to other putative translation start sites and repeats the process.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `In bacteria, the majority of genes have a start codon ATG (orAUG in mRNA; because prediction is done at the DNA level, T is used in place of U), which codes for methionine.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Occasionally, GTG and TTG are used as alternative start codons. But methionine is still the actual amino acid inserted at the first position.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The presence of these codons at The beginning of the frame _____ give a clear indication of the translation initiation site.`,
    options: ["always", "does not necessarily", "does not", "never"],
    correct: "b",
    explanation: `Because there may be multiple ATG, GTG, or TGT codons in a frame, the presence of these codons at the beginning of the frame does not necessarily give a clear indication of the translation initiation site. Instead, to help identify this initiation codon, other features associated with translation are used.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Shine-Delgarno sequence, which is a stretch of purine-rich sequence complementary to 16S rRNA in the ribosome.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is located immediately downstream of the transcription initiation site and slightly upstream of the translation start codon. In many bacteria, it has a consensus motif of AGGAGGT. Identification of the ribosome binding site can help locate the start codon.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `There are ____ possible stop codons, identification of which is straightforward.`,
    options: ["five", "two", "ten", "three"],
    correct: "d",
    explanation: `At the end of the protein coding region is a stop codon that causes translation to stop. There are three possible stop codons, identification of which is straightforward. Many prokaryotic genes are transcribed together as one operon.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is a wrong statement regarding the conventional determination of open reading frames?`,
    options: ["Without the use of specialized programs, prokaryotic gene identification can rely on manual determination of ORFs and major signals related to prokaryotic genes", "Prokaryotic DNA is first subject to conceptual translation in all six possible frames, two frames forward and four frames reverse", "A stop codon occurs in about every twenty codons by chance in a noncoding region", "Prokaryotic DNA is first subject to conceptual translation in all six possible frames, three frames forward and three frames reverse"],
    correct: "b",
    explanation: `Prokaryotic DNA is first subject to conceptual translation in all six possible frames, three frames forward and three frames reverse. Because a stop codon occurs in about every twenty codons by chance in a noncoding region, a frame longer than thirty codons without interruption by stop codons is suggestive of a gene coding region, although the threshold for an ORF is normally set even higher at fifty or sixty codons.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The putative ORF can be translated into a protein sequence, which is then used to search against a protein database.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The putative frame is further manually confirmed by the presence of other signals such as a start codon and Shine–Delgarno sequence. Detection of homologs from this search is probably the strongest indicator of a protein-coding frame.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is a wrong statement regarding TESTCODE method?`,
    options: ["This is based on the nucleotide composition of the third position of a codon", "In practice, because genes can be in any of the six frames, the statistical patterns are computed for all possible frames", "It is implemented in the commercial GCG package", "It exploits the fact that the third codon nucleotides in a coding region fails to repeat themselves"],
    correct: "d",
    explanation: `In a coding sequence, it has been observed that this position has a preference to use G or C over A or T. By plotting the GC composition at this position, regions with values significantly above the random level can be identified, which are indicative of the presence of ORFs. This method exploits the fact that the third codon nucleotides in a coding region tend to repeat themselves.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The conventional determination of open reading methods identify only typical genes and tend to miss atypical genes in which the rule of codon bias is not strictly followed.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These statistical methods, which are based on empirical rules, examine the statistics of a single nucleotide (either G or C). To improve the prediction accuracies, the new generation of prediction algorithms uses more sophisticated statistical models.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is a wrong statement regarding Gene Prediction Using Markov Models and Hidden Markov Models?`,
    options: ["Markov models and HMMs can be very helpful in providing finer statistical description of a gene", "A Markov model describes the probability of the distribution of nucleotides in a DNA sequence", "In a Markov model the conditional probability of a particular sequence position depends on k alternate positions", "A zero-order Markov model assumes each base occurs independently with a given probability"],
    correct: "c",
    explanation: `In a Markov model the conditional probability of a particular sequence position depends on k previous positions. In this case, k is the order of a Markov model. In a zero-order Markov model, it is often the case for noncoding sequences. A first-order Markov model assumes that the occurrence of a base depends on the base preceding it. A second-order model looks at the preceding two bases to determine which base follows, which is more characteristic of codons in a coding sequence.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The use of Markov models in gene finding exploits the fact that oligonucleotide distributions in the coding regions are different from those for the noncoding regions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These can be represented with various orders of Markov models. Since a fixed-order Markov chain describes the probability of a particular nucleotide that depends on previous k nucleotides, the longer the oligomer unit, the more non-randomness can be described for the coding region. Therefore, the higher the order of a Markov model, the more accurately it can predict a gene.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Because a protein-encoding gene is composed of nucleotides in triplets as codons, more effective Markov models are built in sets of three nucleotides, describing nonrandom distributions of trimers or hexamers, and so on.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The parameters of a Markov Model have to be trained using a set of sequences with known gene locations. Once the parameters of the model are established, it can be used to compute the nonrandom distributions of trimers or hexamers in a new sequence to find regions that are compatible with the statistical profiles in the learning set.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue?`,
    options: ["Many entries in the Protein DataBank (PDB) are three-dimensional structures of multiple domains", "The structures in PDB provide experimental information about interactions between domains at atomic detail", "There are comparatively few three-dimensional structures compared to the amount of data available from the lower resolution large-scale experiments", "Many entries in the Protein DataBank (PDB) are two-dimensional structures of multiple domains"],
    correct: "d",
    explanation: `Analysis of structures consisting of multiple domains has uncovered some of the principles of domain interactions in three dimensions. This information can therefore be complementary to the experimental data on protein interactions and to the predicted interactions.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Which of the following is untrue about translation and transcription?`,
    options: ["The first is capping is at the 5\u2019 end of the transcript which involves methylation at the initial residue of the RNA", "The splicing process involves a large RNA-protein complex called spliceosome", "The second event is splicing, which is the process of removing exons and joining introns", "The second event is splicing, which is the process of removing introns and joining exons"],
    correct: "c",
    explanation: `The reaction requires intermolecular interactions between a pair of nucleotides at each end of an intron and the RNA component of the spliceosome. To make the matter even more complex, some eukaryotic genes can have their transcripts spliced and joined in different ways to generate more than one transcript per gene. This is the phenomenon of alternative splicing.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `The main issue in prediction of eukaryotic genes is the identification of exons, introns, and splicing sites.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `From a computational point of view, it is a very complex and challenging problem. Because of the presence of split gene structures, alternative splicing, and very low gene densities, the difficulty of finding genes in such an environment is likened to finding a needle in a haystack.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Most vertebrate genes use __________ as the translation start codon and have a uniquely conserved flanking sequence call a Kozak sequence (CCGCCATGG).`,
    options: ["AAG", "ATG", "AUG", "AGG"],
    correct: "b",
    explanation: `In addition, most of these genes have a high density of CG dinucleotides near the transcription start site. This region is referred to as a CpG island (p refers to the phosphodiester bond connecting the two nucleotides), which helps to identify the transcription initiation site of a eukaryotic gene. The poly-A signal can also help locate the final coding sequence.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is untrue about Ab Initio–Based Programs for Gene Prediction?`,
    options: ["The goal of the ab initio gene prediction programs is to discriminate exons from noncoding sequences", "The goal is joining exons together in the correct order", "The main difficulty is correct identification of exons", "To predict exons, the algorithms rely solely on gene signals"],
    correct: "d",
    explanation: `To predict exons, the algorithms rely on two features, gene signals and gene content. Signals include gene start and stop sites and putative splice sites, recognizable consensus sequences such as poly-A sites.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `In Ab Initio–Based Programs for Gene Prediction– Gene content refers to coding statistics, which includes nonrandom nucleotide distribution, amino acid distribution, synonymous codon usage, and hexamer frequencies.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Among these features, the hexamer frequencies appear to be most discriminative for coding potentials. To derive an assessment for this feature, HMMscan be used, which require proper training. In addition to HMMs, neural network-based algorithms are also common in the gene prediction field.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is untrue about PredictionUsing NeuralNetworks for Gene Prediction?`,
    options: ["A neural network is a statistical model with a special architecture for pattern recognition and classification", "It is composed of a network of mathematical variables", "They resembles ab initio approaches", "The variables in NeuralNetworks resemble the biological nervous system, with variables or nodes connected by weighted functions that are analogous to synapses"],
    correct: "c",
    explanation: `Aspect of the model that makes it look like a biological neural network is its ability to “learn” and then make predictions after being trained. The network is able to process information and modify parameters of the weight functions between variables during the training stage. Once it is trained, it is able to make automatic predictions about the unknown. This is quite different than the ab initio methods.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is untrue about Prediction Using Neural Networks for Gene Prediction?`,
    options: ["A neural network is constructed with multiple layers; the input, output, and hidden layers", "The input is the gene sequence with intron and exon signals", "The model is not fed with a sequence of known gene structure", "The output is the probability of an exon structure"],
    correct: "c",
    explanation: `Between input and output, there may be one or several hidden layers where the machine learning takes place. The machine learning process starts by feeding the model with a sequence of known gene structure. The gene structure information is separated into several classes of features such as hexamer frequencies, splice sites, and GC composition during training. The weight functions in the hidden layers are adjusted during this process to recognize the nucleotide patterns and their relationship with known structures.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `GRAIL is a web-based program that is based on a neural network algorithm Which is trained on several statistical features such as splice junctions, start and stop codons, poly-A sites, promoters, and CpG islands.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The program scans the query sequence with windows of variable lengths and scores for coding potentials and finally produces an output that is the result of exon candidates. The program is currently trained for human, mouse, Arabidopsis, Drosophila, and Escherichia coli sequences.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is untrue about Prediction Using Discriminant Analysis for Gene Prediction?`,
    options: ["QDA draws a curved line based on a quadratic function", "LDA works by drawing a diagonal line that best separates coding signals from noncoding signals based on knowledge learned from training data sets of known gene structures", "Some gene prediction algorithms rely on discriminant analysis, either LDA or quadratic discriminant analysis (QDA), to improve accuracy", "LDA works by plotting a three-dimensional graph of coding signals versus all potential 3\u2019 splice site positions"],
    correct: "d",
    explanation: `QDA draws a curved line based on a quadratic function instead of drawing a straight line to separate coding and noncoding features. This strategy is designed to be more flexible and provide a more optimal separation between the data points.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is untrue about FGENES?`,
    options: ["It stands for FindGenes", "It is a web-based program that uses LDA", "It is used to determine whether a signal is an exon", "It does not make a use of HMMs"],
    correct: "d",
    explanation: `In addition to FGENES, there are many variants of the program. Some programs, such as FGENESH, make use of HMMs. There are others, such as FGENESH C, that are similarity based. Some programs, such as FGENESH+, combine both ab initio and similarity-based approaches.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `GENSCAN is awebbased program that makes predictions based on fifth-order HMMs.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It combines hexamer frequencies with coding signals (initiation codons, TATA box, cap site, poly-A, etc.) in prediction. Putative exons are assigned a probability score (P) of being a true exon. Only predictions with P > 0.5 are deemed reliable. This program is trained for sequences from vertebrates, Arabidopsis, and maize. It has been used extensively in annotating the human genome.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following wrong about HMM GENE?`,
    options: ["It is also an HMM-based web program", "It uses a criterion called the conditional maximum likelihood to discriminate coding from non-coding features", "HMM prediction is unbiased toward the locked region", "If a sequence already has a sub-region identified as coding region, which may be based on similarity with cDNAs or proteins in a database, these regions are locked as coding regions"],
    correct: "c",
    explanation: `An HMM prediction is subsequently made with a bias toward the locked region and is extended from the locked region to predict the rest of the gene coding regions and even neighboring genes. The program is in a way a hybrid algorithm that uses both ab initio-based and homology-based criteria.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue about Homology-Based Programs?`,
    options: ["They are based on the fact that exon structures and exon sequences of related species are less conserved", "This approach assumes that the database sequences are correct", "It is a reasonable assumption in light of the fact that many homologous sequences to be compared with are derived from cDNA or expressed sequence tags (ESTs) of the same species", "Potential coding frames in a query sequence are translated and used to align with closest protein homologs found in databases"],
    correct: "a",
    explanation: `Homology-based programs are based on the fact that exon structures and exon sequences of related species are highly conserved. When potential coding frames in a query sequence are translated and used to align with closest protein homologs found in databases, near perfectly matched regions can be used to reveal the exon boundaries in the query.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `The drawback of Homology-based approach is its reliance on the presence of homologs in databases.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If the homologs are not available in the database, the method cannot be used. Novel genes in a new species cannot be discovered without matches in the database. A number of publicly available programs use this approach.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `GenomeScan is a web-based server that combines GENSCAN prediction results with BLASTX similarity searches.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The user provides genomic DNA and protein sequences from related species. The genomic DNA is translated in all six frames to cover all possible exons. The translated exons are then used to compare with the user-supplied protein sequences.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue about EST2Genome?`,
    options: ["It is a web-based program purely based on the sequence alignment approach to define intron\u2013exon boundaries", "It compares an EST (or cDNA) sequence with a genomic DNA sequence containing the corresponding gene", "The alignment is rarely done using a dynamic programming\u2013based algorithm", "Advantage of the approach is the ability to find very small exons and alternatively spliced exons that are very difficult to predict by any ab initio\u2013type algorithms"],
    correct: "c",
    explanation: `The alignment is done using dynamic programming–based algorithm. Another advantage is that there is no need for model training, which provides much more flexibility for gene prediction. The limitation is that EST or cDNA sequences often contain errors or even introns if the transcripts are not completely spliced before reverse transcription.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Which of the following is untrue about SGP-1?`,
    options: ["The program translates all potential exons in each sequence and does pair wise alignment for the translated protein sequences using a dynamic programming approach", "The near-perfect matches at the protein level define coding regions", "It is a similarity-based web program that aligns two genomic DNA sequences from distinctly related organisms", "It stands for Syntenic Gene Prediction"],
    correct: "c",
    explanation: `It aligns two genomic DNA sequences from closely related organisms. Similar to EST2Genome, there is no training needed. The limitation is the need for two homologous sequences having similar genes with similar exon structures; if this condition is not met, a gene escapes detection from one sequence when there is no counterpart in another sequence.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `TwinScan is also a similarity-based gene-finding Server and it is similar to GenomeScan in that it uses GenScan to predict all possible exons from the genomic sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The putative exons are used for BLAST searching to find closest homologs. The putative exons and homologs from BLAST searching are aligned to identify the best match. Only the closest match from a genome database is used as a template for refining the previous exon selection and exon boundaries.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `Because different prediction programs have different levels of sensitivity and specificity, it makes sense to combine results of multiple programs based on consensus. This idea has prompted development of consensus-based algorithms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These programs work by retaining common predictions agreed by most programs and removing inconsistent predictions. Such an integrated approach may improve the specificity by correcting the false positives and the problem of over prediction. However, since this procedure punishes novel predictions, it may lead to lowered sensitivity and missed predictions.`,
    super_topic: `Gene and Promoter Prediction`,
  },
  {
    question: `In bacteria, transcription is initiated by DNA polymerase.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In bacteria, transcription is initiated by RNA polymerase, which is a multi-subunit enzyme. The σ subunit (e.g., σ70) of the RNA polymerase is the protein that recognizes specific sequences upstream of a gene and allows the rest of the enzyme complex to bind.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The upstream sequence where the σ protein binds constitutes the promoter sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This includes the sequence segments located 35 and 10 base pairs (bp) upstream from the transcription start site. They are also referred to as the −35 and −10 boxes. For the σ70 subunit in Escherichia coli, for example, the −35 box has a consensus sequence of TTGACA. The –10 box has a consensus of TATAAT.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The promoter sequence may determine the expression of one gene or a number of linked genes downstream.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In the latter case, the linked genes form an operon. It is controlled by the promoter.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `In addition to the RNA polymerase, there are also a number of DNA-binding proteins that facilitate the process of transcription.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These proteins are called transcription factors. They bind to specific DNA sequences to either enhance or inhibit the function of the RNA polymerase.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The specific DNA sequences to which the transcription factors bind are referred to as _____`,
    options: ["replication elements", "blocking factors", "transcription factors", "regulatory elements"],
    correct: "d",
    explanation: `The regulatory elements may bind in the vicinity of the promoter or bind to a site several hundred bases away from the promoter. The reason that the binding of the regulatory protein at long distance can still exert their effect is because of the flexible structure of DNA, which is able to bend and exert its effect by bringing the transcription factors in close contact with the RNA polymerase complex.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `In eukaryotes, gene expression is not regulated by a protein complex formed between transcription factors and RNA polymerase.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Here, the gene expression is also regulated by a protein complex formed between transcription factors and RNA polymerase. However, eukaryotic transcription has an added layer of complexity in that there are three different types of RNA polymerase complexes, namely RNA polymerases I, II, and III.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `In eukaryotes, genes often form an operon with a shared promoter.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Unlike in prokaryotes, where genes often form an operon with a shared promoter, each eukaryotic gene has its own promoter. The eukaryotic transcription machinery also requires many more transcription factors than its prokaryotic counterpart to help initiate transcription.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Eukaryotic RNA polymerase II does not directly bind to the promoter, but relies on a dozen or more transcription factors to recognize and bind to the promoter in a specific order before its own binding around the promoter.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The core of many eukaryotic promoters is a so-called TATA box, located 30 bps upstream from the transcription start site, having a consensus motif TATA (A/T) A (A/T). However, not all eukaryotic promoters contain the TATA box. Many genes such as housekeeping genes do not have the TATA box in their promoters.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The TATA box is often used as an indicator of the presence of a promoter.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In addition, many genes have a unique initiator sequence (Inr), which is a pyrimidine rich sequence with a consensus (C/T)(C/T)CA(C/T)(C/T). This site coincides with the transcription start site. Most of the transcription factor binding sites are located within 500 bp upstream of the transcription start site.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Ab initio type of algorithm predicts prokaryotic and eukaryotic promoters and regulatory elements based on characteristic sequences patterns for promoters and regulatory elements.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Some ab initio programs are signal based, relying on characteristic promoter sequences such as the TATA box. Other programs rely on content information such as hexamer frequencies.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The advantage of the ab initio method is that the sequence can be applied as such without having to obtain experimental information.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The limitation is the need for training, which makes the prediction programs species specific. In addition, this type of method has a difficulty in discovering new, unknown motifs.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding the ab initio approaches?`,
    options: ["The conventional approach to detecting a promoter or regulatory site is through matching a consensus sequence pattern represented by regular expressions", "The conventional approach to detecting a promoter or regulatory site is through matching a position-specific scoring matrix constructed from well-characterized binding sites", "The consensus sequences or the matrices are relatively short, covering 6 to 10 bases", "The consensus sequences or the matrices are relatively large, covering 700 to 1000 bases"],
    correct: "d",
    explanation: `To determine whether a query sequence matches a weight matrix, the sequence is scanned through the matrix. Scores of matches and mismatches at all matrix positions are summed up to give a log odds score, which is then evaluated for statistical significance. This simple approach, however, often has difficulty differentiating true promoters from random sequence matches and generates high rates of false positives as a result.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `To improve the specificity of prediction, some algorithms selectively ______ coding regions and focus on the upstream regions________ which are most likely to contain promoters. In that sense, promoter prediction and gene prediction are coupled.`,
    options: ["include, (0.5 to 2.0 kb) only", "include, (0.5 to 2.0 Mb) only", "exclude, (0.5 to 2.0 Mb) only", "exclude, (0.5 to 2.0 kb) only"],
    correct: "d",
    explanation: `To better discriminate true motifs from background noise, a new generation of algorithms has been developed that take into account the higher order correlation of multiple subtle features by using discriminant functions, neural networks, or hidden Markov models (HMMs) that are capable of incorporating more neighboring sequence information.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Operon prediction is less important in prokaryotic promoter prediction.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `One of the unique aspects in prokaryotic promoter prediction is the determination of operon structures because genes within an operon share a common promoter located upstream of the first gene of the operon. Hence, operon prediction is the key in prokaryotic promoter prediction.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Once an operon structure is known ______ for the presence of a promoter and regulatory elements _____ in the operon do not possess such DNA elements.`,
    options: ["only the first gene is predicted, whereas other genes", "only the first hundred genes are predicted, whereas next few genes", "only first two genes are predicted, whereas next few genes", "only first ten genes are predicted, whereas next few genes"],
    correct: "a",
    explanation: `Only the first gene is predicted for the presence of a promoter and regulatory elements, whereas other genes in the operon do not possess such DNA elements. There are a number of methods available for prokaryotic operon prediction. The most accurate is a set of simple rules developed.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is correct regarding the method for prokaryotic operon prediction?`,
    options: ["It relies on two kinds of information: gene orientation and intergenic distances of a pair of genes of interest and conserved linkage of the genes based on comparative genomic analysis", "It relies only on the gene orientation and intergenic distances of a pair of genes of interest", "It relies only on the conserved linkage of the genes based on comparative genomic analysis", "The prediction cannot be done manually using the rules"],
    correct: "a",
    explanation: `A scoring scheme is developed to assign operons with different levels of Confidence. This method is claimed to produce accurate identification of an operon structure, which in turn facilitates the promoter prediction. The prediction can be done manually using the rules. The few dedicated programs for prokaryotic promoter prediction do not apply the rule for historical reasons. The most frequently used program is BPROM.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding BPROM?`,
    options: ["It is a web-based program for prediction of bacterial promoters", "It is a web-based program only for prediction of eukarotic promoters", "It uses a linear discriminant function", "The linear discriminant function is combined with signal and content information"],
    correct: "b",
    explanation: `The linear discriminant function is combined with signal and content Information such as consensus promoter sequence and oligonucleotide composition of the promoter sites. This program first predicts a given sequence for bacterial operon structures by using an intergenic distance of 100 bp as basis for distinguishing genes to be in an operon.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `In BPROM, once the operons are assigned, the program is able to predict putative promoter sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The most bacterial promoters are located within 200 bp of the protein coding region. Hence, the program is most effectively used when about 200 bp of upstream sequence of the first gene of an operon is supplied as input to increase specificity.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding FindTerm?`,
    options: ["It is a program for searching bacterial \u03c1-independent termination signals located at the end of operons", "It is a program for searching bacterial \u03c1-dependent termination signals located within the operons", "The predictions are made based on matching of known profiles of the termination signals combined with energy calculations", "It is available from the same site as FGENES and BPROM"],
    correct: "b",
    explanation: `The predictions are made based on matching of known profiles of the termination signals combined with energy calculations for the derived RNA secondary structures for the putative hairpin-loop structure. The sequence region that scores best in features and energy terms is chosen as the prediction. The information can sometimes be useful in defining an operon.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding the Prediction for Eukaryotes?`,
    options: ["The consensus patterns are only derived from bioinformatics studies", "The experimentally determined DNA binding sites are compiled into profiles and stored in a database for scanning an unknown sequence to find similar conserved patterns", "The consensus patterns are derived from experimentally determined DNA binding sites", "The ab initio method for predicting eukaryotic promoters and regulatory elements relies on searching the input sequences for matching of consensus patterns of known promoters and regulatory elements"],
    correct: "a",
    explanation: `This approach tends to generate very high rate of false positives owing to nonspecific matches with the short sequence patterns. Furthermore, because of the high variability of transcription factor binding sites, the simple sequence matching often misses true promoter sites, creating false negatives.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `To increase the specificity of prediction, a unique feature of eukaryotic promoter is employed, which is the presence of CpG islands.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is known that many vertebrate genes are characterized by a high density of CG dinucleotides near the promoter region overlapping the transcription start site. By identifying the CpG islands, promoters can be traced on the immediate upstream region from the islands. By combining CpG islands and other promoter signals, the accuracy of prediction can be improved. Several programs have been developed based on the combined features to predict the transcription start sites in particular.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The eukaryotic transcription initiation is less dependent on transcription factors.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The eukaryotic transcription initiation requires cooperation of a large number of transcription factors. Co-operativity means that the promoter regions tend to contain a high density of protein-binding sites. Thus, finding a cluster of transcription factor binding sites often enhances the probability of individual binding site prediction.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `CpGProD is a web-based program that predicts promoters containing a high density of CpG islands _______`,
    options: ["in archea genomic sequences", "in mammalian genomic sequences", "in eukaryotic and bacterial genomic sequences", "only in bacterial genomic sequences"],
    correct: "b",
    explanation: `It calculates moving averages of GC% and CpG ratios (observed/expected) over a window of a certain size (usually 200 bp). When the values are above a certain threshold, the region is identified as a CpG island.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding Eponine?`,
    options: ["It is a web-based program that predicts transcription start sites", "It is a web-based program that particularly predicts tranpososons and retropososons", "The regulatory sites include the TATA box, the CCAAT box, and CpG islands", "It is based on a series of pre-constructed PSSMs of several regulatory sites"],
    correct: "b",
    explanation: `The query sequence from a mammalian source is scanned through the PSSMs. The sequence stretches with high-score matching to all the PSSMs, as well as matching of the spacing between the elements, are declared transcription start sites. A Bayesian method is also used in decision making.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding Cluster-Buster?`,
    options: ["It is an HMM-based web-based program", "A query sequence is scanned with a window size of 1 kb for putative regulatory motifs using motif HMMs", "It works by detecting a region of high concentration of unknown transcription factor binding sites and regulatory motifs at the initiation", "It is designed to find clusters of regulatory binding sites"],
    correct: "c",
    explanation: `It works by detecting a region of high concentration of known transcription factor binding sites and regulatory motifs. If multiple motifs are detected within a window, a positive score is assigned to each motif found. The total score of the window is the sum of each motif score subtracting a gap penalty, which is proportional to the distances between motifs. If the score of a certain region is above a certain threshold, it is predicted to contain a regulatory cluster.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding First EF?`,
    options: ["It is a program that predicts promoters for bacterial DNA", "It is a web-based program that predicts promoters for human DNA", "It stands for First Exon Finder", "It integrates gene prediction with promoter prediction"],
    correct: "a",
    explanation: `It uses quadratic discriminant functions (see Chapter 8) to calculate the probabilities of the first exon of a gene and its boundary sites. A segment of DNA (15 kb) upstream of the first exon is subsequently extracted for promoter prediction on the basis of scores for CpG islands.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `McPromoter, a web-based program, uses a neural network to make promoter predictions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It has a unique promoter model containing six scoring segments. The program scans a window of 300 bases for the likelihoods of being in each of the coding, noncoding, and promoter regions.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `The input for the neural network includes parameters for sequence physical properties, such as ______`,
    options: ["DNA bendability", "Signals such as the TATA box", "Signals such as initiator box", "Signals such as CpAA islands"],
    correct: "d",
    explanation: `As seen, the correct answer is CpG in option d. The hidden layer combines all the features to derive an overall likelihood for a site being a promoter. Another unique feature is that McPromoter does not require that certain patterns must be present, but instead the combination of all features is important. For instance, even if the TATA box score is very low, a promoter prediction can still be made if the other features score highly. The program is currently trained for Drosophila and human sequences.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `TSSW is a web program that distinguishes promoter sequences from non-promoter sequences based on a combination of unique content information such as hexamer/trimer frequencies and signal information such the TATA box in the promoter region.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `As mentioned here, TSSW uses unique content information such as hexamer/trimer frequencies and signal information such the TATA box in the promoter region. The values are fed to a linear discriminant function to separate true motifs from background noise.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding CONPRO?`,
    options: ["It is a web-based program that uses a consensus method", "It is used to identify promoter elements for human DNA", "cDNA does not play a role in prediction", "The program uses the information to search the human genome database for the position of the gene"],
    correct: "c",
    explanation: `To use the program, a user supplies the transcript sequence of a gene (cDNA). It then uses the GENSCAN program to predict 5’ untranslated exons in the upstream region. Once the 5’-most exon is located, a further upstream region (1.5 kb) is used for promoter prediction, which relies on a combination of five promoter prediction programs, TSSG, TSSW, NNPP, PROSCAN, and PromFD.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `In CONPRO, for each program, the highest score prediction is taken as the promoter in the region.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If three predictions fall within a 100-bp region, this is considered a consensus prediction If no three-way consensus is achieved, TSSG and PromFD predictions are taken. Because no coding sequence is used in prediction, specificity is improved relative to each individual program.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is incorrect regarding Phylogenetic Footprinting–Based Method?`,
    options: ["It is possible to obtain promoter sequences for a particular gene through comparative analysis", "The conservation from closely related organisms is both at the sequence level and at the level of organization of the elements", "The conservation from closely related organisms is most at the sequence level", "It has been observed that promoter and regulatory elements from closely related organisms such as human and mouse are highly conserved"],
    correct: "c",
    explanation: `The identification of conserved noncoding DNA elements that serve crucial functional roles is referred to as phylogenetic footprinting; the elements are called phylogenetic footprints. This type of method can apply to both prokaryotic and eukaryotic sequences.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `A caveat of phylogenetic footprinting is to extract noncoding sequences Upstream of corresponding genes and focus the comparison to this region only, which helps to prevent false positives.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The predictive value of this method also depends on the quality of the subsequent sequence alignments. Advanced alignment programs can be used. Even more sophisticated expectation maximization (EM) and Gibbs sampling algorithms can be used in detecting weakly conserved motifs.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about ConSite?`,
    options: ["It is a web server that finds putative promoter elements", "It includes comparing two orthologous sequences", "The program does not accept pre-computed alignment", "The program accepts pre-computed alignment"],
    correct: "c",
    explanation: `The user provides two individual sequences which are aligned by ConSite using a global alignment algorithm. Conserved regions are identified by calculating identity scores, which are then used to compare against a motif database of regulatory sites (TRANSFAC). High-scoring sequence segments upstream of genes are returned as putative regulatory elements.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `rVISTA uses two orthologous sequences as input and first identifies all putative regulatory motifs based on TRANSFAC matches.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `rVISTA is a cross-species comparison tool for promoter recognition. It aligns the two sequences using a local alignment strategy. The motifs that have the highest percent identity in the pairwise comparison are presented graphically as regulatory elements.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about Bayes Aligner?`,
    options: ["Posterior probability values, which are considered estimates of the true alignment, are calculated for each alignment", "The method generates a single best alignment", "It aligns two sequences using a Bayesian algorithm which is a unique sequence alignment method", "It is a web-based footprinting program"],
    correct: "b",
    explanation: `Instead of returning a single best alignment, the method generates a distribution of a large number of alignments using a full range of scoring matrices and gap penalties. By studying the distribution, the alignment that has the highest likelihood score, which is in the extreme margin of the distribution, is chosen. Based on this unique alignment searching algorithm, weakly conserved motifs can be identified with high probability scores.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about FootPrinter?`,
    options: ["It is a web-based program for phylogenetic footprinting using multiple input sequences", "The motifs from organisms spanning over the widest evolutionary distances are identified as promoter or regulatory motifs", "The program performs multiple alignment of the input sequences to identify conserved motifs", "The user does not necessarily provides a phylogenetic tree that defines the evolutionary relationship of the input sequences"],
    correct: "d",
    explanation: `The user also needs to provide a phylogenetic tree that defines the evolutionary relationship of the input sequences. One may obtain the tree information from the “Tree of Life” web site, which archives known phylogenetic trees using ribosomal RNAs as gene markers. It identifies unusually well-conserved motifs across a set of orthologous sequences.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about Expression Profiling–Based Method?`,
    options: ["Genes with similar expression profiles are considered coexpressed, which can be identified through a clustering approach", "This approach appears to be less effective for finding transcription factor binding sites", "An advanced alignment-independent profile construction method such as EM and Gibbs motif sampling is often used in finding the subtle sequence motifs", "The basis for coexpression is thought to be due to common promoters and regulatory elements"],
    correct: "b",
    explanation: `This approach is essentially experimentally based and appears to be robust for finding transcription factor binding sites. The problem is that the regulatory elements of coexpressed genes are usually short and weak. Their patterns are difficult to discern using simple multiple sequence alignment approaches.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `INCLUSive is a suite of web based tools designed to streamline the process of microarray data collection and sequence motif detection.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The pipeline processes microarray data, automatically clusters genes according expression patterns, retrieves upstream sequences of coregulated genes and detects motifs using a Gibbs sampling approach (Motif Sampler). To further avoid the problem of getting stuck in a local optimum, each sequence dataset is submitted to Motif Sampler ten times. The results may vary in each run. The results from the ten runs are compiled to derive consensus motifs.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about PhyloCon?`,
    options: ["It stands for Phylogenetic Consensus", "It is used to identify regulatory motifs", "It is a UNIX program that combines phylogenetic footprinting with gene expression profiling analysis", "No conservation among orthologous genes and conservation among coregulated genes is a disadvantage"],
    correct: "d",
    explanation: `This approach takes advantage of conservation among orthologous genes as well as conservation among coregulated genes. For each individual gene in a set of coregulated genes, multiple sequence homologs are aligned to derive profiles. Based on the gene expression data, profiles between coregulated genes are further compared to identify functionally conserved motifs among evolutionarily conserved motifs.`,
    super_topic: `Promoter and Regulatory Elements`,
  },
  {
    question: `Which of the following is untrue about Amino acid conservation?`,
    options: ["It has been known for some time that conservation of residues at the surface of a protein family is often related to function", "Conservation of residues at the core of a protein family is often related to function", "This may be an enzyme active-site or binding site", "Unlike hydrophobicity or electrostatic potential, displaying residue conservation on the molecular surface has no physical or chemical basis"],
    correct: "b",
    explanation: `However, the evolutionary information can sometimes delineate a functional epitope allowing residues to be identified that are important for binding. In order to infer structure-function relationships, the proteins must be structurally related, preferably a large family of proteins with related function.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Which of the following is untrue about Shape Complementarity?`,
    options: ["The complementarity between two proteins or a protein and ligand can be described by surface contact or overlap", "The complementarity between two proteins or a protein and ligand can be described by the overall buried surface area of two molecules in contact", "The complementarity between two proteins or a protein and ligand can be described by the number of adjacent surface points (atoms or residues)", "The hydrophobic effect barely has impact on protein folding"],
    correct: "d",
    explanation: `For example simple atom neighbor counting and the simple surface contact scores are of this type. Whilst these measures are easily calculated they also have some physical basis for being effective scoring functions. The principal driving force for protein folding and binding is the hydrophobic effect, which involves the free energy gain of removing non-polar surface area from water.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The burial of surface area (or the maximization of surface contact) is an approximation of the effect of shape complementarity.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `However, since no distinction is generally made between polar and non-polar surface area this is an approximation. As burial of the former is unfavorable or neutral the sentence is held true.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Which of the following is untrue about Grid Representation?`,
    options: ["To speed up the matching process the topology of the protein can be simplified from atomic level detail to a series of cubic elements", "To speed up the matching process discretizing the 3-dimensional space using a grid is done", "Discretizing allows very fast computer matching using search methods such as Fourier transform", "Fourier transform is hardly used in computer matching"],
    correct: "d",
    explanation: `The shape of a molecule is described by mapping it to a 3-D grid of uniformly spaced points. Clearly the level of detail is controlled by the grid spacing. The larger the grid spacing the cruder the representation with respect to the atomic level.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In Grid representation, in a translational scan the mobile molecule B moves through the grid representing the static molecule A and a signal describing shape complementarity, fc, is generated for each mapping. Mathematically the correlation function, fc, of fA and fB is given by ______ where N is the number of grid points along the cubic axes i, j, and k and α, β, and γ are the translational vectors of the mobile molecule B relative to the static one A.`,
    options: ["fc = \u2211Ni=1,j=1,k=1 (fA,i,j,k * fBi+\u03b1j+\u03b2k+\u0263)", "fc = \u2211Ni=0,j=0,k=0 (fA,i,j,k * fBi+\u03b1j+\u03b2k+\u0263)", "fc = \u2211Ni=1,j=1,k=1 (fA,0,0,0 * fBi+\u03b1j+\u03b2k+\u0263)", "fc = \u2211Ni=0,j=0,k=0 (f0,i,j,k * fBi+\u03b1j+\u03b2k+\u0263)"],
    correct: "a",
    explanation: `The overlap between points representing the surface of the molecules is scored favorably, however, overlap with points representing the core of the static molecule are scored unfavorably. Zero correlation score is given to two molecules not in contact. Negative scores represent surface overlap with the core region of molecule A.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In Grid representation, the lowest score represents the best surface complementarity for a given translational scan.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The highest score represents the best surface complementarity for a given translational scan. Note that the mobile molecule must be mapped differently to the grid for each rotational change applied to the molecule.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In Property-based measures, displaying physical properties on the molecular surface of molecules can help to guide molecular docking.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Indeed in many cases, displaying physical properties on the molecular surface of molecules can help to guide molecular docking. Alternatively sequence conservation might also help, particularly where a homologous family of proteins maintain a specific binding partner.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Which of the following is untrue about Hydrophobicity?`,
    options: ["The hydrophobic effect plays a dominant role in the folding of proteins", "Hydrophobic residues aggregate away from contact with water", "Hydrophobic residues aggregate to form hydrophobic cores with more polar residues", "Hydrophobic residues form the solvent accessible surface but restrict the solubility of the protein"],
    correct: "d",
    explanation: `Clearly a hydrophobic interface will drive the formation of protein-protein or protein-ligand interactions. It has been noted that hydrophobicity is fairly common at protein-protein interfaces particularly in homodimers (two identical protein monomers that associate) and oligomeric proteins.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Oligomers are often obligate complexes meaning that the free-energy cost of dissociation is high and they exist as oligomers under physiological conditions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In some cases biological function is dependent on this. However, many protein interactions are non-obligatory being made/broken according to their environment. These proteins must be independently stable in solution. These are commonly heterodimeric complexes including enzyme inhibitor and antibody-antigen complexes as well as a host of other casual interacting proteins. The hydrophobic effect is often much less dominant and interfaces are more polar in nature partly because of issues relating to protein stability and aggregation. Therefore, hydrophobicity is useful as a guide to molecular complementarity in selective protein-protein interactions.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Which of the following is untrue about Electrostatic Complementarity?`,
    options: ["The electrostatic properties of biomolecules play an important role in determining interactions", "The burial of charged residues at protein-protein/DNA interfaces is thought to be generally net destabilizing with the hydrophobic effect being the primary driving force", "Charged groups involved in the biomolecular interface are often stabilized by other polar or oppositely charged groups on the interacting molecule", "Charged groups involved in the biomolecular interface are often stabilized by similar polar or same charged groups on the interacting molecule"],
    correct: "d",
    explanation: `Therefore charge complementarity can play an important role in determining the specificity of the interaction. In many cases in biology a protein must recognize a highly charged molecule such as poly-anions like DNA or RNA. In order to make a close approach the protein must have charged residues that complement the negative charges present on the phosphate backbone. Electrostatic complementarity is also important in many protein-protein and protein-ligand interactions.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Proteins are dynamic entities that undergo _______`,
    options: ["only fluctuation of flexible loop regions about equilibrium positions when in solution", "only limited conformational change of amino acid side-chains when in solution", "both limited conformational change of amino acid side-chains and fluctuation of flexible loop regions about equilibrium positions when in solution", "illimitable or total conformational change of amino acid side-chains when in solution"],
    correct: "c",
    explanation: `Such flexibility can often be adequately treated by ‘soft’ potentials or limited conformational flexibility and/or refinement of side-chains on docking. However, proteins often undergo more extensive conformational changes which may involve large-scale motions of domains relative to one another or possibly conformational change involving order-disorder transitions.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Typical motions of proteins will be primarily treated by a rigid body model for docking.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `These types of motions will be poorly treated by a rigid body model for docking. Again distinction is often made between the general treatment of protein-protein docking and protein-ligand docking.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In case of Protein-ligand docking, ______ ligands are often _____ in adapting their shape to fit the receptor binding pocket.`,
    options: ["small molecule, highly flexible", "large molecule, highly flexible", "large molecule, more flexible", "small molecule, less flexible"],
    correct: "a",
    explanation: `The degree of conformational flexibility of a small molecule ligand (substrate, cofactor or inhibitor) can be considerable particularly where there are multiple torsion angles. This presents a major challenge in protein-ligand docking and several different approaches have been used to solve this problem.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In Multiple conformation rigid-body method, a ligand is assumed to be able to adopt a number (N) of different _______ that are computed _____ the ligand being docked into the receptor.`,
    options: ["low-energy conformations, after", "low-energy conformations, prior to", "high-energy conformations, prior to", "high-energy conformations, after"],
    correct: "b",
    explanation: `These, N, low-energy conformations are then docked individually into the receptor assuming a rigid conformation using a descriptor-based approach. The scoring function is used to determine which of the resulting solutions is optimal.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The disadvantage of Multiple conformation rigid-body method is that the active conformation may be missed as the result of a minor structural difference not considered in the ______ ligand conformations. Where, the N is the number of low-energy conformations.`,
    options: ["N +1", "N", "N2", "N/2"],
    correct: "b",
    explanation: `The disadvantage is that the active conformation may be missed as the result of a minor structural difference not considered in the N ligand conformations and the advantage of this approach is that the search can be restricted to a smaller number of relevant ligand conformations.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In Stochastic search methods, they include methods such as Monte Carlo simulation, simulated annealing, Tabu search, genetic algorithms and evolutionary programming.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Stochastic processes use a random sampling procedure to search conformational space. The ligand molecule performs a random walk in space in the receptor cavity. Usually, the ligand is placed in a random orientation in the receptor cavity. Then at each step a small displacement is made in any of the degrees of freedom of the ligand molecule (translation, rotation or torsion angle).`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In a Monte Carlo simulation the score (or energy) is calculated at each step and compared to the previous step. The probability of accepting the step is given by _________ where ΔE is the difference in energy; kB is Boltzmann’s constant and T the temperature.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If the new energy is lower the step is accepted, otherwise the result is treated probabilistically by a Boltzmann mechanism. If P (ΔE) is greater than a random number generated between 0 and 1 then the step is accepted. The higher the temperature (or the smaller ΔE at a given T) the higher the likelihood the step is accepted. A conventional Monte Carlo simulation proceeds at constant temperature, whilst in simulated annealing the temperature is gradually cooled during the simulation in an attempt to locate a globally optimal solution. In simulated annealing the computer stores a single solution and generates a new solution randomly.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Genetic methods (genetic algorithms and evolutionary programming) store multiple solutions. These solutions form a population of members.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Each member has an associated score or fitness. During the search for the global optimal solution successive new populations are created by a procedure involving selection of the fittest members. These members then have offspring to create a new population. Differences arise in how the methods generate offspring. In a genetic algorithm two solutions are mated to form a new offspring solution. In evolutionary programming each member of the population generates an offspring by mutation.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Stochastic methods can guarantee reaching a global optimal solution and the methods are computationally costly in comparison to the other methods.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The advantages of stochastic methods are that the ligand is able to explore conformational space in a relatively unconstrained way, frequently leading to the globally optimal solution. The disadvantages are that it cannot guarantee reaching a global optimal solution and the methods are computationally costly in comparison to the other methods.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Which of the following is true regarding Protein flexibility?`,
    options: ["Methods have been described for the introduction of side-chain flexibility to both protein-ligand and protein-protein docking", "Methods have been described for the introduction of side-chain flexibility to protein-ligand docking only", "The Mean Field principle is rarely used in this", "Methods have been described for the introduction of side-chain flexibility to protein-protein docking only"],
    correct: "a",
    explanation: `The Mean Field approach is one type of what are called bounded search methods. Others include the Dead-end-elimination theorem and the A* algorithm. These methods use different approaches to find a solution and a detailed discussion is beyond the scope of this chapter. However, they use a multiple copy representation of protein side chains built using a rotamer library.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In a similar way to structure prediction methods models can be evaluated using RMSD to measure the similarity between two molecular complexes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is only the case if an experimental structure already exists. Several docking methods have been evaluated at ‘Critical Assessment of Structure Prediction 2’ (CASP2) for both protein-ligand and protein-protein interactions.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In case of protein-protein docking, the level of success is dependent on the system under study.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For the protein-protein docking evaluation several of the same docking methodologies were used in both docking challenges. In the first challenge involving a protein inhibitor all the groups successfully predicted the complex.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `There was little degree of success in docking an antibody-antigen complex in the second challenge for the protein-protein docking evaluation.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This may be because modeling molecular recognition is in general more difficult in antibody-antigen than protein-inhibitor systems. In the protein-ligand docking challenge at CASP2 there was generally a good level of success, however, again certain targets proved to be problematic.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The ______ is an ongoing community-wide experiment on the comparative evaluation of protein-protein docking for structure prediction.`,
    options: ["Chronological Assignment of Prediction of Interactions (CAPRI)", "Chronological Assessment of Prediction of Interactions (CAPRI)", "Critical Assignment of Prediction of Interactions (CAPRI)", "Critical Assessment of Prediction of Interactions (CAPRI)"],
    correct: "d",
    explanation: `Clearly there is an ongoing need for assessment of methods. There is currently no one universal method or scoring function that will work on all occasions.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `An understanding of the importance of different factors in a particular interaction is important if confidence in the results is required. Which of the following are not those factors?`,
    options: ["Shape", "Hydrophobicity", "Electrostatics", "pH"],
    correct: "d",
    explanation: `Not that the parameter pH is not related at all but is quite distinctly related to the mentioned factors. Also, pH is relative to the state of the molecule at given instant, other factors being evolutionary relationships and conformational flexibility.
Free 30-Day C++ Certification Bootcamp is Live. Join Now!`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Information about shape, hydrophobicity, electrostatics, evolutionary relationships and conformational flexibility is not always available and the search for a universally applicable scoring function as well as an adequate treatment of conformational flexibility is ongoing.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The technique of virtual screening of small molecule drugs by computer has become commonplace and a very useful tool in narrowing down the very large number of drugs that might need to be screened by experimental methods. In one study virtual screening provided lead compounds where conventional experimental random screening had failed.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `In practical circumstances there exists an experimental structure of the complex.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In practical circumstances an experimental structure of the complex does not already exist. Some other evaluation criteria are needed. This is by necessity experimental validation. The hypothetical protein-biomolecular complex predicts a mode of interaction between the two molecules that can be tested experimentally.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `Visualization methods are very important in viewing molecular properties on molecules. Of particular note is the rendering of molecular surfaces according to their various properties (that can be expressed numerically).`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This approach has been popularized by the GRASP program. Also Virtual Reality Modeling Language (VRML) viewers can provide similar displays.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The popular program RasMol can be made to view molecular properties by assigning those properties to the temperature factor column of the sdf file in question only.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The popular program RasMol can also be made to view molecular properties by assigning those properties to the temperature factor column of the PDB file in question. The molecules are then best viewed in Spacefill mode, however, the color scheme used is not flexible.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The GRASP and VRLM have been incorporated into the GRASS server.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This allows a Web-based interactive exploration of molecules in the PDB allowing the molecular properties to be viewed on the molecular surface. There are also several other popular molecular graphics programs that allow a similar visualization.`,
    super_topic: `Protein–Biomolecular Interactions`,
  },
  {
    question: `The physical contacts between domains are crucial for the functioning of the cellular machinery.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Interactions between domains occur in multidomain proteins, in stable complexes and in transient interactions between proteins that also exist independently. Experimental approaches for the large-scale determination of protein interactions are emerging. Theoretical analyses based on protein structures have unraveled some of the overall principles and features of the way domains evolved to interact with each other.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `There exist three types of interactions between domains. Which of the following is not one of them?`,
    options: ["Stable complex", "Transient interaction", "Multi-domain protein", "Unstable interaction"],
    correct: "d",
    explanation: `Interactions between domains determine the structure of multidomain proteins, in which there are several domains on one polypeptide chain. Given that all proteins consist of domains, interactions between domains also occur between the proteins that are permanently associated in stable complexes and proteins that interact transiently, but also exist independently of each other.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Stable complexes consist of proteins that are _____ associated with each other, like many ____ proteins for instance.`,
    options: ["temporarily, oligomeric", "temporarily, monomeric", "permanently, oligomeric", "permanently, monomeric"],
    correct: "c",
    explanation: `Well-known stable complexes include the histone octamer, the ribosome and DNA and RNA polymerases. Transient interactions on the other hand are all those protein-protein interactions that occur between proteins that also exist independently.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Sets of proteins that are part of stable complexes and sets of proteins involved in transient interactions ____ in terms of the similarity in gene expression among the set of proteins.`,
    options: ["are similar", "differ", "are same", "show similar function"],
    correct: "b",
    explanation: `Proteins permanently associated in a stable complex need to be present or absent in the cell at the same time. Analysis of microarray data by Gerstein and co-workers by methods along the lines, has shown that the members of stable complexes in the yeast Saccharomyces cerevisiae have highly correlated gene expression patterns.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Correlation of gene expression for pairs of transiently interacting proteins is ______ compared to randomly chosen pairs of proteins.`,
    options: ["not significant", "only marginally significant", "totally significant", "significant to much extent"],
    correct: "b",
    explanation: `In prokaryotes, genes are co-regulated if they are a member of the same operon, and many proteins that are members of the same stable complex are part of the same operon. For instance, Ouzounis and Karp determined that over 90% of the enzymes that are in stable complexes in E. coli metabolic pathways are adjacent on the E. coli chromosome.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Membership in a stable complex also differs from transient interaction in terms of evolutionary constraints upon sequence divergence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Thus the proteins in stable complexes are more similar across species, having higher sequence identity between orthologs, than the proteins in transient interactions. A calculation by Teichmann showed that there are significant differences between the average values for sequence identities between S. cerevisiae and S. pombe orthologs in stable complexes, transient interactions and monomers.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `For proteins in stable complexes the average sequence identity is 46%, while for proteins in transient interactions it is 41%.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `(Proteins not known to be involved in any type of interaction have an average sequence identity of 38 %) One of the main reasons for this is the surface area involved in interfaces of stable complexes which is larger than in transient complexes. Sequence divergence may be slower in order to conserve these extensive interfaces.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Which of the following is incorrect about Yeast-two-hybrid screens?`,
    options: ["The yeast-two-hybrid system uses the transcription of a reporter gene driven by the Gal4 transcription factor to monitor whether or not two proteins are interacting", "The DNA-binding domain chimeric protein will not bind upstream of the reporter gene", "If the activation domain chimeric protein interacts with the DNA-binding domain chimeric protein, the reporter gene will be transcribed", "Disadvantages of the method are that only pairwise interactions are tested, and not interactions that can only take place when multiple proteins come together, as well as a high false positive rate"],
    correct: "b",
    explanation: `If the interaction between two proteins, A and B, is being tested, one of their genes would be fused to the DNA-binding domain of the Gal4 transcription factor (Gal4-DBD) while the other would be fused to the activation domain (Gal4-AD). The DNA-binding domain chimeric protein will bind upstream of the reporter gene. This experiment can be carried out hundreds or even thousands of times on microassay plates, as in the case of the study by Uetz and colleagues on S.cerevisiae (yeast) interactions. Each array element on these plates contains yeast cells transformed with a particular combination of two plasmids, one carrying the DNA-binding domain chimeric protein and the other the activation domain chimeric protein.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Which of the following is incorrect about Purification of protein complexes followed by mass spectrometry?`,
    options: ["Isolating protein complexes from cells allows identification of interactions between ensembles of proteins instead of just pairs", "Systematic purification of complexes on a large scale is done by tagging hundreds of genes with an epitope", "UnLike in the yeast-two-hybrid assay, this does not involve chimeric genes", "Affinity purification based on the epitope will then extract all the proteins attached to the bait protein from cell lysates"],
    correct: "c",
    explanation: `Like in the yeast-two-hybrid assay, this is done by making chimeric genes that are introduced into cells. The principle of mass spectrometric identification of proteins is that the protein is chopped into fragments by tryptic digestion, and the mass of each fragment is measured by matrix-assisted laser desorption/ ionization-time-of-flight mass spectrometry (MALDI-TOF MS). This measurement is so accurate that the combination of amino acids in each fragment can be calculated and compared to a database of all the proteins in the proteome of the organism in order to find the correct one.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In protein domain family interaction map, the physical contacts of the domains in different families are represented by the lines between the nodes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Each node in this graph represents a protein domain family. There are a few families that are hubs in the network: these are large families that are functionally versatile, such as Rossmann domains indicated by an ‘R’ here. Most families engage in only one or two types of interactions.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In the Interaction map of domain families, the interactions of one family represent the sum of all the interactions of domains in that family.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `To study the large-scale patterns and evolution of interactions between protein domains, the interactions in terms of the domain families can be summarized. Thus the interactions of one family represent the sum of all the interactions of domains in that family. Precise information about contacts between individual domains can be extracted by analysis of PDB entries.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Most domain families only interact with one or two other families, while a few families are extremely versatile in their interactions and are connected to many families.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The result of the known interactions between members of structural protein families is a graph of connections between families, where the nodes are protein families and the edges represent an interaction between at least one of the domains from each of the two families. This pattern is observed at the level of individual proteins as well, as similar networks can be constructed for the individual proteins in the yeast proteome, for instance.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Almost ______ engage in interactions with domains from their own family when one includes oligomeric proteins.`,
    options: ["one fifith of all known families", "one fourth of all known families", "all of all known families", "half of all known families"],
    correct: "d",
    explanation: `In this case, half of all known families engage in interactions with domains from their own family. Such symmetrical interactions appear to be particularly favorable.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In order to understand the geometry of domain combinations, different structures of homologous pairs of domains must be studied.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is important because though the methods for structure prediction of individual domains are well established, much less is known about assemblies of domains. The network of domain family interactions is a purely two-dimensional map: it lays out the connections between families but does not provide information on the three-dimensional geometry of interactions.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `The investigation (Aloy and Russel) of domain combinations in multidomain proteins by Bashton and Chothia focuses on two-domain proteins belonging to the Rossmann domain family.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These proteins generally consist of one Rossmann domain and one catalytic domain. As for the analysis of transient interactions, all the proteins belonging to one family of catalytic domains form the same type of interface to the Rossmann domains.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `The linkers between the catalytic domain and the Rossmann domain were conserved in each family.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This means that interface conservation within one catalytic family is a result of the direct evolutionary relationship between the proteins that have a particular pair of domains. In other words, each set of Rossmann domain proteins with a particular catalytic domain has descended from one common ancestral recombination event.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Across the different types of catalytic families, the position of the two domains with respect to one another varied, but only within a range of about ______`,
    options: ["20\u00b0", "10\u00b0", "90\u00b0", "80\u00b0"],
    correct: "c",
    explanation: `This is the result of a functional constraint in these enzymes: the catalytic domain can only take up a variety of positions, as the substrate needs to be held sufficiently close to the NAD(P) cofactor of the Rossmann domain. In other multidomain proteins where there is no such strict functional constraint, the domain interfaces of one domain family to other families may well be more variable.

This set of Basic Bioinformatics Questions and Answers focuses on “The Use of Gene Order & Phylogeny to Predict Protein – Protein Interactions”.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Experimentation is most desirable over computational methods by every means.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Computational methods for predicting protein-protein interactions are desirable as experimental determination is time-consuming and expensive. Several prediction methods are based on the observation that if two proteins are part of the same complex, it is favorable for the two interaction partners to be co-expressed and co-regulated.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Interactions between proteins can be predicted computationally by looking for sets of genes that occur as a _______`,
    options: ["single gene in at least one genome", "multiple genes in at least one genome", "multiple genes in various genomes", "single gene in various genomes"],
    correct: "a",
    explanation: `Interactions between proteins can be predicted computationally by looking for sets of genes that occur as a single gene in at least one genome. Or it can be done by looking for prokaryotic genes that have conserved adjacent gene order across several genomes.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Genes that are consistently part of the same operon across different, distantly related genomes are likely to be part of the same protein complex or functional process across all species.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is because they have been selected to remain as a co-regulated unit throughout the extensive shuffling of gene order that takes place in prokaryote genomes. Thus conservation of gene order across different, distantly related genomes has been used as a method for predicting protein interactions.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `When comparing pairs of genes or sets of genes in different genomes for this purpose, it is not mandatory for the genes to be orthologs.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `When comparing pairs of genes or sets of genes in different genomes for this purpose, it is important to ensure that the genes are truly equivalent, in other words that they are orthologs, as opposed to merely similar genes. This is frequently done by only accepting a pair of proteins as orthologs if they are ‘bi-directional best hits’. This means that both proteins are the best match to each other when searching against the other proteome. The other extreme would be to consider proteins as equivalent if they share just one of many domains, for instance.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Members of a stable complex are often co-regulated and thus will be detected by the method of Conservation of gene order.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `As much it is true proteins which are part of the same metabolic pathway or part of the same biological process can also be co-regulated. Therefore, although some of the genes detected as interacting by this method physically interact, others are just functionally associated with each other.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In a quantitative assessment of this method (Conservation of gene order) using the genome of the parasitic organism Mycoplasma genitalium as a benchmark.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Huynen and colleagues found that two-thirds to four-fifths of the general interactions detected correspond to physical interactions and another 13% correspond to a metabolic or non-metabolic pathway. Genes those are physically associated with each other while being regulated and expressed individually will not be detected by this method.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Conservation of gene order due to operon structure is _______ so interactions of proteins specific to eukaryotes cannot be detected by method of Conservation of gene order.`,
    options: ["not applicable to archea genomes", "not applicable to prokaryote genomes", "applicable to eukaryote genomes", "not applicable to eukaryote genomes"],
    correct: "d",
    explanation: `Co-regulation of genes in eukaryotes can be inferred by similarity in the expression patterns of genes. A comparison of co-expressed yeast and worm genes showed that 90% of those pairs of genes with conserved co-expression are members of stable complexes. Thus, the principle of conserved co-regulation across distantly related organisms applies to stable complexes in both prokaryotes and eukaryotes, and can be used as a prediction tool in both.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `An approach for predicting ______ to look for cases across a set of genomes where _____ are part of the same gene in one genome resulted in gene fusion method.`,
    options: ["gene interactions, only three to four orthologs", "gene interactions, two orthologs", "protein interactions, two or more orthologs", "protein interactions, two orthologs"],
    correct: "c",
    explanation: `The prediction is then that the orthologs that are on separate genes in the other genomes interact with each other. In the case of gene fusion, the fused proteins are not only co-regulated, as in conservation of gene order described above, but also permanently colocalized in the cell. The additional requirement of colocalization beyond just co-regulation poses a further limitation on the prediction method.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Domains that are part of a multidomain protein are ______`,
    options: ["nethier co-regulated nor colocalized", "not co-regulated but colocalized", "co-regulated and but not colocalized", "co-regulated and colocalized"],
    correct: "d",
    explanation: `Therefore, members of stable complexes as well as consecutive enzymes may be involved in gene fusions. However elements of signal transduction chains are seldom part of the same gene, for instance, as it is an essential part of their function that they can be regulated and localized independently.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Due to the requirement for co-regulation as well as colocalization, the method is mostly limited to certain classes of protein-protein interactions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Those classes are members of the same stable complex and proteins in the same metabolic pathway. In their assessment of computational methods for prediction of all types of protein interactions, Huynen and colleagues found that two-thirds of the interactions detected in this way were between proteins that physically interact and another 15% between proteins part of the same metabolic pathway. The remaining interactions involved hypothetical proteins of unknown function.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `The phylogenetic profile method relies on detection of orthologs (or homologs, in a variation of the method) in a set of genomes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `If the pattern of ortholog presence or absence is the same in a group of proteins, then these proteins are clustered together as belonging to the same functional class. A method that appears to reliably predict a loose functional correlation between proteins is the phylogenetic profile method developed by Pellegrini and colleagues.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In the assessment of methods to predict protein-protein interactions, one third of such pairs were found to physically interact, and an additional third to belong to the same metabolic pathway or functional process.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `As mentioned, one third of such pairs were found to physically interact. The phylogenetic patterns of clusters of orthologous groups of proteins deposited in the COG database of Koonin and co-workers could in principle be used for prediction in the same way.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `In the phylogenetic profile method for predicting protein interaction, presence or absence of orthologous genes is scored across a variety of genomes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This is represented by presence or absence of a dot in the row. Genes that have the same pattern of presence or absence across genomes are predicted to interact.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Structural analyses on small sets of proteins have shown that the domains from a pair of families bind to each other with the same geometry in multi-domain proteins and in transient interactions.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Most domain families engage in interactions with one or two other types of families, but a few families are very versatile and interact with many families. These versatile families are ubiquitously useful families such as P-loop nucleotide triphosphate hydrolases and Rossmann domains.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `The most detailed experimental information about protein-protein interactions comes from three-dimensional structures.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The 3-D structure gives the most conformation of the structure. It is likely that complexed protein structures will be solved more frequently in the wake of the structural genomics projects.`,
    super_topic: `Protein–Protein Interactions`,
  },
  {
    question: `Which of the following is untrue about DNA sequencing methods?`,
    options: ["Purified fragments of DNA cut from plasmid/phage clones or amplified by polymerase chain reaction (PCR)", "Clones of DNA fragments are denatured to single strands, and one of the strands is hybridized to an oligonucleotide primer", "Taq polymerase is quite heat sensitive", "New strands of DNA are synthesized from the end of the primer"],
    correct: "c",
    explanation: `In an automated procedure, new strands of DNA are synthesized from the end of the primer by heat-resistant Taq polymerase from a pool of deoxyribonucleotide triphosphates (dNTPs) that includes a small amount of one of four chain-terminating nucleotides (ddNTPs).`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Using ddATP, the resulting synthesis creates a set of nested DNA fragments, each one ending at one of the as in the sequence through the substitution of a fluorescent-labeled ddATP.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A similar set of fragments is made for each of the other three bases. But each set is labeled with a different fluorescent ddNTP.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The combined mixture of all labeled DNA fragments is electrophoresed to _____ the fragments by______ and the ladder of fragments is scanned for the presence of each of the four labels.`,
    options: ["separate, size", "separate, pH", "assimilate, pH", "assimilate, size"],
    correct: "a",
    explanation: `A computer program then determines the probable order of the bands and predicts the sequence. Depending on the actual procedure being used, one run may generate a reliable sequence of as many as 500 nucleotides.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The sequence can also be verified by making an oligonucleotide primer complementary to the distal part of the readable sequence and using it to obtain the sequence of the complementary strand on the original DNA template.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For accurate work, a printout of the scan is usually examined for abnormalities that decrease the quality of the sequence, and the sequence may then be edited manually. The first sequence can also be extended by making a second oligonucleotide matching the distal end of the readable sequence and using this primer to read more of the original template.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `When the process is fully automated, a number of priming sites may be used to obtain sequencing results that give optimal separation of bands in each region of the sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `By repeating this procedure, both strands of a DNA fragment several kilobases in length can be sequenced. Sequential sequencing of a DNA molecule using oligonucleotide primers is done later.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `To sequence larger molecules, individual chromosomes are purified and broken into _____ or larger random fragments, which are cloned into vectors designed for large molecules.`,
    options: ["100-Mb", "100-kb", "5000-kb", "600-kb"],
    correct: "b",
    explanation: `To sequence larger molecules, such as human chromosomes, individual chromosomes are purified and broken into 100-kb or larger random fragments, which are cloned into vectors designed for large molecules, such as artificial yeast (YAC) or bacterial (BAC) chromosomes. In a laborious procedure, the resulting library is screened for fragments called contigs, which have overlapping or common sequences, to produce an integrated map of the chromosome.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Many levels of clone redundancy may be required to build a consensus map because individual clones can have _______`,
    options: ["rearrangements", "deletions", "two separate fragments", "vectors"],
    correct: "d",
    explanation: `Option vectors here becomes irrelevant as it has quite less relevancy to redundancy of the clones. These do not reflect the correct map and have to be eliminated.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Once the correct map has been obtained, unique overlapping clones are chosen for sequencing.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `However, these molecules are too large for direct sequencing. One procedure for sequencing these clones is to subclone them further into smaller fragments that are of sizes suitable for sequencing, make a map of these clones and then sequence overlapping clones. However, this method is expensive because it requires a great deal of time to keep track of all the subclones.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `An alternative method is to sequence all the subclones, produce a computer database of the sequences, and then have the computer assemble the sequences from the overlaps that are found.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Up to 10 levels of redundancy are used to get around the problem of a small fraction of abnormal clones. This procedure was first used to obtain the sequence of the 4- Mb chromosome of the bacterium Haemophilus influenzae by The Institute of Genetics Research (TIGR) team. Only a few regions could not be joined because of a problem subcloning those regions into plasmids, requiring manual sequencing of these regions from another library of phage subclones.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is untrue about Shotgun Sequencing?`,
    options: ["When DNA fragments derived from different chromosomal regions have repeats of the same sequence, they will appear to overlap", "When DNA fragments derived from different chromosomal regions have repeats of the same sequence, they will appear to scrutinize", "In a new whole shotgun approach, Celera Genomics is sequencing both ends of DNA fragments of short (2 kb), medium (10 kb), and long (BAC or >100 kb) lengths", "A large number of reads are then assembled by computer"],
    correct: "b",
    explanation: `A controversy has arisen as to whether or not the above shotgun sequencing strategy can be applied to genomes with repetitive sequences such as those likely to be encountered in sequencing the human genome. This method has been used to assemble the genome of the fruit fly Drosophila melanogaster after removal of the most highly repetitive regions and also to assemble a significant proportion of the human genome.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Two common goals in sequence analysis are to identify sequences that encode proteins, which determine all cellular metabolisms, and to discover sequences that regulate the expression of genes or other cellular processes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Genomic sequencing meets both goals. However, only a small percentage of the genomic sequence of many organisms actually encodes proteins because of the presence of introns within coding regions and other noncoding regions in the genome.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `cDNA libraries have been prepared that have the same sequences as the mRNA molecules produced by organisms, or else cDNA copies are sequenced directly by RT-PCR (copying of mRNA by reverse transcriptase followed by sequencing of the cDNA copy by the polymerase chain reaction).`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `There has been a great deal of progress in developing computational methods for analyzing genomic sequences and finding these protein-encoding regions. But these methods are not completely reliable and, furthermore, such genomic sequences are often not available.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Using cDNA sequence with the __________ it is much simpler to locate protein-encoding sequences in these molecules.`,
    options: ["exons taken out", "exons removed", "introns added", "introns removed"],
    correct: "d",
    explanation: `The only possible difficulty is that a gene of interest may be developmentally expressed or regulated in such a way that the mRNA is not present. This problem has been circumvented by pooling mRNA preparations from tissues that express a large proportion of the genome, from a variety of tissues and developing organs or from organisms subjected to several environmental influences.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `An important development for computational purposes was the decision by Craig Venter to prepare databases of partial sequences of the expressed genes, called expressed sequence tags or ESTs.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This was an important development from resolution point of view. This has just enough DNA sequence to give a good idea of the protein sequence.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `The translated sequence can then be compared to a database of protein sequences with the hope of finding a strong similarity to a protein of known function, and hence to identify the function of the cloned EST.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The translated sequence can then be compared in the mentioned way hence to identify the function of the cloned EST. The corresponding cDNA clone of the gene of interest can then be obtained and the gene completely sequenced.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Investigators are encouraged to submit their newly obtained sequences directly to a member of the International Nucleotide Sequence Database Collaboration, such as the NCBI, DDBJ, and EMBL.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `NCBI stands for National Center for Biotechnology Information. It manages GenBank. DDBJ and EMBL stand for DNA Database Bank of Japan and European Molecular Biology Laboratory respectively.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `NCBI reviews new entries and updates existing ones, as requested.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `A database accession number, which is required to publish the sequence, is provided. New sequences are exchanged daily by the GenBank, EMBL, and DDBJ databases.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the given statements is incorrect?`,
    options: ["The processes of domain assortment and gene duplication produce families of proteins in organisms", "Following speciation, a newly derived genome will inherit the families of ancestor organisms, but will also develop new ones to meet evolutionary challenges", "Comparison of each of the proteins encoded by an organism with every protein, an all-against-all comparison, reveals which protein families have been amplified and what rearrangements have occurred as steps in the evolutionary process", "When two or more proteins in the proteome share a high degree of similarity they are least likely to be paralogs"],
    correct: "d",
    explanation: `When two or more proteins in the proteome share a high degree of similarity because they share the same set of domains, they are likely to be paralogs, genes that arose by gene duplication events. Proteins that align over shorter regions share some domains, but also may not share others. Although gene duplication events could have created such variation, other rearrangements may have also occurred, blurring the evolutionary history.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is untrue?`,
    options: ["Sequence and other data files that contain non-ASCII characters also may not be transferred correctly from one machine to another and may cause unpredictable behavior of the communications software", "The ASCII mode is useful for transferring text files, and the binary mode is useful for transferring compressed data files, which also contain non-ASCII characters", "ASCII and binary modes cannot be set by the user", "Most sequence analysis programs also require not only that a DNA or protein sequence file be a standard ASCII file, but also that the file be in a particular format such as the FASTA format"],
    correct: "b",
    explanation: `The file transfer program (FTP) has ASCII and binary modes, which may be set by the user. Some communications software can be set to ignore such control character. The use of windows on a computer has simplified such problems, since one merely has to copy a sequence from one window, for example, a window that is running a Web browser on the ENTREZ Web site, and paste it into another, for example, that of a translation program.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Granting agency requires a certain level of accuracy in case of errors. Which of the given statements is untrue regarding it?`,
    options: ["In other laboratories, such as those performing a single-attempt sequencing of ESTs, the error rate may be much higher, approximately 1 in 100, including incorrectly identified bases and inserted or deleted bases", "Incorrect bases always translate to the right amino acid", "Base insertions/deletions will cause frame-shifts in the sequence", "Making alignment with a protein sequence becomes difficult because of frameshifts"],
    correct: "b",
    explanation: `In translating EST sequences in GenBank and other databases, incorrect bases may translate to the wrong amino acid. Another type of database sequence that is error-prone is a fragment of sequence from the immunological variant of a pathogenic organism, such as the regions in the protein coat of the human immunodeficiency virus (HIV). Although this low level of accuracy may be suitable for some purposes such as identification, for more detailed analyses, e.g., evolutionary analyses, the accuracy of such sequence fragments should be verified.`,
    super_topic: `Uncategorized`,
  },
  {
    question: `Which of the following is wrong about GenBank DNA Sequence Entry?`,
    options: ["The information is organized into fields, each with an identifier, shown as the first text on each line", "In some entries, these identifiers may be abbreviated to two letters, e.g., RF for reference", "Some identifiers may have additional subfields", "The CDS subfield in the field FEATURES does not offer the amino acid sequence"],
    correct: "d",
    explanation: `The CDS subfield in the field FEATURES gives the amino acid sequence obtained by translation of known and potential open reading frames. The format of a database entry in GenBank, the NCBI nucleic acid and protein sequence database, is as follows: Information describing each sequence entry is given, including literature references, information about the function of the sequence, locations of mRNAs and coding regions, and positions of important mutations.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `A consecutive set of three-letter words that could be codons specifying the amino acid sequence of a protein. The sequence entry is assumed by computer programs to lie between the identifiers “ORIGIN” and “//”.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The sequence includes numbers on each line so that sequence positions can be located by eye. Because the sequence count or a sequence checksum value may be used by the computer program to verify the sequence composition, the sequence count should not be modified except by programs that also modify the count. The GenBank sequence format often has to be changed for use with sequence analysis software.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In Organization of the GenBank database and the search procedure used by ENTREZ—each row is another sequence entry and each column another GenBank field.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `When one sequence entry is retrieved, all of these fields will be displayed. Search for the term “SOS regulon and coli” in all fields will find two matching sequences. Finding these sequences is simple because indexes have been made listing all of the sequences that have any given term, one index for each field. Similarly, a search for transcriptional regulator will find three sequences.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is wrong about European Molecular Biology Laboratory Data Library Format?`,
    options: ["EMBL maintains DNA and protein sequence databases", "As with GenBank entries, a large amount of information describing each sequence entry is given", "Sequence entry includes literature references and information about the function of the sequence, but not locations of mRNAs and coding regions", "Information is organized into fields, each with an identifier, shown as the first text on each line"],
    correct: "c",
    explanation: `Sequence entry includes literature references and information about the function of the sequence, locations of mRNAs and coding regions and positions of important mutations. The sequence count or a checksum value for the sequence may be used by computer programs to make sure that the sequence is complete and accurate. For this reason, the sequence part of the entry should usually not be modified except with programs that also modify this count.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The format of an entry in the SwissProt protein sequence database is very similar to the EMBL format.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The format is quite similar to the EMBL format, except that considerably more information about the physical and biochemical properties of the protein is provided. Also, the output of a DDBJ DNA sequence entry is almost identical to that of GenBank.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is wrong about FASTA Sequence Format?`,
    options: ["The FASTA sequence format includes a comment line identified by a \u201c>\u201d character in the first column followed by the name and origin of the sequence", "The FASTA sequence format includes the sequence in standard one-letter symbols", "This format provides a very convenient way to copy just the sequence part from one window to another because there are no numbers or other nonsequence characters within the sequence", "The presence of \u2018*\u2019 is not quite essential for reading the sequence correctly by some sequence analysis programs"],
    correct: "d",
    explanation: `The FASTA sequence format includes an optional ‘*’ which indicates end of sequence and which may or may not be present and its presence maybe essential. The FASTA sequence format is similar to the protein information resource (NBRF) format except that the NBRF format includes a first line with a “>” character in the first column followed by information about the sequence, a second line containing an identification name for the sequence, and the third to last lines containing the sequence.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is wrong about National Biomedical Research Foundation/Protein Information Resource Sequence Format?`,
    options: ["Sequences retrieved from the PIR database are not in this compact format, but in an expanded format with much more information about the sequence", "The NBRF format is similar to the FASTA sequence format but with significant differences", "This is different than PIR format", "The first line includes an initial \u201c>\u201d character followed by a two-letter code such as P for complete sequence or F for fragment, followed by a 1 or 2 to indicate type of sequence, then a semicolon, then a four- to six-character unique name for the entry"],
    correct: "c",
    explanation: `This sequence format, which is sometimes also called the PIR format. It has been used by the National Biomedical Research Foundation/Protein Information Resource (NBRF) and also by other sequence analysis programs.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In Stanford University/Intelligenetics Sequence Format At the end of the sequence, a 1 is placed if the sequence is linear, and a 2 if the sequence is circular.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is started by a molecular genetics group at Stanford University, and subsequently continued by a company, Intelligenetics, the IG format is similar to the PIR format, except that a semicolon is usually placed before the comment line. The identifier on the second line is also present.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is wrong about Genetics Computer Group Sequence Format?`,
    options: ["Earlier versions of the Genetics Computer Group (GCG) programs require a unique sequence format and include programs that convert other sequence formats into GCG format", "Information about the sequence in the GenBank entry is not included but the line information is carried out", "If one or more sequence characters become changed through error, a program reading the sequence will be able to determine that the change has occurred because the checksum value in the sequence entry will no longer be correct", "Lines of information are terminated by two periods, which mark the end of information and the start of the sequence on the next line"],
    correct: "b",
    explanation: `Information about the sequence in the GenBank entry is first included, followed by a line of information about the sequence and a checksum value. This value (not shown) is provided as a check on the accuracy of the sequence by the addition of the ASCII values of the sequence. If the sequence has not been changed, this value should stay the same.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is wrong about Abstract Syntax Notation Sequence Format?`,
    options: ["The information is much more difficult to read by eye than a GenBank formatted sequence", "Abstract Syntax Notation (ASN.1) is a formal data description language that has been developed by the computer industry", "All the information found in other forms of sequence storage, e.g., the GenBank format, is present. For example, sequences can be retrieved in this format by ENTREZ", "Taxonomic information and bibliographic information cannot be encoded with this format"],
    correct: "d",
    explanation: `ASN.1 has been adopted by the National Center for Biotechnology Information (NCBI) to encode data such as sequences, maps, taxonomic information, molecular structures, and bibliographic information. These data sets may then be easily connected and accessed by computers. The ASN.1 sequence format is a highly structured and detailed format especially designed for computer access to the data.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about ASCII and Hexadecimal?`,
    options: ["Computers store sequence information as simple rows of sequence characters called strings, which are similar to the sequences shown on the computer terminal", "Each character is stored in binary code in the smallest unit of memory, called a byte", "Each character is stored in binary code in the smallest unit of memory, called a bit", "By convention, many of these combinations have a specific definition, called their ASCII equivalent"],
    correct: "b",
    explanation: `Each byte comprises 8 bits, with each bit having a possible value of 0 or 1, producing 255 possible combinations. Some ASCII values are defined as keyboard characters, others as special control characters, such as signaling the end of a line (a line feed and a carriage return), or the end of a file full of text (end-of-file character). A file with only ASCII characters is called an ASCII file.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `According to standard amino acid code letters which of the given pair is not right?`,
    options: ["K- lysine", "Y- tyrosine", "Q- glutamine", "R- serine"],
    correct: "d",
    explanation: `In addition to the standard four base symbols, A, T, G, and C, the Nomenclature Committee of the International Union of Biochemistry has established a standard code to represent bases in a nucleic acid sequence that is uncertain or ambiguous. R is represented by arginine.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `For computer analysis of proteins, it is more convenient to use single-letter than three letter amino acid codes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For example, GenBank DNA sequence entries contain a translated sequence in single-letter code. The standard, single-letter amino acid code was established by a joint international committee.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `which of the given statements is incorrect about Block multiple sequence alignment format?`,
    options: ["Identification starts contain a short identifier for the group of sequences from which the block was made and often is the original Prosite group ID", "The identifier is terminated by a comma, and \u201cBLOCK\u201d indicates the entry type", "AC contains the block number, a seven-character group number for sequences from which the block was made, followed by a letter (A\u2013Z) indicating the order of the block in the sequences", "The block number is a 5-digit number preceded by BL (BLOCKS database) or PR (PRINTS database)"],
    correct: "b",
    explanation: `The identifier is terminated by a semicolon, and “BLOCK” indicates the entry type. Min, max is the minimum, maximum number of amino acids from the previous blocks or from the sequence starting. DE describes sequences from which the block was made.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `BL contains information about the block: xxx is the amino acids in the spaced triplet found by MOTIF upon which the block is based.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In addition to this, w is the width of the sequence segments (columns) in the block. s is the number of sequence segments (rows) in the block. Other values (n1, n2) describe statistical features of the block. Sequence id is a list of sequences. Each sequence line contains a sequence identifier, the offset from the beginning of the sequence to the block in parentheses, the sequence segment, and a weight for the segment.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about READSEQ?`,
    options: ["It is an extremely useful sequence formatting program developed by D. G. Gilbert at Indiana University, Bloomington", "It was developed at Indiana University, Bloomington", "It can recognize a DNA or protein sequence file in any of the formats", "It can recognize a DNA or protein sequence file in some particular formats"],
    correct: "d",
    explanation: `It can identify the format, and write a new file with an alternative format. Some of these formats are used for special types of analyses such as multiple sequence alignment and phylogenetic analysis.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Data files that have multiple sequences, such as those required for multiple sequence alignment and phylogenetic analysis using parsimony (PAUP), are not converted in READSEQ.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Data files with such multiple sequences as mentioned are converted in READSEQ. Options to reverse-complement and to remove gaps from sequences are included. SEQIO and another sequence conversion program for a UNIX machine.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The “from” programs convert sequence files from GCG format into the named format, and the “to” programs convert the alternative format into GCG format.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In addition, the GCG programs include the following sequence formatting programs: (1) GETSEQ, which converts a simple ASCII file being received from a remote PC to GCG format; (2) REFORMAT, which will format a GCG file that has been edited, and will also perform other functions; and (3) SPEW, which sends a GCG sequence file as an ASCII file to a remote PC.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The Common Object Request Broker Architecture (CORBA) is the Object Management Group’s interface for objects.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It allows different computer applications to communicate with each other through a common language, Interface Definition Language (IDL). To plan an object-oriented database by defining the classes of objects and the relationships among these objects, a specific set of procedures called the Unified Modeling Language (UML) has been devised by the OMG group.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The FASTA format is readily converted into other formats and also is smaller and simpler.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It contains just a line of sequence identifiers followed by the sequence without numbers, is very useful for browsing and analyzing purposes. One browser window may retrieve sequences from a database and a second may analyze these sequences.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Each DNA or protein sequence database entry has much information, including ______`,
    options: ["an assigned accession number(s)", "source organism", "name of locus", "reference number type(s)"],
    correct: "d",
    explanation: `In addition to these keywords that apply to sequence, features in the sequence such as coding regions, intron splice sites, and mutations; and finally the sequence itself is given the sequence database entry. The above information is organized into a tabular form very much like that found in a relational database.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Database Types?`,
    options: ["Relational databases are more useful in the development of biological databases", "The tables in relational database are carefully indexed and cross-referenced with each other, sometimes using additional tables, so that each item in the database has a unique set of identifying features", "The relational database orders data in tables made up of rows giving specific items in the database, and columns giving the features as attributes of those items", "The two principal types of DBs are the relational and object-oriented databases"],
    correct: "a",
    explanation: `The object-oriented database structure has been useful in the development of biological databases. The objects, such as genetic maps, genes, or proteins, each have an associated set of utilities for analysis and display of the object and a set of attributes such as identifying name or references.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is incorrect about ENTREZ?`,
    options: ["There is no simple way to find the correct sequence without manually checking the information provided in each sequence, but this usually takes longer time", "Before leaving ENTREZ, it is often useful to check for sequence database entries that are similar to the one of interest, called \u201cneighbors\u201d by ENTREZ", "The expanded query searches other database entries of interest, such as the same protein in another organism, a large chromosomal sequence that includes the gene, or members of the same gene family", "While visiting the site, note that ENTREZ has been adapted to search through a number of other biological databases, and also through Medline, and these searches are available from the initial ENTREZ Web page"],
    correct: "a",
    explanation: `Opposite to what is mentioned in option a, this takes shorter time. It is important to look through the sequences to locate the one intended. There may be several different copies of the sequence because it may have been sequenced from more than one organism, or the sequence may be a mutant sequence, a particular clone, or a fragment.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The databases Genbank, EMBL and DDBJ are updated daily.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The mentioned database centers are updated daily and exchange new sequences daily, so that it is only necessary to access one of them. The EMBL stands for European Molecular Biology Laboratory and DDBJ for DNA DataBank of Japan.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Using boolean logic, the search looks for database entries that include the first term __________ the second, and subsequent terms repeated until the last term.`,
    options: ["AND", "OR", "ExOR", "NAND"],
    correct: "a",
    explanation: `On the ENTREZ form, make a selection in the data entry window after the term “Search,” then enter search terms in the longer data entry window after “for.” The database will be searched for sequence database entries that contain all of these terms or related ones.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `To assist in finding suitable terms, for each field, ENTREZ provides a list of index entries.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `When searching for terms in a particular field, some knowledge of the terms that are in the database can be helpful. The “Limits” link on the ENTREZ form page is used to limit the GenBank field to be searched, and various logical combinations of search terms may be designed by this method. These fields refer to the GenBank fields.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `For a protein search, which of the following is the wrong choice for fields?`,
    options: ["Accession (number)", "E. C. number", "Issue", "Journal number"],
    correct: "d",
    explanation: `Other fields being- author name, journal name, keyword, modification date. Also, it includes organism, page number, primary accession (number), properties, protein name, publication date (of reference), seqID string, sequence length, substance name, text word, title word, volume, and sequence ID. Similar fields are shown for the DNA database search.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The results of searches in separate fields may be combined to narrow down the choices.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The number of terms to be searched for and the field to be searched is the main decisions to be made. In doing so, it is important to be as specific as possible, or else there may be a great many possibilities.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Knowing ________ should be enough to find the required entry quickly.`,
    options: ["publication date, protein name, journal name", "accession number, protein name, or name of gene", "publication date, protein name, or volume", "properties, protein name, or title word"],
    correct: "b",
    explanation: `If the same protein has been sequenced in several organisms, providing an organism name is also helpful. When the chosen search terms and fields have been decided and submitted, a database comprising all of the currently available sequences (called the non redundant or NR database) will be searched. Other database selections can also be made.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The program returns the number of matches found and provides an opportunity to narrow this list by including more terms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `When the number of matching sequences has been narrowed to a reasonable number, the sequence may be retrieved in a chosen format in several straightforward steps. This helps in getting to the required data in less number of steps.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following is incorrect about Retrieving a Specific Sequence?`,
    options: ["It can be difficult to retrieve the sequence of a specific gene or protein simply because of the sheer number of sequences in the Gen-Bank database and the complex problem of indexing them", "Other projects may benefit from the availability of better curated and annotated protein sequence databases, but not PIR and SwissProt", "For projects that require the most currently available sequences, the NR databases should be searched", "The genomic databases can also provide the sequence of a particular gene or protein. Protein sequences in the Genpro database are generated by automatic translation of DNA sequences"],
    correct: "b",
    explanation: `Curated and annotated protein sequence databases include PIR and SwissProt. When read from cDNA copies of mRNA sequences, they provide a reliable sequence, given a certain amount of uncertainty as to the translational start site. Many protein sequences are now predicted by translation of genomic sequences, requiring a prediction of exons, a somewhat error-prone step.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The chromosomes comprised linear DNA molecules in a tightly compact form that was wrapped around protein complexes, called the nucleosome.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Nuclei and chromosomes were not observed in bacteria (a prokaryotic cell), but when bacterial DNA was eventually detected, the molecule was usually circular and was also in a compacted form. The following sections outline the structure and composition of prokaryotic and eukaryotic genomes.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The first bacterial genome to be sequenced was that of ________ a mild human pathogen.`,
    options: ["Hemophilus influenzae", "Lactobacillus", "Vibrio cholarae", "Clostridium botulinum"],
    correct: "a",
    explanation: `This project was carried out at the Institute of Genomics Research. It was carried out in part to prove a new genome sequencing method—the shotgun method.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `While sequencing of the first bacterial genome–A large number of random overlapping fragments were sequenced and then a consensus sequence of the entire ______ chromosome of Hemophilus was assembled by computer.`,
    options: ["8.6 x 109 bp", "1.8 x 106 bp", "6.9 x 105 bp", "1.8 x 104 bp"],
    correct: "b",
    explanation: `It was done excepting several regions that had to be assembled manually. Once available, open reading frames were identified, and these were compared to the existing proteins by a database similarity search.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `While sequencing of The first bacterial genome—Approximately ____ of the ____ predicted genes matched genes of another species, the bacterial species E. coli K-12 that had been the subject of many years of genetic and biochemical research.`,
    options: ["46%, 1500", "58%, 1496", "72%, 1743", "58%, 1743"],
    correct: "d",
    explanation: `The function of the other 42% of the Hemophilus genes could not be identified, although some of them were similar to the 38% of E. coli genes that were also of unknown function. Other unique sequences that appeared to be associated with the ability of the organism to behave as a human pathogen were also found.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `After sequencing the Hemophilus genome and Organisms were selected for sequencing based on minimum criteria. Which of the following is not one of them?`,
    options: ["They had been subjected to a good deal of biological analysis", "They were model eukaryotic organisms", "They were an important human pathogen, e.g., Mycobacterium tuberculosis (tuberculosis)", "They were of phylogenetic interest"],
    correct: "b",
    explanation: `There were model prokaryotic organisms. For e.g., E. coli and Bacillus subtilis. The success of sequencing the Hemophilus genome in a relatively short time and with a modest budget heralded the sequencing of a large number of additional prokaryotic organisms.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Analysis of the ribosomal RNA molecules of prokaryotes and eukaryotes had led to the prediction of three main branches in the tree of life.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The three branches are represented by Archaea, the Bacteria, and the Eukarya. Analysis of the ribosomal for genome sequencing projects, organisms have been sampled from throughout the tree, including some that are in deeper branches of the tree and that have growth properties reminiscent of an ancient environment.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Annotation involves identifying open reading frames in the genome sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `It is done by using the predicted protein as query sequences in a database similarity search. It further adds significant matches to the genome sequence entry in the sequence database.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `A simple way to retrieve sequences of viral and other extra-chromosomal genetic elements such as organelles is through the National Center for Biotechnology Information (NCBI).`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Prior to the sequencing of H. influenzae, the first free-living organism to be sequenced, a large number of viruses had been sequenced. Many of these organisms also serve as model systems for studying replication and gene expression. As an example, the nucleotide sequence of bacteriophage lambda was completed by Sanger.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given is wrongly matched?`,
    options: ["Escherichia coli \u2013 Bacteria", "Methanococcus jannaschii \u2013 Archaea", "Synechocystis sp. \u2013 Archaea", "Aquifex aeolicus \u2013 Bacteria"],
    correct: "c",
    explanation: `Synechocystis sp. – Bacteria is the correct pair. It is an ancient organism that produces oxygen by light-harvesting.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In examining the results of analysis, it is important to look for the method used the statistical significance of the result, and the overall degree of confidence in the alignments.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The analysis should be repeated if necessary. Annotation errors occur when the above criteria are not followed.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Computational resources can facilitate the analysis of bacterial genomes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `GeneQuiz is an example of such a resource. There are Web sites that provide a complete annotation of the prokaryotic genomes that have been sequenced.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Telomeres hold newly replicated daughter chromosomes together.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Centromeres hold newly replicated daughter chromosomes together. They serve as a point of attachment for pulling the chromosomes apart during cell division.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Prokaryotic genomes commonly have tandem repeats of sequences and include introns in protein-coding genes.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Eukaryotic genomes commonly have tandem repeats of sequences. In addition to this, they having linear chromosomes within a nucleus, and differing from prokaryotic genomes in this respect.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The sequences of satellite DNA fall into different types, each with a different repeat unit of length _____`,
    options: ["5\u2013400 Mbp", "3\u2013300 kbp", "6\u2013900 Mbp", "5\u2013200 bp"],
    correct: "d",
    explanation: `Because of the skewed base composition of regions that have repeats, they may be purified by virtue of having different buoyant densities. The repeat unit of length is 5–200 bp and not in the scale if Mega.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Most of the repetitive DNA is found near the open ends of chromosomes.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Some of it is found near the open ends of chromosomes- telomeres but Most of this repetitive DNA is found near the centromere. It is because they serve as a point of attachment for pulling the chromosomes apart during cell division.
Free 30-Day C++ Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Minisatellites are made up of repeat units of up to ____ and microsatellites compose of repeat units of ____ or less.`,
    options: ["25 bp, 10 bp", "70 bp, 6 bp", "80 bp, 9 bp", "25 bp, 4 bp"],
    correct: "d",
    explanation: `They are also found in eukaryotic Genomes. Microsatellite repeats are found at the ends of eukaryotic chromosomes at the telomeres, which in humans comprise hundreds of copies of a 6-bp repeat TTAGGG.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In nondividing cells, a mixture of lightly and darkly stained chromosomal regions called euchromatin and heterochromatin respectively, are observed.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In nondividing cells, a mixture of lightly and darkly stained chromosomal regions called heterochromatin and euchromatin respectively, are observed. The centromeric and telomeric regions are located in the heterochromatin, which is in a compact configuration and is thought not to be transcribed.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Genes that are transcribed are located in the ______`,
    options: ["euchromatin", "heterochromatin", "heterochromatin and euchromatin", "tightly bound DNA"],
    correct: "a",
    explanation: `They are located in the less compact euchromatin. This gives the regulatory proteins access to the genetic material.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `______ can comprise a large proportion of the eukaryotic genome as ______`,
    options: ["transposable elements, single copy sequences", "transposable elements, repetitive sequences", "macrosatellite DNA elements, single copy sequences", "satellite DNA elements, single copy sequences"],
    correct: "b",
    explanation: `Transposable elements (TEs) are thought to play an important role in the evolution of these genomes. TEs are DNA sequences that can move from one chromosomal location to another faster than the chromosome can replicate.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `(Transposable Elements) TEs have the potential to _____ in number until they comprise a ____ proportion of the genome sequence.`,
    options: ["decrease, large", "decrease, micro", "increase, micro", "increase, large"],
    correct: "d",
    explanation: `It is a feature already observed in many plants and animals. They remain detectable in the genome until they blend into the background sequence by mutation.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `More than _____ of the human genome consists of interspersed repetitive sequences derived from TEs (transposable elements).`,
    options: ["one-third", "one-eighth", "one-fifth", "half"],
    correct: "a",
    explanation: `The presence of these elements may be demonstrated using programs for detection of low-complexity regions in sequences. For e.g. in the fruit fly Drosophila has 15% of genome that is made up of transposable elements.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The retroposons include short _________ interspersed nuclear elements (SINES).`,
    options: ["90\u20134000 bp long", "80\u2013500 Mbp long", "80\u2013300 bp long", "100\u20133000 bp long"],
    correct: "c",
    explanation: `There exists also (6–8 kbp long) interspersed nuclear elements (LINES). Different types of transposable elements are present in high copy numbers in mammalian genomes in varying manner.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `_____ of the human genome comprises one particular family of the SINE Element, designated Alu (1.2 million copies).`,
    options: ["10%", "20%", "60%", "40%"],
    correct: "a",
    explanation: `Ten percent of the human genome comprises one particular family of the SINE Element. And 14.6% of one particular LINE designated LINE1 (593,000 copies) are present.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Vertebrate chromosomes have long (>300 kb) regions of distinct GC richness, repeat content, and gene density, designated isochores in a model of genome organization proposing that genomes are made up of distinct segments of unique composition.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Human and mouse chromosomal regions that have a low density of genes are AT-rich and have more Alu or B1/B2 (SINES) than LINE1 elements. Whereas the reverse is true for regions that have a high gene density, and those regions are more GC-rich.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The human genome contains about _________ of class II of elements that probably predate human evolution (Smit 1996).`,
    options: ["2,000 copies", "200,000 copies", "2,00,00,000 copies", "20,00,000 copies"],
    correct: "b",
    explanation: `The class of TEs, class II, is made up of elements that employ a DNA-based mechanism of transposition. Class II elements also include the Activation-Dissociation (Ac-Ds) family in maize and the P element in Drosophila.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `A third category of TEs has features of both class I and class II TEs. These miniature, inverted repeat TEs (MITES) are ________ in length.`,
    options: ["400 bp", "500 Mbp", "300 kbp", "600 kbp"],
    correct: "a",
    explanation: `They were discovered in diverse flowering plants where they are frequently associated with regulatory regions of genes. Hence, they could be exerting an influence on regulation of gene expression.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given features is incorrect?`,
    options: ["TEs are present in few particular chromosomes", "TEs are present in all of the chromosomes", "Abundance of TEs varies", "TEs can comprise a large portion of the genomes of higher eukaryotes, both plants and animals"],
    correct: "a",
    explanation: `TEs are present in all of the chromosomes, ranging from bacteria to humans, but their abundance varies. They can comprise a large portion of the genomes of higher eukaryotes, thus, only a small fraction of the genome of these organisms carries gene sequences.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Eukaryotic genes that encode proteins are interrupted by ________`,
    options: ["exons of varying length and number", "introns of varying length and number", "exons of varying length and but same number", "introns of varying number but same length"],
    correct: "b",
    explanation: `In S. cerevisiae (budding yeast), only a small fraction of the genes contain introns, and there are a total of 239 introns in the entire genome. In contrast, in individual human genes, introns may be present in numbers exceeding 100 and comprise more than 95% of the gene.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Introns can remain at a corresponding position in a eukaryotic gene for long periods of evolutionary time.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The origin of introns in eukaryotic genes is not understood but has been accounted for by two models. The “introns-early” view proposes that introns were used to assemble the first genes from sets of ancient conserved exons, whereas the “introns-late” view proposes that introns broke up previously continuous genes by inserting into them.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The intron structure of genes in a particular eukaryote is used for predicting the location of genes of genome sequences.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Other features of eukaryotic genes in a particular organism that are useful for gene prediction include the consensus sequences at exon–intron and intron–exon splice junctions, base composition, codon usage, and preference for neighboring codons. Computational methods incorporate this information into a gene model that may be used to predict the presence of genes in a genome sequence.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Sequencing of genomes depends on the assembly of a large number of DNA reads into a linear, contiguous DNA sequence.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The cost and efficiency of this process has been greatly improved by automatic methods of sequence assembly, first used for the sequencing of the bacterium H. influenza. This same method of assembly was also used, in part, to complete the sequencing of the Drosophila and human genomes in a timely manner.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Each genome sequence is scanned for protein-encoding genes using gene models trained on known gene sequences from the same organism.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `For a new genome, each predicted gene is translated into a protein sequence; the collection of protein sequences encoded by the genome is the proteome of the organism. every protein in the proteome is then used as a query sequence in a database similarity search. Matching database sequences are realigned with the query sequence to evaluate the extent and significance of the alignment.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Screening the predicted protein sequences against ______ library confirms the prediction and expression of the gene.`,
    options: ["expressed sequence tag (EST)", "tags", "palindromes", "proteomes"],
    correct: "a",
    explanation: `The collective information on proteome function can then be further analyzed by self-comparison to find duplicated genes (paralogs) and by a proteome-by-proteome comparison to identify orthologs, genes that have maintained the same function through speciation, and other sequence and evolutionary relationships that are important for metabolic, regulatory, and cellular functions.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In case of genome sequence assembly which of the given statement is incorrect?`,
    options: ["Full chromosomal sequences are assembled from the overlaps in a highly redundant set of fragments by an automatic computational method or from the fragment order on a physical map", "Chromosome cloning is carried out in bacterial artifical chromosomes (BACs)", "Chromosomes of a target organism are purified, fragmented, and subcloned in fragments of size hundreds of bp", "Genome sequences are assembled from DNA sequence fragments of approximate length 500 bp obtained using DNA sequencing machines"],
    correct: "c",
    explanation: `Chromosomes of a target organism are purified, fragmented, and subcloned in fragments of size hundreds of kbp and not bp. The BAC fragments are then further subcloned as smaller fragments into plasmid vectors for DNA sequencing.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `TEs (Transposable Elements) can at most comprise one-fourth of the genome sequence.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `TEs (Transposable Elements) can comprise one-half or more of the genome sequence. Eukaryotic genomes comprise classes of repeated elements, including tandem repeats present in centromeres and telomeres, dispersed tandem repeats (minisatellites and macrosatellites), and interdispersed TEs.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Gene identification in prokaryotic organisms is simplified by their lacking _________`,
    options: ["exons", "introns", "coding segments", "useful nucleotide sequences"],
    correct: "b",
    explanation: `Once the sequence patterns that are characteristic of the genes in a particular prokaryotic organism (e.g, codon usage, codon neighbor preference) have been found, gene locations in the genome sequence can be predicted quite accurately. The presence of introns in eukaryotic genomes makes gene prediction more involved because, in addition to the above features, locations of intron–exon and exon–intron splice junctions must also be predicted.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect?`,
    options: ["In a given organism or species, genes are found in a given order that is maintained on the chromosomes from one generation to the next", "Genes with a related function are frequently found to be distorted on a chromosome", "A possibility is that there is genetic variation (alleles) within each gene in a cluster of a given species and that only certain allelic combinations of different genes are compatible", "Clustering of related genes presumably provides an evolutionary advantage to a species"],
    correct: "b",
    explanation: `Genetic analysis has revealed that genes with a related function are frequently found to be clustered at one chromosomal location. As genome-by-genome comparisons of the chromosomes of related species are made and the rearrangements are discovered, a further challenge to computational and evolutionary biologists is to estimate the number and types of rearrangements that have occurred and also to determine when they occurred. For example, a comparison of the mouse and human chromosomes reveals many rearrangements.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the following information is not directly obtained by microarray analysis?`,
    options: ["Which genes are expressed at a particular stage of the cell cycle", "Which genes are expressed at a particular stage of developmental cycle of an organism", "Which genes are depleted at what time", "Genes that respond to a given environmental signal to the same extent"],
    correct: "c",
    explanation: `For chronological information there are other numerous techniques that can be followed. This type of information provides an indication as to which genes share a related biological function or may act in the same biochemical pathway and may thereby give clues that will assist in gene identification.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect about Functional Genomics?`,
    options: ["Functional genomics involves the preparation of mutant or transgenic organisms with a mutant form of a particular gene usually designed to prevent expression of the gene", "An abnormal properties of the mutant organism does not reveal the gene function", "When two or more members of a gene family are found, rather than a single match to a known gene, the biological activity of these members may be analyzed by functional genomics to look for diversification of function in the family", "A more detailed analysis of the relative amount of sequence variability in a chromosomal region within populations of closely related species can reveal the presence of genes that are under selection"],
    correct: "b",
    explanation: `The gene function is revealed by any abnormal properties of the mutant organism. This methodology provides a way to test a gene function that is predicted by sequence similarity to be the same as that of a gene of known function in another organism. If the other organism is very different biologically (comparing a predicted plant or animal gene to a known yeast gene), then functional genomics can also shed light on any newly acquired biological role.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect about the gene map?`,
    options: ["Gene order in two related organisms reflects the order that was present in a common ancestor genome. Chromosomal breaks followed by reassembly of fragments in a different order can produce new gene maps", "Gene order is only revealed by the physical order of genes on the chromosome", "Sequence variations (polymorphisms) that are close to (tightly linked) a trait may be used to trace the trait by virtue of the fact that the polymorphism and the trait are seldom separated from one generation to the next", "These types of evolutionary changes in genomes have been modeled by computational methods"],
    correct: "b",
    explanation: `Gene order is revealed not only by the physical order of genes on the chromosome but also by genetic analysis. Populations of an organism show sequence variations that are readily detected by DNA sequencing and other analysis methods. The inheritance of genetic diseases in humans and animals (e.g., cancer and heart disease), and of desirable traits in plants, can be traced genetically by pedigree analysis or genetic crosses.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In the program COGNITOR each protein in the proteome is used as a query of a database of protein clusters.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The database was made by performing an all-by-all genome comparison across a spectrum of prokaryotic organisms and a portion of the yeast proteome. Orthologous pairs of sequence were then merged with clusters or orthologous pairs (COGs) for multiple proteomes.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `WU-BLAST produces P scores and BLAST (NCBI) produces E scores where _________`,
    options: ["E = ln (1 + P2)", "E = ln (1 \u2013 P2)", "E = ln (1 + P)", "E = ln (1 \u2013 P)"],
    correct: "d",
    explanation: `For values less than 0.05, E = P. The choice of a < 10-20 score is a conservative one for identification of orthologs that should have a similar domain structure.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In Proteolysis and fragment sequencing, Protein spots may be excised from a two-dimensional protein gel and subjected to a combination of amino acid sequencing and cleavage analyses using the techniques of mass spectrometry and high-pressure liquid chromatography.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Genome regions that encode these sequences can then be identified and the corresponding gene located. A similar method may be used to identify the gene that encodes a particular protein that has been purified and characterized in the laboratory.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In Protein 2D gel Electrophoresis, Individual proteins produced by the genome can be separated to _____ by this method and specific ones identified by various _______`,
    options: ["smaller extent, biochemical and immunological tests", "a large extent, biochemical and immunological tests", "a large extent, biochemical tests only", "smaller extent, purely mechanical tests"],
    correct: "b",
    explanation: `Moreover, changes in levels of proteins in response to an environment signal can be monitored in much the same way as a microarray analysis is performed. Microarrays only detect untranslated mRNAs, whereas a two-dimensional gel protein analysis detects translation products, thus revealing an additional level of regulation.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In Metabolic pathways and regulation, as genes are identified in a new genome sequence, some will be found that are known to act sequentially in a metabolic pathway or to have a known role in gene regulation in other organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `From this information, the metabolic pathways and metabolic activities of the organism will become apparent. In some cases, the apparent absence of a gene in a well-represented pathway may lead to a more detailed search for the gene. Clustering of genes in the pathway on the genome of a related organism can provide a further hint as to where the gene may be located.
Free 30-Day C Certification Bootcamp is Live. Join Now!`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `_______ of the Drosophila sequence is composed of TEs and _____ is heterochromatic regions that do not include genes.`,
    options: ["one-fourth, one-third", "one-fifth, one-fourth", "one-sixth, one-eighth", "one-sixth, one-third"],
    correct: "d",
    explanation: `Hence, in the euchromatic regions, the gene density in the Drosophila genome is one gene per 9 kb. Despite the fact that the lower number of predicted genes in Drosophila is smaller than that of the other genomes, the amount of functional diversity, as evidenced by protein family representation, is similar.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Yeast is about ______ compact than E. coli.`,
    options: ["fivefold, less", "threefold, more", "twofold, less", "twofold, more"],
    correct: "c",
    explanation: `Of the remaining genomes, C. elegans and A. thaliana have approximately the same density of genes (one gene per 6 kb). Drosophila is the least dense in this comparison (one gene per 14 kb).`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The ________ are _______ by genetic structure to retroviruses.`,
    options: ["STR retrotransposons, related", "LTR retrotransposons, related", "STR transposons, related", "LTR retrotransposons, not related"],
    correct: "b",
    explanation: `There are three main subclasses of these TEs—the long terminal repeat (LTR) retrotransposons, retroposons, and retrovirus-like elements with LTRs. Class I elements encode reverse transcriptase and use RNA mediated mechanisms of transcription.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The higher the E value, the more significant the alignment between a pair of matching sequences.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `The lower the E value, the more significant the alignment between a pair of matching sequences. The E value of an alignment score is the probability that an alignment score as good as the one found would be observed between two random or unrelated sequences in a search of a database of the same size.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect about proteome analyses?`,
    options: ["For BLAST, setting an effective database size appropriate for each search and program is important", "Due to the large number of comparisons that must be made in these types of analyses and due to the volume of program output, the procedure must be automated on a local machine using Perl scripts or a similar method and a database system", "BLAST is used for obtaining a correct statistical evaluation of alignment scores", "BLAST does not give statistical evaluation of alignment scores"],
    correct: "d",
    explanation: `Each protein encoded by the genome is used as a query in database similarity searches to identify similar database proteins, some having a known structure or function. Additional searches of EST databases can be used to identify additional relatives of the query sequence.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `An all-against-all analysis requires first making a database of the proteome. This database is then sequentially searched by each individual protein sequence of the proteome using a rapid database similarity search tool such as _______`,
    options: ["XPBLAST", "WU-BLAST", "BLAST", "FASTA"],
    correct: "a",
    explanation: `P values of WU-BLAST are similar to E values of NCBI BLAST (Rubin et al. 2000) for values of P and E < 0.05. This analysis generates a matrix of alignment scores, each with an E value and corresponding alignment for each pair of proteins. Also, the E value of an alignment score is the probability that an alignment score as good as the one found would be observed between two random or unrelated sequences in a search of a database of the same size.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Evolutionary modeling can include various types of analyses. Which is mostly not one of them?`,
    options: ["The prediction of chromosomal rearrangements", "Eu/Hetero-chromatin structures", "Duplications at gene, chromosomal and full genome level", "Duplications at the protein domain level"],
    correct: "b",
    explanation: `Option Eu/Hetero-chromatin structures indicate structural studies. Also as mentioned, analysis of the prediction of chromosomal rearrangements that preceded the present arrangement is done (e.g., a comparison of mouse and human chromosomes).`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect about Clusters of functionally related genes?`,
    options: ["In microbial genomes, genes specifying a metabolic pathway may be contiguous on the genome where they are coregulated transcriptionally in an operon by a common promoter", "In related organisms, gene order on the chromosome is least likely to be conserved", "As the relationship between the organisms decreases, local groups of genes remain clustered together, but chromosomal rearrangements move the clusters to other locations", "The function of a particular gene can sometimes be predicted, given the known function of a neighboring, closely linked gene"],
    correct: "b",
    explanation: `In related organisms, both gene content of the genome and gene order on the chromosome are likely to be conserved.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statement is incorrect about Orthologs?`,
    options: ["In comparing two proteomes, a common standard is to require that for each pair of orthologs, the first of the pair is the best hit when the second is used to query the proteome of the first", "To identify orthologs, each protein in the proteome of an organism is used as a query in a similarity search of a database comprising the proteomes of only one different organism", "The best hit in each proteome is likely to be with an ortholog of the query gene", "Orthologs are genes that are so highly conserved by sequence in different genomes that the proteins they encode are strongly predicted to have the same structure and function and to have arisen from a common ancestor through speciation"],
    correct: "b",
    explanation: `To identify orthologs, each protein in the proteome of an organism is used as a query in a similarity search of a database comprising the proteomes of one or more different organisms.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In protein/domain analysis, each protein in the predicted proteome is again used as a query of a curated protein sequence database such as ____ in order to locate similar domains and sequences. To find orthologs, very low E value scores (E<10<20) for the alignment score and an alignment that includes 60–80% of the query sequence are generally required in order to avoid matches to paralogs.`,
    options: ["PubChem", "Genbank", "MeSH", "SwissProt"],
    correct: "d",
    explanation: `The domain composition of each protein is also determined by searching for matches in domain databases such as Interpro. The analysis reveals how many domains and domain combinations are present in the proteome, and reveals any unusual representation that might have biological significance. The number of expressed genes in each family can also be compared to the number in other organisms to determine whether or not there has been an expansion of the family in the genome.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `In all-against-all self comparison, A comparison is made in which every protein is used as a query in a similarity search against a database composed of the rest of the proteome, and the significant matches are identified by a low expect value.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Many proteins comprise different combinations of a common set of domains, proteins that align along most of their lengths (80% identity is a conservative choice). Hence they are chosen to select those that have a conserved domain structure.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Processed pseudogenes are also derived from a functional gene and they contain introns and a promoter.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Processed pseudogenes are also derived from a functional gene, but they do not contain introns and lack a promoter; hence, they are not expressed. The origin of these pseudogenes is probably due to reverse transcription of the mRNA of the functional gene and insertion of the cDNA copy into a new chromosomal location by a LINE1 reverse transcriptase.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Pseudogenes are DNA sequences that were derived from distinct genes but which have acquired mutations that are deleterious to function in the same period of time.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `Pseudogenes are DNA sequences that were derived from a functional copy of a gene but which have acquired mutations that are deleterious to function. For e.g. the pseudogene TRY5 is similar to the nearby functional gene TRY4.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `New gene functions are thought to be gained by duplication of an existing gene creating two tandem copies.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Functional differentiation then occurs between the copies by mutation and selection. However, because most mutations are deleterious, and because only one gene copy may be needed for function, there is a strong tendency of one copy to accumulate mutations that render the gene nonfunctional.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Comparative genomics includes a comparison of gene number, gene content, and gene location in both prokaryotic and eukaryotic groups of organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The availability of complete genome sequences makes possible a comparison of all of the proteins encoded by one genome, the proteome of that organism, with those of another. Because the genome sequence provides both the sequence and the map location of each gene, both the sequence and location can be compared.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the following information Sequence comparisons do not provide?`,
    options: ["Gene relationships", "Function history", "Evolutionary history", "Gene locations"],
    correct: "d",
    explanation: `Map locations of orthologous genes may also be compared. If a set of genes is grouped together at a particular chromosomal location, and if a set of similar genes is also grouped together in the genome of another organism, these groups share an evolutionary history.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about All-against-all Self-comparison?`,
    options: ["A comparison of each protein in the proteome with all other proteins distinguishes unique proteins from proteins that have arisen from gene duplication, and also reveals the number of protein families but the domain content of these proteins cannot be known", "In all-against-all proteome comparison, each protein is used as a query in a similarity search against the remaining proteome", "In all-against-all proteome comparison, the similar sequences are ranked by the quality and length of the alignments found", "In all-against-all proteome comparison, The search is conducted with each alignment score receiving a statistical evaluation (P or E value)"],
    correct: "a",
    explanation: `The domain content of these proteins may also be analyzed. in all-against-all proteome comparison, a match between a query sequence and another proteome sequence with the same domain structure will produce a high-scoring, highly significant alignment. These proteins are designated paralogs because they have almost certainly originated from a gene duplication event.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Cluster analysis?`,
    options: ["Clustering organizes the proteins into groups by some objective criterion", "One criterion for a matching protein pair is the statistical significance of their alignment score", "The P or E value from BLAST searches cannot be the criterion for a matching protein pair", "A criterion for clustering proteins is the distance between each pair of sequences in a multiple sequence alignment"],
    correct: "c",
    explanation: `Option c and b mean the same yet are different by the negation in option c.The lower this value, the better the alignment. There will be a cutoff P or E value at which the matches in the BLAST search are no longer considered significant. A value of P or E = 0.01–0.05 is usually the point at which the alignment score is no longer considered to be significant in order to focus on a more closely related group of proteins.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Clustering by making subgraphs?`,
    options: ["Each sequence is a vertex and each pair of sequences that is matched with a significant alignment score is joined by an edge that is weighted according to the statistical significance of the alignment score", "One way to identify the most strongly supported clusters is simply to add the most weakly supported edges in the graph", "One way to identify the most strongly supported clusters is simply to remove the most weakly supported edges in the graph", "An edge is weighted according to the statistical significance of the alignment score"],
    correct: "c",
    explanation: `As weaker and weaker links are removed, the remaining combinations of vertices and edges represent most strongly linked sequences. This type of analysis was performed on an initial collection of E. coli genes by Labedan and Riley (1995).`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Clustering by single linkage?`,
    options: ["In First step, a group of related sequences found in the all-against-all proteome comparison is subjected to a multiple sequence alignment usually by CLUSTALW", "A neighbor-joining algorithm is rarely used in this method", "This procedure and the algorithms are the same as those used to make a phylogenetic tree by the distance methods", "A distance matrix that shows the number of amino acid changes between each pair of sequences is made"],
    correct: "b",
    explanation: `The matrix is then used to cluster the sequences by a neighbor-joining algorithm. These methods produce a tree or a different representation of the tree called a dendrogram, which minimizes the number of amino acid changes that would generate the group of sequences.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `The all-against-all analyses provide an indication as to the number of protein/gene families in an organism. This number represents the core proteome of the organism from which all biological functions have diversified.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `In Hemophilus, 1247 of the total number of 1709 proteins do not have paralogs.
The core proteomes of the worm and fly are similar in size but with a greater number of duplicated genes in the worm. It is quite remarkable that the core proteome of the multicellular organisms (worm and fly) is only twice that of yeast.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Grouping Sequences?`,
    options: ["The problem of deciding which sequences to include in the same group or cluster and which to separate into different groups or clusters is a recurring one", "Divergence is necessary, but the sequences chosen should be clearly related based on inspection of each pair-wise alignment and a statistical analysis", "The conservative approach is to group distinct sequences", "The adventurous approach is to choose a set of marginally alignable sequences to pursue the difficult task of making a multiple sequence alignment and then to make profile models that may recognize divergence but will also give false predictions"],
    correct: "c",
    explanation: `The conservative approach is to group only very similar sequences together. However, in making a conservative multiple sequence alignment with only very alike sequences, it is not possible to analyze the evolutionary divergence that may have occurred in a family of proteins. Furthermore, if a matrix or profile model is made from this alignment, that model will not be useful for identifying more divergent members of a family.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Clusters of orthologous groups?`,
    options: ["Paralogs may include a best hit or a high-scoring match of one of the sequences by another, but the reciprocal match can have low similarity that does not have to be significant", "Paralogs defined by sets of three matching sequences in the selected organisms were kept separated from the clusters", "Orthologous pairs were first defined by the best hits in reciprocal searches", "To produce COGs, similarity searches were performed among the proteomes of phylogenetically distinct clades of prokaryotes"],
    correct: "b",
    explanation: `Paralogs defined by sets of three matching sequences in the selected organisms were also added to these clusters. Sixty percent of the original set of 720 COGs does not include paralogs, or includes paralogs from one lineage only, suggesting that there has not been extensive duplication of this group.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about the Comparison of proteomes to EST databases of an organism?`,
    options: ["ESTs are single DNA sequence reads that contain a small fraction of incorrect base assessments, insertions, and deletions", "Many sequences arise from near the 5\u2019 end of the mRNA, although every effort is usually made to read as far 3\u2019 as possible into the upstream portion of the cDNA", "EST libraries are useful for preliminary identification of genes by database similarity searches", "An EST database of an organism can be analyzed for the presence of gene families,"],
    correct: "b",
    explanation: `Many sequences arise from near the 3’ end of the mRNA, although every effort is usually made to read as far 5’ as possible into the upstream portion of the cDNA. Because not all of the genes may be expressed in the tissues chosen for analysis, the library will often not be complete.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Searching for orthologs to a protein family in an EST database?`,
    options: ["Searches of EST databases for matches to a query sequence routinely produce minimal amounts of output that must be searched manually for significant hits", "ESTs with a high percent identity with the query sequence, a long alignment with the query sequence, and a very low E value of the alignment score represent groups of paralogous and orthologous genes", "To identify orthologs as the most closely related sequence, ESTs were aligned using the amino acid alignment as a guide", "To identify orthologs as the most closely related sequence, a phylogenetic tree was produced by the maximum likelihood method"],
    correct: "a",
    explanation: `The Searches of EST databases for matches to a query sequence routinely produce large amounts of output that must be searched manually for significant hits. an automatic method was described in 1999 utilizing a computer script, FAST-PAN, that scans EST databases with multiple queries from a protein family, sorts the alignment scores, and produces charts and alignments of the matches found.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Family and Domain Analysis?`,
    options: ["Gene identification of predicted proteins in the genome is designed to discover the metabolic features of an organism", "In a particular organism or group of organisms, one particular domain can be expanded to perform a particular function", "Comparison of the domain content of an entire proteome with that of another proteome cannot help in revealing the biological roles of diverse domains in different organisms", "Different proteins are mosaics of domains that occur in different combinations in a given protein"],
    correct: "b",
    explanation: `In a particular organism or group of organisms, various domains can be expanded to perform a particular function. More than 2000 fly and worm proteins are multidomain proteins, compared to about one-third this number in yeast.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Ancient Conserved Regions?`,
    options: ["The method involves database similarity searches of the SwissProt database with human, worm, yeast, or E. coli genes and identification of matches with sequences from a different phylum than the query sequence", "An analysis of ACRs that predate the radiation of the major animal phyla some 580\u2013540 million years ago suggested that 50\u201360% of coding sequences are ACRs", "These ACRs may represent proteins present at the time of the prokaryotic\u2013eukaryotic divergence", "Phylogenetically diverse groups of organisms have been analyzed for the presence of conserved proteins and protein domains that have been conserved over long periods of evolutionary time, called ancient conserved regions or ACRs"],
    correct: "b",
    explanation: `The analysis of ACRs 580–540 million years ago suggested that 20–40% of coding sequences are ACRs. For example, a search with 1916 E. coli proteins detected 266 ACRs found in 439 sequences, roughly one-quarter of the SwissProt database.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Which of the given statements is incorrect about Horizontal Gene Transfer?`,
    options: ["It is a significant source of genome variation in bacteria, allowing them to exploit new environments", "Such transfer is rendered possible by a variety of natural mechanisms in bacteria for transferring DNA from one species to another", "Detection of HT is made possible by the fact that each genome of each bacterial species has a unique base composition", "The time of transfer of DNA cannot be estimated by the composition of the HT DNA"],
    correct: "d",
    explanation: `The time of transfer of DNA may be estimated by the degree to which the composition of the HT DNA has blended into that of the recipient genome. Transfer of a portion of a genome from one organism to another can generally be detected as an island of sequence of different composition in the recipient. If the amino acid composition of transferred genes is typical, these islands may be detected by a codon usage analysis.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `Annotation is based on finding significant alignment to sequences of known function in database similarity searches.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Accurate annotation of genome sequences is an important first step in genome analysis. Matches of lesser significance provide only a tentative or hypothetical prediction and should be used as a working hypothesis of function.`,
    super_topic: `Collecting & Storing Sequences`,
  },
  {
    question: `GeneMultiple Choice Questions & Answers (MCQs) focuses on deriving a predicted protein function, based on a variety of available evidence, including the evaluation of the similarity to the closest homolog in a database.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `GeneQuiz is an integrated system for large-scale biological sequence analysis that uses a variety of search and analysis methods using current sequence databases. By applying expert rules to the results of the different methods, GeneQuiz creates a compact summary of findings.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statement is incorrect regarding MAGPIE?`,
    options: ["It analyzes the genome using a set of automated processes", "It is designed for high-throughput genome sequence analysis", "It is unable to locate potential promoters", "It automatically annotates genomic sequence data and maintains a daily up-to-date record in response to user queries about one or more genomes"],
    correct: "c",
    explanation: `The system also uses a set of rules in logic programming to make decisions that may be used to interpret information from various sources. It has been used to locate potential promoters, terminators, start codons, Shine-Dalgarno sites, DNA motif sites, co-transcription units, and putative operons in microbial genomes. These sites are shown on a map display of the genome that may be edited.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `The designation ECa.b.c.d conveys information. Which of the following is not one of it?`,
    options: ["One of twelve main classes of biochemical reactions", "The group of substrate molecule", "The nature of chemical bond that is involved in the reaction", "Designation for acceptor molecules (cofactors)"],
    correct: "a",
    explanation: `Option “One of twelve main classes of biochemical reactions” should be ‘one of six main classes of biochemical reactions’. The Enzyme Commission numbers formulated by the Enzyme Commission of the International Union of Biochemistry and Molecular Biology provide a detailed way to classify enzymes based on the biochemical reactions they catalyze.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `An approach to classification of genes that encode enzymes is to examine relationships among multiple enzymes that perform the same biochemical function in the same organism.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Although catalyzing the same reaction, these enzymes showed variations in metabolic regulation of their activity. More than one-half of multiple enzymes in E. coli share significant sequence similarity; i.e., they are paralogs. However, the remainder do not share any sequence similarity.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Other functional classification schemes for genes include a broader category for genes involved in the same biological process, e.g., a three-group scheme for energy-related, information-related, and communication-related genes has also been used.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `By this scheme, plants devote more than one-half of their genome to energy metabolism. Whereas, animals devote one-half of their genome to communication-related functions.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Two species that have recently diverged from a common ancestor might be expected to have a ________ set of genes and ________ chromosomes with these genes positioned along the chromosomes in the same order.`,
    options: ["distinct, similar", "similar, distinct", "similar, dissimilar", "similar, similar"],
    correct: "d",
    explanation: `Over evolutionary time, the sequence of each pair of genes will slowly diverge, as the species diverge and other changes such as gene duplication and gene loss change the gene content. In addition, the order of genes also changes over evolutionary time as a result of chromosomal rearrangements.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statement is incorrect about the observations made with regard to gene order?`,
    options: ["Order is highly conserved in closely related species", "Order in closely related species becomes changed by rearrangements over evolutionary time", "As more and more rearrangements occur, there will no longer be any correspondence in the order of orthologous genes on the chromosome of one organism with that of a second organism", "Order is less conserved in closely related species"],
    correct: "d",
    explanation: `Order is more conserved in closely related species. Another observation is that the groups of genes that have a similar biological function tend to remain localized in a group or cluster.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statement is incorrect about the Chromosomal Rearrangements?`,
    options: ["Comparison of the number of rearrangements in a given period of evolutionary history may vary significantly from one organism to the next", "If gene A has a neighboring gene B, then if an ortholog of A occurs in another genome, there is an increased probability of an ortholog of B also occurring in the other organism", "If gene A has a neighboring gene B, then if an ortholog of A occurs in another genome, the B ortholog is more likely to be a neighbor of the A ortholog of the genome of the second species if the two species are more divergent", "In general, the order of orthologs is not well conserved in prokaryotes when the genomes have diverged sufficiently that the orthologs have < 50% identity"],
    correct: "c",
    explanation: `The B ortholog is less likely to be a neighbor of the A ortholog of the genome of the second species if the two species are more divergent. By classifying genes using a nine class functional classification scheme, several genes falling into the same functional category are clustered together on the chromosomes of both of these organisms, and the clusters are in a similar order.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statements is incorrect about global gene regulation?`,
    options: ["One way to obtain useful information about a genome is to determine which genes are induced or repressed in response to a phase of the cell cycle", "Sets of a gene whose expression rises and falls under the same condition are likely to have a related function", "Sets of a gene whose expression rises and falls under the same condition are likely to have dissimilar functions", "Cell cycle is a developmental phase, or a response to the environment"],
    correct: "c",
    explanation: `In addition, a pattern of gene expression may also be an indicator of abnormal cellular regulation and is a useful tool in cancer diagnosis. Because genomes, especially eukaryotic genomes, are so large, a new technology has been developed for studying the regulation of thousands of genes on a microscope slide.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statements is incorrect about Microarray (or microchip) analysis?`,
    options: ["It is a new technology in which all of the genes of an organism are represented by oligonucleotide sequences spread out in an 80 x 80 array on microscope slides", "The oligonucleotide sequences cannot be synthesized directly on the slide", "The oligonucleotides are collectively hybridized to a labeled cDNA library prepared by reverse-transcribing mRNA from cells", "The amount of label binding to each oligonucleotide spot reflects the amount of mRNA in the cell"],
    correct: "b",
    explanation: `The oligonucleotide sequences can also be synthesized directly on the slide at densities of up to one million per square centimeter. Genes that are responding the same way to an environmental signal, in this case the addition of serum to serum-starved skin cells are clustered together in a display. From this analysis, a set of genes that responds in an identical manner may be identified.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Once a set of genes that are co-regulated has been found, the promoter regions of these genes may be analyzed for conserved patterns that represent sites of interaction with specific transcription factors.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Automatic methods for clustering related sets of genes have been devised. The first of these methods, hierarchical clustering, is commonly used, but the other two methods are better designed to detect differences in patterns over a set of time points or samples.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statements is incorrect about Microarray Analysis?`,
    options: ["It is designed to detect global changes in transcription in a genome", "It provides information about the levels of protein products of the genes", "The proteins are first separated in a column on the basis of size and then across a second dimension on a slab on the basis of charge", "Labeled protein samples may also be extracted from treated cells and separated by two-dimensional gel electrophoresis"],
    correct: "b",
    explanation: `Microarray analysis is designed to detect global changes in transcription in a genome but does not provide information about the levels of protein products of the genes, which may also be subject to translational regulation. This method also can resolve thousands of proteins based on size and charge. There are databases of the patterns found in different organisms.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In cluster analysis of microarray data– If Xi is the log odds value for gene X at time i, then for two genes X and Y and N observations, a similarity score is calculated. S(X,Y) is also known as the Pearson correlation coefficent. Xoffset and Yoffset can be the mean of the observations on X or Y, respectively, in which case is the standard deviation, or else Xoffset and Yoffset can be set to zero when a reference state is used. Which of the following best represents it?`,
    options: ["S(X,Y) = 1N\u22122 \u2211i=1,N . (Xi \u2013 Xoffset) (Yi + Yoffset)/\u03d5xQY", "S(X,Y) = 1N \u2211i=1,N . (Xi \u2013 Xoffset) (Yi \u2013 Yoffset)/\u03d5xQY", "S(X,Y) = 1N\u22121 \u2211i=1,N . (Xi + Xoffset) (Yi + Yoffset)/\u03d5xQY", "S(X,Y) = 1N \u2211i=1,N+2 . (Xi + Xoffset) (Yi \u2013 Yoffset)/\u03d5xQY"],
    correct: "b",
    explanation: `After values of S(X,Y) have been calculated for all gene combinations, the most closely related pairs are identified in an above-diagonal scoring matrix. The object of clustering is to identify genes that respond the same way to the environmental treatment. Each gene is compared to every other gene and a gene similarity score (metric) is produced.
Free 30-Day Java Certification Bootcamp is Live. Join Now!`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In cluster analysis of microarray data– A node is created between the _________ scoring pair, and the gene expressed profiles of these two genes are averaged and the joined elements are weighted by the _________ of elements they contain.`,
    options: ["lowest, frequency", "average, sequence", "lowest, number", "highest, number"],
    correct: "d",
    explanation: `The node is created as mentioned. The matrix is then updated replacing the two joined elements by the node.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In cluster analysis of microarray data– For n genes, the process is repeated ________ times until a single element remains.`,
    options: ["n2", "n", "n-1", "n-4"],
    correct: "c",
    explanation: `This number of iterations gives the best results. In the final dendrogram, the order of genes within a cluster is determined by simple weighting schemes, e.g., average dendrogram level.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `The hierarchical clustering method generates a similarity score [S(X,Y)] for all gene combinations, places the scores in a matrix, joins those genes that have the highest score, and then continues to join progressively less similar pairs.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The disadvantage of this method is that it fails to discriminate between different patterns of variation. For example, a gene expression pattern for which a high value is found at an intermediate time point will be clustered with another for which a high value is found at a late time point in the experiment. These variations have to be separated in a subsequent step.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In Self-organizing maps a choice is made of a number of clusters by which to organize the data.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The object is to move each node to the center of a cluster of data points. At each iteration a data point P is selected, and the node closest to that point is identified.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `SVMs (Support vector machines) are a binary classification method to discriminate one set of data points from another.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `They are similar to the types of discriminant analyses. For microarray analysis, sets of genes are identified that represent a target pattern of gene expression.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In SVMs (Support vector machines) Data points are log-transformed and normalized as in method A, where for N observations of a gene i, the log transform Xi of the expression level Ei and reference level Ri is?`,
    options: ["Xi = Log(Ei/Ri)\u2211j=1,NLogz\u22122(Ej/Rj)\u221a", "Xj = Log(Ej/Ri)\u2211j=1,NLogz(Ej/Rj)\u221a", "Xi = Log(Ei/Ri)\u2211j=1,(N\u22121)Logz(Ej/Rj)\u221a", "Xi = Log(Ei/Ri)\u2211j=1,NLogz(Ej/Rj)\u221a"],
    correct: "a",
    explanation: `SVMs were used to categorize genes based on 79 different sets of data points from studies of the yeast cell cycle and are particularly useful for such complex data sets. Gene combinations averaged over all experimental conditions are then examined by a multidimensional analysis.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `When two proteins share a considerable degree of sequence identity throughout the sequence alignment, they are least likely to share the same function.`,
    options: ["True", "False", "", ""],
    correct: "b",
    explanation: `In the mentioned case they are more likely to share the same function. A considerable fraction of a genome may encode proteins whose function may not be identified in this manner because the proteins are not related to another of known function.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Other types of evidence for a relationship between two genes are also given that are not dependent in sequence similarity. Which of the following is a wrong statement?`,
    options: ["genes are closely linked on the same chromosomes", "genes are transcribed from the same DNA strand", "gene fusions are observed between otherwise separate genes", "phylogenetic profiles show the genes are not that commonly present in organisms"],
    correct: "d",
    explanation: `Phylogenetic profiles reveal the genes are both commonly present in many organisms implying they have interdependent metabolic functions. Option a and b imply coordinated regulation in an operon-like structure. Option c suggests the encoded proteins are physically associated in a common complex.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In Genome-wide prediction of protein functions by a combinatorial method– Each point represents a protein, and branches between proteins indicate a relationship by one of several criteria indicated in the legend.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `Branch lengths are shorter for closely related proteins and thicker when two or more prediction methods indicate a relationship. The links are based on experimental data, proteins whose homologs are known to operate sequentially in metabolic pathways, proteins that evolved in a correlated fashion as evidenced by presence in fully sequenced genomes, proteins whose homologs are fused into a single protein in another organism, and proteins whose mRNA expression profiles are similar under a range of cellular and environmental conditions.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `Which of the given statement is untrue about functional genomics?`,
    options: ["Known functions are derived from experimental evidence in molecular biology and genetic studies with model organisms", "Non-Orthologous genes between biologically distinct species can be identified, and it is strong evidence for a related function", "Sequence-based methods of gene prediction can be augmented by the types of genome comparisons that are designed to identify related genes based on common patterns of expression, evolutionary profiles, chromosomal locations, and other features", "Genome analysis depends to a large extent on sequence analysis methods that identify gene function based on similarity between proteins of unknown function and proteins of known function"],
    correct: "b",
    explanation: `Orthologous genes between biologically distinct species (for example, yeast and fruit flies) can be identified, and the high sequence similarity between them is strong evidence for a related function. Given the more complex multicellular biology of flies, the fly gene could have an additional function that is not predictable by the yeast model. In other cases, the occurrence of families of paralogous genes that share common domains can make a precise guess of function of one of these proteins more difficult because all match a model protein to some degree.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In case of functional genomics– Two general types of approaches are used—one in which a genetic construct is made that interferes with the expression of a particular gene (and sometimes a set of related genes) and a second in which a large number of random mutations are generated in a population of organisms.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `The individual with a mutation in a particular gene is then identified. Once mutants are obtained, the effect of the mutant genes on phenotype is determined. The gene function may then be predicted on the basis of the observed alterations. Because such extreme genetic experiments cannot be performed with humans, the mouse model for the human genome serves the same purpose.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `A genome database may also be interfaced with other types of data, such as clinical data.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `These types of organization, termed data warehousing, can facilitate the search for novel relationships among the data by data-mining methods. These methods include genetic algorithms, neuronetworks, and others.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `The ultimate step in genome analysis is to collect the information found on gene and protein sequences, alignments, gene function and location, protein families and domains, relationships of genes to those in other organisms, chromosomal rearrangements, and so on, into a comprehensive database.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This database should be logically organized so that all types of information are readily accessible and easily retrievable by users who have widely divergent knowledge of the organism. This goal is best achieved by using controlled vocabularies that can identify the same genetic or biochemical function in different organisms without ambiguity.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In addition to the care needed in organizing genome databases, a great deal of human input is needed to annotate the genome manually with information.`,
    options: ["True", "False", "", ""],
    correct: "a",
    explanation: `This information can be about individual genes and proteins, effects of mutations in these genes, and other types of genome variations that cannot be readily incorporated into the database by automated methods. For the human genome, this activity will occupy the time of many scientists for many years to come.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `In Reverse-genetics analysis of gene function– Even though a particular gene may be _____ ortholog of a gene of known function in another organism, that gene may be acquired by a _____ function.`,
    options: ["a highly predicted, similar", "a highly predicted, same", "a highly predicted, novel", "less predicted, novel"],
    correct: "c",
    explanation: `For example, a defect in a plant or animal gene that is a homolog of a yeast gene may have an effect on a developmental process or other biologically unique function of multicellular organisms. Information on knockout mutants in model organisms is available through the genome Web sites.`,
    super_topic: `Genome Analysis`,
  },
  {
    question: `The level of complexity of a system should NOT influence the methodology used to reverse engineer it`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The complexity of a system is a fundamental factor in choosing an appropriate reverse engineering methodology, as more complex systems may require more advanced or modular techniques.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect about Oligonucleotide Design in a microarray?`,
    options: ["DNA microarrays are generated by fixing oligonucleotides onto a solid support", "The oligonucleotides don\u2019t react with cDNA samples", "The oligonucleotide array slide represents thousands of preselected genes from an organism", "The length of oligonucleotides is typically in the range of twenty-five to seventy bases long"],
    correct: "The oligonucleotides don’t react with cDNA samples",
    explanation: `This is incorrect because oligonucleotides are specifically designed to hybridize with cDNA in microarrays, which is essential for detecting gene expression.`,
    super_topic: `Quizzone`,
  },
  {
    question: `A complete diagnostic process automatization is not generally feasible at present`,
    options: ["False", "True"],
    correct: "True",
    explanation: `Full automation of the diagnostic process is still limited by technological, interpretative, and validation challenges, making it currently unfeasible in many medical contexts.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In eukaryotes, gene expression is not regulated by a protein complex formed between transcription factors and RNA polymerase`,
    options: ["True", "False"],
    correct: "False",
    explanation: `In eukaryotes, transcription factors form complexes with RNA polymerase to regulate gene expression, making the statement false.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Marker 3D position cannot be precisely reconstructed using triangulation`,
    options: ["False", "True"],
    correct: "False",
    explanation: `Triangulation is a common method for reconstructing 3D positions of markers with high precision in motion capture systems.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The length of the reads is one of the biggest obstacles in sequencing genomes`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Short read lengths can complicate genome assembly, especially in repetitive regions, making longer reads more desirable.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The shotgun approach ______ sequences clones from ______ of cloned DNA.`,
    options: ["specifically, one end", "randomly, one end", "specifically, both ends", "randomly, both ends"],
    correct: "randomly, both ends",
    explanation: `Shotgun sequencing randomly sequences fragments from both ends to allow reassembly based on overlaps.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Automated sleep stage classification`,
    options: ["Is difficult because the data set is very imbalanced", "achieves high performance especially for the N1 stage detection", "is commonly used in the clinical practice, and manual screening is now obsolete", "is not very useful because manual screening is little time consuming"],
    correct: "Is difficult because the data set is very imbalanced",
    explanation: `Sleep stage classification suffers from class imbalance, especially for N1 stage, making automation more difficult.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The SANGER method can be used to discover the amminoacid sequence of a protein:`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The Sanger method is used for DNA sequencing, not for determining protein sequences.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The shape of a protein is encoded into its amminoacid sequence`,
    options: ["False", "True"],
    correct: "True",
    explanation: `The primary amino acid sequence determines the folding and final 3D structure of a protein.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Despite the multiple coverage, sometimes certain genomic regions remain unsequenced, mainly owing to cloning difficulties`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Certain genomic regions, such as highly repetitive or GC-rich regions, are difficult to clone and sequence even with high coverage, leading to gaps in the assembly.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Base pairing, in RNA, is A-G and U-C`,
    options: ["True", "False"],
    correct: "False",
    explanation: `In RNA, canonical base pairing is A-U and G-C. A-G and U-C are incorrect and do not represent Watson-Crick base pairs.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Motor complications in Parkinson's disease include:`,
    options: ["Freezing of gait, tremor", "Bradykinesia, postural instability", "Dyskinesia, freezing of gait", "OFF periods, bradykinesia"],
    correct: "Dyskinesia, freezing of gait",
    explanation: `Dyskinesia and freezing of gait are common motor complications associated with long-term treatment and disease progression in Parkinson's disease.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Machine Learning (vs Deep learning) requires more data but reduces user dependency, requires much computational power, less interpretable`,
    options: ["True", "False"],
    correct: "False",
    explanation: `This description actually fits deep learning. Machine learning generally requires less data, is more interpretable, and often requires more manual feature engineering.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is untrue about distance based methods`,
    options: ["pairwise distance scores in matrix, a phylogenetic tree can be constructed for all the taxa involved", "computed evo distances can be used to construct a matrix of distances between all pairs of taxa", "clustering algos compute tree based on distance matrix starting from the most similar sequence pairs", "Clustering is the only method among the algorithms for the distance-based tree-building method"],
    correct: "Clustering is the only method among the algorithms for the distance-based tree-building method",
    explanation: `Clustering is one of several possible algorithms for distance-based tree construction; it's incorrect to say it's the only one.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Voice can represent an early marker of neurodegeneration`,
    options: ["False", "True"],
    correct: "True",
    explanation: `Changes in vocal features like pitch, articulation, and rhythm may signal early neurological decline, particularly in diseases like Parkinson’s or Alzheimer’s.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The 'central dogma' says that a mRNA molecule can be translated back to DNA`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The central dogma describes DNA → RNA → Protein. mRNA is not typically reverse-translated to DNA, although reverse transcription exists in retroviruses.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect about Self-Organizing Maps`,
    options: ["The data points are initially assigned to the nodes at random", "It starts by defining a number of nodes", "It doesn't involve neural networks", "Clustering by SOMs is in principle similar to the k-means method"],
    correct: "It doesn't involve neural networks",
    explanation: `SOMs are a type of artificial neural network used for unsupervised learning and clustering, so stating that they don't involve neural networks is incorrect.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The shotgun approach does not require knowledge of physical mapping of the clone fragments, but rather a robust computer assembly program to join pieces of random fragments into a single, whole-genome sequence`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Shotgun sequencing relies on random fragmentation and computational assembly, bypassing the need for a predefined physical map.`,
    super_topic: `Quizzone`,
  },
  {
    question: `For many eukaryotic proteins, newly synthesized protein precursors have to be transported to specific membrane-bound compartments and be proteolytically processed to become functional`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Protein precursors often contain signal peptides that direct them to organelles like the ER or mitochondria, where they are processed into mature, functional forms.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The degree of sequence variation in the alignment reveals evolutionary relatedness of different sequences, whereas the conservation between sequences reflects the changes that have occurred during evolution in the form of subs, insertions, and deletions`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The degree of conservation between sequences indicates evolutionary relatedness, not the variation. Substitutions, insertions, and deletions cause variation, but conservation shows preserved, functionally important regions.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Hand tracking poses is less challenging than body tracking`,
    options: ["False", "True"],
    correct: "False",
    explanation: `Hand tracking is more challenging than body tracking due to finer articulation, faster movements, and frequent occlusion of fingers compared to limbs.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In Protein – Protein Interactions the physical contacts between domains are crucial for the functioning of the cellular machinery.`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Physical contacts between domains enable molecular recognition, signaling, and structural assembly, making them essential for many cellular functions.`,
    super_topic: `Quizzone`,
  },
  {
    question: `During Data Cleaning, outliers should be always removed from datasets`,
    options: ["False", "True"],
    correct: "False",
    explanation: `Outliers are not always errors—they can represent rare but important phenomena. Blindly removing them may eliminate meaningful biological signals.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Reverse engineering using a top-down approach`,
    options: ["is best for linear systems", "Allows to identify relationships between the parts of the system", "is a sequencing technique", "is the methodology used by most Life Science researchers"],
    correct: "Allows to identify relationships between the parts of the system",
    explanation: `A top-down approach starts with the whole system and breaks it down to identify relationships and interactions between its parts.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Reverse engineering, is the processes of extracting knowledge from something and reproducing it based on the extracted information`,
    options: ["False", "True"],
    correct: "True",
    explanation: `Reverse engineering involves analyzing an object to extract knowledge and replicate it, often used in system biology and software analysis.`,
    super_topic: `Quizzone`,
  },
  {
    question: `TATA box is often used as an indicator of the presence of a promoter`,
    options: ["True", "False"],
    correct: "True",
    explanation: `The TATA box is a conserved DNA sequence in promoter regions that helps position RNA polymerase for transcription initiation.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Dissociated sleep means:`,
    options: ["excessive skeletal muscle atonia during REM stage", "lack of smooth muscle atonia during REM stage", "lack of skeletal muscle atonia during REM stage", "lack of eye muscle atonia during REM stage"],
    correct: "lack of skeletal muscle atonia during REM stage",
    explanation: `This condition refers to the lack of skeletal muscle atonia during REM sleep, which may lead to acting out dreams, as seen in REM sleep behavior disorder.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Phylogenetic relationship can be shown by`,
    options: ["Gene Bank", "Data retrieving tool", "Dendrogram", "Data search tool"],
    correct: "Dendrogram",
    explanation: `Dendrograms are tree-like diagrams used to visualize evolutionary relationships based on sequence similarity or trait data.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of these sentences on cross-individual transfer learning is correct?`,
    options: ["It is not feasible at all, a merely abstract concept", "It impairs the performance of most algorithms", "It means to train the network on Patient 1 and partially reuse on Patient 2", "It means to train the network on Patient 1 and totally reuse on Patient 2"],
    correct: "It means to train the network on Patient 1 and partially reuse on Patient 2",
    explanation: `In transfer learning, knowledge from one subject (e.g., Patient 1) is reused in another (e.g., Patient 2), often by reusing parts of the model like shared feature extractors.`,
    super_topic: `Quizzone`,
  },
  {
    question: `A good estimation metric for balanced dataset is accuracy`,
    options: ["False", "True"],
    correct: "True",
    explanation: `Accuracy is a reliable metric when class distributions are balanced. In imbalanced datasets, it can be misleading.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In microarray data-If replicated datasets are available, statistical tests such as t-test and ANOVA can be performed to test the null hypothesis that a given data point is not significantly different from the mean of the data distribution.`,
    options: ["False", "True"],
    correct: "True",
    explanation: `When replicates are available in microarray data, statistical tests like t-test or ANOVA can assess whether observed gene expression differences are statistically significant.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Structures are not much important when it comes to the study of the functions`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The structure of a biological molecule is closely related to its function. For example, the 3D shape of an enzyme determines its substrate specificity. Understanding structure is essential for functional studies.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Unlike DNA, which is mainly double stranded, RNA is single stranded, although an RNA molecule can self-hybridize at certain regions to form partial double-stranded structures`,
    options: ["False", "True"],
    correct: "True",
    explanation: `RNA is typically single-stranded but can fold back and form intramolecular base-pairing, resulting in structures like hairpins or stems, which are crucial for its function (e.g., tRNA, ribozymes).`,
    super_topic: `Quizzone`,
  },
  {
    question: `The cost of Data Cleaning is negligible when compared to the cost of running lab experiments`,
    options: ["True", "False"],
    correct: "False",
    explanation: `Data cleaning can be a significant part of data analysis, especially with large biological datasets. It requires time, computational resources, and domain knowledge, and its cost is often underestimated.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Petri nets`,
    options: ["have been defined to represent biological data", "have been created in the late 90'", "do not allow to model stochasticity", "can simulate the dynamics of a system"],
    correct: "can simulate the dynamics of a system",
    explanation: `Petri nets are a mathematical modeling tool used to represent and simulate biological systems, especially those involving concurrent processes like metabolic or signaling pathways.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect regarding pairwise sequence alignment?`,
    options: ["It's an important first step toward structural and functional analysis of newly determined sequences", "The most fundamental process in this type of comparison is sequence alignment", "...patterns and establishing residue-residue correspondence among related sequences", "It is the process of aligning multiple sequences"],
    correct: "It is the process of aligning multiple sequences",
    explanation: `Pairwise sequence alignment compares two sequences, not multiple ones. Aligning multiple sequences is called multiple sequence alignment (MSA), which is a separate method.`,
    super_topic: `Quizzone`,
  },
  {
    question: `An Attractor`,
    options: ["is a stable state of a boolean network", "is an unstable state in a complex system", "is the process that makes two cells move toward each others", "is a stable state in a protein-to-protein interaction network"],
    correct: "is a stable state of a boolean network",
    explanation: `In boolean network models, an attractor represents a stable pattern of gene activity (or states) that the system tends to evolve towards, often representing a cell fate or behavior.`,
    super_topic: `Quizzone`,
  },
  {
    question: `When the two domains are located in two different proteins, to preserve the same functionality, their close proximity and interaction have to be conserved as well`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Protein domains that function together often require physical proximity. When separated into different proteins, their interaction must be maintained to preserve the original function.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Which of the following is incorrect about k-Means Clustering`,
    options: ["It classifies data through a single step partition", "In this method, data are partitioned into k-clusters, which are prespecified at the outset", "k-means clustering produces a dendrogram", "It is a divisive approach"],
    correct: "k-means clustering produces a dendrogram",
    explanation: `k-Means clustering partitions data into a fixed number of clusters without creating a hierarchical tree. A dendrogram is a product of hierarchical clustering, not k-means.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The REM sleep is characterized by frequent limb movements`,
    options: ["False", "True"],
    correct: "False",
    explanation: `REM (Rapid Eye Movement) sleep is associated with vivid dreams and muscle atonia — the body is typically paralyzed to prevent acting out dreams. Limb movements are more frequent in non-REM sleep.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The promoter sequence may determine the expression of one gene or a number of linked genes downstream`,
    options: ["False", "True"],
    correct: "True",
    explanation: `In prokaryotes especially, a single promoter can control the expression of an operon — a group of genes transcribed together. Even in eukaryotes, promoter configuration can affect multiple transcripts.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In motion capture systems, markers can be either passive or active`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Motion capture markers can be passive (reflecting light detected by infrared cameras) or active (emitting their own signals). Both are used in biomechanics and animation.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In a linear system some functionalities emerge from the interaction of its parts`,
    options: ["False", "True"],
    correct: "False",
    explanation: `Emergent properties typically arise in non-linear systems where the whole exhibits behaviors not predictable from the parts. Linear systems follow additive, predictable rules.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The ‘central dogma’ says that a protein can be translated back to mRNA`,
    options: ["True", "False"],
    correct: "False",
    explanation: `The central dogma of molecular biology describes the flow of genetic information as DNA → RNA → Protein. Once a protein is formed, it cannot be translated back into mRNA. This process is unidirectional.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Machine Learning algorithms are always extremely efficient when dealing with Biological data`,
    options: ["True", "False"],
    correct: "False",
    explanation: `Biological data are often noisy, high-dimensional, and unbalanced. While machine learning can be powerful, it requires careful preprocessing and tuning. Efficiency is not guaranteed and heavily depends on the data and problem formulation.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Deep Learning (vs Machine learning) requires much more data but improves generalization capability, requires much computational power, less interpretable`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Deep learning models can generalize well on complex tasks but require large amounts of data and computational resources. However, they are often considered "black boxes" due to low interpretability, especially in biomedical contexts.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The specific DNA sequences to which the transcription factors bind are referred to as`,
    options: ["regulatory elements", "blocking factors", "transcription factors", "replication elements"],
    correct: "regulatory elements",
    explanation: `Regulatory elements are short DNA sequences that transcription factors recognize and bind to in order to regulate gene expression. These include promoters, enhancers, silencers, and insulators.`,
    super_topic: `Quizzone`,
  },
  {
    question: `In addition to RNA polymerase, there are also a number of DNA-binding proteins that facilitate the process of transcription`,
    options: ["False", "True"],
    correct: "True",
    explanation: `Transcription is facilitated not only by RNA polymerase but also by transcription factors and other DNA-binding proteins that assist in the initiation and regulation of the transcription process.`,
    super_topic: `Quizzone`,
  },
  {
    question: `A good estimation metric for balanced dataset is F-score`,
    options: ["False", "True"],
    correct: "False",
    explanation: `The F-score is particularly useful for imbalanced datasets as it balances precision and recall. For balanced datasets, metrics like accuracy can be more straightforward and appropriate.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Chronic patients should be always treated at home`,
    options: ["False", "True"],
    correct: "True",
    explanation: `While not universally true, there is a strong shift toward home-based care for chronic conditions when clinically appropriate, to improve quality of life and reduce hospital burden. However, this must be evaluated case-by-case.`,
    super_topic: `Quizzone`,
  },
  {
    question: `The gene is a union of genomic sequences encoding a coherent set of potentially overlapping functional products`,
    options: ["True", "False"],
    correct: "True",
    explanation: `This is a modern and widely accepted definition. Genes can produce multiple transcripts (and proteins) through alternative splicing, leading to overlapping functional products from the same genomic locus.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Voice can support clinicians in the objective assessment of many neurodegenerative condition`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Voice analysis is being increasingly used to detect and monitor conditions like Parkinson's, Alzheimer's, and ALS, offering a non-invasive and objective assessment of neurological decline.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Many deep learning solutions are possible for 2D and 3D body tracking/pose estimation`,
    options: ["True", "False"],
    correct: "True",
    explanation: `Deep learning has enabled accurate 2D and 3D pose estimation using CNNs, transformers, and other architectures. Applications span healthcare, sports, rehabilitation, and more.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Deep learning in sleep disorder classification:`,
    options: ["is always to be preferred to machine learning methods", "is infeasible because polysomnographic data are scarce", "lacks interpretability, which is a key issue in the clinical practice", "is validated by numerous controlled trials"],
    correct: "lacks interpretability, which is a key issue in the clinical practice",
    explanation: `Despite its power, deep learning lacks transparency. In clinical sleep disorder diagnosis (e.g., sleep apnea), interpretability is crucial for trust, validation, and regulatory acceptance.`,
    super_topic: `Quizzone`,
  },
  {
    question: `Threshold processing of inertial data for fall prevention:`,
    options: ["is feasible and simple but yields many false negatives", "always outperforms statistical methods", "is feasible and simple but yields many false positives", "is infeasible"],
    correct: "is feasible and simple but yields many false positives",
    explanation: `Threshold-based approaches are easy to implement and low-cost, but often trigger many false alarms due to their sensitivity. Advanced statistical or ML methods are often preferred to improve precision.`,
    super_topic: `Quizzone`,
  },
];
